{
  "cells": [
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Jzns4LpZoanp",
        "outputId": "8b11f27c-ffcc-4367-87eb-ee7c259bd7f6"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: transformers==4.0.0 in /usr/local/lib/python3.7/dist-packages (4.0.0)\n",
            "Requirement already satisfied: packaging in /usr/local/lib/python3.7/dist-packages (from transformers==4.0.0) (21.3)\n",
            "Requirement already satisfied: regex!=2019.12.17 in /usr/local/lib/python3.7/dist-packages (from transformers==4.0.0) (2022.6.2)\n",
            "Requirement already satisfied: sacremoses in /usr/local/lib/python3.7/dist-packages (from transformers==4.0.0) (0.0.53)\n",
            "Requirement already satisfied: tqdm>=4.27 in /usr/local/lib/python3.7/dist-packages (from transformers==4.0.0) (4.64.1)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from transformers==4.0.0) (1.21.6)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.7/dist-packages (from transformers==4.0.0) (3.8.0)\n",
            "Requirement already satisfied: requests in /usr/local/lib/python3.7/dist-packages (from transformers==4.0.0) (2.23.0)\n",
            "Requirement already satisfied: tokenizers==0.9.4 in /usr/local/lib/python3.7/dist-packages (from transformers==4.0.0) (0.9.4)\n",
            "Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from packaging->transformers==4.0.0) (3.0.9)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests->transformers==4.0.0) (1.25.11)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests->transformers==4.0.0) (2022.6.15)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests->transformers==4.0.0) (3.0.4)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests->transformers==4.0.0) (2.10)\n",
            "Requirement already satisfied: six in /usr/local/lib/python3.7/dist-packages (from sacremoses->transformers==4.0.0) (1.15.0)\n",
            "Requirement already satisfied: click in /usr/local/lib/python3.7/dist-packages (from sacremoses->transformers==4.0.0) (7.1.2)\n",
            "Requirement already satisfied: joblib in /usr/local/lib/python3.7/dist-packages (from sacremoses->transformers==4.0.0) (1.1.0)\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: datasets in /usr/local/lib/python3.7/dist-packages (2.4.0)\n",
            "Requirement already satisfied: fsspec[http]>=2021.11.1 in /usr/local/lib/python3.7/dist-packages (from datasets) (2022.8.2)\n",
            "Requirement already satisfied: xxhash in /usr/local/lib/python3.7/dist-packages (from datasets) (3.0.0)\n",
            "Requirement already satisfied: aiohttp in /usr/local/lib/python3.7/dist-packages (from datasets) (3.8.1)\n",
            "Requirement already satisfied: requests>=2.19.0 in /usr/local/lib/python3.7/dist-packages (from datasets) (2.23.0)\n",
            "Requirement already satisfied: dill<0.3.6 in /usr/local/lib/python3.7/dist-packages (from datasets) (0.3.5.1)\n",
            "Requirement already satisfied: pandas in /usr/local/lib/python3.7/dist-packages (from datasets) (1.3.5)\n",
            "Requirement already satisfied: importlib-metadata in /usr/local/lib/python3.7/dist-packages (from datasets) (4.12.0)\n",
            "Requirement already satisfied: pyarrow>=6.0.0 in /usr/local/lib/python3.7/dist-packages (from datasets) (6.0.1)\n",
            "Requirement already satisfied: tqdm>=4.62.1 in /usr/local/lib/python3.7/dist-packages (from datasets) (4.64.1)\n",
            "Requirement already satisfied: packaging in /usr/local/lib/python3.7/dist-packages (from datasets) (21.3)\n",
            "Requirement already satisfied: numpy>=1.17 in /usr/local/lib/python3.7/dist-packages (from datasets) (1.21.6)\n",
            "Requirement already satisfied: responses<0.19 in /usr/local/lib/python3.7/dist-packages (from datasets) (0.18.0)\n",
            "Requirement already satisfied: multiprocess in /usr/local/lib/python3.7/dist-packages (from datasets) (0.70.13)\n",
            "Requirement already satisfied: huggingface-hub<1.0.0,>=0.1.0 in /usr/local/lib/python3.7/dist-packages (from datasets) (0.9.1)\n",
            "Requirement already satisfied: asynctest==0.13.0 in /usr/local/lib/python3.7/dist-packages (from aiohttp->datasets) (0.13.0)\n",
            "Requirement already satisfied: yarl<2.0,>=1.0 in /usr/local/lib/python3.7/dist-packages (from aiohttp->datasets) (1.8.1)\n",
            "Requirement already satisfied: charset-normalizer<3.0,>=2.0 in /usr/local/lib/python3.7/dist-packages (from aiohttp->datasets) (2.1.1)\n",
            "Requirement already satisfied: attrs>=17.3.0 in /usr/local/lib/python3.7/dist-packages (from aiohttp->datasets) (22.1.0)\n",
            "Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /usr/local/lib/python3.7/dist-packages (from aiohttp->datasets) (4.0.2)\n",
            "Requirement already satisfied: aiosignal>=1.1.2 in /usr/local/lib/python3.7/dist-packages (from aiohttp->datasets) (1.2.0)\n",
            "Requirement already satisfied: typing-extensions>=3.7.4 in /usr/local/lib/python3.7/dist-packages (from aiohttp->datasets) (4.1.1)\n",
            "Requirement already satisfied: frozenlist>=1.1.1 in /usr/local/lib/python3.7/dist-packages (from aiohttp->datasets) (1.3.1)\n",
            "Requirement already satisfied: multidict<7.0,>=4.5 in /usr/local/lib/python3.7/dist-packages (from aiohttp->datasets) (6.0.2)\n",
            "Requirement already satisfied: filelock in /usr/local/lib/python3.7/dist-packages (from huggingface-hub<1.0.0,>=0.1.0->datasets) (3.8.0)\n",
            "Requirement already satisfied: pyyaml>=5.1 in /usr/local/lib/python3.7/dist-packages (from huggingface-hub<1.0.0,>=0.1.0->datasets) (6.0)\n",
            "Requirement already satisfied: pyparsing!=3.0.5,>=2.0.2 in /usr/local/lib/python3.7/dist-packages (from packaging->datasets) (3.0.9)\n",
            "Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.7/dist-packages (from requests>=2.19.0->datasets) (3.0.4)\n",
            "Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.7/dist-packages (from requests>=2.19.0->datasets) (2.10)\n",
            "Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.7/dist-packages (from requests>=2.19.0->datasets) (2022.6.15)\n",
            "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.7/dist-packages (from requests>=2.19.0->datasets) (1.25.11)\n",
            "Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.7/dist-packages (from importlib-metadata->datasets) (3.8.1)\n",
            "Requirement already satisfied: python-dateutil>=2.7.3 in /usr/local/lib/python3.7/dist-packages (from pandas->datasets) (2.8.2)\n",
            "Requirement already satisfied: pytz>=2017.3 in /usr/local/lib/python3.7/dist-packages (from pandas->datasets) (2022.2.1)\n",
            "Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.7/dist-packages (from python-dateutil>=2.7.3->pandas->datasets) (1.15.0)\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: xmltodict in /usr/local/lib/python3.7/dist-packages (0.13.0)\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Collecting torch==1.6.0\n",
            "  Downloading torch-1.6.0-cp37-cp37m-manylinux1_x86_64.whl (748.8 MB)\n",
            "\u001b[K     |████████████████████████████████| 748.8 MB 17 kB/s \n",
            "\u001b[?25hRequirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from torch==1.6.0) (1.21.6)\n",
            "Requirement already satisfied: future in /usr/local/lib/python3.7/dist-packages (from torch==1.6.0) (0.16.0)\n",
            "Installing collected packages: torch\n",
            "  Attempting uninstall: torch\n",
            "    Found existing installation: torch 1.12.1+cu113\n",
            "    Uninstalling torch-1.12.1+cu113:\n",
            "      Successfully uninstalled torch-1.12.1+cu113\n",
            "\u001b[31mERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.\n",
            "torchvision 0.13.1+cu113 requires torch==1.12.1, but you have torch 1.6.0 which is incompatible.\n",
            "torchtext 0.13.1 requires torch==1.12.1, but you have torch 1.6.0 which is incompatible.\n",
            "torchaudio 0.12.1+cu113 requires torch==1.12.1, but you have torch 1.6.0 which is incompatible.\n",
            "fastai 2.7.9 requires torch<1.14,>=1.7, but you have torch 1.6.0 which is incompatible.\u001b[0m\n",
            "Successfully installed torch-1.6.0\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: sklearn in /usr/local/lib/python3.7/dist-packages (0.0)\n",
            "Requirement already satisfied: scikit-learn in /usr/local/lib/python3.7/dist-packages (from sklearn) (1.0.2)\n",
            "Requirement already satisfied: joblib>=0.11 in /usr/local/lib/python3.7/dist-packages (from scikit-learn->sklearn) (1.1.0)\n",
            "Requirement already satisfied: numpy>=1.14.6 in /usr/local/lib/python3.7/dist-packages (from scikit-learn->sklearn) (1.21.6)\n",
            "Requirement already satisfied: scipy>=1.1.0 in /usr/local/lib/python3.7/dist-packages (from scikit-learn->sklearn) (1.7.3)\n",
            "Requirement already satisfied: threadpoolctl>=2.0.0 in /usr/local/lib/python3.7/dist-packages (from scikit-learn->sklearn) (3.1.0)\n",
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: tqdm in /usr/local/lib/python3.7/dist-packages (4.64.1)\n"
          ]
        }
      ],
      "source": [
        "!pip install transformers==4.0.0\n",
        "!pip install datasets\n",
        "!pip install xmltodict \n",
        "!pip install torch==1.6.0\n",
        "!pip install sklearn\n",
        "!pip install tqdm "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "sLVlLhLXrX6l",
        "outputId": "1f3f9dde-9749-4713-aa18-8d86c4bb0768"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Mounted at /content/drive\n"
          ]
        }
      ],
      "source": [
        "from google.colab import drive \n",
        "drive.mount('/content/drive')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "QWXo5NrLreu9"
      },
      "outputs": [],
      "source": [
        "import xmltodict\n",
        "import datasets\n",
        "from datasets import Dataset, concatenate_datasets\n",
        "import glob\n",
        "import pandas as pd\n",
        "import torch\n",
        "import transformers "
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "cfD86ZF1riGu",
        "outputId": "08f4333b-8e8c-46a1-d974-12e90c42ceb8"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py:27: SettingWithCopyWarning: \n",
            "A value is trying to be set on a copy of a slice from a DataFrame.\n",
            "Try using .loc[row_indexer,col_indexer] = value instead\n",
            "\n",
            "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
            "/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py:28: SettingWithCopyWarning: \n",
            "A value is trying to be set on a copy of a slice from a DataFrame.\n",
            "Try using .loc[row_indexer,col_indexer] = value instead\n",
            "\n",
            "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
            "/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py:30: SettingWithCopyWarning: \n",
            "A value is trying to be set on a copy of a slice from a DataFrame.\n",
            "Try using .loc[row_indexer,col_indexer] = value instead\n",
            "\n",
            "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "922\n",
            "212\n",
            "0.2299349240780911\n",
            "0.20715835140997832\n",
            "0.18655097613882862\n",
            "0.12472885032537961\n",
            "0.07266811279826464\n",
            "0.04772234273318872\n"
          ]
        }
      ],
      "source": [
        "#!/usr/bin/env python\n",
        "import xmltodict\n",
        "import datasets\n",
        "from datasets import Dataset, concatenate_datasets\n",
        "import glob\n",
        "import pandas as pd\n",
        "import re\n",
        "\n",
        "\n",
        "\n",
        "dataframe_list = []\n",
        "#for file in glob.glob(\"/content/drive/MyDrive/all_earnings_sub/*.xml\"):\n",
        "for file in glob.glob(\"/content/drive/MyDrive/original_pubmed/*.xml\"):\n",
        "  #print(file)\n",
        "  with open(file, 'r', encoding='utf-8') as file: #'pubmed_1_copy.xml'\n",
        "      my_xml = file.read()\n",
        "      clean_xml = re.sub(\"[&;]\",\"\",my_xml)\n",
        "      xml_dict = xmltodict.parse(my_xml)\n",
        "      df1 = pd.DataFrame(xml_dict, columns=xml_dict.keys())\n",
        "      order_dict = df1[\"rst\"].iloc[0]\n",
        "      segments_list_of_dicts = order_dict[\"segment\"]\n",
        "      xml_dataframe = pd.DataFrame.from_dict(segments_list_of_dicts, orient='columns')\n",
        "      selected_df = xml_dataframe[~xml_dataframe.isnull().any(axis=1)] #removes null\n",
        "      selected_df.sort_values(by='@id',ascending=True)\n",
        "      #print(selected_df.head(10))\n",
        "      #df1['Score_diff'] = df1['score1'] - df1['score2']\n",
        "      selected_df[\"@id\"] = selected_df[\"@id\"].astype(float)\n",
        "      selected_df[\"@parent\"] = selected_df[\"@parent\"].astype(float)\n",
        "\n",
        "      selected_df[\"edu_distance\"] = abs(selected_df[\"@id\"] - selected_df[\"@parent\"])\n",
        "      df_formatted = selected_df.assign(parent_text=selected_df[\"#text\"].shift(1))\n",
        "      df_formatted[\"parent_text\"].fillna(value=df_formatted.iloc[0][\"#text\"],inplace=True) #fillna parent\n",
        "      df_formatted[\"parent_text\"].fillna(value=\"Introduction\",inplace=True) #fillna parent\n",
        "      dataframe_list.append(df_formatted)\n",
        "      all_pubmed_sub_df = pd.concat(dataframe_list)\n",
        "    \n",
        "#selected_df.groupby('a').count()\n",
        "#print(all_earnings_df.head(15))\n",
        "#print(len(all_pubmed_sub_df))\n",
        "print(len(all_pubmed_sub_df[\"edu_distance\"]))\n",
        "print(len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 1]))\n",
        "#.value_counts()\n",
        "\n",
        "print(len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 1]) / len(all_pubmed_sub_df[\"edu_distance\"]))\n",
        "print(len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 3]) / len(all_pubmed_sub_df[\"edu_distance\"]))\n",
        "print(len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 5]) / len(all_pubmed_sub_df[\"edu_distance\"]))\n",
        "print(len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 10]) / len(all_pubmed_sub_df[\"edu_distance\"]))\n",
        "print(len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 15]) / len(all_pubmed_sub_df[\"edu_distance\"]))\n",
        "print(len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 20]) / len(all_pubmed_sub_df[\"edu_distance\"]))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "uJy8F4IqSP_X",
        "outputId": "d2ea66f1-afdd-4e89-ae56-2a7824f4a2fe"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "416\n",
            "64\n",
            "0.15384615384615385\n",
            "0.13942307692307693\n",
            "0.12980769230769232\n",
            "0.11538461538461539\n",
            "0.10336538461538461\n",
            "0.09134615384615384\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py:28: SettingWithCopyWarning: \n",
            "A value is trying to be set on a copy of a slice from a DataFrame.\n",
            "Try using .loc[row_indexer,col_indexer] = value instead\n",
            "\n",
            "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
            "/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py:29: SettingWithCopyWarning: \n",
            "A value is trying to be set on a copy of a slice from a DataFrame.\n",
            "Try using .loc[row_indexer,col_indexer] = value instead\n",
            "\n",
            "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n",
            "/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py:31: SettingWithCopyWarning: \n",
            "A value is trying to be set on a copy of a slice from a DataFrame.\n",
            "Try using .loc[row_indexer,col_indexer] = value instead\n",
            "\n",
            "See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy\n"
          ]
        }
      ],
      "source": [
        "#!/usr/bin/env python\n",
        "import datasets\n",
        "from datasets import Dataset, concatenate_datasets\n",
        "import glob\n",
        "import pandas as pd\n",
        "import re\n",
        "import transformers \n",
        "import xmltodict\n",
        "\n",
        "\n",
        "\n",
        "dataframe_list = []\n",
        "for file in glob.glob(\"/content/drive/MyDrive/all_earnings_sub/*.xml\"):\n",
        "#for file in glob.glob(\"/content/drive/MyDrive/original_pubmed/*.xml\"):\n",
        "  #print(file)\n",
        "  with open(file, 'r', encoding='utf-8') as file: #'pubmed_1_copy.xml'\n",
        "      my_xml = file.read()\n",
        "      clean_xml = re.sub(\"[&;]\",\"\",my_xml)\n",
        "      xml_dict = xmltodict.parse(my_xml)\n",
        "      df1 = pd.DataFrame(xml_dict, columns=xml_dict.keys())\n",
        "      order_dict = df1[\"rst\"].iloc[0]\n",
        "      segments_list_of_dicts = order_dict[\"segment\"]\n",
        "      xml_dataframe = pd.DataFrame.from_dict(segments_list_of_dicts, orient='columns')\n",
        "      selected_df = xml_dataframe[~xml_dataframe.isnull().any(axis=1)] #removes null\n",
        "      selected_df.sort_values(by='@id',ascending=True)\n",
        "      #print(selected_df.head(10))\n",
        "      #df1['Score_diff'] = df1['score1'] - df1['score2']\n",
        "      selected_df[\"@id\"] = selected_df[\"@id\"].astype(float)\n",
        "      selected_df[\"@parent\"] = selected_df[\"@parent\"].astype(float)\n",
        "\n",
        "      selected_df[\"edu_distance\"] = abs(selected_df[\"@id\"] - selected_df[\"@parent\"])\n",
        "      df_formatted = selected_df.assign(parent_text=selected_df[\"#text\"].shift(1))\n",
        "      df_formatted[\"parent_text\"].fillna(value=df_formatted.iloc[0][\"#text\"],inplace=True) #fillna parent\n",
        "      df_formatted[\"parent_text\"].fillna(value=\"Introduction\",inplace=True) #fillna parent\n",
        "      dataframe_list.append(df_formatted)\n",
        "      all_earnings_sub_df = pd.concat(dataframe_list)\n",
        "    \n",
        "#selected_df.groupby('a').count()\n",
        "#print(all_earnings_df.head(15))\n",
        "#print(len(all_pubmed_sub_df))\n",
        "print(len(all_earnings_sub_df[\"edu_distance\"]))\n",
        "print(len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] > 1]))\n",
        "#.value_counts()\n",
        "\n",
        "print(len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] > 1]) / len(all_earnings_sub_df[\"edu_distance\"]))\n",
        "print(len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] > 3]) / len(all_earnings_sub_df[\"edu_distance\"]))\n",
        "print(len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] > 5]) / len(all_earnings_sub_df[\"edu_distance\"]))\n",
        "print(len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] > 10]) / len(all_earnings_sub_df[\"edu_distance\"]))\n",
        "print(len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] > 15]) / len(all_earnings_sub_df[\"edu_distance\"]))\n",
        "print(len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] > 20]) / len(all_earnings_sub_df[\"edu_distance\"]))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "4QqwYfrnSmtN",
        "outputId": "189d3f5c-3e53-49dd-d4a4-9e9bbfa3e958"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "dict_values([0.8461538461538461, 0.01201923076923077, 0.002403846153846154, 0.004807692307692308, 0.004807692307692308, 0.002403846153846154, 0.0, 0.004807692307692308, 0.007211538461538462, 0.0, 0.002403846153846154, 0.0, 0.0, 0.0, 0.009615384615384616, 0.0, 0.004807692307692308, 0.0, 0.002403846153846154, 0.004807692307692308, 0.002403846153846154, 0.0, 0.0, 0.002403846153846154, 0.002403846153846154])\n",
            "{'equal_to_one_EDU': 0.8461538461538461, 'two_edus': 0.01201923076923077, 'three_edus': 0.002403846153846154, 'four_edus': 0.004807692307692308, 'five_edus': 0.004807692307692308, 'six_edus': 0.002403846153846154, 'seven_edus': 0.0, 'eight_edus': 0.004807692307692308, 'nine_edus': 0.007211538461538462, 'ten_edus': 0.0, 'eleven_edus': 0.002403846153846154, 'twelve_edus': 0.0, 'thirteen_edus': 0.0, 'fourteen_edus': 0.0, 'fifteen_edus': 0.009615384615384616, 'sixteen_edus': 0.0, 'seventeen_edus': 0.004807692307692308, 'eighteen_edus': 0.0, 'nineteen_edus': 0.002403846153846154, 'twenty_edus': 0.004807692307692308, 'twenty_one_edus': 0.002403846153846154, 'twenty_two_edus': 0.0, 'twenty_three_edus': 0.0, 'twenty_four_edus': 0.002403846153846154, 'twenty_five_edus': 0.002403846153846154}\n",
            "[0.8461538461538461, 0.01201923076923077, 0.002403846153846154, 0.004807692307692308, 0.004807692307692308, 0.002403846153846154, 0.0, 0.004807692307692308, 0.007211538461538462, 0.0, 0.002403846153846154, 0.0, 0.0, 0.0, 0.009615384615384616, 0.0, 0.004807692307692308, 0.0, 0.002403846153846154, 0.004807692307692308, 0.002403846153846154, 0.0, 0.0, 0.002403846153846154, 0.002403846153846154]\n"
          ]
        }
      ],
      "source": [
        "all_earnings_sub_df[\"equal_to_1\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 1]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_2\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 2]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_3\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 3]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_4\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 4]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_5\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 5]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_6\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 6]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_7\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 7]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_8\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 8]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_9\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 9]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_10\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 10]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_11\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 11]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_12\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 12]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_13\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 13]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_14\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 14]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_15\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 15]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_16\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 16]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_17\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 17]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_18\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 18]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_19\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 19]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_20\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 20]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_21\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 21]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_22\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 22]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_23\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 23]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_24\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 24]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "all_earnings_sub_df[\"equal_to_25\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 25]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "\n",
        "\n",
        "percentage_dict = {\"equal_to_one_EDU\": all_earnings_sub_df[\"equal_to_1\"].iloc[0],\n",
        "                   \"two_edus\": all_earnings_sub_df[\"equal_to_2\"].iloc[0],\n",
        "                   \"three_edus\": all_earnings_sub_df[\"equal_to_3\"].iloc[0],\n",
        "                   \"four_edus\": all_earnings_sub_df[\"equal_to_4\"].iloc[0],\n",
        "                   \"five_edus\": all_earnings_sub_df[\"equal_to_5\"].iloc[0],\n",
        "                   \"six_edus\": all_earnings_sub_df[\"equal_to_6\"].iloc[0],\n",
        "                   \"seven_edus\": all_earnings_sub_df[\"equal_to_7\"].iloc[0],\n",
        "                   \"eight_edus\": all_earnings_sub_df[\"equal_to_8\"].iloc[0],\n",
        "                   \"nine_edus\": all_earnings_sub_df[\"equal_to_9\"].iloc[0],\n",
        "                   \"ten_edus\": all_earnings_sub_df[\"equal_to_10\"].iloc[0],\n",
        "                   \"eleven_edus\": all_earnings_sub_df[\"equal_to_11\"].iloc[0],\n",
        "                   \"twelve_edus\": all_earnings_sub_df[\"equal_to_12\"].iloc[0],\n",
        "                   \"thirteen_edus\": all_earnings_sub_df[\"equal_to_13\"].iloc[0],\n",
        "                   \"fourteen_edus\": all_earnings_sub_df[\"equal_to_14\"].iloc[0],\n",
        "                   \"fifteen_edus\": all_earnings_sub_df[\"equal_to_15\"].iloc[0],\n",
        "                   \"sixteen_edus\": all_earnings_sub_df[\"equal_to_16\"].iloc[0],\n",
        "                   \"seventeen_edus\": all_earnings_sub_df[\"equal_to_17\"].iloc[0],\n",
        "                   \"eighteen_edus\": all_earnings_sub_df[\"equal_to_18\"].iloc[0],\n",
        "                   \"nineteen_edus\": all_earnings_sub_df[\"equal_to_19\"].iloc[0],\n",
        "                   \"twenty_edus\": all_earnings_sub_df[\"equal_to_20\"].iloc[0],\n",
        "                   \"twenty_one_edus\": all_earnings_sub_df[\"equal_to_21\"].iloc[0],\n",
        "                   \"twenty_two_edus\": all_earnings_sub_df[\"equal_to_22\"].iloc[0],\n",
        "                   \"twenty_three_edus\": all_earnings_sub_df[\"equal_to_23\"].iloc[0],\n",
        "                   \"twenty_four_edus\": all_earnings_sub_df[\"equal_to_24\"].iloc[0],\n",
        "                   \"twenty_five_edus\": all_earnings_sub_df[\"equal_to_25\"].iloc[0],\n",
        "\n",
        "                    \n",
        "}\n",
        "\n",
        "print(percentage_dict.values())\n",
        "\n",
        "#all_earnings_sub_df[\"more_than_10\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] > 10]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "#all_earnings_sub_df[\"more_than_15\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] > 15]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "#all_earnings_sub_df[\"more_than_20\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] > 20]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "#all_earnings_sub_df[\"equal_to_1\"] = len(all_earnings_sub_df[all_earnings_sub_df[\"edu_distance\"] == 1]) / len(all_earnings_sub_df[\"edu_distance\"])\n",
        "\n",
        "\n",
        "#people_dict = {'bob': 'boy', 'Mike': 'boy', \n",
        "           #'Katie': 'girl', 'Stacey': 'girl'}  \n",
        "#df['people'] = people_dict\n",
        "\n",
        "\"\"\"\n",
        "percentage_dict = {\"equal_to_one_EDU\": all_earnings_sub_df[\"equal_to_1\"].iloc[0],\n",
        "                   \"2-3 EDUs\": all_earnings_sub_df[\"more_than_1\"].iloc[0],\n",
        "                   \"more_than_3_EDUs\":all_earnings_sub_df[\"more_than_3\"].iloc[0],\n",
        "                   \"5\":all_earnings_sub_df[\"more_than_5\"].iloc[0],\n",
        "                   \"more_than_10\":all_earnings_sub_df[\"more_than_10\"].iloc[0], \n",
        "                   \"more_than_15\":all_earnings_sub_df[\"more_than_15\"].iloc[0],\n",
        "                   \"more_than_20\":all_earnings_sub_df[\"more_than_20\"].iloc[0], \n",
        "}\n",
        "\"\"\"\n",
        "#print(all_pubmed_sub_df[\"equal_to_1\"].iloc[0])\n",
        "\n",
        "print(percentage_dict)\n",
        "\n",
        "print(list(percentage_dict.values()))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 436
        },
        "id": "Nip7lnoaVdFM",
        "outputId": "f2d92bd6-3224-4ceb-82f5-1b0d5dfab849"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "/usr/local/lib/python3.7/dist-packages/ipykernel_launcher.py:6: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.\n",
            "  \n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 432x288 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAXQAAAFrCAYAAADB43IdAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4yLjIsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy+WH4yJAAAgAElEQVR4nO3debwcVZn/8c83CWFfzQXJAgkkKCGiYgTUUVFRAgjRHyhhBEE2wQkyoo4wKsNEcZcZFxRREcGBgLjFMQ6uOOKAJiAuAdEYEYKoETdwg8jz++Oc5lbq9lad7nsvxff9evXrdlU9VXVOneqnq05V9VVEYGZmD38TxroAZmbWH07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEXiDpQklv7NOydpF0n6SJefhaSSf1Y9l5eV+UdFy/lldhvW+W9BtJvxztdTcpyypJB4zyOkPS7C7iZubYST2sY0zmHS2Sjpd0XWH4Pkm7jWWZ6uIRk9Al3S7pL5LulfR7Sf8n6VRJD22DiDg1It7U5bIObBcTEXdExFYR8fc+lP1cSZ8oLf/giPj4xi67Yjl2AV4NzI2IRzeZfoCkB/MHtPh6yiDKExF7RcS1g1h2XRU+B8X2eX+edrykvxfG/0zSxyTtUZj/AElrmyy35wOW/DlZ06HcTddrG3rEJPTssIjYGtgVeBvwOuCj/V7JeD462ki7APdExK/bxPwif0CLr+urrqjG23A8OKzUPosL066PiK2AbYEDgb8AN0qaNyYltUoeaQkdgIj4Q0QsA44CjmvsrJIukfTm/H6KpP/OR/O/lfRNSRMkXUZKbJ/PRzH/UjjNPVHSHcDXWpz67i7pO5L+KOlzknbI6xpx9NE4C5C0APhX4Ki8vu/l6Q8dEeVyvUHSzyX9WtKlkrbN0xrlOE7SHbm75PWtto2kbfP86/Ly3pCXfyDwZWBqLsclVbe7pJdJujWfJa2R9PLCtAMkrZX0utyd87F8ZnJVLs+9uYtlfnkb5fedYveR9N087ZOSruzU1l3U59C8zD9KulPSuU3CTpD0C0l3S3pNYd4Jks6S9FNJ9+Sy79BiPdtK+mhexl1K3V6NrryJkt6V23UNcGincncjIv4eET+NiFcA3wCa1a0rkh4laVneTt8Bdi9Nf6gbS9Ihkm7J7XSXpNdI2hL4IsP73n2SpkraV9L1ud3ulvR+SZNLyz1V0k9yzAWSVJh+cmF/vEXSPnn8VEmfyp+Bn0l6ZWGefSWtzHX5laTze90uAxERj4gXcDtwYJPxdwCn5feXAG/O798KXAhskl9PB9RsWcBMIIBLgS2BzQvjJuWYa4G7gHk55lPAJ/K0A4C1rcpL+jB9ojT9WuCk/P4EYDWwG7AV8GngslLZPpzL9Xjgb8CeLbbTpcDngK3zvD8GTmxVztK8naYfSvowC3gm8Gdgn8K864G3A5vmsp4L/BU4BJiY2+SGNtuoaSwwGfg5cEZuy/8H3N9NWzepQwCzC2V+HOnAaG/gV8ALStv9itzejwPWFcp7BnADMD3X90PAFaV5G/vOZ/L0LYEdge8AL8/TTgV+BMwAdgC+Xpy3289BnnY8cF2T8ScAv2rXxhT2xybTlgJX5fLPI30OrmuxTe8Gnp7fb1/aP8qfkScB+wOT8ja7Ffjn0nL/G9iOdBC2DliQp70ol+PJpP1xNunMfQJwI3BO3m92A9YAB+X5rgeOze+3AvYf69xWfD0ij9BLfkH6IJQ9AOwM7BoRD0TENyO3YhvnRsSfIuIvLaZfFhE/jIg/AW8EXtw40tpILwHOj4g1EXEfcDawSBueHfx7RPwlIr4HfI+U2DeQy7IIODsi7o2I24F3A8dWKMvUfDRUfG0JEBFfiHTUFxHxDeBLpOTZ8CDwbxHxt8I2vC4ilke6FnFZs3IXtIptfOjfm9vy06Sk2NBLWxMR10bEDyLiwYj4Pil5P7MU9u95n/gB8DHg6Dz+VOD1EbE2Iv5G+kI6stRmSNqJ9CX1z3k5vwb+g9ROAC8G/jMi7oyI35K+nDr5bKl9Tu4Q3+oz0lHep44Azsnl/yHQ7trPA8BcSdtExO8i4qZWgRFxY0TcEBHr8776IUZu/7dFxO8j4g7Sl90T8viTgHdExIq8P66OiJ+TEvxQRCyJiPsj9e1/mOHt/QAwW9KUiLgvIm6otEEGzAkdpgG/bTL+naSj3i/l7oGzuljWnRWm/5x0NDilq1K2NzUvr7jsScBOhXHFu1L+TDq6KJuSy1Re1rQKZflFRGxXev0JQNLBkm7I3Rq/JyWqYv3XRcRfS8srl3uzctLrInYqcFcpSRfbope2RtJ+kr6eT83/QErS5fYst/nU/H5X4DONpEo6uvw7G7ZZI24T4O5C7IdIR+rk5ZXX0ckLSu3z4Q7xxc/I+lyesk1Iya5siLQvdlvGI0j7xc8lfUNtLqhL2iN3lf1S0h+BtzBy+7fa72cAP22y2F0pHZSQujwb7XIisAfwI0krJD2/TV1G3SM6oUt6Mmlnva48LR+hvjoidgMOB86U9JzG5BaL7HRUN6PwfhfSB+A3wJ+ALQrlmkj6IHS73F+QdsTisteTugCq+E0uU3lZd1VczgiSNiV1M70L2CkitgOWk053Gwb10593A9OK/acU2qJDW7dzObAMmBER25K6bVSKKbf5L/L7O4GDS4l1s4gob+s7SV1kUwpx20TEXoW6ldfRby8Evpnf3wFMkfTQAUHerrvSPFGvI+2LXZUxHzEvJH1hfZbUVQPN940Pkrqb5kTENqTEW97+rdxJqS+/MP5npXbZOiIOyeX7SUQcncv3duDqxhnoePCITOiStsnfrEtJfdM/aBLzfEmz8876B9LR04N58q9IfWtVHSNprqQtgCXA1bl74Meko8lDJW0CvIHUr9rwK2CmWl+ouwJ4laRZ+YP2FuDKiFhfpXC5LFcB50naWtKuwJnAJ9rP2ZXJpDqtA9ZLOhh4Xh+W243rSe23WNIkSQuBfRsTO7R1O1sDv42Iv0raF/jHJjFvlLSFpL2AlwFX5vEXkrbzrrkMQ7lcG4iIu0ldU+/O++0ESbtLanQtXAW8UtJ0SdsDXZ1ddKJ0sXWWpPeR+q//PZfnDuDbwNslbZW/qF9LOhAY0f2Q96lPA+fm7TAXaPr8hKTJkl4iaduIeAD4Ixt+5h6lfLE/2zrH3CfpscBpFar4EeA1kp6kZHZui+8A9ypdnN88b4d5+eAPScdIGoqIB4Hf52V1s6+MikdaQv+8pHtJ38KvB84nfciamQN8BbiPlBA+EBFfz9PeCrwhn5K9psX8zVxGuvD6S2Az4JWQ7roBXkHaye4iHbEX73r5ZP57j6RmfYoX52X/L/Az0sXB0yuUq+j0vP41pDOXy/Pyu1W8E6HxOiIi7iXV9yrgd6Tkt6zHMlYSEfeTLoSeSPoQHkO6WPa3HNKurdt5BbAk71PnMHw0WfQNUnfOV4F3RcSX8vj3kOr/pTz/DcB+LdbzUtIX4i2kbXc1qc8fUv/uNaTrIjeRkmcnny+1z2cK054i6T5SorwW2AZ4cumg5yjSEepq0v76HODQJt1lDYtJXR2/JO3/H2tTtmOB23MXyqmk60NExI9IBy5r8uduKvAa0n50b94OVzZf5EgR8UngPNL+fS/pbGCH/AX0fFJf+89IZ60fId3GCbAAWJW30XuARW2umY26xl0bZo8okr4NXBgR7ZKL2cPKI+0I3R6hJD1T0qNzl8txpNsM/2esy2XWT34azx4pHsPwvdBrgCNz/7RZbbjLxcysJtzlYmZWE07oZmY1MWZ96FOmTImZM2eO1erNzB6Wbrzxxt9ExFCzaWOW0GfOnMnKlSvHavVmZg9Lklr+dIK7XMzMasIJ3cysJpzQzcxqwgndzKwmnNDNzGrCCd3MrCac0M3MasIJ3cysJh6Wv7Y486wvdBV3+9sOHXBJzMzGDx+hm5nVhBO6mVlNOKGbmdWEE7qZWU04oZuZ1YQTuplZTTihm5nVRFcJXdICSbdJWi3prCbTd5H0dUnflfR9SYf0v6hmZtZOx4QuaSJwAXAwMBc4WtLcUtgbgKsi4onAIuAD/S6omZm1180R+r7A6ohYExH3A0uBhaWYALbJ77cFftG/IpqZWTe6SejTgDsLw2vzuKJzgWMkrQWWA6c3W5CkUyStlLRy3bp1PRTXzMxa6ddF0aOBSyJiOnAIcJmkEcuOiIsiYn5EzB8aavpPq83MrEfdJPS7gBmF4el5XNGJwFUAEXE9sBkwpR8FNDOz7nST0FcAcyTNkjSZdNFzWSnmDuA5AJL2JCV096mYmY2ijgk9ItYDi4FrgFtJd7OskrRE0uE57NXAyZK+B1wBHB8RMahCm5nZSF39HnpELCdd7CyOO6fw/hbgaf0tmpmZVeEnRc3MasIJ3cysJpzQzcxqwgndzKwmnNDNzGrCCd3MrCac0M3MasIJ3cysJpzQzcxqwgndzKwmnNDNzGrCCd3MrCac0M3MasIJ3cysJpzQzcxqoquELmmBpNskrZZ0VpPp/yHp5vz6saTf97+oZmbWTsd/cCFpInAB8FxgLbBC0rL8Ty0AiIhXFeJPB544gLKamVkb3Ryh7wusjog1EXE/sBRY2Cb+aNK/oTMzs1HUTUKfBtxZGF6bx40gaVdgFvC1jS+amZlV0e+LoouAqyPi780mSjpF0kpJK9etW9fnVZuZPbJ1k9DvAmYUhqfncc0sok13S0RcFBHzI2L+0NBQ96U0M7OOuknoK4A5kmZJmkxK2svKQZIeC2wPXN/fIpqZWTc6JvSIWA8sBq4BbgWuiohVkpZIOrwQughYGhExmKKamVk7HW9bBIiI5cDy0rhzSsPn9q9YZmZWlZ8UNTOrCSd0M7OacEI3M6sJJ3Qzs5pwQjczqwkndDOzmnBCNzOrCSd0M7OacEI3M6sJJ3Qzs5pwQjczqwkndDOzmnBCNzOrCSd0M7OacEI3M6sJJ3Qzs5roKqFLWiDpNkmrJZ3VIubFkm6RtErS5f0tppmZddLxPxZJmghcADwXWAuskLQsIm4pxMwBzgaeFhG/k7TjoApsZmbNdXOEvi+wOiLWRMT9wFJgYSnmZOCCiPgdQET8ur/FNDOzTrpJ6NOAOwvDa/O4oj2APSR9S9INkhY0W5CkUyStlLRy3bp1vZXYzMya6tdF0UnAHOAA4Gjgw5K2KwdFxEURMT8i5g8NDfVp1WZmBt0l9LuAGYXh6Xlc0VpgWUQ8EBE/A35MSvBmZjZKuknoK4A5kmZJmgwsApaVYj5LOjpH0hRSF8yaPpbTzMw66JjQI2I9sBi4BrgVuCoiVklaIunwHHYNcI+kW4CvA6+NiHsGVWgzMxup422LABGxHFheGndO4X0AZ+aXmZmNAT8pamZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTXSV0SQsk3SZptaSzmkw/XtI6STfn10n9L6qZmbXT8R9cSJoIXAA8l/S/Q1dIWhYRt5RCr4yIxQMoo5mZdaGbI/R9gdURsSYi7geWAgsHWywzM6uqm4Q+DbizMLw2jys7QtL3JV0taUZfSmdmZl3r10XRzwMzI2Jv4MvAx5sFSTpF0kpJK9etW9enVZuZGXSX0O8Cikfc0/O4h0TEPRHxtzz4EeBJzRYUERdFxPyImD80NNRLec3MrIVuEvoKYI6kWZImA4uAZcUASTsXBg8Hbu1fEc3MrBsd73KJiPWSFgPXABOBiyNilaQlwMqIWAa8UtLhwHrgt8DxAyyzmZk10TGhA0TEcmB5adw5hfdnA2f3t2hmZlaFnxQ1M6sJJ3Qzs5pwQjczqwkndDOzmnBCNzOrCSd0M7OacEI3M6sJJ3Qzs5pwQjczqwkndDOzmnBCNzOrCSd0M7OacEI3M6sJJ3Qzs5pwQjczqwkndDOzmugqoUtaIOk2SaslndUm7ghJIWl+/4poZmbd6JjQJU0ELgAOBuYCR0ua2yRua+AM4Nv9LqSZmXXWzRH6vsDqiFgTEfcDS4GFTeLeBLwd+Gsfy2dmZl3qJqFPA+4sDK/N4x4iaR9gRkR8od2CJJ0iaaWklevWratcWDMza22jL4pKmgCcD7y6U2xEXBQR8yNi/tDQ0Mau2szMCrpJ6HcBMwrD0/O4hq2BecC1km4H9geW+cKomdno6iahrwDmSJolaTKwCFjWmBgRf4iIKRExMyJmAjcAh0fEyoGU2MzMmuqY0CNiPbAYuAa4FbgqIlZJWiLp8EEX0MzMujOpm6CIWA4sL407p0XsARtfLDMzq8pPipqZ1YQTuplZTTihm5nVhBO6mVlNOKGbmdWEE7qZWU04oZuZ1YQTuplZTTihm5nVhBO6mVlNOKGbmdWEE7qZWU04oZuZ1YQTuplZTTihm5nVRFcJXdICSbdJWi3prCbTT5X0A0k3S7pO0tz+F9XMzNrpmNAlTQQuAA4G5gJHN0nYl0fE4yLiCcA7SP802szMRlE3R+j7AqsjYk1E3A8sBRYWAyLij4XBLYHoXxHNzKwb3fwLumnAnYXhtcB+5SBJ/wScCUwGnt2X0pmZWdf6dlE0Ii6IiN2B1wFvaBYj6RRJKyWtXLduXb9WbWZmdJfQ7wJmFIan53GtLAVe0GxCRFwUEfMjYv7Q0FD3pTQzs466SegrgDmSZkmaDCwClhUDJM0pDB4K/KR/RTQzs2507EOPiPWSFgPXABOBiyNilaQlwMqIWAYslnQg8ADwO+C4QRbazMxG6uaiKBGxHFheGndO4f0ZfS6XmZlV5CdFzcxqwgndzKwmnNDNzGrCCd3MrCac0M3MasIJ3cysJpzQzcxqwgndzKwmnNDNzGrCCd3MrCac0M3MasIJ3cysJpzQzcxqwgndzKwmnNDNzGrCCd3MrCa6SuiSFki6TdJqSWc1mX6mpFskfV/SVyXt2v+implZOx0TuqSJwAXAwcBc4GhJc0th3wXmR8TewNXAO/pdUDMza6+bI/R9gdURsSYi7geWAguLARHx9Yj4cx68AZje32KamVkn3ST0acCdheG1eVwrJwJfbDZB0imSVkpauW7duu5LaWZmHfX1oqikY4D5wDubTY+IiyJifkTMHxoa6ueqzcwe8SZ1EXMXMKMwPD2P24CkA4HXA8+MiL/1p3hmZtatbo7QVwBzJM2SNBlYBCwrBkh6IvAh4PCI+HX/i2lmZp10TOgRsR5YDFwD3ApcFRGrJC2RdHgOeyewFfBJSTdLWtZicWZmNiDddLkQEcuB5aVx5xTeH9jncpmZWUV+UtTMrCac0M3MasIJ3cysJpzQzcxqwgndzKwmnNDNzGrCCd3MrCac0M3MasIJ3cysJpzQzcxqwgndzKwmnNDNzGrCCd3MrCac0M3MasIJ3cysJrpK6JIWSLpN0mpJZzWZ/gxJN0laL+nI/hfTzMw66ZjQJU0ELgAOBuYCR0uaWwq7AzgeuLzfBTQzs+508x+L9gVWR8QaAElLgYXALY2AiLg9T3twAGU0M7MudNPlMg24szC8No8zM7NxZFQviko6RdJKSSvXrVs3mqs2M6u9bhL6XcCMwvD0PK6yiLgoIuZHxPyhoaFeFmFmZi10k9BXAHMkzZI0GVgELBtssczMrKqOCT0i1gOLgWuAW4GrImKVpCWSDgeQ9GRJa4EXAR+StGqQhTYzs5G6ucuFiFgOLC+NO6fwfgWpK8bMzMaInxQ1M6sJJ3Qzs5pwQjczqwkndDOzmnBCNzOrCSd0M7OacEI3M6sJJ3Qzs5pwQjczqwkndDOzmnBCNzOrCSd0M7OacEI3M6sJJ3Qzs5pwQjczqwkndDOzmujqH1xIWgC8B5gIfCQi3laavilwKfAk4B7gqIi4vb9F3Tgzz/pCx5jb33boKJTEzGwwOiZ0SROBC4DnAmuBFZKWRcQthbATgd9FxGxJi4C3A0cNosA2fnTzJQn+ovR2stHSzRH6vsDqiFgDIGkpsBAoJvSFwLn5/dXA+yUpIqKPZR3Xqn5ox+OHfDyWyboz3s5AvS+NDXXKuZKOBBZExEl5+Fhgv4hYXIj5YY5Zm4d/mmN+U1rWKcApefAxwG39qggwBfhNx6je40djHXUok+swPuJHYx2uw+DmaWfXiBhqOiUi2r6AI0n95o3hY4H3l2J+CEwvDP8UmNJp2f18ASsHGT8a66hDmVyH8RE/HsvkOgz+1c1dLncBMwrD0/O4pjGSJgHbki6OmpnZKOkmoa8A5kiaJWkysAhYVopZBhyX3x8JfC3yV5OZmY2OjhdFI2K9pMXANaTbFi+OiFWSlpBOJZYBHwUuk7Qa+C0p6Y+2iwYcPxrrqEOZXIfxET8a63AdBjdPTzpeFDUzs4cHPylqZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXhhG5mVhNO6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjVR64Qu6RmSHtXDfCq8n1QlvuryR6NMg4gvz1M1vtt19GNd3cxfNb4fZaoaPxp1qLqOfi9/NNYx6Dr3Mk8v26mZWid04AjgR5J26HYGSYqIyO9PBA6VtGmX8cdL2l/SxH7F96FMfY/vpR69rKPFvI+puL3mS9qkMdyP+D6UaTzWoeo6+l6H0VjHoOs8GnVoKyJq/QLeBawBHtVl/MT89xXATcBuXcYvBm4GZvczfiPLNJD4jax31+tosoxXAZ8ApnUZ/8/AMmDXwjj1K77HMo3HOlRdx0DrMB63U9XyjNZ2GrGMKsEPhxfwaGD70rj/AH4G7NBmvmeQkz6wC/AtYC9gE+DFwCuBpxXinwxsl9/PAW4AZuXhw4AXAXv0Gt9jmQYa32O9K6+jMO8mhfdHA//XaFvgUcDkUvzWhfcLc9m2ycMzgM3y+wm9xPdYpvFYh6rrGGgdxuN2qlqe0dpOnV6V+jHHO0lTgXcA10j6fET8HiAiXiXpwTz+WRFxX5PZjwA+JemxEXGHpG8C5wP3AA8Cvwe2BL6V+7eOAfaX9DxgNemo89/ytO2AvwNbAT+uGt9LmUYjvsd6VC1Toy2fCLxI0nsi4lfArsC1wExJi4Gn5yIdFhF/lfRY4ERJV0fEt4Eh4BZgL0mHAM8GdpT0+Ij4c9X4Hss0HutQdR0DrcN4bOuq5RmNOtCtbjP/w+UFnABcAixi5JH6JcDCNvM2ume2AaYCLwVm5mmnAZcz3HUwGXg38BVgM+CpwBnAnnn6a4ELyadMVeN7KdMoxVeuR9V15PFTga8CbwK2Bubl4a8BxwOPBS4id/UAU/Lw23LsNsA1wHLSmcBk4OPAk3uJ77FM47EOVdcx0DqMx+1UtTyjtZ26yn9jnYD7lMR3Ip3+N07tDyEliQ2SOnAB8KLCcLPumfcAPyF3zwDKDXIzsB8bniZOBt6bG6J4+nRsjn9yxfg9eyjTMwYcvxewbcV6PL3qOlq0687A54ElwA7AJPJpK/BC4IfAToX4R+X1vAt4bGlZLyQdAU3tNb7HMo3HOlRdx0DrMB63U9XyjNZ26pgLqwSPxxdwKOnU5n9J34iXA9sDzwMuA04hfTs+H/gO+WIc6Rv1E6QktF1pme8GVpC6DnYB3gIcBHwR+AdGJrfzgatIR6x7kb55q8bP66FMzxpw/DxgVsV6VF5HYfpc4J9K80wFPgecR/oS2Ax4CXArqd/xHaX4KcB/Au/MZd+E9MV+K/CCivHzeijTYeOwDvMqznP4gOvQy3Yd9HaqWue9RqEO84qxXeXDsU7IG/MCFgA3As8BNgf2JiWZb5OS+tNJpzTLSUeTjyvN36l75vn5fePixL+RvoGfwobdEDsBlwJPysNb9RLfY5kGGt9jvauWScBE0qnmx4FTS/PsDFwHnJWH9wRmAvOBq4HzmnxILm2Mz8O79BBftUy7jcM6qOI8M0ehDuOtrXup88Dr0FNO7GWm8fACHgf8jZwcCuO3ICX1i/POvAnpYt22eXqV7pmjSN0NxVOxU4H/ISW3SYXxnyAdlVeK77FMA43P76vWu/I6mrTpZNJRzIcZeeQzH1jaZJ6984fh7aXxu5HOLLbcyPhKZRqPdag6zyjVYVxtpx7LM/A6VH09LO9ykTQB+C3p2/EgSddHxD2SJkS6Sv1J0inPtpHudPl9nu9Q0kW7CcADkn4F/BOwntRFsI2k/wVmk/q/bwSWAttJuo90W9F5wAPAG4D3Svou8CRgD1L/d5X4W3so003AlQOMf3e+yn56hXrsDJxdZR2Fttwf2Id0Ien2iLgyt+9Bkl4RER/IoY8HJkg6gHQhdgpwZ0T8h6T3AK+Q9PaIeF2O3490x81+eR3dxq/voUxPy+/HUx2eTjpD7XaefSQ9boB16GW7Dno7Va3zJFKiHmQd1rMRGndgPGxIWggcALwa2J105DgFOCMnbyRtDXwKODbSLURIWkBKSv9Cuj90Dumi3NNIXTfzSP3xe5NOpz5NegjmX4Dvkjb4waSunJeRbsd7Nunix/bAFRXjTwemVSzTZ4CTBxj/GlIf4Dsq1GNp1XVExA9ymxxM6j+8gnSV/whS185ZeR0vyOO/SroW8sHc7hflZZ0IfIP0BbEZ8DrSmcJyUr/lh4AzK8S/lHT/b5UyXZjXMZ7qMJ10sa3beSlQIeEAABB8SURBVC4iPQQzqDr0sl0HvZ2q1vkYUlfOIOvw0oj4IRtjYw7vR/tFutB5I3BIYdw0UhfLpQzfxH8C8GXyHRhU7555JumofkEp/rHAB4DX5+GdSAnwORXjh3oo01MGHL8t6Yi7Sj2eUXUdhZjDSfee/0Nh3HbAKuD9eXhH4FzSbZGnkC6oPr0QvyXpS+sjhXFnks4KTqoY/5geyvSKcViHgyvOc9qA69DLdh30dqpa5z1HoQ6P6UuOHETiHcSLlDRvB/bLw7sBr83v55Bu/Xkf6Yh9BfkKMakbYBrpW/J9DPfxNp4IewrpXtHtCut6Mqk75wzyk1uFaYtIXxbaiPhKZRp0fC/16HUdedomuS0/XRjXuEC6XZ5WvL10IvBN4OrCuMbtYJuTHnB6Za/xPZZpPNah6joGWofxuJ2qlme0tlO/Xg+LH+eStDnpdrD/A74naQrplsQHASLiJ6TTm61IN/YfFxE/zN0z7wbuJt0WdD9wvqTtIuLBvPgfkvqtNpV0mKQ3RsQK0sW+WcCZkrYqFOe6/HfLqvG5LlXLNND4XKaq9a68jgZJe0TEA6Sj+/mS3gQQ6em5zSN1m32NdMaApL1J3TsvBnaX9JYcf7+kTSPiL6Run616ie+xTOOxDlXXMdA6jMftVLU8o7Wd+ulhkdDzRvgY8FPSbYjXAZdExLulh35m8m7gjcDjI+IWpUfTzwG+HBEP5qTfeOT8vZK2z/O9iNQtsD/pdGlFHv8V0i1G2wGvzf3ykE5p/0Tqx68S//ceyzTI+D/nMlWtd6V15GGULrYuzR+SO0j3tp8gaQk81M4AfwD+kvvYLwEeHRF3k/riD5F0Xo7/W46fCNxbNb7HMo3HOlRdx0DrMB7bump5RqMOeR09/UxuS4M47B/Ui+H7zD/Hhr9IdjLptH9SHq7aPXMycBv5R6JI94y+NL9/LunHvc7IcTf1ED+vhzKdOOD4ebmsVepReR2FNlpAOsN6Xh5udA/tAtwFLMnDLyH1Sx5H6rNsxDe6dKYC3wPenIePIT1R97KK8bN7LNN4rEPVdQyyDr1u10Fup6p1njUKdej4K6s95cixTtItEne7n7Hci/SE4RLSlfP/R3qQaO88fXPSlejLSVeSp+QN/erCMnYnHfGvI/Ubfwa4IE97dG7I0/LwBOBA0q1/a3uIn9tDmZ444Pi5pKvxVepReR2F8fNI3TIvKMR9i3Q2RW7Hn5P6Hb9Juof9QeCIUvxTCx+Sm4Cvk26prBq/Vw9lWjgO67BHxXkGXYdetuugt1PVOo9GHZr+1EWdE/oWjaTSYvrjSX3l/006wpzbYvp/Aj8CTi5+UZDuwphOfnCGdMvRR0n3WN9MfvKrEC9SQtull/geyzTQ+B7rXXkdefws0m1lbyedZX2FdPsiDP/Y2XTSHUx7k34D472kL5x9SBdjX12K34n029F7VY3vsUzjsQ5V1zHQOozHtq5antGow0Bz51gn7ybJ+jDS/c/75uF2Sf08WtzuQ5fdM4XxB5D6ji9nw8feXw68pcnyK8V3WyY2vBumm/gJVetcWkfVelfaroXps0ldOPcx/Eh0446Yp5Ee1CjWZSLpfvgHGb5dshF/IKm7R73G91im8ViHqusYaB3G43aqWp7R2k6DeI15Ai9txHmkI7+LSd9+nZL6Jk3GFRuybfdMedmkix6Xku4vnky6sPdthk+1yo3eNr5FmVuVab+K8QfS/Ef2O9a5ar2rbtdSbPH9LOD9pDuSGj/FcDzpR9N2blKXSflD8mVgSil+lybL7xjfpO5Vy9T3OvQSX6pDN9upUp173K7jqq03pp1Ho60H8RrzJF7aIDsxfFHuNNLFhBFJnZFHTfMpHF2WllnunjmM9ODQji3in0X6Qvkv4AekixjHNlt3i/i5eR1ntKlnuUyL847Tqkzl+JOB6xn+5chymUZ0SZHugDk2/232RVCux4K8XScwfNo4sd06ijt2adnF+WaTLp6+P7fxN2nzq3KFD8nyvJ1GxNM8uRXj9yrXuck+VC7TU/K0EUdVLeIPAoYq1OGoxjq6jN+L9CV+fIV59mXD3yvpVOfyl/7ELso03tr68W3im5aH4Queg2rrgXazbLD+0VpR1wXa8IefTiUl9cZdFbuz4b95ej7wfdLFhisYfpiofArY6J45Kcd/FvgChf/1V2r4Z5OOWPchXcG+hcIvqjGya6QR/1jS06w3A89tUrcJTcp0DOmspFl8OYGeR3oK9mbS3SbvabMdH+qSIj3l9n3SQ0NXA3NarKNRj3/K2/1TpP7yV5ATQ4syPaYw7uC8jrPZ8IuwON/upC+P35Dv1+2wT0wk/ajXvbn+Hyf1+7+wi/i9cptcDbwVWFSIU5sy3Qfs36ZMxfhTSRfBdinFlLsJGmU6ndSt+OxSTPkUvliH55IuTn+BdFtcp3q/nPSz0ldSOLhoU+eTSc8gnAEc06LdymUab219csV23ot00DLIth61ZB4xDhN6k415Kulo9HzS76s0Tn2eSvrN4Cfm4Q8AF7dZznNI/xatccT/GeDAYmOUPlCb5r//Qvo9hkuBV5WWuUF8LtOvCuvYlvSvqLZk+Ch3g/+jSHr0t3HBZSrpg7tfoZ7FD8dBpKfMGv+T80vAM9rUeRPSj+hfw/CX3cWkLpUdGX7arbiOnUm/+jY3D59AugXxjQz/lEJx5y1+we5L+icWLyH9T8XvU+iHL61nF9JR6oOkL46O/2Q6L3NV3ifOICWgPdvE70g60/ge6ezklcB7y3GlMu1MOlv5S673P5TiVYo/Ou+HTY/oGZmk55ISbeMgpXxwUE4MO+Z2vzG3xVKG99sRXZF5noWks6xDSF9mF3ao8wtzHY4ifcn8AHhDmzrsOA7b+qiq7ZzfD7Ktm55xD/I1Zom6UiHTt+46NuzTfSqF00/S76N8lpyIy41C+j2GZ+X3jwZ+keM/ROrnGtENkWPPJB2lPod0FnA+6QhAjDzdfAzpKGohKZF+nXTqdRXpXthmff6vZDih/19ex2V5By7/lvhBDN8KtR3pIuRpxXo2Wf62pH/+cSTpVsU1pB8Uuhx4MyN/cnRb0mniswvjPkn6saejO6zrH4APF4Z3Jv1z7vMK4xrtsTnpx4leSjqlvZQ2H3TSbZKfBJ6Th3ciHX0+q808O+blHpCHn0d6qu9I8i1peXy5y2oT0i9gnkJ6mG12sw9nboMryAcSefiNpAT0knKdC/X4MulaxfTcDv/FhsmwmBhm5vZ7eh4+g5TcWyYLUpJt7FP7kM4GziY/M9BkHUez4e2n/0p6IOz1bdZRpa23qNjWQ1Xautd2rtjWjR/g67qtx+I1ZivuuoApkd7CyH9OMZHh/5A9MX84vkvu28qN3Opi6uvJRyCkZL6U1qexuzN8lfvVeUe/oE15H09KmmtJp4ATSEdWV9Ck3430I1235TK8LI/bjfQrfge1WEfj6vkC4JflbdMk/sicBG4A3pjHPZv0VNuIC7iko6LGfx06L79/OfDRJrHFWxyfRLo1a4fC9EeT7sF9cZN5ZxXm/yDp9HqPNvV4BikhNup/HvCvHeo+Lf99FOlU+SJS19u3afJlmPel7UhfxFNJX853kZ6SncXIo7JDcjnemvfTd5ASxHeAU5pso0mkpPYs0j37p5NuDV1B6b/ZNKnDBNIX7geBg0v7QrEOLyGd1S7O++JbSUnuduDfmsSfmPePxl1QZ+dyfZMWZ4CkrorPV2zrRlm7beuhbtsamN6pnZvMM5GUqKu09VvatfVYv8a8AB0LmL75d+0QM4n02whfLezQHyTfz97FOpYD+7SYNjV/8E4mnWKek3fkl7dZ3lxgcWnc/wBPaBF/GOnoZklh3Icp9GW2WdeS/AEULb7Actz2pN9deX5h3KeAw5vEbpu34cXA+YXx/83IH+3aojT8QeA7pXEnAqe3iC8mlgvzB33LvP4XtGtD0hfzOwvb8IBm6yjsRwcWhhcx/PTeQ889MJx8TyWd1c0gfWneSf6/j03qcDDpiHJxYdw/Mvw04RbFupJ+pvhbpGcAGtNmkPrHtyivg5GJ5a3A5zu0w0mks8LLC+OeSPoSn9gk/r9Iye9y4No87k3AYYWYxg0Im+Th93Vo68bv3ze7CN+srWeTunI2Kde9WVsX4jctxE9t1c6lOmzG8JdFu7ZurKPxZffcdm091q8xL0BfK5N21reSjjaaHrU2+XAckeObHqHnmCXAHY2dm3R0NaNCuRrr2KnF9Emk09E1+QNxIrCSDheQCsu+jkIfYZvYg0lfTs8jXSi9CZjZJr54Wv5SUpdQ8a6JDZ4ZKIy/PCeHR+fh1+UP7+E0ecaADfs330rqj72ddDG2ZTypq+C0/CH7DulsasRzDOU2L5TpoiZlaiz7OFJf8hrSXVHH5jY8okWd55S211mk7rxW2+jfSWdmzyZ1v7yY1BWzWbM6lPddUnfeSR3aYRbpzK8x/ymkL+UXtojfH3gCw8nrneTExYY3IFxF3jfz8pu19WE0v2FhcmF9xbY+rRC/tBDf+PIot/VJzZbfpp0nMPImisflmBPatHUj/kryxX9S12qzth7T7paImiR00tHpZFIf2B0U7uJoM8+mpMS5qtXOUIidQf6/mcUPWJflOoF0etbxajepv/MtpF8ybNuNUprvKtok5kLcdqQ++2+QLpS2vF++NF+jDo8rjGv6zEBh+vtIF7GXMvwPcls+Y8BwIj0B+F2b+GJSez7pgta3Gf5nyB2fY8gf2JtISadpHUhHsZdROIMhXbdpWedC3DGkhNly+TnudaQnDC8jdYft3akOhe10NulH1Z7QYR2fyG33TtJFw6bbtUkdTiQ93j6b5jcgXNamrV/SJP7i4rYttXXb+CZtfUyn+FI779miDh9rbNsmbX1Qk/iPt2nrphdsR/s15gXoa2VSf3hXtwmRLoYcQoUflqfiNzApoR9APn0bQH17OiIgPeW2TYX4XSldxKLNMwOFmD1Ip9wz28UznKy2Jp1a790hvpEQDgJ+3Shbl+t4KsP3ULeLF8PdH5tUWP4TSUfabZdf2EaPIh3dT+t2Hfn9nqTrRt2s4x9JSXOPLuuwK+lGgHmFbXZ8YXmNGxCK/9z8MYW27njDAukCfaOtW8Yz3OWygHQH2ewul/800sXkvTrUYfM83Pi7SYf44tPU+5DuNBvVWxPbflbHugB9rcw4OOV5JL1o/8zATEY+zNMufjfSkdKkLuMbTwPuVGEdu+a/23WI3z8Pb/DcQzd1zn+37TJ+FqW7srqYZ3Z5ng7xMxh5N1bbMuW/xcTV6QaEKaVldrxhoViOLuJ3KLZ1F/GNXzrcvosyNcqyIyPvuW9b5/K+NB5eY14Av+rzosUzA13Gf4YOZw2l+M+R74uvOE+VdVStw2f7vfw+lKmvdaDiDQgt4i+sGP9h8hF0l/EXtYrvYx3armOsXmNeAL/q9aLJMwNjGT8ey1STOlxChxsQxnP8aK1jtF9jXgC/6vOixTMDYxU/Hsv0cK8DFW9AGG/xo7WOsXo1LjiYbTRJO5P6zX8+HuLHY5nqUIc8z/HAiohY9XCMH611jDYndDOrTJKiQvIYb/GjtY7R5oRuZlYTE8a6AGZm1h9O6GZmNeGEbmZWE07oZmY14YRuZlYTTuhmZjXx/wHiV2yPkP2MywAAAABJRU5ErkJggg==\n"
          },
          "metadata": {
            "needs_background": "light"
          }
        }
      ],
      "source": [
        "import numpy as np\n",
        "import matplotlib.pyplot as plt\n",
        "plt.bar(*zip(*percentage_dict.items()))\n",
        "#plt.hist(x =list(percentage_dict.values()) , bins= [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25])\n",
        "\n",
        "ax1 = plt.subplot()\n",
        "ax1.tick_params(axis='x', which='major', pad=75)\n",
        "ax1.set_xticklabels([\"1 EDU\",\"2 EDUs\",\"3 EDUs\",\"4 EDUs\", \"5 EDUs\",\"6 EDUs\",\"7 EDUs\",\"8 EDUs\",\"9 EDUs\",\"10 EDUs\",\"11 EDUs\",\"12 EDUs\",\"13 EDUs\",\"14 EDUs\", \"15 EDUs\",\"16 EDUs\",\"17 EDUs\", \"18 EDUs\",\"19 EDUs\",\"20 EDUs\",\"21 EDUs\",\"22 EDUs\",\"23 EDUs\",\"24 EDUs\",\"25 EDUs\"], rotation=45)\n",
        "#plt.xticks(ticks=np.arange(min(percentage_dict.values()), max(percentage_dict.values())+2, 10.0))\n",
        "plt.title(label=\"Distribution of Earnings labeled EDU distances\", fontdict=None, loc='center', pad=None)\n",
        "plt.show()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 321
        },
        "id": "MRcHmFucLCSp",
        "outputId": "94e52f14-8403-4921-ccb7-c68b88ba3f94"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "{'equal_to_one': 0.7700650759219089, 'more_than_1': 0.2299349240780911, 'more_than_3': 0.20715835140997832, 'more_than_5': 0.18655097613882862, 'more_than_10': 0.12472885032537961, 'more_than_15': 0.07266811279826464, 'more_than_20': 0.04772234273318872}\n",
            "[0.7700650759219089, 0.2299349240780911, 0.20715835140997832, 0.18655097613882862, 0.12472885032537961, 0.07266811279826464, 0.04772234273318872]\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 432x288 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAW4AAAD4CAYAAADM6gxlAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4yLjIsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy+WH4yJAAALhklEQVR4nO3db4hldR3H8c+nXaVapX9eRPzTVIQQQSoXIxIpQ9GK/kCEC0VFMD3IUAjKepI9CCoq7EEEU1lGZvRHK4r+CCkVlHV320p3tUxWcjH3ioSuDxL104M5i9N479xzpzlz53vv+wXD3Llzdvb748jbu785Z8ZJBACo41mzHgAAMB3CDQDFEG4AKIZwA0AxhBsAitndxRc95ZRTsrS01MWXBoC5tG/fvoeS9Noc20m4l5aWNBgMuvjSADCXbN/X9li2SgCgGMINAMUQbgAohnADQDGEGwCKIdwAUMzEcNs+2/aBNW+P2L5qO4YDADzTxOu4k9wt6RxJsr1L0hFJN3c8FwBgjGm3St4g6R9JWl8oDgDYWtPeOXm5pBtHfcL2sqRlSTrrrLM2PdDS1T+d6vjDn37Tpv8uAKio9Stu2ydKeouk7436fJKVJP0k/V6v1e32AIBNmGar5DJJ+5M82NUwAIDJpgn3Xo3ZJgEAbJ9W4ba9R9LFkm7qdhwAwCStvjmZ5DFJL+p4FgBAC9w5CQDFEG4AKIZwA0AxhBsAiiHcAFAM4QaAYgg3ABRDuAGgGMINAMUQbgAohnADQDGEGwCKIdwAUAzhBoBiCDcAFEO4AaAYwg0AxRBuACiGcANAMYQbAIpp+1ven2/7+7bvsn3I9mu6HgwAMFqr3/Iu6YuSfp7kHbZPlPTcDmcCAGxgYrhtP0/ShZLeK0lJHpf0eLdjAQDGabNV8hJJQ0lft/0n21+1vWf9QbaXbQ9sD4bD4ZYPCgBY1SbcuyWdJ+nLSc6V9Jikq9cflGQlST9Jv9frbfGYAIDj2oT7fkn3J7m9+fj7Wg05AGAGJoY7yb8k/dP22c1Tb5B0sNOpAABjtb2q5EOSbmiuKLlX0vu6GwkAsJFW4U5yQFK/41kAAC1w5yQAFEO4AaAYwg0AxRBuACiGcANAMYQbAIoh3ABQDOEGgGIINwAUQ7gBoBjCDQDFEG4AKIZwA0AxhBsAiiHcAFAM4QaAYgg3ABRDuAGgGMINAMUQbgAohnADQDGtfsu77cOSHpX0pKQnkvAb3wFgRlqFu/H6JA91NgkAoBW2SgCgmLbhjqRf2t5ne3nUAbaXbQ9sD4bD4dZNCAD4H23DfUGS8yRdJumDti9cf0CSlST9JP1er7elQwIAntYq3EmONO+PSrpZ0vldDgUAGG9iuG3vsX3y8ceSLpF0R9eDAQBGa3NVyamSbrZ9/PhvJ/l5p1MBAMaaGO4k90p61TbMAgBogcsBAaAYwg0AxRBuACiGcANAMYQbAIoh3ABQDOEGgGIINwAUQ7gBoBjCDQDFEG4AKIZwA0AxhBsAiiHcAFAM4QaAYgg3ABRDuAGgGMINAMUQbgAohnADQDGtw217l+0/2f5JlwMBADY2zSvuKyUd6moQAEA7rcJt+wxJb5L01W7HAQBM0vYV97WSPiLpqXEH2F62PbA9GA6HWzIcAOCZJobb9pslHU2yb6Pjkqwk6Sfp93q9LRsQAPC/2rzifq2kt9g+LOk7ki6y/a1OpwIAjDUx3Ek+luSMJEuSLpf0qyTv6nwyAMBIXMcNAMXsnubgJLdJuq2TSQAArfCKGwCKIdwAUAzhBoBiCDcAFEO4AaAYwg0AxRBuACiGcANAMYQbAIoh3ABQDOEGgGIINwAUQ7gBoBjCDQDFEG4AKIZwA0AxhBsAiiHcAFAM4QaAYgg3ABRDuAGgmInhtv1s23+w/Wfbd9r+5HYMBgAYbXeLY/4j6aIkx2yfIOm3tn+W5PcdzwYAGGFiuJNE0rHmwxOat3Q5FABgvFZ73LZ32T4g6aikW5LcPuKYZdsD24PhcLjVcwIAGq3CneTJJOdIOkPS+bZfOeKYlST9JP1er7fVcwIAGlNdVZLk35JulXRpN+MAACZpc1VJz/bzm8fPkXSxpLu6HgwAMFqbq0pOk3S97V1aDf13k/yk27EAAOO0uarkL5LO3YZZAAAtcOckABRDuAGgGMINAMUQbgAohnADQDGEGwCKIdwAUAzhBoBiCDcAFEO4AaAYwg0AxRBuACiGcANAMYQbAIoh3ABQDOEGgGIINwAUQ7gBoBjCDQDFEG4AKGZiuG2faftW2wdt32n7yu0YDAAw2sTf8i7pCUkfTrLf9smS9tm+JcnBjmcDAIww8RV3kgeS7G8ePyrpkKTTux4MADDaVHvctpcknSvp9i6GAQBM1jrctk+S9ANJVyV5ZMTnl20PbA+Gw+FWzggAWKNVuG2foNVo35DkplHHJFlJ0k/S7/V6WzkjAGCNNleVWNLXJB1K8oXuRwIAbKTNK+7XSnq3pItsH2je3tjxXACAMSZeDpjkt5K8DbMAAFrgzkkAKIZwA0AxhBsAiiHcAFAM4QaAYgg3ABRDuAGgGMINAMUQbgAohnADQDGEGwCKIdwAUAzhBoBiCDcAFEO4AaAYwg0AxRBuACiGcANAMYQbAIoh3ABQDOEGgGImhtv2dbaP2r5jOwYCAGyszSvub0i6tOM5AAAtTQx3kl9LengbZgEAtLBle9y2l20PbA+Gw+FWfVkAwDpbFu4kK0n6Sfq9Xm+rviwAYB2uKgGAYgg3ABTT5nLAGyX9TtLZtu+3/f7uxwIAjLN70gFJ9m7HIACAdtgqAYBiCDcAFEO4AaAYwg0AxRBuACiGcANAMYQbAIoh3ABQDOEGgGIINwAUQ7gBoBjCDQDFEG4AKIZwA0AxhBsAiiHcAFAM4QaAYgg3ABRDuAGgGMINAMUQbgAoplW4bV9q+27b99i+uuuhAADjTQy37V2SviTpMkmvkLTX9iu6HgwAMFqbV9znS7onyb1JHpf0HUlv7XYsAMA4u1scc7qkf675+H5Jr15/kO1lScvNh8ds373JmU6R9FDbg/2ZTf4tO9NUa59Di7x+1r64jq//xW3/QJtwt5JkRdLK//t1bA+S9LdgpHIWee3SYq+ftS/m2qXNrb/NVskRSWeu+fiM5jkAwAy0CfcfJb3c9ktsnyjpckk/7nYsAMA4E7dKkjxh+wpJv5C0S9J1Se7scKb/e7ulsEVeu7TY62fti2vq9TtJF4MAADrCnZMAUAzhBoBidky4F/22etuHbf/V9gHbg1nP0yXb19k+avuONc+90PYttv/evH/BLGfs0pj1X2P7SHP+D9h+4yxn7IrtM23favug7TttX9k8P/fnf4O1T33ud8Qed3Nb/d8kXazVG3z+KGlvkoMzHWwb2T4sqZ9k7m9EsH2hpGOSvpnklc1zn5X0cJJPN//jfkGSj85yzq6MWf81ko4l+dwsZ+ua7dMknZZkv+2TJe2T9DZJ79Wcn/8N1v5OTXnud8orbm6rXyBJfi3p4XVPv1XS9c3j67X6H/RcGrP+hZDkgST7m8ePSjqk1buz5/78b7D2qe2UcI+6rX5TCyoskn5pe1/z4wMWzalJHmge/0vSqbMcZkausP2XZitl7rYK1rO9JOlcSbdrwc7/urVLU577nRJuSBckOU+rP4Xxg80/pxdSVvfvZr+Ht72+LOllks6R9ICkz892nG7ZPknSDyRdleSRtZ+b9/M/Yu1Tn/udEu6Fv60+yZHm/VFJN2t1+2iRPNjsAR7fCzw643m2VZIHkzyZ5ClJX9Ecn3/bJ2g1XDckual5eiHO/6i1b+bc75RwL/Rt9bb3NN+skO09ki6RdMfGf2ru/FjSe5rH75H0oxnOsu2OR6vxds3p+bdtSV+TdCjJF9Z8au7P/7i1b+bc74irSiSpuQTmWj19W/2nZjzStrH9Uq2+ypZWfwzBt+d5/bZvlPQ6rf44ywclfULSDyV9V9JZku6T9M4kc/kNvDHrf51W/6kcSYclfWDNnu/csH2BpN9I+qukp5qnP67Vvd65Pv8brH2vpjz3OybcAIB2dspWCQCgJcINAMUQbgAohnADQDGEGwCKIdwAUAzhBoBi/gtmMfeCKUu8OgAAAABJRU5ErkJggg==\n"
          },
          "metadata": {
            "needs_background": "light"
          }
        }
      ],
      "source": [
        "all_pubmed_sub_df[\"equal_to_1\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] == 1]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_1\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 1]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_3\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 3]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_5\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 5]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_10\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 10]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_15\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 15]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_20\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 20]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"equal_to_1\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] == 1]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "\n",
        "\n",
        "#people_dict = {'bob': 'boy', 'Mike': 'boy', \n",
        "           #'Katie': 'girl', 'Stacey': 'girl'}  \n",
        "#df['people'] = people_dict\n",
        "\n",
        "\n",
        "percentage_dict = {\"equal_to_one\": all_pubmed_sub_df[\"equal_to_1\"].iloc[0],\n",
        "                   \"more_than_1\": all_pubmed_sub_df[\"more_than_1\"].iloc[0],\n",
        "                   \"more_than_3\":all_pubmed_sub_df[\"more_than_3\"].iloc[0],\n",
        "                   \"more_than_5\":all_pubmed_sub_df[\"more_than_5\"].iloc[0],\n",
        "                   \"more_than_10\":all_pubmed_sub_df[\"more_than_10\"].iloc[0], \n",
        "                   \"more_than_15\":all_pubmed_sub_df[\"more_than_15\"].iloc[0],\n",
        "                   \"more_than_20\":all_pubmed_sub_df[\"more_than_20\"].iloc[0], \n",
        "}\n",
        "\n",
        "#print(all_pubmed_sub_df[\"equal_to_1\"].iloc[0])\n",
        "\n",
        "print(percentage_dict)\n",
        "\n",
        "print(list(percentage_dict.values()))\n",
        "\n",
        "x = list(percentage_dict.values())\n",
        "\n",
        "plt.hist(x,bins = [i for i in range(25)])\n",
        "plt.show()"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "percentage_dict = {\"equal_to_one_EDU\": all_earnings_sub_df[\"equal_to_1\"].iloc[0],\n",
        "                   \"two_edus\": all_earnings_sub_df[\"equal_to_2\"].iloc[0],\n",
        "                   \"three_edus\": all_earnings_sub_df[\"equal_to_3\"].iloc[0],\n",
        "                   \"four_edus\": all_earnings_sub_df[\"equal_to_4\"].iloc[0],\n",
        "                   \"five_edus\": all_earnings_sub_df[\"equal_to_5\"].iloc[0],\n",
        "                   \"six_edus\": all_earnings_sub_df[\"equal_to_6\"].iloc[0],\n",
        "                   \"seven_edus\": all_earnings_sub_df[\"equal_to_7\"].iloc[0],\n",
        "                   \"eight_edus\": all_earnings_sub_df[\"equal_to_8\"].iloc[0],\n",
        "                   \"nine_edus\": all_earnings_sub_df[\"equal_to_9\"].iloc[0],\n",
        "                   \"ten_edus\": all_earnings_sub_df[\"equal_to_10\"].iloc[0],\n",
        "                   \"eleven_edus\": all_earnings_sub_df[\"equal_to_11\"].iloc[0],\n",
        "                   \"twelve_edus\": all_earnings_sub_df[\"equal_to_12\"].iloc[0],\n",
        "                   \"thirteen_edus\": all_earnings_sub_df[\"equal_to_13\"].iloc[0],\n",
        "                   \"fourteen_edus\": all_earnings_sub_df[\"equal_to_14\"].iloc[0],\n",
        "                   \"fifteen_edus\": all_earnings_sub_df[\"equal_to_15\"].iloc[0],\n",
        "                   \"sixteen_edus\": all_earnings_sub_df[\"equal_to_16\"].iloc[0],\n",
        "                   \"seventeen_edus\": all_earnings_sub_df[\"equal_to_17\"].iloc[0],\n",
        "                   \"eighteen_edus\": all_earnings_sub_df[\"equal_to_18\"].iloc[0],\n",
        "                   \"nineteen_edus\": all_earnings_sub_df[\"equal_to_19\"].iloc[0],\n",
        "                   \"twenty_edus\": all_earnings_sub_df[\"equal_to_20\"].iloc[0],\n",
        "                   \"twenty_one_edus\": all_earnings_sub_df[\"equal_to_21\"].iloc[0],\n",
        "                   \"twenty_two_edus\": all_earnings_sub_df[\"equal_to_22\"].iloc[0],\n",
        "                   \"twenty_three_edus\": all_earnings_sub_df[\"equal_to_23\"].iloc[0],\n",
        "                   \"twenty_four_edus\": all_earnings_sub_df[\"equal_to_24\"].iloc[0],\n",
        "                   \"twenty_five_edus\": all_earnings_sub_df[\"equal_to_25\"].iloc[0],\n",
        "\n",
        "                    \n",
        "}\n",
        "\n",
        "print(list(percentage_dict.values()))\n",
        "\n",
        "x = list(percentage_dict.values())\n",
        "\n",
        "plt.hist(x,bins = [i for i in range(25)])\n",
        "plt.show()\n",
        "\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 303
        },
        "id": "xhreQfUIHTaU",
        "outputId": "57788d46-fde8-4a27-e3fd-5dc9ed953518"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "[0.8461538461538461, 0.01201923076923077, 0.002403846153846154, 0.004807692307692308, 0.004807692307692308, 0.002403846153846154, 0.0, 0.004807692307692308, 0.007211538461538462, 0.0, 0.002403846153846154, 0.0, 0.0, 0.0, 0.009615384615384616, 0.0, 0.004807692307692308, 0.0, 0.002403846153846154, 0.004807692307692308, 0.002403846153846154, 0.0, 0.0, 0.002403846153846154, 0.002403846153846154]\n"
          ]
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "<Figure size 432x288 with 1 Axes>"
            ],
            "image/png": "iVBORw0KGgoAAAANSUhEUgAAAXQAAAD4CAYAAAD8Zh1EAAAABHNCSVQICAgIfAhkiAAAAAlwSFlzAAALEgAACxIB0t1+/AAAADh0RVh0U29mdHdhcmUAbWF0cGxvdGxpYiB2ZXJzaW9uMy4yLjIsIGh0dHA6Ly9tYXRwbG90bGliLm9yZy+WH4yJAAALeUlEQVR4nO3cX6ik9X3H8fenWXujXqzsYVmsdluRghS6hoMpKMGSJhh7od5I9yLdQmC9UFDIRcWbeFOwJZreFGFFyRaMJaBWoaGNiGADRXJWlri6pIawUpd194gF9Sqo316cR3rY7jh/zzme77xfcJiZZ5458/3xyNvZ58xMqgpJ0u73Ozs9gCRpMQy6JDVh0CWpCYMuSU0YdElqYs92Ptm+ffvq4MGD2/mUkrTrnThx4v2qWhm337YG/eDBg6ytrW3nU0rSrpfknUn285SLJDVh0CWpCYMuSU0YdElqwqBLUhMGXZKaGBv0JNckeSXJW0neTHL/sP3hJGeTnBx+bt/6cSVJo0zyPvRPgO9V1etJrgROJHlpuO+HVfWDrRtPkjSpsUGvqnPAueH6R0lOA1dv9WCSpOlM9UnRJAeBG4HXgJuB+5L8FbDGxqv4/7nEY44CRwGuvfbamQc9+OC/Tv2YM4/8xczPJ0m7zcR/FE1yBfAs8EBVfQg8DlwHHGLjFfyjl3pcVR2rqtWqWl1ZGftVBJKkGU0U9CSXsRHzp6vqOYCqOl9Vn1bVZ8ATwE1bN6YkaZxJ3uUS4EngdFU9tmn7gU273QWcWvx4kqRJTXIO/WbgO8AbSU4O2x4CDic5BBRwBrhnSyaUJE1kkne5/BzIJe766eLHkSTNyk+KSlITBl2SmjDoktSEQZekJgy6JDVh0CWpCYMuSU0YdElqwqBLUhMGXZKaMOiS1IRBl6QmDLokNWHQJakJgy5JTRh0SWrCoEtSEwZdkpow6JLUhEGXpCYMuiQ1YdAlqQmDLklNGHRJasKgS1ITBl2SmjDoktSEQZekJgy6JDVh0CWpCYMuSU0YdElqYmzQk1yT5JUkbyV5M8n9w/arkryU5O3hcu/WjytJGmWSV+ifAN+rqhuAPwXuTXID8CDwclVdD7w83JYk7ZCxQa+qc1X1+nD9I+A0cDVwB3B82O04cOdWDSlJGm+qc+hJDgI3Aq8B+6vq3HDXe8D+EY85mmQtydr6+voco0qSvsjEQU9yBfAs8EBVfbj5vqoqoC71uKo6VlWrVbW6srIy17CSpNEmCnqSy9iI+dNV9dyw+XySA8P9B4ALWzOiJGkSk7zLJcCTwOmqemzTXS8CR4brR4AXFj+eJGlSeybY52bgO8AbSU4O2x4CHgF+kuS7wDvA3VszoiRpEmODXlU/BzLi7m8sdhxJ0qz8pKgkNWHQJakJgy5JTRh0SWrCoEtSEwZdkpow6JLUhEGXpCYMuiQ1YdAlqQmDLklNGHRJasKgS1ITBl2SmjDoktSEQZekJgy6JDVh0CWpCYMuSU0YdElqwqBLUhMGXZKaMOiS1IRBl6QmDLokNWHQJakJgy5JTRh0SWrCoEtSEwZdkpow6JLUhEGXpCbGBj3JU0kuJDm1advDSc4mOTn83L61Y0qSxpnkFfqPgNsusf2HVXVo+PnpYseSJE1rbNCr6lXgg22YRZI0h3nOod+X5JfDKZm9C5tIkjSTWYP+OHAdcAg4Bzw6asckR5OsJVlbX1+f8ekkSePMFPSqOl9Vn1bVZ8ATwE1fsO+xqlqtqtWVlZVZ55QkjTFT0JMc2HTzLuDUqH0lSdtjz7gdkjwD3ArsS/Iu8H3g1iSHgALOAPds4YySpAmMDXpVHb7E5ie3YBZJ0hz8pKgkNWHQJakJgy5JTRh0SWrCoEtSEwZdkpow6JLUhEGXpCYMuiQ1YdAlqQmDLklNGHRJasKgS1ITBl2SmjDoktSEQZekJgy6JDVh0CWpCYMuSU0YdElqwqBLUhMGXZKaMOiS1IRBl6QmDLokNWHQJakJgy5JTRh0SWrCoEtSEwZdkpow6JLUhEGXpCYMuiQ1MTboSZ5KciHJqU3brkryUpK3h8u9WzumJGmcSV6h/wi47aJtDwIvV9X1wMvDbUnSDhob9Kp6Ffjgos13AMeH68eBOxc8lyRpSrOeQ99fVeeG6+8B+0ftmORokrUka+vr6zM+nSRpnLn/KFpVBdQX3H+sqlaranVlZWXep5MkjTBr0M8nOQAwXF5Y3EiSpFnMGvQXgSPD9SPAC4sZR5I0q0netvgM8J/AHyV5N8l3gUeAbyZ5G/jz4bYkaQftGbdDVR0ecdc3FjyLJGkOflJUkpow6JLUhEGXpCYMuiQ1YdAlqQmDLklNGHRJasKgS1ITBl2SmjDoktSEQZekJgy6JDVh0CWpCYMuSU0YdElqwqBLUhMGXZKaMOiS1IRBl6QmDLokNWHQJakJgy5JTRh0SWrCoEtSEwZdkpow6JLUhEGXpCYMuiQ1YdAlqQmDLklNGHRJasKgS1ITe+Z5cJIzwEfAp8AnVbW6iKEkSdObK+iDP6uq9xfweyRJc/CUiyQ1MW/QC/hZkhNJjl5qhyRHk6wlWVtfX5/z6SRJo8wb9Fuq6qvAt4F7k3z94h2q6lhVrVbV6srKypxPJ0kaZa6gV9XZ4fIC8Dxw0yKGkiRNb+agJ7k8yZWfXwe+BZxa1GCSpOnM8y6X/cDzST7/PT+uqn9byFSSpKnNHPSq+g3wJwucRZI0B9+2KElNGHRJasKgS1ITBl2SmjDoktSEQZekJgy6JDVh0CWpCYMuSU0YdElqwqBLUhMGXZKaMOiS1IRBl6QmDLokNWHQJakJgy5JTRh0SWrCoEtSEwZdkpow6JLUhEGXpCYMuiQ1YdAlqQmDLklNGHRJasKgS1ITBl2SmjDoktSEQZekJgy6JDVh0CWpCYMuSU3MFfQktyX5VZJfJ3lwUUNJkqY3c9CTfAX4R+DbwA3A4SQ3LGowSdJ05nmFfhPw66r6TVX9Fvhn4I7FjCVJmtaeOR57NfDfm26/C3zt4p2SHAWODjc/TvKrGZ9vH/D+NA/I3834TF9OU6+/Ede+vJZ5/ZvX/vuTPGCeoE+kqo4Bx+b9PUnWqmp1ASPtSsu8fte+nGuH5V7/LGuf55TLWeCaTbd/b9gmSdoB8wT9F8D1Sf4gye8Cfwm8uJixJEnTmvmUS1V9kuQ+4N+BrwBPVdWbC5vs/5v7tM0ut8zrd+3La5nXP/XaU1VbMYgkaZv5SVFJasKgS1ITuyLoy/wVA0nOJHkjyckkazs9z1ZL8lSSC0lObdp2VZKXkrw9XO7dyRm3yoi1P5zk7HD8Tya5fSdn3CpJrknySpK3kryZ5P5h+7Ic+1Hrn+r4f+nPoQ9fMfBfwDfZ+PDSL4DDVfXWjg62TZKcAVaraik+XJHk68DHwD9V1R8P2/4e+KCqHhn+h763qv5mJ+fcCiPW/jDwcVX9YCdn22pJDgAHqur1JFcCJ4A7gb9mOY79qPXfzRTHfze8QvcrBpZIVb0KfHDR5juA48P142z8h97OiLUvhao6V1WvD9c/Ak6z8Wn0ZTn2o9Y/ld0Q9Et9xcDUC93FCvhZkhPD1ygso/1VdW64/h6wfyeH2QH3JfnlcEqm5SmHzZIcBG4EXmMJj/1F64cpjv9uCPqyu6WqvsrGt1reO/yzfGnVxjnCL/d5wsV6HLgOOAScAx7d2XG2VpIrgGeBB6rqw833LcOxv8T6pzr+uyHoS/0VA1V1dri8ADzPximoZXN+OMf4+bnGCzs8z7apqvNV9WlVfQY8QePjn+QyNmL2dFU9N2xemmN/qfVPe/x3Q9CX9isGklw+/IGEJJcD3wJOffGjWnoRODJcPwK8sIOzbKvPYza4i6bHP0mAJ4HTVfXYpruW4tiPWv+0x/9L/y4XgOGtOv/A/33FwN/u8EjbIskfsvGqHDa+puHH3dee5BngVja+OvQ88H3gX4CfANcC7wB3V1W7Px6OWPutbPxzu4AzwD2bzim3keQW4D+AN4DPhs0PsXEeeRmO/aj1H2aK478rgi5JGm83nHKRJE3AoEtSEwZdkpow6JLUhEGXpCYMuiQ1YdAlqYn/BUWg8cLCOccZAAAAAElFTkSuQmCC\n"
          },
          "metadata": {
            "needs_background": "light"
          }
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 240
        },
        "id": "mXMj1WYFGjPP",
        "outputId": "f4cac3be-2ca1-404c-fa63-c4e65f1577a1"
      },
      "outputs": [
        {
          "ename": "NameError",
          "evalue": "ignored",
          "output_type": "error",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-2-d0615f6c8dcf>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;31m#D = {u'Label1':26, u'Label2': 17, u'Label3':30}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0mnumpy\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mnp\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 3\u001b[0;31m \u001b[0mplt\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mbar\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0mzip\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0mpercentage_dict\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mitems\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      4\u001b[0m \u001b[0max1\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mplt\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msubplot\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      5\u001b[0m \u001b[0max1\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mset_xticklabels\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m\"=1\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\">1\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\">3\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\">5\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\">10\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\">15\"\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\">20\"\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mrotation\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m45\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mNameError\u001b[0m: name 'plt' is not defined"
          ]
        }
      ],
      "source": [
        "#D = {u'Label1':26, u'Label2': 17, u'Label3':30}\n",
        "import numpy as np\n",
        "plt.bar(*zip(*percentage_dict.items()))\n",
        "ax1 = plt.subplot()\n",
        "ax1.set_xticklabels([\"=1\",\">1\",\">3\",\">5\",\">10\",\">15\",\">20\"], rotation=45)\n",
        "#plt.xticks(ticks=np.arange(min(percentage_dict.values()), max(percentage_dict.values())+2, 10.0))\n",
        "plt.title(label=\"Distribution of Pubmed labeled EDU distances\", fontdict=None, loc='center', pad=None)\n",
        "plt.show()\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "ubQAGsXaRnZP"
      },
      "outputs": [],
      "source": [
        "all_pubmed_sub_df[\"equal_to_1\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] == 1]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_1\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 1]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_3\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 3]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_5\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 5]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_10\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 10]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_15\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 15]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"more_than_20\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 20]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "all_pubmed_sub_df[\"equal_to_1\"] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] == 1]) / len(all_pubmed_sub_df[\"edu_distance\"])\n",
        "\n",
        "\n",
        "#people_dict = {'bob': 'boy', 'Mike': 'boy', \n",
        "           #'Katie': 'girl', 'Stacey': 'girl'}  \n",
        "#df['people'] = people_dict\n",
        "\n",
        "\n",
        "percentage_dict = {\"equal_to_one\": all_pubmed_sub_df[\"equal_to_1\"].iloc[0],\n",
        "                   \"more_than_1\": all_pubmed_sub_df[\"more_than_1\"].iloc[0],\n",
        "                   \"more_than_3\":all_pubmed_sub_df[\"more_than_3\"].iloc[0],\n",
        "                   \"more_than_5\":all_pubmed_sub_df[\"more_than_5\"].iloc[0],\n",
        "                   \"more_than_10\":all_pubmed_sub_df[\"more_than_10\"].iloc[0], \n",
        "                   \"more_than_15\":all_pubmed_sub_df[\"more_than_15\"].iloc[0],\n",
        "                   \"more_than_20\":all_pubmed_sub_df[\"more_than_20\"].iloc[0], \n",
        "}\n",
        "\n",
        "#print(all_pubmed_sub_df[\"equal_to_1\"].iloc[0])\n",
        "\n",
        "print(percentage_dict)\n",
        "\n",
        "print(list(percentage_dict.values()))\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "hlH0XpoSpybV"
      },
      "outputs": [],
      "source": [
        "#df.plot(x=\"X\", y=[\"A\", \"B\", \"C\"], kind=\"bar\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "jkeT_nMVNVR6"
      },
      "outputs": [],
      "source": [
        "df = pd.DataFrame(data=all_pubmed_sub_df,\n",
        "              columns=['more_than_1', 'more_than_3', 'more_than_5', 'more_than_10', 'more_than_15','more_than_20','Total_distance'],\n",
        "             index = ['ITEM1', 'ITEM2', 'ITEM3', 'ITEM4', 'ITEM5'])\n",
        "df['percent_more_than_1'] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 1]) / len(all_pubmed_sub_df[\"edu_distance\"]) * 100\n",
        "df['percent_more_than_3'] = len(all_pubmed_sub_df[all_pubmed_sub_df[\"edu_distance\"] > 3]) / len(all_pubmed_sub_df[\"edu_distance\"]) * 100\n",
        "df[['percent_more_than_1', 'percent_more_than_3']].plot(kind='bar')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "i2o8XbMXL3Eh"
      },
      "outputs": [],
      "source": [
        "df = pd.DataFrame(data=all_pubmed_sub_df,\n",
        "              columns=['A', 'B', 'Total'],\n",
        "             index = ['ITEM1', 'ITEM2', 'ITEM3', 'ITEM4', 'ITEM5'])\n",
        "df['percA'] = df['A']/df['Total']*100\n",
        "df['percB'] = df['B']/df['Total']*100\n",
        "df[['percA', 'percB']].plot(kind='bar')"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "HEm3EbTmRf00",
        "outputId": "0e8279b9-4194-4f82-d3df-67dc6edf2add"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "1844\n"
          ]
        }
      ],
      "source": [
        "#!/usr/bin/env python\n",
        "import xmltodict\n",
        "import datasets\n",
        "from datasets import Dataset, concatenate_datasets\n",
        "import glob\n",
        "import pandas as pd\n",
        "import re\n",
        "import transformers \n",
        "\n",
        "\n",
        "\n",
        "dataframe_list = []\n",
        "for file in glob.glob(\"/content/drive/MyDrive/original_pubmed/*.xml\"):\n",
        "  #print(file)\n",
        "  with open(file, 'r', encoding='utf-8') as file: #'pubmed_1_copy.xml'\n",
        "      my_xml = file.read()\n",
        "      clean_xml = re.sub(\"[&;]\",\"\",my_xml)\n",
        "      xml_dict = xmltodict.parse(my_xml)\n",
        "      df1 = pd.DataFrame(xml_dict, columns=xml_dict.keys())\n",
        "      order_dict = df1[\"rst\"].iloc[0]\n",
        "      segments_list_of_dicts = order_dict[\"segment\"]\n",
        "      xml_dataframe = pd.DataFrame.from_dict(segments_list_of_dicts, orient='columns')\n",
        "      selected_df = xml_dataframe[~xml_dataframe.isnull().any(axis=1)] #removes null\n",
        "      selected_df.sort_values(by='@id',ascending=True)\n",
        "      df_formatted = selected_df.assign(parent_text=selected_df[\"#text\"].shift(1))\n",
        "      df_formatted[\"parent_text\"].fillna(value=df_formatted.iloc[0][\"#text\"],inplace=True) #fillna parent\n",
        "      dataframe_list.append(df_formatted)\n",
        "      #df_formatted[\"parent_text\"].fillna(value=\"Introduction\",inplace=True) #fillna parent\n",
        "      dataframe_list.append(df_formatted)\n",
        "      all_pubmed_sub_df = pd.concat(dataframe_list)\n",
        "\n",
        "#print(all_earnings_df.head(15))\n",
        "print(len(all_pubmed_sub_df))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "N7xHCWyarliK"
      },
      "outputs": [],
      "source": [
        "import pandas as pd\n",
        "#l = ['michael','michael','alice','carter']\n",
        "#pd.Series(l).astype('category').cat.codes.values\n",
        "\n",
        "rel_labels = list(all_pubmed_sub_df[\"@relname\"])\n",
        "rel_2_int = pd.Series(rel_labels).astype('category').cat.codes.values\n",
        "print(rel_2_int)\n",
        "for val in rel_2_int:\n",
        "  print(val)\n",
        "#print(len(set(rel_2_int)))"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "6koFvcoEroEH"
      },
      "outputs": [],
      "source": [
        "#hr_df['candidates'] = candidates\n",
        "import torch\n",
        "all_pubmed_sub_df[\"int_relation\"] = rel_2_int.astype(float)\n",
        "all_pubmed_sub_df[\"int_relation\"] = torch.Tensor.long(torch.Tensor(rel_2_int))\n",
        "all_pubmed_sub_df.head(5)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "XGwQ-HSvrqIQ"
      },
      "outputs": [],
      "source": [
        "import datasets \n",
        "from datasets import Dataset\n",
        "data_to_dset = all_pubmed_sub_df[[\"#text\",\"parent_text\",\"int_relation\"]]\n",
        "\n",
        "raw_datasets = Dataset.from_pandas(data_to_dset)\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yVCLPcIsrr4B",
        "outputId": "3927a927-299a-4d99-ed8e-ed8c3e4b289f"
      },
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['Background: Neoadjuvant chemotherapy', 'and radiation', 'followed by surgical resection', 'of the rectum', 'is a standard treatment', 'for locally advanced rectal cancer.', 'A subset of rectal cancer is caused', 'by a deficiency in mismatch repair.', 'is responsive to programmed death 1', '(PD-1)', 'blockade in the context of metastatic disease,', 'it was hypothesized', 'that checkpoint blockade could be effective in patients', 'with mismatch repair-deficient,', 'locally advanced rectal cancer.', 'Methods: We initiated a prospective phase 2 study', 'an anti-PD-1 monoclonal antibody,', 'was administered', 'every 3 weeks for 6 months', 'in patients with mismatch repair-deficient stage II', 'This treatment was to be followed', 'by standard chemoradiotherapy', 'Patients who had a clinical complete response', 'after completion of dostarlimab therapy would proceed', '12 months after completion of dostarlimab therapy', 'or pathological complete response', 'after completion of dostarlimab therapy with', 'and overall response', 'to neoadjuvant dostarlimab therapy', 'Results: A total of 12 patients have completed treatment', 'with dostarlimab', 'All 12 patients (100%; 95% confidence interval, 74 to 100)', 'had a clinical complete response,', 'with no evidence of tumor', 'on magnetic resonance imaging,', 'endoscopic evaluation,', 'digital rectal examination,', 'or biopsy.', 'no patients had received chemoradiotherapy', 'or undergone surgery,', 'and no cases of progression', 'or recurrence had been reported during follow-up', 'No adverse events of grade 3', 'Conclusions: Mismatch repair-deficient,', 'locally advanced rectal cancer was highly sensitive', 'Background: Neoadjuvant chemotherapy', 'and radiation', 'followed by surgical resection', 'of the rectum', 'is a standard treatment', 'for locally advanced rectal cancer.', 'A subset of rectal cancer is caused', 'by a deficiency in mismatch repair.', 'is responsive to programmed death 1', '(PD-1)', 'blockade in the context of metastatic disease,', 'it was hypothesized', 'that checkpoint blockade could be effective in patients', 'with mismatch repair-deficient,', 'locally advanced rectal cancer.', 'Methods: We initiated a prospective phase 2 study', 'an anti-PD-1 monoclonal antibody,', 'was administered', 'every 3 weeks for 6 months', 'in patients with mismatch repair-deficient stage II', 'This treatment was to be followed', 'by standard chemoradiotherapy', 'Patients who had a clinical complete response', 'after completion of dostarlimab therapy would proceed', '12 months after completion of dostarlimab therapy', 'or pathological complete response', 'after completion of dostarlimab therapy with', 'and overall response', 'to neoadjuvant dostarlimab therapy', 'Results: A total of 12 patients have completed treatment', 'with dostarlimab', 'All 12 patients (100%; 95% confidence interval, 74 to 100)', 'had a clinical complete response,', 'with no evidence of tumor', 'on magnetic resonance imaging,', 'endoscopic evaluation,', 'digital rectal examination,', 'or biopsy.', 'no patients had received chemoradiotherapy', 'or undergone surgery,', 'and no cases of progression', 'or recurrence had been reported during follow-up', 'No adverse events of grade 3', 'Conclusions: Mismatch repair-deficient,', 'locally advanced rectal cancer was highly sensitive', 'Abstract', 'Objective:', 'First, to evaluate the risks of stillbirth', 'and neonatal death by gestational age', 'in twin pregnancies with different levels', 'of growth discordance', 'and in relation to small', 'and on this basis', 'to establish optimal gestational ages', 'Second,', 'to compare these optimal gestational ages', 'for twin pregnancies not complicated', 'by fetal growth restriction,', 'which, in a previous individual patient meta-analysis,', 'and 36 0/7 weeks for monochorionic pregnancies.', 'Data sources:', 'A search of MEDLINE, EMBASE, ClinicalTrials.gov,', 'and Ovid between 2015', 'and 2018 was performed of cohort studies reporting risks of stillbirth', 'Studies from a previous meta-analysis', 'using a similar search strategy (from inception to 2015) were combined.', 'Methods of study selection: Overall, of 57 eligible studies, 20 cohort studies', 'that contributed original data reporting on 7,474 dichorionic', 'Tabulation, integration,', 'and results:', 'We performed an individual participant data meta-analysis', 'to calculate the risk of perinatal death (risk difference between prospective stillbirth', 'Analyses were stratified by chorionicity,', 'levels of growth discordance,', 'For both dichorionic', 'the absolute risks of stillbirth', 'and neonatal death were higher when one', 'or both twins were SGA', 'and increased with greater levels', 'Regardless of level of growth discordance', 'and birth weight, perinatal risk balanced between 36 0/7-6/7', 'and 37 0/7-6/7 weeks of gestation in both dichorionic', 'and monochorionic twin pregnancies,', 'than neonatal death from 37 0/7-6/7 weeks onward.', 'Conclusion: Growth discordance', 'or SGA', 'However, balancing these two risks,', 'we did not find evidence', 'that the optimal timing of delivery is changed', 'Abstract', 'Objective:', 'First, to evaluate the risks of stillbirth', 'and neonatal death by gestational age', 'in twin pregnancies with different levels', 'of growth discordance', 'and in relation to small', 'and on this basis', 'to establish optimal gestational ages', 'Second,', 'to compare these optimal gestational ages', 'for twin pregnancies not complicated', 'by fetal growth restriction,', 'which, in a previous individual patient meta-analysis,', 'and 36 0/7 weeks for monochorionic pregnancies.', 'Data sources:', 'A search of MEDLINE, EMBASE, ClinicalTrials.gov,', 'and Ovid between 2015', 'and 2018 was performed of cohort studies reporting risks of stillbirth', 'Studies from a previous meta-analysis', 'using a similar search strategy (from inception to 2015) were combined.', 'Methods of study selection: Overall, of 57 eligible studies, 20 cohort studies', 'that contributed original data reporting on 7,474 dichorionic', 'Tabulation, integration,', 'and results:', 'We performed an individual participant data meta-analysis', 'to calculate the risk of perinatal death (risk difference between prospective stillbirth', 'Analyses were stratified by chorionicity,', 'levels of growth discordance,', 'For both dichorionic', 'the absolute risks of stillbirth', 'and neonatal death were higher when one', 'or both twins were SGA', 'and increased with greater levels', 'Regardless of level of growth discordance', 'and birth weight, perinatal risk balanced between 36 0/7-6/7', 'and 37 0/7-6/7 weeks of gestation in both dichorionic', 'and monochorionic twin pregnancies,', 'than neonatal death from 37 0/7-6/7 weeks onward.', 'Conclusion: Growth discordance', 'or SGA', 'However, balancing these two risks,', 'we did not find evidence', 'that the optimal timing of delivery is changed', 'Abstract', 'Purpose: Pancreatic cancer (PC) is a common,', 'Autophagy is involved in the occurrence', 'This study aims to explore the feasibility', 'of using an autophagy-related long noncoding RNA', '(lncRNA)', 'Methods: We obtained RNA sequencing', 'and clinical data of patients from the TCGA website.', 'The prognostic model,', 'generated through univariate', 'and multivariate Cox regression analyses,', 'Receiver operating characteristic', '(ROC) curves', 'and forest plots were generated for univariate', 'and multivariate Cox regression analyses,', 'Gene set enrichment analysis (GSEA) was used to screen enriched gene sets.', 'Results: Twenty-eight autophagy-related lncRNAs were filtered out through univariate Cox regression analysis', 'Ten autophagy-related lncRNAs,', 'including 4 poor prognosis factors', 'were further screened via multivariate Cox regression analysis.', 'GSEA results demonstrated', 'Conclusion: A signature based on ten autophagy-related lncRNAs was identified.', 'This signature could be potentially used for evaluating clinical prognosis', 'Abstract', 'Purpose: Pancreatic cancer (PC) is a common,', 'Autophagy is involved in the occurrence', 'This study aims to explore the feasibility', 'of using an autophagy-related long noncoding RNA', '(lncRNA)', 'Methods: We obtained RNA sequencing', 'and clinical data of patients from the TCGA website.', 'The prognostic model,', 'generated through univariate', 'and multivariate Cox regression analyses,', 'Receiver operating characteristic', '(ROC) curves', 'and forest plots were generated for univariate', 'and multivariate Cox regression analyses,', 'Gene set enrichment analysis (GSEA) was used to screen enriched gene sets.', 'Results: Twenty-eight autophagy-related lncRNAs were filtered out through univariate Cox regression analysis', 'Ten autophagy-related lncRNAs,', 'including 4 poor prognosis factors', 'were further screened via multivariate Cox regression analysis.', 'GSEA results demonstrated', 'Conclusion: A signature based on ten autophagy-related lncRNAs was identified.', 'This signature could be potentially used for evaluating clinical prognosis', 'Rare Alleles and Signatures of Selection on the Immunodominant Domains of Pfs230', 'and Pfs48/45 in Malaria Parasites From Western Kenya', 'Abstract', 'Background: Malaria elimination', 'and eradication efforts can be advanced', 'by including transmission-blocking or reducing vaccines', 'Key transmission-blocking vaccine candidates,', 'such as Pfs230 domain one and Pfs48/45 domain 3,', 'We evaluated genetic polymorphism', 'and temporal stability of Pfs230 domain one', 'Methods: Dry blood spots on filter paper were collected from febrile malaria patients reporting', 'to community health facilities in endemic areas of Homa Bay', 'and an epidemic-prone area of Kisii County in 2018 and 2019.', 'Plasmodium speciation was performed using eluted DNA and real-time PCR.', 'We sequenced Pfs230 domain one on 156 clinical isolates', 'and Pfs48/45 domain three on 118 clinical isolates to infer the levels of genetic variability,', 'Results: Pfs230 domain one had low nucleotide diversity', '(π = 0.15 × 10-2)', 'Six polymorphic sites with nonsynonymous mutations', 'I539T was a novel variant, whereas G605S was nearing fixation.', 'high conservation index, and three segregating sites,', 'resulting in nonsynonymous mutation', 'Some loci of Pfs230 D1 were in positive', 'or positive selection signatures,', 'had a history of recombination.', 'Mutated loci pairs in Pfs48/45 domain three had negative linkage disequilibrium,', 'and some had negative', 'Conclusion: The two transmission blocking vaccine candidates have low nucleotide diversity,', 'a small number of zone-specific variants,', 'high nucleotide conservation index,', 'With the near fixation a polymorphic site', 'it will be necessary', 'to continue monitoring sequence modifications of these domains when designing TBVs', 'Rare Alleles and Signatures of Selection on the Immunodominant Domains of Pfs230', 'and Pfs48/45 in Malaria Parasites From Western Kenya', 'Abstract', 'Background: Malaria elimination', 'and eradication efforts can be advanced', 'by including transmission-blocking or reducing vaccines', 'Key transmission-blocking vaccine candidates,', 'such as Pfs230 domain one and Pfs48/45 domain 3,', 'We evaluated genetic polymorphism', 'and temporal stability of Pfs230 domain one', 'Methods: Dry blood spots on filter paper were collected from febrile malaria patients reporting', 'to community health facilities in endemic areas of Homa Bay', 'and an epidemic-prone area of Kisii County in 2018 and 2019.', 'Plasmodium speciation was performed using eluted DNA and real-time PCR.', 'We sequenced Pfs230 domain one on 156 clinical isolates', 'and Pfs48/45 domain three on 118 clinical isolates to infer the levels of genetic variability,', 'Results: Pfs230 domain one had low nucleotide diversity', '(π = 0.15 × 10-2)', 'Six polymorphic sites with nonsynonymous mutations', 'I539T was a novel variant, whereas G605S was nearing fixation.', 'high conservation index, and three segregating sites,', 'resulting in nonsynonymous mutation', 'Some loci of Pfs230 D1 were in positive', 'or positive selection signatures,', 'had a history of recombination.', 'Mutated loci pairs in Pfs48/45 domain three had negative linkage disequilibrium,', 'and some had negative', 'Conclusion: The two transmission blocking vaccine candidates have low nucleotide diversity,', 'a small number of zone-specific variants,', 'high nucleotide conservation index,', 'With the near fixation a polymorphic site', 'it will be necessary', 'to continue monitoring sequence modifications of these domains when designing TBVs', 'Salivary Progesterone Levels in Female Patients with a History of Idiopathic Scoliosis:', 'Abstract', 'However, in addition to the spinal curvature,', 'it has also been reported that patients with idiopathic scoliosis can display other abnormal physiologic parameters,', 'such as hormone imbalances,', 'genetic variants,', 'The present study evaluates the salivary progesterone levels', 'A group of female patients with a history of idiopathic scoliosis is compared', 'Salivary progesterone levels were 49% higher in non-scoliosis patients', 'compared to the scoliosis group', 'When compared by the menstrual cycling status,', 'while the progesterone level among non-cycling,', 'These results suggest a potential relationship between salivary progesterone', 'Salivary Progesterone Levels in Female Patients with a History of Idiopathic Scoliosis:', 'Abstract', 'However, in addition to the spinal curvature,', 'it has also been reported that patients with idiopathic scoliosis can display other abnormal physiologic parameters,', 'such as hormone imbalances,', 'genetic variants,', 'The present study evaluates the salivary progesterone levels', 'A group of female patients with a history of idiopathic scoliosis is compared', 'Salivary progesterone levels were 49% higher in non-scoliosis patients', 'compared to the scoliosis group', 'When compared by the menstrual cycling status,', 'while the progesterone level among non-cycling,', 'These results suggest a potential relationship between salivary progesterone', 'Abstract', 'Introduction: Epidermoid cysts', '(EC)', 'A primary brain lymphoma development related', 'to a EC is presented,', 'Case presentation:', 'A woman 40 years old,', 'harbouring a EC for more than 20 years,', 'Surgery was performed', 'Discussion:', 'usually to squamous cells carcinoma,', 'Inflammatory mechanisms have been advocated', 'Chronic inflammation', 'and lymphoma genesis are related,', 'Conclusions: Even being benign lesions,', 'EC can develop malignant tumours due', 'Abstract', 'Introduction: Epidermoid cysts', '(EC)', 'A primary brain lymphoma development related', 'to a EC is presented,', 'Case presentation:', 'A woman 40 years old,', 'harbouring a EC for more than 20 years,', 'Surgery was performed', 'Discussion:', 'usually to squamous cells carcinoma,', 'Inflammatory mechanisms have been advocated', 'Chronic inflammation', 'and lymphoma genesis are related,', 'Conclusions: Even being benign lesions,', 'EC can develop malignant tumours due', 'Cerebral Blood Flow', 'and Other Predictors of Responsiveness', 'Abstract', 'Introduction: Monoclonal antibodies', '(mAbs)', 'The aim of this study was', 'to check if baseline clinical parameters', '(CBF)', 'measured by transcranial Doppler', '(TCD)', 'may help predict mAbs efficacy.', 'Methods: Electronic charts of migraineurs treated with erenumab', 'with baseline TCD evaluations were collected,', 'including data on migraine type,', 'pain localization,', 'monthly migraine days', '(MMD),', 'medication overuse headache', 'mean blood flow velocity', '(Vm),', 'Results: A total of 123 patients were enrolled,', 'mean age 38,', '75 years,', '61 with MOH,', '72 were good responders (GR),', '43 ≥75% reduction in MMD.', 'Baseline Vm values in MCAs were significantly lower in GR', 'MAbs responsiveness ≥50% was positively associated with unilateral pain localization', '(OR: 6.53, 95% CI: 2.01-23.93; p = 0.003)', 'whereas negatively associated with Vm in right MCA', '(OR: 0.96, 95% CI: 0.92-0.99; p = 0.012),', 'and having no relatives with migraine', 'Conclusions: Baseline Vm in MCA is lower in mAbs GR', 'as compared with non-responders', 'Simple clinical features', 'and baseline CBF in anterior circulation might help', 'Cerebral Blood Flow', 'and Other Predictors of Responsiveness', 'Abstract', 'Introduction: Monoclonal antibodies', '(mAbs)', 'The aim of this study was', 'to check if baseline clinical parameters', '(CBF)', 'measured by transcranial Doppler', '(TCD)', 'may help predict mAbs efficacy.', 'Methods: Electronic charts of migraineurs treated with erenumab', 'with baseline TCD evaluations were collected,', 'including data on migraine type,', 'pain localization,', 'monthly migraine days', '(MMD),', 'medication overuse headache', 'mean blood flow velocity', '(Vm),', 'Results: A total of 123 patients were enrolled,', 'mean age 38,', '75 years,', '61 with MOH,', '72 were good responders (GR),', '43 ≥75% reduction in MMD.', 'Baseline Vm values in MCAs were significantly lower in GR', 'MAbs responsiveness ≥50% was positively associated with unilateral pain localization', '(OR: 6.53, 95% CI: 2.01-23.93; p = 0.003)', 'whereas negatively associated with Vm in right MCA', '(OR: 0.96, 95% CI: 0.92-0.99; p = 0.012),', 'and having no relatives with migraine', 'Conclusions: Baseline Vm in MCA is lower in mAbs GR', 'as compared with non-responders', 'Simple clinical features', 'and baseline CBF in anterior circulation might help', 'Abstract', 'Objective: To explore the application value of pelvic magnetic resonance imaging', '(MRI) scan combined with serum pyruvate kinase isozyme M2 (PKM2),', 'and soluble leptin receptor', 'detection in diagnosing endometrial carcinoma', 'Methods: The clinical data of 45 patients with pathologically confirmed EC treated in our hospital from May 2019 to May 2020 were retrospectively analyzed.', 'All patients received pelvic MRI scan,', 'serum PKM2,', 'NGAL', 'detection was performed,', 'and the combination of the two was performed', 'Results: Compared with the joint detection,', 'the number of true positive cases,', 'sensitivity, specificity,', 'and accuracy rate obtained by a single application of pelvic MRI or serum PKM2,', 'NGAL,', 'the area under the ROC curve of the joint detection was obviously larger than that of single detection;', 'the results of the joint detection were better than those of single detection', 'the combined diagnosis obtained the highest sensitivity.', 'Conclusion: Combining pelvic MRI with serum PKM2,', 'NGAL,', 'and sOB-4 detection can effectively promote the diagnostic accuracy for EC,', 'Abstract', 'Objective: To explore the application value of pelvic magnetic resonance imaging', '(MRI) scan combined with serum pyruvate kinase isozyme M2 (PKM2),', 'and soluble leptin receptor', 'detection in diagnosing endometrial carcinoma', 'Methods: The clinical data of 45 patients with pathologically confirmed EC treated in our hospital from May 2019 to May 2020 were retrospectively analyzed.', 'All patients received pelvic MRI scan,', 'serum PKM2,', 'NGAL', 'detection was performed,', 'and the combination of the two was performed', 'Results: Compared with the joint detection,', 'the number of true positive cases,', 'sensitivity, specificity,', 'and accuracy rate obtained by a single application of pelvic MRI or serum PKM2,', 'NGAL,', 'the area under the ROC curve of the joint detection was obviously larger than that of single detection;', 'the results of the joint detection were better than those of single detection', 'the combined diagnosis obtained the highest sensitivity.', 'Conclusion: Combining pelvic MRI with serum PKM2,', 'NGAL,', 'and sOB-4 detection can effectively promote the diagnostic accuracy for EC,', 'Abstract', 'Objective: To explore the application value of pelvic magnetic resonance imaging', '(MRI) scan combined with serum pyruvate kinase isozyme M2 (PKM2),', 'and soluble leptin receptor', 'detection in diagnosing endometrial carcinoma', 'Methods: The clinical data of 45 patients with pathologically confirmed EC treated in our hospital from May 2019 to May 2020 were retrospectively analyzed.', 'All patients received pelvic MRI scan,', 'serum PKM2,', 'NGAL', 'detection was performed,', 'and the combination of the two was performed', 'Results: Compared with the joint detection,', 'the number of true positive cases,', 'sensitivity, specificity,', 'and accuracy rate obtained by a single application of pelvic MRI or serum PKM2,', 'NGAL,', 'the area under the ROC curve of the joint detection was obviously larger than that of single detection;', 'the results of the joint detection were better than those of single detection', 'the combined diagnosis obtained the highest sensitivity.', 'Conclusion: Combining pelvic MRI with serum PKM2,', 'NGAL,', 'and sOB-4 detection can effectively promote the diagnostic accuracy for EC,', 'Abstract', 'Objective: To explore the application value of pelvic magnetic resonance imaging', '(MRI) scan combined with serum pyruvate kinase isozyme M2 (PKM2),', 'and soluble leptin receptor', 'detection in diagnosing endometrial carcinoma', 'Methods: The clinical data of 45 patients with pathologically confirmed EC treated in our hospital from May 2019 to May 2020 were retrospectively analyzed.', 'All patients received pelvic MRI scan,', 'serum PKM2,', 'NGAL', 'detection was performed,', 'and the combination of the two was performed', 'Results: Compared with the joint detection,', 'the number of true positive cases,', 'sensitivity, specificity,', 'and accuracy rate obtained by a single application of pelvic MRI or serum PKM2,', 'NGAL,', 'the area under the ROC curve of the joint detection was obviously larger than that of single detection;', 'the results of the joint detection were better than those of single detection', 'the combined diagnosis obtained the highest sensitivity.', 'Conclusion: Combining pelvic MRI with serum PKM2,', 'NGAL,', 'and sOB-4 detection can effectively promote the diagnostic accuracy for EC,', 'Seven cases of acute sphenoid sinusitis were reviewed.', 'Two happened after deep sea diving.', 'Headache of variable location', 'Purulent discharge in the cavum was present in three.', 'Correct diagnosis was delayed from six', 'to twenty-four days after beginning of symptoms,', 'when a neurologic deficit became apparent:', 'mainly paralysis of the sixth', 'and third cranial nerves.', 'Computed axial tomography was the most useful radiologic procedure', 'Two patients died,', 'three had complete recovery,', 'one had persistent epileptic focus after brain abscess.', 'Last patient had paraparesis', 'and incontinence of urine after study of cerebrospinal fluid', 'Cannulation of the sphenoid sinus was performed', 'Gram negative microorganisms', 'and Staphylococcus aureus were isolated from the sphenoid sinus of only three patients.', 'Blood cultures were negatives in every cases', 'and C. S. F. in six.', 'First choice antibiotics was an association of aminoglucosides', 'and broad-spectrum penicillin in six cases.', 'Seven cases of acute sphenoid sinusitis were reviewed.', 'Two happened after deep sea diving.', 'Headache of variable location', 'Purulent discharge in the cavum was present in three.', 'Correct diagnosis was delayed from six', 'to twenty-four days after beginning of symptoms,', 'when a neurologic deficit became apparent:', 'mainly paralysis of the sixth', 'and third cranial nerves.', 'Computed axial tomography was the most useful radiologic procedure', 'Two patients died,', 'three had complete recovery,', 'one had persistent epileptic focus after brain abscess.', 'Last patient had paraparesis', 'and incontinence of urine after study of cerebrospinal fluid', 'Cannulation of the sphenoid sinus was performed', 'Gram negative microorganisms', 'and Staphylococcus aureus were isolated from the sphenoid sinus of only three patients.', 'Blood cultures were negatives in every cases', 'and C. S. F. in six.', 'First choice antibiotics was an association of aminoglucosides', 'and broad-spectrum penicillin in six cases.', 'Disturbed noradrenaline', '(NA)', 'metabolism is thought to play a causal role in certain types of endogenic depression.', 'This study was based on data from 88 patients with depression.', 'The metabolism of NA was investigated by measuring urinary MHPG', 'and plasma DOPEG concentrations', 'and platelet nonoamine oxidase activity', 'to determine if there were any differences in subgroups of depression defined', 'by the DSM3.', 'There was no difference in plasma DOPEG concentrations', 'or of MAO activity in the different subgroups of depression,', 'especially between episodes of major', 'and non-major depression.', 'On the other hand,', 'depressed patients with an episode of major depression had', 'significantly higher urinary MHPG concentrations', 'than those with non-major depression.', 'Disturbed noradrenaline', '(NA)', 'metabolism is thought to play a causal role in certain types of endogenic depression.', 'This study was based on data from 88 patients with depression.', 'The metabolism of NA was investigated by measuring urinary MHPG', 'and plasma DOPEG concentrations', 'and platelet nonoamine oxidase activity', 'to determine if there were any differences in subgroups of depression defined', 'by the DSM3.', 'There was no difference in plasma DOPEG concentrations', 'or of MAO activity in the different subgroups of depression,', 'especially between episodes of major', 'and non-major depression.', 'On the other hand,', 'depressed patients with an episode of major depression had', 'significantly higher urinary MHPG concentrations', 'than those with non-major depression.', 'Apart from the rarity of the causal agent,', 'this case is of interest', 'because of the absence of cervico-facial involvement', 'and the presence of multiple cutaneous nodules', 'at a distance from the visceral localisation.', 'Apart from the rarity of the causal agent,', 'this case is of interest', 'because of the absence of cervico-facial involvement', 'and the presence of multiple cutaneous nodules', 'at a distance from the visceral localisation.', 'We report a new case of granuloma inguinale in France.', 'A 47 years old Turkish man living in France for eight years', 'had an ulcerovegetative lesion on the corona sulcus', 'and inner aspect of the prepuce for three months.', 'Donovanosis was established by granulation time smears', 'Complete resolution occurred', 'after treatment with ampicillin for three weeks', 'and later doxycycline for a month.', 'Granuloma inguinale is rare in France', 'clinical manifestations', 'and therapeutic indications are reviewed.', 'We report a new case of granuloma inguinale in France.', 'A 47 years old Turkish man living in France for eight years', 'had an ulcerovegetative lesion on the corona sulcus', 'and inner aspect of the prepuce for three months.', 'Donovanosis was established by granulation time smears', 'Complete resolution occurred', 'after treatment with ampicillin for three weeks', 'and later doxycycline for a month.', 'Granuloma inguinale is rare in France', 'clinical manifestations', 'and therapeutic indications are reviewed.', 'Vascular purpura and haemoptysis developed in a patient suffering from accelerated hypertension', 'and rapidly progressive renal failure due to cholesterol embolism.', 'Alveolar haemorrhage was diagnosed on the association of clinical', 'and endoscopic results,', 'and on the outcome of the disease.', 'This syndrome is usually associated with other immunological disorders', 'and raises the question of immunological phenomena being the cause of some of the clinical manifestations observed', 'after cholesterol embolism.', 'Vascular purpura and haemoptysis developed in a patient suffering from accelerated hypertension', 'and rapidly progressive renal failure due to cholesterol embolism.', 'Alveolar haemorrhage was diagnosed on the association of clinical', 'and endoscopic results,', 'and on the outcome of the disease.', 'This syndrome is usually associated with other immunological disorders', 'and raises the question of immunological phenomena being the cause of some of the clinical manifestations observed', 'after cholesterol embolism.', 'This research attempts to examine a possible connection', 'between neuroticism (level of anxiety)', 'and the meaning of life.', 'A research population composed of 446 people of whom 140 were imprisoned criminals', 'and 306 were non-criminals,', 'was examined.', 'A significant negative correlation was found', 'between the meaning of life and neuroticism,', 'which means that the higher the neuroticism is the lower the meaning of life is.', 'This negative connection was stronger for the criminal group', 'which on the average had a lower score for the meaning of life.', 'According to these results,', 'the processes allowing a positive meaning of life can diminish the effects of psychogenic anxiety.', 'No connection was found between extraversion', 'and the meaning of life.', 'Likewise, socioeconomic factors did not influence the negative correlation', 'In specific cases we found', 'that criminal solutions could be a substitute for the meaning of life.', 'This research attempts to examine a possible connection', 'between neuroticism (level of anxiety)', 'and the meaning of life.', 'A research population composed of 446 people of whom 140 were imprisoned criminals', 'and 306 were non-criminals,', 'was examined.', 'A significant negative correlation was found', 'between the meaning of life and neuroticism,', 'which means that the higher the neuroticism is the lower the meaning of life is.', 'This negative connection was stronger for the criminal group', 'which on the average had a lower score for the meaning of life.', 'According to these results,', 'the processes allowing a positive meaning of life can diminish the effects of psychogenic anxiety.', 'No connection was found between extraversion', 'and the meaning of life.', 'Likewise, socioeconomic factors did not influence the negative correlation', 'In specific cases we found', 'that criminal solutions could be a substitute for the meaning of life.', 'The cytomorphologic findings of a vitrectomy specimen', 'from the right eye of an 80-year-old woman', 'with an unsuspected primary choroidal malignant melanoma are described.', 'The patient had undergone a closed vitrectomy because of chronic vitreous hemorrhage.', 'Histocytology of the vitreous fluid specimens revealed melanoma cells of variable shape', 'and size (from 30-150 microns)', 'with eccentric nuclei.', 'Many of these cells were binucleated', 'or multinucleated with small, uniform, evenly dispersed intracytoplasmic melanin granules.', 'The histocytologic findings together with the postoperative tumor characteristics by ultrasonography', 'and fluorescein angiography suggested the diagnosis of choroidal malignant melanoma.', 'The cytomorphologic findings of a vitrectomy specimen', 'from the right eye of an 80-year-old woman', 'with an unsuspected primary choroidal malignant melanoma are described.', 'The patient had undergone a closed vitrectomy because of chronic vitreous hemorrhage.', 'Histocytology of the vitreous fluid specimens revealed melanoma cells of variable shape', 'and size (from 30-150 microns)', 'with eccentric nuclei.', 'Many of these cells were binucleated', 'or multinucleated with small, uniform, evenly dispersed intracytoplasmic melanin granules.', 'The histocytologic findings together with the postoperative tumor characteristics by ultrasonography', 'and fluorescein angiography suggested the diagnosis of choroidal malignant melanoma.', 'Sexual impotence developed in four patients', 'who were receiving carbonic anhydrase inhibitor therapy for glaucoma.', 'Whereas decreased libido has been reported as a side effect of carbonic anhydrase inhibitor therapy,', 'organic impotence has not been documented as an isolated finding.', 'Sexual impotence developed in four patients', 'who were receiving carbonic anhydrase inhibitor therapy for glaucoma.', 'Whereas decreased libido has been reported as a side effect of carbonic anhydrase inhibitor therapy,', 'organic impotence has not been documented as an isolated finding.', 'Multiple epiphyseal dysplasias may present as congenital', 'or late-onset (tarda) forms', 'or as the variant pseudo-achondroplastic type.', 'While the association between cataract', 'and skin anomalies is well known in the congenital type,', 'it has not been reported for the late-onset type of multiple epiphyseal dysplasia.', 'A 5 1/2-year-old child with the pseudo-achondroplastic variant of multiple epiphyseal dysplasia has been observed', 'to have the unusual association of cataracts', 'and vitiligo,', 'a condition which lacked previous documentation.', 'Multiple epiphyseal dysplasias may present as congenital', 'or late-onset (tarda) forms', 'or as the variant pseudo-achondroplastic type.', 'While the association between cataract', 'and skin anomalies is well known in the congenital type,', 'it has not been reported for the late-onset type of multiple epiphyseal dysplasia.', 'A 5 1/2-year-old child with the pseudo-achondroplastic variant of multiple epiphyseal dysplasia has been observed', 'to have the unusual association of cataracts', 'and vitiligo,', 'a condition which lacked previous documentation.', 'Cystic lesions constituted 37% of all the conjunctival specimens received over a period of 6 1/2 years.', 'Epithelial inclusion cysts were the most common,', 'followed by dermoid', 'and parasitic cysts.', 'A clinicopathologic correlation,', 'along with an attempt at classification', 'of cystic lesions of conjunctiva,', 'is made in this article.', 'Cystic lesions constituted 37% of all the conjunctival specimens received over a period of 6 1/2 years.', 'Epithelial inclusion cysts were the most common,', 'followed by dermoid', 'and parasitic cysts.', 'A clinicopathologic correlation,', 'along with an attempt at classification', 'of cystic lesions of conjunctiva,', 'is made in this article.', 'Ticks commonly infest humans but rarely bite the eyes.', 'They are vectors of disease;', 'therefore, rapid and complete removal is important.', 'Their attachment can be so secure', 'that removal is difficult or hazardous.', \"We excised a larval form of Amblyomma americanum from a woodsman's conjunctiva.\", 'Ticks commonly infest humans but rarely bite the eyes.', 'They are vectors of disease;', 'therefore, rapid and complete removal is important.', 'Their attachment can be so secure', 'that removal is difficult or hazardous.', \"We excised a larval form of Amblyomma americanum from a woodsman's conjunctiva.\", 'A two-year-old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration.', 'Orbital roof fractures may occur via a blow-in mechanism rather than by direct transmission of an outside force.', 'As is the case with those caused by floor fractures,', 'motility disturbances from roof fractures merit clinical observation', 'before surgery is performed,', 'since some motility defects will resolve spontaneously.', 'A two-year-old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration.', 'Orbital roof fractures may occur via a blow-in mechanism rather than by direct transmission of an outside force.', 'As is the case with those caused by floor fractures,', 'motility disturbances from roof fractures merit clinical observation', 'before surgery is performed,', 'since some motility defects will resolve spontaneously.', 'A two-year-old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration.', 'Orbital roof fractures may occur via a blow-in mechanism rather than by direct transmission of an outside force.', 'As is the case with those caused by floor fractures,', 'motility disturbances from roof fractures merit clinical observation', 'before surgery is performed,', 'since some motility defects will resolve spontaneously.', 'A two-year-old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration.', 'Orbital roof fractures may occur via a blow-in mechanism rather than by direct transmission of an outside force.', 'As is the case with those caused by floor fractures,', 'motility disturbances from roof fractures merit clinical observation', 'before surgery is performed,', 'since some motility defects will resolve spontaneously.', 'In this case report the authors present a patient in', 'whom loss of vision developed in the left eye in the early stage of herpes zoster ophthalmicus.', 'He suffered from optic neuropathy', 'followed by central retinal artery occlusion.', 'Ophthalmologists should be aware of this rare', 'and early complication of herpes zoster.', 'In this case report the authors present a patient in', 'whom loss of vision developed in the left eye in the early stage of herpes zoster ophthalmicus.', 'He suffered from optic neuropathy', 'followed by central retinal artery occlusion.', 'Ophthalmologists should be aware of this rare', 'and early complication of herpes zoster.', 'A case of limbal vernal keratoconjunctivitis associated with a hypertrophic mass lesion measuring', '8 X 5 X 3 mm is reported.', 'The histopathology of this mass,', 'which consisted of hyperplastic epithelium with eosinophilic infiltration', 'and thickened subepithelial stroma of irregular hyperplastic collagenous connective tissue interspersed with numerous eosinophils', 'and inflammatory cells, is presented.', 'A large limbal-mass lesion', 'such as this has not been previously described in association with limbal vernal keratoconjunctivitis.', 'A case of limbal vernal keratoconjunctivitis associated with a hypertrophic mass lesion measuring', '8 X 5 X 3 mm is reported.', 'The histopathology of this mass,', 'which consisted of hyperplastic epithelium with eosinophilic infiltration', 'and thickened subepithelial stroma of irregular hyperplastic collagenous connective tissue interspersed with numerous eosinophils', 'and inflammatory cells, is presented.', 'A large limbal-mass lesion', 'such as this has not been previously described in association with limbal vernal keratoconjunctivitis.', 'In a previous work on a group of patients with diabetic retinopathy,', 'the effect of laser photocoagulation on the serum zinc', 'and copper levels was evaluated.', 'In many of those patients,', 'an increase in the serum zinc', 'and copper levels was found one day after the laser treatment.', 'changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit.', 'A significant rise was found one day after the treatment in the level of serum copper', '(P less than .005),', 'and one week thereafter in the level of serum zinc', '(P less than .005).', 'The significance of these results and their connection to the retinal pigment epithelium', 'and choroid is discussed.', 'In a previous work on a group of patients with diabetic retinopathy,', 'the effect of laser photocoagulation on the serum zinc', 'and copper levels was evaluated.', 'In many of those patients,', 'an increase in the serum zinc', 'and copper levels was found one day after the laser treatment.', 'changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit.', 'A significant rise was found one day after the treatment in the level of serum copper', '(P less than .005),', 'and one week thereafter in the level of serum zinc', '(P less than .005).', 'The significance of these results and their connection to the retinal pigment epithelium', 'and choroid is discussed.', 'A case of tuberculous uveitis', 'that presented 34 years after a primary articular focus is evaluated.', 'The pathogenesis of the infection', 'and the role of the tuberculin skin test in the diagnosis are discussed.', 'Isoniazid was used with good therapeutic results.', 'Fluorescein angiography was found to be essential in the follow-up of the choroidal lesions.', 'Subretinal neovascularization was managed with argon laser photocoagulation.', 'A case of tuberculous uveitis', 'that presented 34 years after a primary articular focus is evaluated.', 'The pathogenesis of the infection', 'and the role of the tuberculin skin test in the diagnosis are discussed.', 'Isoniazid was used with good therapeutic results.', 'Fluorescein angiography was found to be essential in the follow-up of the choroidal lesions.', 'Subretinal neovascularization was managed with argon laser photocoagulation.', 'In a previous work on a group of patients with diabetic retinopathy,', 'the effect of laser photocoagulation on the serum zinc', 'and copper levels was evaluated.', 'In many of those patients,', 'an increase in the serum zinc', 'and copper levels was found one day after the laser treatment.', 'changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit.', 'A significant rise was found one day after the treatment in the level of serum copper', '(P less than .005),', 'and one week thereafter in the level of serum zinc', '(P less than .005).', 'The significance of these results and their connection to the retinal pigment epithelium', 'and choroid is discussed.', 'In a previous work on a group of patients with diabetic retinopathy,', 'the effect of laser photocoagulation on the serum zinc', 'and copper levels was evaluated.', 'In many of those patients,', 'an increase in the serum zinc', 'and copper levels was found one day after the laser treatment.', 'changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit.', 'A significant rise was found one day after the treatment in the level of serum copper', '(P less than .005),', 'and one week thereafter in the level of serum zinc', '(P less than .005).', 'The significance of these results and their connection to the retinal pigment epithelium', 'and choroid is discussed.', 'The best method for preventing medical malpractice litigation is the nurturing', 'and maintenance of the physician-patient relationship.', 'This relationship can be developed by means of care', 'and compassion for the patient as a person', 'and patient education through a free', 'and open discussion of informed consent.', 'The relationship can be maintained by recognizing \"poor result/litigious patient\" situations with one\\'s own patients', 'as well as with patients of other physicians.', 'The best method for preventing medical malpractice litigation is the nurturing', 'and maintenance of the physician-patient relationship.', 'This relationship can be developed by means of care', 'and compassion for the patient as a person', 'and patient education through a free', 'and open discussion of informed consent.', 'The relationship can be maintained by recognizing \"poor result/litigious patient\" situations with one\\'s own patients', 'as well as with patients of other physicians.', 'Abstract', 'Orbital cellulitis as a complication of orbital fracture is discussed.', 'Three cases of orbital cellulitis', 'that followed trauma to the orbit are presented.', 'A description of the proposed mechanism by which an orbital fracture predisposes to cellulitis follows.', 'Suggestions are made for prophylactic management.', 'Abstract', 'Orbital cellulitis as a complication of orbital fracture is discussed.', 'Three cases of orbital cellulitis', 'that followed trauma to the orbit are presented.', 'A description of the proposed mechanism by which an orbital fracture predisposes to cellulitis follows.', 'Suggestions are made for prophylactic management.', 'Always frequent and often misleading,', 'chronic sinusitis are multifactorial.', 'Consequently, the role of teeth apical infection is largely underestimated.', 'The difficulties of interpretation of the sinus radiographic imaging are well established,', 'particularly in the study of the dental with sinus border.', 'The clinical circumstances in which the responsibility of teeth can be involved,', 'are studied.', 'Diagnosis insufficiency are frequent,', 'and inadequate procedures can be undertaken.', 'The particular conditions to obtain good polycyclicidal tomographs pictures are described', 'and demonstrated.', 'In special cases,', 'with the eventual contribution of the bacterial-sensitivity evaluation,', 'these sinusal-dental tomographs allow a relevant diagnosis.', 'Always frequent and often misleading,', 'chronic sinusitis are multifactorial.', 'Consequently, the role of teeth apical infection is largely underestimated.', 'The difficulties of interpretation of the sinus radiographic imaging are well established,', 'particularly in the study of the dental with sinus border.', 'The clinical circumstances in which the responsibility of teeth can be involved,', 'are studied.', 'Diagnosis insufficiency are frequent,', 'and inadequate procedures can be undertaken.', 'The particular conditions to obtain good polycyclicidal tomographs pictures are described', 'and demonstrated.', 'In special cases,', 'with the eventual contribution of the bacterial-sensitivity evaluation,', 'these sinusal-dental tomographs allow a relevant diagnosis.', 'The authors used scanning electronic microscopy to investigate', 'and define morphological characteristics of the musculus stapedius tendon:', 'the opening through which the tendon protrudes at the eminentia pyramidalis', 'and it can be found at its base.', 'Fibers of the tendon may directly be attached to the bony surface of the stages', 'or its prolongation of various forms and dimensions.', 'In some cases the tendon fibrous bundles are more or less ossified.', 'In one patient who was operate we found a congenital malformation manifested as absence of the tendon', 'and deformation of the stapes.', 'The tendon absence is found in 0.5% of cases.', 'The authors used scanning electronic microscopy to investigate', 'and define morphological characteristics of the musculus stapedius tendon:', 'the opening through which the tendon protrudes at the eminentia pyramidalis', 'and it can be found at its base.', 'Fibers of the tendon may directly be attached to the bony surface of the stages', 'or its prolongation of various forms and dimensions.', 'In some cases the tendon fibrous bundles are more or less ossified.', 'In one patient who was operate we found a congenital malformation manifested as absence of the tendon', 'and deformation of the stapes.', 'The tendon absence is found in 0.5% of cases.', 'The insertion of autogenous fascia lata to suspend the paralyzed cheek', 'and lips remains one of the most widely practiced procedures in correction of long term paralysis of the lower facial muscles.', 'For 34 months we used thin layers', '(1 mm)', 'of expanded polytetrafluoroethylene', '(E-PTFE Gore-Tex Soft Tissue Patch)', 'to replace autogenous fascia lata,', 'in 20 patients,', 'with satisfactory results.', '13 patients were treated by classical techniques of insertion-suspensions of the paralyzed side with a perioral loop', 'and slings of PTFE suspended to the zygomatic arch', 'and the infraorbital rim,', 'by way of nasolabial angle or rhytidectomy incisions.', \"In 7 patients, an eyelid suspension was performed with PTFE by Arion's technique,\", 'but by replacing the classical silicon thread by E-PTFE', 'and transposing the medial part of the temporalis muscle on the external canthus,', 'and fixing the lateral end of the sling to the muscle.', 'This technique assures a good corneal coverage with healing of the previous ulceration', 'and allows a voluntary occlusion of the eyelids.', 'The PTFE soft tissue is notably successful as an implant.', 'This biocompatible material shows excellent tissue tolerance,', 'the porous microstructure encouraging tissue attachment', 'and infiltration.', 'E-PTFE holds its shape,', 'resists to infection,', 'and permits a notably reduction of the hospitalization', 'because it avoids a second surgical site.', 'We have experience of the material as a reconstructive substance in 6 cases of either bone', 'or subcutaneous deficits,', 'where no other satisfactory solution was available.', 'The insertion of autogenous fascia lata to suspend the paralyzed cheek', 'and lips remains one of the most widely practiced procedures in correction of long term paralysis of the lower facial muscles.', 'For 34 months we used thin layers', '(1 mm)', 'of expanded polytetrafluoroethylene', '(E-PTFE Gore-Tex Soft Tissue Patch)', 'to replace autogenous fascia lata,', 'in 20 patients,', 'with satisfactory results.', '13 patients were treated by classical techniques of insertion-suspensions of the paralyzed side with a perioral loop', 'and slings of PTFE suspended to the zygomatic arch', 'and the infraorbital rim,', 'by way of nasolabial angle or rhytidectomy incisions.', \"In 7 patients, an eyelid suspension was performed with PTFE by Arion's technique,\", 'but by replacing the classical silicon thread by E-PTFE', 'and transposing the medial part of the temporalis muscle on the external canthus,', 'and fixing the lateral end of the sling to the muscle.', 'This technique assures a good corneal coverage with healing of the previous ulceration', 'and allows a voluntary occlusion of the eyelids.', 'The PTFE soft tissue is notably successful as an implant.', 'This biocompatible material shows excellent tissue tolerance,', 'the porous microstructure encouraging tissue attachment', 'and infiltration.', 'E-PTFE holds its shape,', 'resists to infection,', 'and permits a notably reduction of the hospitalization', 'because it avoids a second surgical site.', 'We have experience of the material as a reconstructive substance in 6 cases of either bone', 'or subcutaneous deficits,', 'where no other satisfactory solution was available.', 'Of the many proposed classifications for staging maxillary sinus cancer,', 'none has been adopted universally and none is known to be superior to the others.', 'This study identified the best of six currently used classifications using data from 53 previously untreated patients with squamous cell carcinoma of the maxillary sinus.', \"Analysis of each classification's ability to stage the majority of patients,\", 'produce a balanced distribution of T stages,', 'and correlate T stage with treatment', \"and prognosis revealed Harrison's classification to be the best.\", \"Harrison's classification should be adopted worldwide as the classification of choice for staging squamous cell carcinoma of the maxillary sinus.\", 'Of the many proposed classifications for staging maxillary sinus cancer,', 'none has been adopted universally and none is known to be superior to the others.', 'This study identified the best of six currently used classifications using data from 53 previously untreated patients with squamous cell carcinoma of the maxillary sinus.', \"Analysis of each classification's ability to stage the majority of patients,\", 'produce a balanced distribution of T stages,', 'and correlate T stage with treatment', \"and prognosis revealed Harrison's classification to be the best.\", \"Harrison's classification should be adopted worldwide as the classification of choice for staging squamous cell carcinoma of the maxillary sinus.\", \"Twenty-four patients with non Hodgkin's lymphoma of nasopharynx were treated in the Institut Gustave-Roussy,\", 'France,', 'between 1976 and 1983.', 'Mean age of patients was 53 years', 'and the male/female sex ratio was 2.', 'Symptomatology was immediately suggestive of a lesion in the nasopharynx in 21 of the 24 patients,', 'and histology gave an unfavorable prognosis', '(diffuse large cell tumors)', 'in 71% of cases.', 'Complete investigations showed the lymphoma to be relatively limited in extent in the majority of cases', '(68% of stages I and II).', 'These findings are in agreement with published reports.', 'The 4 patient with stage I disease were treated by irradiation alone', 'and are all alive without recurrence after 43 months.', 'and at 42-month follow up 8 were alive without recurrence', '(1 patient died of intercurrent illness).', 'Results of conventional chemotherapy', '(CHVmP)', 'in patients with stages III', 'and IV were disappointing,', 'however: only one patient survived for 60 months', '(after a cerebral recurrence treated by irradiation).', 'Although current therapy is effective for stage I', 'and II lesions,', 'patients with stages III', 'and IV', 'require more aggressive therapy recently developed:', 'heavy multiple chemotherapy,', 'and even intensive chemotherapy-total body irradiation', 'and bone marrow graft.', \"Twenty-four patients with non Hodgkin's lymphoma of nasopharynx were treated in the Institut Gustave-Roussy,\", 'France,', 'between 1976 and 1983.', 'Mean age of patients was 53 years', 'and the male/female sex ratio was 2.', 'Symptomatology was immediately suggestive of a lesion in the nasopharynx in 21 of the 24 patients,', 'and histology gave an unfavorable prognosis', '(diffuse large cell tumors)', 'in 71% of cases.', 'Complete investigations showed the lymphoma to be relatively limited in extent in the majority of cases', '(68% of stages I and II).', 'These findings are in agreement with published reports.', 'The 4 patient with stage I disease were treated by irradiation alone', 'and are all alive without recurrence after 43 months.', 'and at 42-month follow up 8 were alive without recurrence', '(1 patient died of intercurrent illness).', 'Results of conventional chemotherapy', '(CHVmP)', 'in patients with stages III', 'and IV were disappointing,', 'however: only one patient survived for 60 months', '(after a cerebral recurrence treated by irradiation).', 'Although current therapy is effective for stage I', 'and II lesions,', 'patients with stages III', 'and IV', 'require more aggressive therapy recently developed:', 'heavy multiple chemotherapy,', 'and even intensive chemotherapy-total body irradiation', 'and bone marrow graft.', 'The auditory brain stem response to click stimuli was investigated in a group of 50 full-term healthy newborns,', 'as well as in a group of 20 older children and adults.', 'The stimulus parameters of click level', 'and click repetition rate were varied systematically', 'to quantify the characteristics of the auditory brain stem response in the full-term newborn infant.', 'The results reveal increased latencies for waves I,', 'III,', 'and V for all conditions among the newborns,', 'relative to the older age group.', 'The results suggest that the neurological system is the primary source of differences between newborns', 'and older subjects,', 'but do not rule out the possibility that external ear,', 'middle ear,', 'or cochlear mechanisms may also contribute to the differences observed.', 'The auditory brain stem response to click stimuli was investigated in a group of 50 full-term healthy newborns,', 'as well as in a group of 20 older children and adults.', 'The stimulus parameters of click level', 'and click repetition rate were varied systematically', 'to quantify the characteristics of the auditory brain stem response in the full-term newborn infant.', 'The results reveal increased latencies for waves I,', 'III,', 'and V for all conditions among the newborns,', 'relative to the older age group.', 'The results suggest that the neurological system is the primary source of differences between newborns', 'and older subjects,', 'but do not rule out the possibility that external ear,', 'middle ear,', 'or cochlear mechanisms may also contribute to the differences observed.', 'Cervical spondylosis', 'and ankylosing hyperostosis of the cervical vertebrae are common findings.', 'Although these hypertrophic changes can be completely asymptomatic,', 'Laryngotracheal symptoms due to cervical hyperostosis are less frequent', 'and may be managed initially as tumors of the esophagus,', 'trachea,', 'or thyroid gland.', 'The management of two severe cases of dyspnea due to cervical ankylosing hyperostosis are discussed.', 'Cervical spondylosis', 'and ankylosing hyperostosis of the cervical vertebrae are common findings.', 'Although these hypertrophic changes can be completely asymptomatic,', 'Laryngotracheal symptoms due to cervical hyperostosis are less frequent', 'and may be managed initially as tumors of the esophagus,', 'trachea,', 'or thyroid gland.', 'The management of two severe cases of dyspnea due to cervical ankylosing hyperostosis are discussed.', 'The value of a perceptual-physiologic approach to both the evaluation', 'and treatment of voice disorders is described', 'Objective measures are obtained to facilitate the understanding of the underlying pathophysiology', 'and to document change over time.', 'Evaluation data are recorded', 'and interpreted collectively by a speech-otolaryngology team', 'Finally, the evaluation methods, particularly videolaryngoscopy,', 'The value of a perceptual-physiologic approach to both the evaluation', 'and treatment of voice disorders is described', 'Objective measures are obtained to facilitate the understanding of the underlying pathophysiology', 'and to document change over time.', 'Evaluation data are recorded', 'and interpreted collectively by a speech-otolaryngology team', 'Finally, the evaluation methods, particularly videolaryngoscopy,', 'The postnatal development of the eustachian tube', 'and its surrounding structures', '(tubal cartilage, tubal gland, tensor veli palatini muscle,', 'and levator veli palatini muscle)', 'was investigated in serial vertical histologic sections from 12 normal temporal bones of individuals', 'After projecting tissue sections onto paper and tracing the structures,', 'several measurements were made in order to analyze this development.', 'Findings revealed that the eustachian tube', 'and its accessory structures developed postnatally up to the age of 19 years.', 'The lumen area in a 19-year-old specimen was 4.7 times', 'that of the 1-day-old infant.', 'The development of the tube was greatest in the pharyngeal part.', 'Postnatal development of the eustachian tube appears', 'The cartilage area in the 19-year-old specimen was 3.6 times that', 'The mucosal acinar cells were predominant in infants but no greater', 'The areas of the tensor', 'and levator veli palatini muscles in a 19-year-old specimen were 5.1', 'respectively,', 'those of the 1-day-old infant.', 'This preliminary study reports the postnatal development of the eustachian tube', 'and its accessory structures,', 'However, because of the limited number of cases available,', 'further investigation of a greater number of cases should be performed so that the relationship between tubal development', 'The postnatal development of the eustachian tube', 'and its surrounding structures', '(tubal cartilage, tubal gland, tensor veli palatini muscle,', 'and levator veli palatini muscle)', 'was investigated in serial vertical histologic sections from 12 normal temporal bones of individuals', 'After projecting tissue sections onto paper and tracing the structures,', 'several measurements were made in order to analyze this development.', 'Findings revealed that the eustachian tube', 'and its accessory structures developed postnatally up to the age of 19 years.', 'The lumen area in a 19-year-old specimen was 4.7 times', 'that of the 1-day-old infant.', 'The development of the tube was greatest in the pharyngeal part.', 'Postnatal development of the eustachian tube appears', 'The cartilage area in the 19-year-old specimen was 3.6 times that', 'The mucosal acinar cells were predominant in infants but no greater', 'The areas of the tensor', 'and levator veli palatini muscles in a 19-year-old specimen were 5.1', 'respectively,', 'those of the 1-day-old infant.', 'This preliminary study reports the postnatal development of the eustachian tube', 'and its accessory structures,', 'However, because of the limited number of cases available,', 'further investigation of a greater number of cases should be performed so that the relationship between tubal development', 'The eustachian tubes of 29 influenza', 'A virus-infected chinchillas were examined for histopathologic signs at intervals up to 21 days after inoculation to elucidate the pathologic basis of negative middle ear pressure,', 'In the animal model,', 'eardrum inflammation', 'and negative middle ear pressure mirrored epithelial damage in the eustachian tube', 'Epithelial damage was greatest in the proximal two thirds of the tube near the nasopharynx,', 'whereas goblet cell metaplasia', 'and increased secretory activity was greatest in the distal,', 'These results provide a morphologic correlate to the development of negative middle ear pressure,', 'The eustachian tubes of 29 influenza', 'A virus-infected chinchillas were examined for histopathologic signs at intervals up to 21 days after inoculation to elucidate the pathologic basis of negative middle ear pressure,', 'In the animal model,', 'eardrum inflammation', 'and negative middle ear pressure mirrored epithelial damage in the eustachian tube', 'Epithelial damage was greatest in the proximal two thirds of the tube near the nasopharynx,', 'whereas goblet cell metaplasia', 'and increased secretory activity was greatest in the distal,', 'These results provide a morphologic correlate to the development of negative middle ear pressure,', 'Since 1980 we have used middle ear implants of aluminum oxide ceramic', 'We had opportunities to examine 108 ears', '(91 patients);', 'Follow-up examinations showed that the rates of extrusion', 'and liability to extrusion can be reduced by interposing a small disk of autologous cartilage', 'Moreover, interposing cartilage trends to improve audiometric results.', 'On the basis of our study,', 'we strongly recommend the interposition of a disk of autologous cartilage', 'and perichondrium between the middle ear implant', 'Since 1980 we have used middle ear implants of aluminum oxide ceramic', 'We had opportunities to examine 108 ears', '(91 patients);', 'Follow-up examinations showed that the rates of extrusion', 'and liability to extrusion can be reduced by interposing a small disk of autologous cartilage', 'Moreover, interposing cartilage trends to improve audiometric results.', 'On the basis of our study,', 'we strongly recommend the interposition of a disk of autologous cartilage', 'and perichondrium between the middle ear implant', 'Pneumosinus dilatans is an extremely rare condition', 'in which facial deformity is caused by gross enlargement of apparently normal paranasal sinuses.', 'Pneumosinus dilatans is an extremely rare condition', 'in which facial deformity is caused by gross enlargement of apparently normal paranasal sinuses.', 'The internal jugular vein is an uncommon site of deep venous thrombosis.', 'Infection,', 'prolonged central venous catheterization,', 'We present an unusual case of spontaneous thrombosis of the internal jugular vein.', 'The evaluation of this case included a search for possible anatomic,', 'hematologic,', 'and oncologic conditions which could predispose', 'to hypercoagulability', 'Anticoagulation therapy resulted in complete resolution of thrombosis', 'The internal jugular vein is an uncommon site of deep venous thrombosis.', 'Infection,', 'prolonged central venous catheterization,', 'We present an unusual case of spontaneous thrombosis of the internal jugular vein.', 'The evaluation of this case included a search for possible anatomic,', 'hematologic,', 'and oncologic conditions which could predispose', 'to hypercoagulability', 'Anticoagulation therapy resulted in complete resolution of thrombosis', 'Our previous investigation using normal guinea pigs disclosed that,', 'like respiratory mucosa,', 'the middle ear lining has ciliary activity,', 'In this experimental study,', 'the effects of Staphylococcus aureus on the middle ear lining was examined from functional', 'In conclusion,', 'the ciliary activity at the entrance to the eustachian tube', 'and that more distal to the tube present a similar pattern of reaction', 'the ciliary activity of the middle ear lining displays a varying pattern of reaction according to the locations within the tympanic cavity.', 'Our previous investigation using normal guinea pigs disclosed that,', 'like respiratory mucosa,', 'the middle ear lining has ciliary activity,', 'In this experimental study,', 'the effects of Staphylococcus aureus on the middle ear lining was examined from functional', 'In conclusion,', 'the ciliary activity at the entrance to the eustachian tube', 'and that more distal to the tube present a similar pattern of reaction', 'the ciliary activity of the middle ear lining displays a varying pattern of reaction according to the locations within the tympanic cavity.', 'An otherwise healthy male patient had a bilateral,', 'slowly progressive hearing loss first noticed in early childhood', 'Audiometric studies at the age of 32 showed a moderately severe,', 'bilateral, mixed type hearing loss.', 'Studies of the temporal bones showed bilateral hypoplasia', 'The history and findings are consistent with a genetically determined hypoplasia', 'An otherwise healthy male patient had a bilateral,', 'slowly progressive hearing loss first noticed in early childhood', 'Audiometric studies at the age of 32 showed a moderately severe,', 'bilateral, mixed type hearing loss.', 'Studies of the temporal bones showed bilateral hypoplasia', 'The history and findings are consistent with a genetically determined hypoplasia', \"Epithelial malignancy in a Warthin's tumor exists in three forms.\", 'The most common is a coexistent, separate neoplasm.', \"Metastasis to the lymphoid component of Warthin's tumor follows,\", 'and the least common is a primary carcinoma arising from the ductal component of the tumor.', \"Epithelial malignancy in a Warthin's tumor exists in three forms.\", 'The most common is a coexistent, separate neoplasm.', \"Metastasis to the lymphoid component of Warthin's tumor follows,\", 'and the least common is a primary carcinoma arising from the ductal component of the tumor.', 'Clinic and histologic analysis of 92 phototests of various photodermatosis', '(14 benign summer light eruption', '47 polymorphous light eruption', '20 persistent light reactor photodermatitis', '11 drug induced photodermatitis', 'Clinically,', 'a phototoxic aspect was observed in 30 cases,', 'it was photoallergic in 36 cases', 'Histologically,', 'a phototoxic picture was present in 32 cases,', 'a photoallergic one in 48 cases,', 'Only one case of BSLE was associated with photoallergic aspect,', 'Photoallergic type was observed in 70 p. 100 of iterative exposure phototest', 'of PMLE;', 'clinical and histological aspects were concordant;', 'in contrast phototoxic aspects were constantly observed when single exposure phototest', 'In PLRP histological aspect of IEP was constantly of photoallergic type;', 'whereas clinical aspect was uncertain in 50 p. 100 of the cases;', 'In DIP histological aspect of IEP was constantly photoallergic,', 'whereas 5 of the 6 cases exhibited uncertain clinical aspect;', 'only one of the 5 SEP gave a photoallergic reaction both clinically', 'Comparison of histological features revealed a more typical photoallergic picture in PLRP', 'We conclude that:', 'the reproducibility of clinical', 'and histological lesions is better with the iterative exposure phototest', 'histologic examination is more reliable', 'than clinical findings in persistent light reactor photodermatitis', 'Clinic and histologic analysis of 92 phototests of various photodermatosis', '(14 benign summer light eruption', '47 polymorphous light eruption', '20 persistent light reactor photodermatitis', '11 drug induced photodermatitis', 'Clinically,', 'a phototoxic aspect was observed in 30 cases,', 'it was photoallergic in 36 cases', 'Histologically,', 'a phototoxic picture was present in 32 cases,', 'a photoallergic one in 48 cases,', 'Only one case of BSLE was associated with photoallergic aspect,', 'Photoallergic type was observed in 70 p. 100 of iterative exposure phototest', 'of PMLE;', 'clinical and histological aspects were concordant;', 'in contrast phototoxic aspects were constantly observed when single exposure phototest', 'In PLRP histological aspect of IEP was constantly of photoallergic type;', 'whereas clinical aspect was uncertain in 50 p. 100 of the cases;', 'In DIP histological aspect of IEP was constantly photoallergic,', 'whereas 5 of the 6 cases exhibited uncertain clinical aspect;', 'only one of the 5 SEP gave a photoallergic reaction both clinically', 'Comparison of histological features revealed a more typical photoallergic picture in PLRP', 'We conclude that:', 'the reproducibility of clinical', 'and histological lesions is better with the iterative exposure phototest', 'histologic examination is more reliable', 'than clinical findings in persistent light reactor photodermatitis', 'Radioactive iodine', '(I-131)', 'uptakes', '(RAIU)', 'and serum T4 levels were observed in 72 patients referred for radioiodine therapy.', 'Of 65 patients whose results could be evaluated 47 had these tests while on carbiomazole', '(CMZ).', 'In the remaining 18 patients of the 65,', 'The differences in the results were again not significant.', 'The incidence of hypothyroidism at 1 year was noted in the group receiving radioiodine therapy while on CMZ as compared to those stopping the CMZ for 1-12 days.', 'Serum T4 levels were measured in 21 patients,', 'Radioactive iodine', '(I-131)', 'uptakes', '(RAIU)', 'and serum T4 levels were observed in 72 patients referred for radioiodine therapy.', 'Of 65 patients whose results could be evaluated 47 had these tests while on carbiomazole', '(CMZ).', 'In the remaining 18 patients of the 65,', 'The differences in the results were again not significant.', 'The incidence of hypothyroidism at 1 year was noted in the group receiving radioiodine therapy while on CMZ as compared to those stopping the CMZ for 1-12 days.', 'Serum T4 levels were measured in 21 patients,', 'A prospective study of 17 patients with fracture neck of femur was undertaken to study the vascularity of the femoral head by bone scan using Technetium-99m MDP,', 'as well as to study the viability of the femoral head by histopathology,', 'and therefore determine correlation,', 'Treatment was by primary endoprosthetic replacement', 'In 16 of the 17 patients the bone scan showed decreased vascularity of the head,', 'It is likely that,', 'since the majority of the patients were operated on within 4 days,', 'there was insufficient time for histopathological evidence of ischemia to develop,', 'A prospective study of 17 patients with fracture neck of femur was undertaken to study the vascularity of the femoral head by bone scan using Technetium-99m MDP,', 'as well as to study the viability of the femoral head by histopathology,', 'and therefore determine correlation,', 'Treatment was by primary endoprosthetic replacement', 'In 16 of the 17 patients the bone scan showed decreased vascularity of the head,', 'It is likely that,', 'since the majority of the patients were operated on within 4 days,', 'there was insufficient time for histopathological evidence of ischemia to develop,', 'Serum thyroglobulin antibodies', '(TgAb)', 'and serum thyroglobulin', '(Tg)', 'The TgAb and Tg levels were measured in the patients when they were on replacement thyroxine therapy prior to the whole body radioiodine I-131 scan (WBS),', 'The TgAb level in normal controls was from 0-4.8 micrograms/ml,', 'with a mean of 0.6 microgram/ml,', 'while the Tg level in these controls was from 0.0-48.0 ng/ml,', 'In the patients with thyroid cancer,', 'Their Tg levels were normal except for 7 patients who had elevated values.', '2 of these 7 had no evidence of disease on X-ray,', 'bone scan,', 'or WBS (I-131)', 'However, 4 patients had metastatic disease with normal Tg levels,', 'and 2 of these patients had residual thyroid in the neck and raised TgAb levels,', 'while in the remaining 2 patients,', 'Serum Tg is a useful tumour marker for recurrent or metastatic thyroid cancer,', 'Serum thyroglobulin antibodies', '(TgAb)', 'and serum thyroglobulin', '(Tg)', 'The TgAb and Tg levels were measured in the patients when they were on replacement thyroxine therapy prior to the whole body radioiodine I-131 scan (WBS),', 'The TgAb level in normal controls was from 0-4.8 micrograms/ml,', 'with a mean of 0.6 microgram/ml,', 'while the Tg level in these controls was from 0.0-48.0 ng/ml,', 'In the patients with thyroid cancer,', 'Their Tg levels were normal except for 7 patients who had elevated values.', '2 of these 7 had no evidence of disease on X-ray,', 'bone scan,', 'or WBS (I-131)', 'However, 4 patients had metastatic disease with normal Tg levels,', 'and 2 of these patients had residual thyroid in the neck and raised TgAb levels,', 'while in the remaining 2 patients,', 'Serum Tg is a useful tumour marker for recurrent or metastatic thyroid cancer,', 'Hepatic lithiasis is a major disease in Asia,', 'especially East Asia.', 'The radionuclide methods ranging from sequential scintigraphy', 'Hepatic lithiasis is a major disease in Asia,', 'especially East Asia.', 'The radionuclide methods ranging from sequential scintigraphy', 'Cancers of childhood tend to be aggressive and to be associated with poor prognoses.', 'Early diagnosis is essential and simple methods of assessing response to therapy are critical.', 'Nuclear Medicine procedures have proved invaluable.', 'Achieving these objectives has become even more important by improvement in therapeutic regimes', 'which have reduced the pessimistic prognoses.', 'Recent advances in Nuclear Medicine equipment have markedly increased the sensitivity of the investigations', 'and the introduction of new radiopharmaceuticals has greatly raised the specificity,', 'Recent research indicates the probability of localizing the radioactive labelled agents,', 'such as monoclonal antibodies,', 'Cancers of childhood tend to be aggressive and to be associated with poor prognoses.', 'Early diagnosis is essential and simple methods of assessing response to therapy are critical.', 'Nuclear Medicine procedures have proved invaluable.', 'Achieving these objectives has become even more important by improvement in therapeutic regimes', 'which have reduced the pessimistic prognoses.', 'Recent advances in Nuclear Medicine equipment have markedly increased the sensitivity of the investigations', 'and the introduction of new radiopharmaceuticals has greatly raised the specificity,', 'Recent research indicates the probability of localizing the radioactive labelled agents,', 'such as monoclonal antibodies,', 'Flow cytofluorometry and drug targeting with labelled neoglycoproteins are used as tools to probe for membrane lectins in two human adenocarcinoma cell lines.', 'Both cell lines express activities for galactosides,', 'Affinity chromatography on gels with immobilized sugar leads to purification of an alpha-galactoside-binding protein at an apparent molecular weight of 64 kDa that also binds to lactose,', 'maltose and fucose', 'and exhibits Ca2+-requirement for binding,', 'a beta-galactoside-binding protein without Ca2+-requirement at an apparent molecular weight of 14 kDa,', 'The description of membrane lectins,', 'documented here for the first human tumor cell lines,', 'Flow cytofluorometry and drug targeting with labelled neoglycoproteins are used as tools to probe for membrane lectins in two human adenocarcinoma cell lines.', 'Both cell lines express activities for galactosides,', 'Affinity chromatography on gels with immobilized sugar leads to purification of an alpha-galactoside-binding protein at an apparent molecular weight of 64 kDa that also binds to lactose,', 'maltose and fucose', 'and exhibits Ca2+-requirement for binding,', 'a beta-galactoside-binding protein without Ca2+-requirement at an apparent molecular weight of 14 kDa,', 'The description of membrane lectins,', 'documented here for the first human tumor cell lines,', 'Evaluations were made of the expression of blood group antigens,', 'DNA content and S-phase rate in a prospective study of 97 patients with squamous cell carcinoma of the uterine cervix.', 'No correlation was found between loss of ABH expression and ploidy level or the degree of proliferation,', 'In 11 (42%) of 26 tumors with two or three detectable stemlines,', 'distinct areas with disparate ABH staining patterns were found,', 'In tumors with one stemline significantly fewer such tumors were found', 'Twenty-four early recurrences', '(within 20-36 months)', 'High S-phase rates', '(greater than 20%)', 'were associated with a significantly increased recurrence rate', '(p less than 0.05).', 'Evaluations were made of the expression of blood group antigens,', 'DNA content and S-phase rate in a prospective study of 97 patients with squamous cell carcinoma of the uterine cervix.', 'No correlation was found between loss of ABH expression and ploidy level or the degree of proliferation,', 'In 11 (42%) of 26 tumors with two or three detectable stemlines,', 'distinct areas with disparate ABH staining patterns were found,', 'In tumors with one stemline significantly fewer such tumors were found', 'Twenty-four early recurrences', '(within 20-36 months)', 'High S-phase rates', '(greater than 20%)', 'were associated with a significantly increased recurrence rate', '(p less than 0.05).', 'This study investigates the phenomenon of unscheduled DNA synthesis', '(UDS)', 'in cells whose DNA contains Bromodeoxyuridine', '(BrdU)', 'The technique utilized is capable of detecting two probes simultaneously from the DNA of individual cells.', 'However, after adjusting our experimental protocol to eliminate the artifacts produced by the incomplete suppression of DNA synthesis by HU,', 'This study investigates the phenomenon of unscheduled DNA synthesis', '(UDS)', 'in cells whose DNA contains Bromodeoxyuridine', '(BrdU)', 'The technique utilized is capable of detecting two probes simultaneously from the DNA of individual cells.', 'However, after adjusting our experimental protocol to eliminate the artifacts produced by the incomplete suppression of DNA synthesis by HU,', 'The effects of the 18 carbon chain length series of fatty acids on the viability of a 3T6 derived mouse fibroblast cell line were tested.', 'An overall concentration dependent cytotoxic effect was demonstrated with the saturated fatty acid,', 'and cis- and trans- mono- and dienoic fatty acids;', 'however, within a narrow range of concentrations,', 'As a result of the increase in cell viability over this concentration range,', 'The most dramatic cytotoxic effects were shown by cis-C18:3 omega 3 and cis-C18:3', 'omega 6 with ID50 values of +/- 30 micrograms/ml.', 'cis-C18:4 omega 3 showed an initial cytotoxic effect at fatty acid concentrations of 0-20 micrograms/ml,', 'Furthermore, the effects observed do not appear to be dependent upon the 48 hour incubation period or the presence of fetal bovine serum,', 'The effects of the 18 carbon chain length series of fatty acids on the viability of a 3T6 derived mouse fibroblast cell line were tested.', 'An overall concentration dependent cytotoxic effect was demonstrated with the saturated fatty acid,', 'and cis- and trans- mono- and dienoic fatty acids;', 'however, within a narrow range of concentrations,', 'As a result of the increase in cell viability over this concentration range,', 'The most dramatic cytotoxic effects were shown by cis-C18:3 omega 3 and cis-C18:3', 'omega 6 with ID50 values of +/- 30 micrograms/ml.', 'cis-C18:4 omega 3 showed an initial cytotoxic effect at fatty acid concentrations of 0-20 micrograms/ml,', 'Furthermore, the effects observed do not appear to be dependent upon the 48 hour incubation period or the presence of fetal bovine serum,', 'Data obtained during post-surgical monitoring', '(2 to 28 months)', 'It was observed that:', 'whatever the grade of malignancy,', '100% of the patients whose tumor recurred showed,', 'on at least one occasion,', 'abnormal red blood cell', 'spermidine', '(greater than 14 nmoles/8.10(9) RBC)', 'or spermine (SPM)', '30% of the patients whose tumor recurred had abnormal RBC polyamine concentrations,', 'In patients with normal RBC polyamine values,', 'During individual follow-up,', 'RBC SPD levels of each patient were generally significantly correlated to those of SPM,', 'In such conditions we established the regression equation of the group of all recurring patients taken together.', 'We propose a graphic model,', 'including a space corresponding to RBC SPD and SPM levels observed in case of tumor recurrence,', 'Data obtained during post-surgical monitoring', '(2 to 28 months)', 'It was observed that:', 'whatever the grade of malignancy,', '100% of the patients whose tumor recurred showed,', 'on at least one occasion,', 'abnormal red blood cell', 'spermidine', '(greater than 14 nmoles/8.10(9) RBC)', 'or spermine (SPM)', '30% of the patients whose tumor recurred had abnormal RBC polyamine concentrations,', 'In patients with normal RBC polyamine values,', 'During individual follow-up,', 'RBC SPD levels of each patient were generally significantly correlated to those of SPM,', 'In such conditions we established the regression equation of the group of all recurring patients taken together.', 'We propose a graphic model,', 'including a space corresponding to RBC SPD and SPM levels observed in case of tumor recurrence,', 'Gestodene, a new synthetic progestogen, binds to progesterone receptor', '(PR)', 'endometrium and both normal', 'PR in malignant breast could not be quantified on account of the binding of gestodene to oestrogen receptor', 'Sucrose density ultracentrifugation using 3H-oestradiol and 3H-gestodene in normal and malignant breast demonstrated that the gestodene-ER complex in addition to exhibiting the 4S and 5S peaks showed an additional peak', 'Dissociation kinetics of gestodene from ER which was either heat-activated or molybdate-stabilised were comparable to the triphenylethylene class of antioestrogens in that the rate of dissociation,', 'unlike that of oestradiol from ER,', 'The binding of ER-gestodene to DNA-cellulose was also investigated and was found to be approximately 30% less than that of ER-oestradiol.', 'Gestodene, a new synthetic progestogen, binds to progesterone receptor', '(PR)', 'endometrium and both normal', 'PR in malignant breast could not be quantified on account of the binding of gestodene to oestrogen receptor', 'Sucrose density ultracentrifugation using 3H-oestradiol and 3H-gestodene in normal and malignant breast demonstrated that the gestodene-ER complex in addition to exhibiting the 4S and 5S peaks showed an additional peak', 'Dissociation kinetics of gestodene from ER which was either heat-activated or molybdate-stabilised were comparable to the triphenylethylene class of antioestrogens in that the rate of dissociation,', 'unlike that of oestradiol from ER,', 'The binding of ER-gestodene to DNA-cellulose was also investigated and was found to be approximately 30% less than that of ER-oestradiol.', 'This study examines the suitability of the intracranial route of transplantation of human cell lines in nude mice for preclinical testing of cytostatic drugs.', 'Animals in which intramuscularly or intracranially transplanted human tumours developed were treated with cytostatic drugs.', 'Three different cell line generated tumours', '(T-ALL L735, Hodgkin cell line L540 and Burkitt Lymphoma Line BJAB)', 'were treated with four different drugs', 'The data show a good correspondence between the results of treatment of intramuscularly and intracranially transplanted tumours.', 'This study examines the suitability of the intracranial route of transplantation of human cell lines in nude mice for preclinical testing of cytostatic drugs.', 'Animals in which intramuscularly or intracranially transplanted human tumours developed were treated with cytostatic drugs.', 'Three different cell line generated tumours', '(T-ALL L735, Hodgkin cell line L540 and Burkitt Lymphoma Line BJAB)', 'were treated with four different drugs', 'The data show a good correspondence between the results of treatment of intramuscularly and intracranially transplanted tumours.', 'A method is described', 'that applies a low concentration of thymidine to achieve maximal hormonal synchronization in cultures of an established human breast cancer cell line,', 'The method consists of sequential treatments with tamoxifen,', 'estradiol,', 'The increase', '(3.8 fold)', 'in the S-phase fraction achieved by this method is higher than that', '(2.6 fold)', 'The MCF-7 cells,', 'which have an enhanced labeling index', '(S-phase fraction)', 'after being treated with the synchronizing method,', 'have a markedly increased sensitivity to an S-phase-specific cytotoxic agent,', 'This method may have potentially important implications for clinical chemotherapeutic strategy in the treatment of breast cancer.', 'A method is described', 'that applies a low concentration of thymidine to achieve maximal hormonal synchronization in cultures of an established human breast cancer cell line,', 'The method consists of sequential treatments with tamoxifen,', 'estradiol,', 'The increase', '(3.8 fold)', 'in the S-phase fraction achieved by this method is higher than that', '(2.6 fold)', 'The MCF-7 cells,', 'which have an enhanced labeling index', '(S-phase fraction)', 'after being treated with the synchronizing method,', 'have a markedly increased sensitivity to an S-phase-specific cytotoxic agent,', 'This method may have potentially important implications for clinical chemotherapeutic strategy in the treatment of breast cancer.', 'The glutathione transferase', '(EC 2.5.1.18)', 'activity of multidrug-resistant', 'Increase in activity was found in all multidrug-resistant sublines.', 'Electrophoretic separation of cytosolic proteins and immunodetection,', 'using antisera against Class Alpha,', 'Class Mu', 'and Class Pi glutathione transferase,', 'The role of glutathione transferase of Class Pi in the multidrug resistance', 'Reversal of the resistance,', 'using the calcium antagonist verapamil,', 'The glutathione transferase', '(EC 2.5.1.18)', 'activity of multidrug-resistant', 'Increase in activity was found in all multidrug-resistant sublines.', 'Electrophoretic separation of cytosolic proteins and immunodetection,', 'using antisera against Class Alpha,', 'Class Mu', 'and Class Pi glutathione transferase,', 'The role of glutathione transferase of Class Pi in the multidrug resistance', 'Reversal of the resistance,', 'using the calcium antagonist verapamil,', 'Cytostatic treatment of liver metastases from colorectal cancer has been of limited value.', 'It was the aim of this investigation to examine the effect of PALA', '(N-phosphonacetyl-L-aspartate)', 'and D-glucosamine on the level of uracil nucleotides in the liver', 'and in an N-methyl-N-nitrosoguanidine-induced adenocarcinoma of the colon transplanted to the liver of rats,', 'and on the incorporation of 3H-FUrd into the acid-soluble fraction,', 'Combined treatment with PALA and D-glucosamine reduced the UTP pool in the liver and the tumor.', 'Pretreatment with PALA and D-glucosamine increased incorporation of 3H-FUrd into RNA of the liver and kidney,', 'and into the DNA fraction of the liver,', 'Cytostatic treatment of liver metastases from colorectal cancer has been of limited value.', 'It was the aim of this investigation to examine the effect of PALA', '(N-phosphonacetyl-L-aspartate)', 'and D-glucosamine on the level of uracil nucleotides in the liver', 'and in an N-methyl-N-nitrosoguanidine-induced adenocarcinoma of the colon transplanted to the liver of rats,', 'and on the incorporation of 3H-FUrd into the acid-soluble fraction,', 'Combined treatment with PALA and D-glucosamine reduced the UTP pool in the liver and the tumor.', 'Pretreatment with PALA and D-glucosamine increased incorporation of 3H-FUrd into RNA of the liver and kidney,', 'and into the DNA fraction of the liver,', 'A 2,5-pyrrolidinedione linked nitrogen mustard derivative,', \"(R,S)3-[N,N-bis(2-chloroethyl)]-amino-1-(2'-methoxyphenyl)pyrrolidine- 2,5-dione hydrochloride (I)\", 'Since (I) is also active against L1210 and P388 leukaemias,', 'its toxicity in mice was evaluated.', \"Another derivative, (R,S)3-[N,N-bis-(2-chloroethyl)]amino-1-(4'-n-butoxyphenyl)pyrrolidine- 2,5-dione dihydrate (II),\", 'which also possessed significant anticancer effect on P388 and L1210 leukaemias,', 'A 2,5-pyrrolidinedione linked nitrogen mustard derivative,', \"(R,S)3-[N,N-bis(2-chloroethyl)]-amino-1-(2'-methoxyphenyl)pyrrolidine- 2,5-dione hydrochloride (I)\", 'Since (I) is also active against L1210 and P388 leukaemias,', 'its toxicity in mice was evaluated.', \"Another derivative, (R,S)3-[N,N-bis-(2-chloroethyl)]amino-1-(4'-n-butoxyphenyl)pyrrolidine- 2,5-dione dihydrate (II),\", 'which also possessed significant anticancer effect on P388 and L1210 leukaemias,', 'The fourth i.p. passage of the plasmacytoma', '\"PR\"', 'The development of \"PR\" ascitic tumours was slower and the survival time of inoculated mice was longer than that of MOPC-315 inoculated mice.', 'Moreover,', 'the myeloma protein secreted by the \"PR\" tumour cells,', 'differed from MOPC-315 secreted myeloma protein in its electrophoretic mobility', '(fast gamma-2)', 'Chemotherapy by a single injection of melphalan 7.5 mg/kg was effective in preventing the development of both MOPC-315 ascitic tumours and \"PR\" ascitic tumours.', 'Mice cured from MOPC-315 tumours,', 'On the other hand,', 'mice cured from \"PR\" tumour by melphalan were highly susceptible to challenge', 'The fourth i.p. passage of the plasmacytoma', '\"PR\"', 'The development of \"PR\" ascitic tumours was slower and the survival time of inoculated mice was longer than that of MOPC-315 inoculated mice.', 'Moreover,', 'the myeloma protein secreted by the \"PR\" tumour cells,', 'differed from MOPC-315 secreted myeloma protein in its electrophoretic mobility', '(fast gamma-2)', 'Chemotherapy by a single injection of melphalan 7.5 mg/kg was effective in preventing the development of both MOPC-315 ascitic tumours and \"PR\" ascitic tumours.', 'Mice cured from MOPC-315 tumours,', 'On the other hand,', 'mice cured from \"PR\" tumour by melphalan were highly susceptible to challenge', 'Melphalan-induced DNA cross-linking was compared in a human melanoma cell line', '(RPMI 8322)', 'and in phytohaemagglutinin', 'In both cell types a delayed induction of DNA cross-links was observed,', 'Significantly higher peak levels of DNA cross-links were found in PHA-stimulated lymphocytes,', 'The intracellular content of free melphalan was 1.3-fold higher in RPMI 8322 cells,', 'RPMI 8322 cells had a 1.8-fold higher level of glutathione,', 'Melphalan-induced DNA cross-linking was compared in a human melanoma cell line', '(RPMI 8322)', 'and in phytohaemagglutinin', 'In both cell types a delayed induction of DNA cross-links was observed,', 'Significantly higher peak levels of DNA cross-links were found in PHA-stimulated lymphocytes,', 'The intracellular content of free melphalan was 1.3-fold higher in RPMI 8322 cells,', 'RPMI 8322 cells had a 1.8-fold higher level of glutathione,', '[Surgical treatment of functional tumors of the adrenal gland]', '- PMID: 3566191', \"The authors evaluated and operated 40 cases of Cushing's syndrome, 8 cases of Conn's syndrome and 19 cases of phaeochromocytoma.\", \"Cancer of the adrenal gland was observed in 17% of cases of Cushing's syndrome,\", \"There was a female predominance for both Cushing's syndrome and Conn's syndrome.\", 'Four malignant lesions were observed in the series of 19 tumours of the adrenal medulla', 'CT scan was found to be the most accurate of modern diagnostic techniques', 'and was able to define the site and the dimensions of the adrenal tumour.', 'Arteriography was particularly useful in cases with large,', 'highly vascular tumours. The preferred surgical approach was lumbotomy,', 'and, in cases of invasive tumour,', 'Removal of the tumour by adrenalectomy led to resolution of the symptoms and,', '[Surgical treatment of functional tumors of the adrenal gland]', '- PMID: 3566191', \"The authors evaluated and operated 40 cases of Cushing's syndrome, 8 cases of Conn's syndrome and 19 cases of phaeochromocytoma.\", \"Cancer of the adrenal gland was observed in 17% of cases of Cushing's syndrome,\", \"There was a female predominance for both Cushing's syndrome and Conn's syndrome.\", 'Four malignant lesions were observed in the series of 19 tumours of the adrenal medulla', 'CT scan was found to be the most accurate of modern diagnostic techniques', 'and was able to define the site and the dimensions of the adrenal tumour.', 'Arteriography was particularly useful in cases with large,', 'highly vascular tumours. The preferred surgical approach was lumbotomy,', 'and, in cases of invasive tumour,', 'Removal of the tumour by adrenalectomy led to resolution of the symptoms and,', 'The authors report their experience of percutaneous nephrolithotomy over a period of three years.', 'with serious complications in 1% of cases,', 'and an acceptable complete success rate of 83%.', 'The indications for percutaneous nephrolithotomy have been modified since the opening of an extracorporeal lithotripsy unit.', 'the volume,', 'chemical composition and site of the stone and the morphology of the upper urinary tract.', 'Percutaneous nephrolithotomy is the treatment of choice for large stones', '(greater than 2.5 cm),', 'hard stones or those formed proximally to a stenotic lesion of the urinary tract,', 'which can then be dilated at the same procedure.', 'On the other hand, very large staghorn calculi with caliceal branches are best treated by open surgery.', 'The authors report their experience of percutaneous nephrolithotomy over a period of three years.', 'with serious complications in 1% of cases,', 'and an acceptable complete success rate of 83%.', 'The indications for percutaneous nephrolithotomy have been modified since the opening of an extracorporeal lithotripsy unit.', 'the volume,', 'chemical composition and site of the stone and the morphology of the upper urinary tract.', 'Percutaneous nephrolithotomy is the treatment of choice for large stones', '(greater than 2.5 cm),', 'hard stones or those formed proximally to a stenotic lesion of the urinary tract,', 'which can then be dilated at the same procedure.', 'On the other hand, very large staghorn calculi with caliceal branches are best treated by open surgery.', 'The authors describe their technique to preserve the erector nerves during a radical retropubic prostatectomy.', \"The 2nd part consists of discussion based on the author's experiences\", '(20 cases of nerve preservation in a series of 115 radical prostatectomies). The authors stress the insufficient oncological follow-up and the risk of an incorrect indication due to understaging of the cancer in pre-operative period.', 'The authors describe their technique to preserve the erector nerves during a radical retropubic prostatectomy.', \"The 2nd part consists of discussion based on the author's experiences\", '(20 cases of nerve preservation in a series of 115 radical prostatectomies). The authors stress the insufficient oncological follow-up and the risk of an incorrect indication due to understaging of the cancer in pre-operative period.', 'The authors report a case of differentiated myxoid liposarcoma with a survival of 27 years as a result of repeated resections.', 'well differentiated and/or myxoid forms of liposarcomas appear to have a more favourable long term prognosis.', 'This prognosis justifies repeated operations.', 'The post-operative follow-up by computed tomography ensures earlier detection of recurrences.', 'The time of re-operation depends on the volume of the tumour,', 'the rate of growth,', 'The authors report a case of differentiated myxoid liposarcoma with a survival of 27 years as a result of repeated resections.', 'well differentiated and/or myxoid forms of liposarcomas appear to have a more favourable long term prognosis.', 'This prognosis justifies repeated operations.', 'The post-operative follow-up by computed tomography ensures earlier detection of recurrences.', 'The time of re-operation depends on the volume of the tumour,', 'the rate of growth,', 'The author reports a case of rare malformation affecting all of the elements derived from the Wolffian duct,', 'The clinical picture,', 'as in the majority of previously published cases,', 'was initially suggestive of acute prostatitis,', 'but clinical,', 'The treatment is surgical and most authors prefer an intra-peritoneal transvesical approach.', 'The author reports a case of rare malformation affecting all of the elements derived from the Wolffian duct,', 'The clinical picture,', 'as in the majority of previously published cases,', 'was initially suggestive of acute prostatitis,', 'but clinical,', 'The treatment is surgical and most authors prefer an intra-peritoneal transvesical approach.', 'Sub-trigonal phenol injection ensures a satisfactory response in about 50% of patients with detrusor instability.', 'this minimally invasive technique has been found,', 'in our series of 11 patients,', 'at least temporarily.', 'Sub-trigonal phenol injection ensures a satisfactory response in about 50% of patients with detrusor instability.', 'this minimally invasive technique has been found,', 'in our series of 11 patients,', 'at least temporarily.', 'The authors report their experience of retropubic radical prostatectomy for prostatic cancer.', 'Their indications are essentially stage B', 'nd stage C1', 'The post-operative morbidity was low and no cases of incontinence were observed.', \"However, erection was preserved in two cases when Walsh's technique was used.\", 'The authors report their experience of retropubic radical prostatectomy for prostatic cancer.', 'Their indications are essentially stage B', 'nd stage C1', 'The post-operative morbidity was low and no cases of incontinence were observed.', \"However, erection was preserved in two cases when Walsh's technique was used.\", 'Two populations of patients with invasive epithelial bladder carcinoma are considered.', 'received adjuvant postoperative chemotherapy with methotrexate,', 'adriamycin,', '5 fluoro-uracil and cisplatinum.', 'Group B includes 24 patients treated in the same institution but without chemotherapy.', 'In group A,', 'One death was not related to disease progression.', 'The 3-year-disease-free survival rate is 67.7%.', 'Acceptability and tolerance of chemotherapy are satisfactory.', 'In group B,', 'Twenty patients died from distant metastases.', 'The 3-year-disease-free survival rate is 18.5%.', 'No treatment related deaths occurred in either group.', 'Although biased selection in such conditions is inevitable,', 'Two populations of patients with invasive epithelial bladder carcinoma are considered.', 'received adjuvant postoperative chemotherapy with methotrexate,', 'adriamycin,', '5 fluoro-uracil and cisplatinum.', 'Group B includes 24 patients treated in the same institution but without chemotherapy.', 'In group A,', 'One death was not related to disease progression.', 'The 3-year-disease-free survival rate is 67.7%.', 'Acceptability and tolerance of chemotherapy are satisfactory.', 'In group B,', 'Twenty patients died from distant metastases.', 'The 3-year-disease-free survival rate is 18.5%.', 'No treatment related deaths occurred in either group.', 'Although biased selection in such conditions is inevitable,', 'Intraperitoneal infection of mice with M. anthritidis resulted in development of necrotic lesions in the omentum,', 'The effect of 2,4,6-trimethyloxypyrimidine on such affections was studied.', 'Intraperitoneal infection of mice with M. anthritidis resulted in development of necrotic lesions in the omentum,', 'The effect of 2,4,6-trimethyloxypyrimidine on such affections was studied.', 'Antimicrobial effect of lysozyme in combination with a wide set of antimicrobial drugs', 'was studied with respect to 74 bacterial cultures.', 'It was shown that synergism of the antimicrobial effect in the presence of lysozyme was variable for drugs differing in the mechanism of their action', 'and depended on the pathogen species.', 'ampiox,', 'morphocycline,', 'erythromycin and others.', 'The potentiation effect of lysozyme was less pronounced with respect to Coli bacteria and Pseudomonas.', 'tobramycin,', 'sisomicin', 'and amikacin resulted in increasing antimicrobial effect with respect to practically all the microbial cultures tested.', 'The clinical trials of the efficient combinations of the antibiotics', 'Thus, in children with acute pneumonia', '(92 observations)', 'it resulted in more rapid elimination of the temperature reaction,', 'toxic and cardiorespiratiry syndromes,', 'In treatment of 83 children with pyelonephritis complete clinico-laboratory remission was observed in 81 per cent of the cases against 56.4 per cent in the patients treated with the antibiotics without lysozyme.', 'Antimicrobial effect of lysozyme in combination with a wide set of antimicrobial drugs', 'was studied with respect to 74 bacterial cultures.', 'It was shown that synergism of the antimicrobial effect in the presence of lysozyme was variable for drugs differing in the mechanism of their action', 'and depended on the pathogen species.', 'ampiox,', 'morphocycline,', 'erythromycin and others.', 'The potentiation effect of lysozyme was less pronounced with respect to Coli bacteria and Pseudomonas.', 'tobramycin,', 'sisomicin', 'and amikacin resulted in increasing antimicrobial effect with respect to practically all the microbial cultures tested.', 'The clinical trials of the efficient combinations of the antibiotics', 'Thus, in children with acute pneumonia', '(92 observations)', 'it resulted in more rapid elimination of the temperature reaction,', 'toxic and cardiorespiratiry syndromes,', 'In treatment of 83 children with pyelonephritis complete clinico-laboratory remission was observed in 81 per cent of the cases against 56.4 per cent in the patients treated with the antibiotics without lysozyme.', 'The etiologic profile and state of the immune system of the patients with different forms of sinusitis treated at the Moscow Research Institute of the Ear,', 'The treatment of the patients with a combination of penicillin', 'Differences between the humoral factors of the nonspecific defence', 'and between the indices of the cell reactivity of the patients before', 'The etiologic profile and state of the immune system of the patients with different forms of sinusitis treated at the Moscow Research Institute of the Ear,', 'The treatment of the patients with a combination of penicillin', 'Differences between the humoral factors of the nonspecific defence', 'and between the indices of the cell reactivity of the patients before']\n"
          ]
        }
      ],
      "source": [
        "raw_datasets.rename_column(\"#text\",\"text\")\n",
        "raw_datasets.train_test_split(test_size=0.2)\n",
        "\n",
        "print(raw_datasets[\"#text\"])"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!pip3 install torch==1.6\n",
        "import torch"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "GTwFAROD4Rz9",
        "outputId": "8f8faead-c99b-4513-998e-3c338ff13297"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/\n",
            "Requirement already satisfied: torch==1.6 in /usr/local/lib/python3.7/dist-packages (1.6.0)\n",
            "Requirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (from torch==1.6) (1.21.6)\n",
            "Requirement already satisfied: future in /usr/local/lib/python3.7/dist-packages (from torch==1.6) (0.16.0)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 81,
          "referenced_widgets": [
            "b4fdb559d38444efbff9e1261d7e6fc9",
            "8f43742fc54c44e89dfd75c0c6855c02",
            "c5e41eaf1a204ab2ac67b2530d6051ae",
            "0f33ae297a984ac5b48e1e642a93d564",
            "08ce54756ea8417980a906da0897438b",
            "cb35af01c76441d08fba5c46bed3bc14",
            "44892a2a92d44a89b9e1e15a943a621e",
            "fd771a5502e44de789fcdcb36102a3d1",
            "3ef0ae063c5943fe9f3cce6487289870",
            "2f6ef6d4056c4d088664ac8d7b169355",
            "6ca7e15e48164f118022784ded3593da",
            "b1684cf338d246dd8907741e464965fa",
            "dda92dca05ca4897bf3bc5693af46f2a",
            "a49db3987a3248199f9c6625d8c836c7",
            "ed18e1b9370046f3895facecf86567e2",
            "4f0d4c20e52c4acdbafb66c024f293ac",
            "c91cda385987475ba2329d9b77ff1b66",
            "a393377c162045a5bcfaf003de3c5980",
            "c972ab1ab39f4d899728ad5fbad711ae",
            "77716b15c2a24aa6a1c9878093019fe9",
            "0f1d3d53aa094f57bf926f60ea16797a",
            "7c1df77dee5e464c93a72a46df370cd8"
          ]
        },
        "id": "2Zu5tbAlr7BE",
        "outputId": "eed5e700-29c8-4773-ab8d-e7fec373cf62"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Downloading:   0%|          | 0.00/385 [00:00<?, ?B/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "b4fdb559d38444efbff9e1261d7e6fc9"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Downloading:   0%|          | 0.00/228k [00:00<?, ?B/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "b1684cf338d246dd8907741e464965fa"
            }
          },
          "metadata": {}
        }
      ],
      "source": [
        "from transformers import AutoModelForSequenceClassification, AutoTokenizer,DataCollatorWithPadding\n",
        "bio_model = \"dmis-lab/biobert-v1.1\"\n",
        "scibert = \"allenai/scibert_scivocab_uncased\"\n",
        "#roberta_model = \"roberta-base\"\n",
        "#scibert_tokenizer = AutoModelForSequenceClassification.from_pretrained(scibert,num_labels=len(set(all_earnings_sub_df[\"int_relation\"])))\n",
        "scibert_tokenizer = AutoTokenizer.from_pretrained(scibert,add_prefix_space=True, use_fast=False)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 105,
          "referenced_widgets": [
            "ed4b7ab068d24cebb625f5f49b6f80c8",
            "b613b49bc52e4acc88160bbb06e2fd15",
            "fd3a5a0ab26b47428dd77379c4082b2a",
            "4ace639ae8e94fea99cc160c9d251c11",
            "99fa9bcd15b748b29d359e2489775178",
            "6a69ddf311fe48f9b9574a4b0b94b33b",
            "c51b724a70ca4a5d994e936c8fc15cdf",
            "9ec8fedfc7024b6ca2eb938e7cb7b73c",
            "eb55b4968f0f498fa933ce800c773d7f",
            "66d88631b5fd4325be2be15aba751d24",
            "9f5f06454fd84c84a1ab873085fcd0bb"
          ]
        },
        "id": "ftcaXR33r9gO",
        "outputId": "89940dc5-475c-4e54-cf11-7522564b341e"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "  0%|          | 0/2 [00:00<?, ?ba/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "ed4b7ab068d24cebb625f5f49b6f80c8"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Asking to truncate to max_length but no maximum length is provided and the model has no predefined maximum length. Default to no truncation.\n"
          ]
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "['[CLS] background : neoadjuvant chemotherapy [SEP] background : neoadjuvant chemotherapy [SEP]', '[CLS] and radiation [SEP] background : neoadjuvant chemotherapy [SEP]', '[CLS] followed by surgical resection [SEP] and radiation [SEP]', '[CLS] of the rectum [SEP] followed by surgical resection [SEP]', '[CLS] is a standard treatment [SEP] of the rectum [SEP]', '[CLS] for locally advanced rectal cancer. [SEP] is a standard treatment [SEP]', '[CLS] a subset of rectal cancer is caused [SEP] for locally advanced rectal cancer. [SEP]', '[CLS] by a deficiency in mismatch repair. [SEP] a subset of rectal cancer is caused [SEP]', '[CLS] is responsive to programmed death 1 [SEP] by a deficiency in mismatch repair. [SEP]', '[CLS] ( pd - 1 ) [SEP] is responsive to programmed death 1 [SEP]', '[CLS] blockade in the context of metastatic disease, [SEP] ( pd - 1 ) [SEP]', '[CLS] it was hypothesized [SEP] blockade in the context of metastatic disease, [SEP]', '[CLS] that checkpoint blockade could be effective in patients [SEP] it was hypothesized [SEP]', '[CLS] with mismatch repair - deficient, [SEP] that checkpoint blockade could be effective in patients [SEP]', '[CLS] locally advanced rectal cancer. [SEP] with mismatch repair - deficient, [SEP]', '[CLS] methods : we initiated a prospective phase 2 study [SEP] locally advanced rectal cancer. [SEP]', '[CLS] an anti - pd - 1 monoclonal antibody, [SEP] methods : we initiated a prospective phase 2 study [SEP]', '[CLS] was administered [SEP] an anti - pd - 1 monoclonal antibody, [SEP]', '[CLS] every 3 weeks for 6 months [SEP] was administered [SEP]', '[CLS] in patients with mismatch repair - deficient stage ii [SEP] every 3 weeks for 6 months [SEP]', '[CLS] this treatment was to be followed [SEP] in patients with mismatch repair - deficient stage ii [SEP]', '[CLS] by standard chemoradiotherapy [SEP] this treatment was to be followed [SEP]', '[CLS] patients who had a clinical complete response [SEP] by standard chemoradiotherapy [SEP]', '[CLS] after completion of dostarlimab therapy would proceed [SEP] patients who had a clinical complete response [SEP]', '[CLS] 12 months after completion of dostarlimab therapy [SEP] after completion of dostarlimab therapy would proceed [SEP]', '[CLS] or pathological complete response [SEP] 12 months after completion of dostarlimab therapy [SEP]', '[CLS] after completion of dostarlimab therapy with [SEP] or pathological complete response [SEP]', '[CLS] and overall response [SEP] after completion of dostarlimab therapy with [SEP]', '[CLS] to neoadjuvant dostarlimab therapy [SEP] and overall response [SEP]', '[CLS] results : a total of 12 patients have completed treatment [SEP] to neoadjuvant dostarlimab therapy [SEP]', '[CLS] with dostarlimab [SEP] results : a total of 12 patients have completed treatment [SEP]', '[CLS] all 12 patients ( 100 % ; 95 % confidence interval, 74 to 100 ) [SEP] with dostarlimab [SEP]', '[CLS] had a clinical complete response, [SEP] all 12 patients ( 100 % ; 95 % confidence interval, 74 to 100 ) [SEP]', '[CLS] with no evidence of tumor [SEP] had a clinical complete response, [SEP]', '[CLS] on magnetic resonance imaging, [SEP] with no evidence of tumor [SEP]', '[CLS] endoscopic evaluation, [SEP] on magnetic resonance imaging, [SEP]', '[CLS] digital rectal examination, [SEP] endoscopic evaluation, [SEP]', '[CLS] or biopsy. [SEP] digital rectal examination, [SEP]', '[CLS] no patients had received chemoradiotherapy [SEP] or biopsy. [SEP]', '[CLS] or undergone surgery, [SEP] no patients had received chemoradiotherapy [SEP]', '[CLS] and no cases of progression [SEP] or undergone surgery, [SEP]', '[CLS] or recurrence had been reported during follow - up [SEP] and no cases of progression [SEP]', '[CLS] no adverse events of grade 3 [SEP] or recurrence had been reported during follow - up [SEP]', '[CLS] conclusions : mismatch repair - deficient, [SEP] no adverse events of grade 3 [SEP]', '[CLS] locally advanced rectal cancer was highly sensitive [SEP] conclusions : mismatch repair - deficient, [SEP]', '[CLS] background : neoadjuvant chemotherapy [SEP] background : neoadjuvant chemotherapy [SEP]', '[CLS] and radiation [SEP] background : neoadjuvant chemotherapy [SEP]', '[CLS] followed by surgical resection [SEP] and radiation [SEP]', '[CLS] of the rectum [SEP] followed by surgical resection [SEP]', '[CLS] is a standard treatment [SEP] of the rectum [SEP]', '[CLS] for locally advanced rectal cancer. [SEP] is a standard treatment [SEP]', '[CLS] a subset of rectal cancer is caused [SEP] for locally advanced rectal cancer. [SEP]', '[CLS] by a deficiency in mismatch repair. [SEP] a subset of rectal cancer is caused [SEP]', '[CLS] is responsive to programmed death 1 [SEP] by a deficiency in mismatch repair. [SEP]', '[CLS] ( pd - 1 ) [SEP] is responsive to programmed death 1 [SEP]', '[CLS] blockade in the context of metastatic disease, [SEP] ( pd - 1 ) [SEP]', '[CLS] it was hypothesized [SEP] blockade in the context of metastatic disease, [SEP]', '[CLS] that checkpoint blockade could be effective in patients [SEP] it was hypothesized [SEP]', '[CLS] with mismatch repair - deficient, [SEP] that checkpoint blockade could be effective in patients [SEP]', '[CLS] locally advanced rectal cancer. [SEP] with mismatch repair - deficient, [SEP]', '[CLS] methods : we initiated a prospective phase 2 study [SEP] locally advanced rectal cancer. [SEP]', '[CLS] an anti - pd - 1 monoclonal antibody, [SEP] methods : we initiated a prospective phase 2 study [SEP]', '[CLS] was administered [SEP] an anti - pd - 1 monoclonal antibody, [SEP]', '[CLS] every 3 weeks for 6 months [SEP] was administered [SEP]', '[CLS] in patients with mismatch repair - deficient stage ii [SEP] every 3 weeks for 6 months [SEP]', '[CLS] this treatment was to be followed [SEP] in patients with mismatch repair - deficient stage ii [SEP]', '[CLS] by standard chemoradiotherapy [SEP] this treatment was to be followed [SEP]', '[CLS] patients who had a clinical complete response [SEP] by standard chemoradiotherapy [SEP]', '[CLS] after completion of dostarlimab therapy would proceed [SEP] patients who had a clinical complete response [SEP]', '[CLS] 12 months after completion of dostarlimab therapy [SEP] after completion of dostarlimab therapy would proceed [SEP]', '[CLS] or pathological complete response [SEP] 12 months after completion of dostarlimab therapy [SEP]', '[CLS] after completion of dostarlimab therapy with [SEP] or pathological complete response [SEP]', '[CLS] and overall response [SEP] after completion of dostarlimab therapy with [SEP]', '[CLS] to neoadjuvant dostarlimab therapy [SEP] and overall response [SEP]', '[CLS] results : a total of 12 patients have completed treatment [SEP] to neoadjuvant dostarlimab therapy [SEP]', '[CLS] with dostarlimab [SEP] results : a total of 12 patients have completed treatment [SEP]', '[CLS] all 12 patients ( 100 % ; 95 % confidence interval, 74 to 100 ) [SEP] with dostarlimab [SEP]', '[CLS] had a clinical complete response, [SEP] all 12 patients ( 100 % ; 95 % confidence interval, 74 to 100 ) [SEP]', '[CLS] with no evidence of tumor [SEP] had a clinical complete response, [SEP]', '[CLS] on magnetic resonance imaging, [SEP] with no evidence of tumor [SEP]', '[CLS] endoscopic evaluation, [SEP] on magnetic resonance imaging, [SEP]', '[CLS] digital rectal examination, [SEP] endoscopic evaluation, [SEP]', '[CLS] or biopsy. [SEP] digital rectal examination, [SEP]', '[CLS] no patients had received chemoradiotherapy [SEP] or biopsy. [SEP]', '[CLS] or undergone surgery, [SEP] no patients had received chemoradiotherapy [SEP]', '[CLS] and no cases of progression [SEP] or undergone surgery, [SEP]', '[CLS] or recurrence had been reported during follow - up [SEP] and no cases of progression [SEP]', '[CLS] no adverse events of grade 3 [SEP] or recurrence had been reported during follow - up [SEP]', '[CLS] conclusions : mismatch repair - deficient, [SEP] no adverse events of grade 3 [SEP]', '[CLS] locally advanced rectal cancer was highly sensitive [SEP] conclusions : mismatch repair - deficient, [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] objective : [SEP] abstract [SEP]', '[CLS] first, to evaluate the risks of stillbirth [SEP] objective : [SEP]', '[CLS] and neonatal death by gestational age [SEP] first, to evaluate the risks of stillbirth [SEP]', '[CLS] in twin pregnancies with different levels [SEP] and neonatal death by gestational age [SEP]', '[CLS] of growth discordance [SEP] in twin pregnancies with different levels [SEP]', '[CLS] and in relation to small [SEP] of growth discordance [SEP]', '[CLS] and on this basis [SEP] and in relation to small [SEP]', '[CLS] to establish optimal gestational ages [SEP] and on this basis [SEP]', '[CLS] second, [SEP] to establish optimal gestational ages [SEP]', '[CLS] to compare these optimal gestational ages [SEP] second, [SEP]', '[CLS] for twin pregnancies not complicated [SEP] to compare these optimal gestational ages [SEP]', '[CLS] by fetal growth restriction, [SEP] for twin pregnancies not complicated [SEP]', '[CLS] which, in a previous individual patient meta - analysis, [SEP] by fetal growth restriction, [SEP]', '[CLS] and 36 0 / 7 weeks for monochorionic pregnancies. [SEP] which, in a previous individual patient meta - analysis, [SEP]', '[CLS] data sources : [SEP] and 36 0 / 7 weeks for monochorionic pregnancies. [SEP]', '[CLS] a search of medline, embase, clinicaltrials. gov, [SEP] data sources : [SEP]', '[CLS] and ovid between 2015 [SEP] a search of medline, embase, clinicaltrials. gov, [SEP]', '[CLS] and 2018 was performed of cohort studies reporting risks of stillbirth [SEP] and ovid between 2015 [SEP]', '[CLS] studies from a previous meta - analysis [SEP] and 2018 was performed of cohort studies reporting risks of stillbirth [SEP]', '[CLS] using a similar search strategy ( from inception to 2015 ) were combined. [SEP] studies from a previous meta - analysis [SEP]', '[CLS] methods of study selection : overall, of 57 eligible studies, 20 cohort studies [SEP] using a similar search strategy ( from inception to 2015 ) were combined. [SEP]', '[CLS] that contributed original data reporting on 7, 474 dichorionic [SEP] methods of study selection : overall, of 57 eligible studies, 20 cohort studies [SEP]', '[CLS] tabulation, integration, [SEP] that contributed original data reporting on 7, 474 dichorionic [SEP]', '[CLS] and results : [SEP] tabulation, integration, [SEP]', '[CLS] we performed an individual participant data meta - analysis [SEP] and results : [SEP]', '[CLS] to calculate the risk of perinatal death ( risk difference between prospective stillbirth [SEP] we performed an individual participant data meta - analysis [SEP]', '[CLS] analyses were stratified by chorionicity, [SEP] to calculate the risk of perinatal death ( risk difference between prospective stillbirth [SEP]', '[CLS] levels of growth discordance, [SEP] analyses were stratified by chorionicity, [SEP]', '[CLS] for both dichorionic [SEP] levels of growth discordance, [SEP]', '[CLS] the absolute risks of stillbirth [SEP] for both dichorionic [SEP]', '[CLS] and neonatal death were higher when one [SEP] the absolute risks of stillbirth [SEP]', '[CLS] or both twins were sga [SEP] and neonatal death were higher when one [SEP]', '[CLS] and increased with greater levels [SEP] or both twins were sga [SEP]', '[CLS] regardless of level of growth discordance [SEP] and increased with greater levels [SEP]', '[CLS] and birth weight, perinatal risk balanced between 36 0 / 7 - 6 / 7 [SEP] regardless of level of growth discordance [SEP]', '[CLS] and 37 0 / 7 - 6 / 7 weeks of gestation in both dichorionic [SEP] and birth weight, perinatal risk balanced between 36 0 / 7 - 6 / 7 [SEP]', '[CLS] and monochorionic twin pregnancies, [SEP] and 37 0 / 7 - 6 / 7 weeks of gestation in both dichorionic [SEP]', '[CLS] than neonatal death from 37 0 / 7 - 6 / 7 weeks onward. [SEP] and monochorionic twin pregnancies, [SEP]', '[CLS] conclusion : growth discordance [SEP] than neonatal death from 37 0 / 7 - 6 / 7 weeks onward. [SEP]', '[CLS] or sga [SEP] conclusion : growth discordance [SEP]', '[CLS] however, balancing these two risks, [SEP] or sga [SEP]', '[CLS] we did not find evidence [SEP] however, balancing these two risks, [SEP]', '[CLS] that the optimal timing of delivery is changed [SEP] we did not find evidence [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] objective : [SEP] abstract [SEP]', '[CLS] first, to evaluate the risks of stillbirth [SEP] objective : [SEP]', '[CLS] and neonatal death by gestational age [SEP] first, to evaluate the risks of stillbirth [SEP]', '[CLS] in twin pregnancies with different levels [SEP] and neonatal death by gestational age [SEP]', '[CLS] of growth discordance [SEP] in twin pregnancies with different levels [SEP]', '[CLS] and in relation to small [SEP] of growth discordance [SEP]', '[CLS] and on this basis [SEP] and in relation to small [SEP]', '[CLS] to establish optimal gestational ages [SEP] and on this basis [SEP]', '[CLS] second, [SEP] to establish optimal gestational ages [SEP]', '[CLS] to compare these optimal gestational ages [SEP] second, [SEP]', '[CLS] for twin pregnancies not complicated [SEP] to compare these optimal gestational ages [SEP]', '[CLS] by fetal growth restriction, [SEP] for twin pregnancies not complicated [SEP]', '[CLS] which, in a previous individual patient meta - analysis, [SEP] by fetal growth restriction, [SEP]', '[CLS] and 36 0 / 7 weeks for monochorionic pregnancies. [SEP] which, in a previous individual patient meta - analysis, [SEP]', '[CLS] data sources : [SEP] and 36 0 / 7 weeks for monochorionic pregnancies. [SEP]', '[CLS] a search of medline, embase, clinicaltrials. gov, [SEP] data sources : [SEP]', '[CLS] and ovid between 2015 [SEP] a search of medline, embase, clinicaltrials. gov, [SEP]', '[CLS] and 2018 was performed of cohort studies reporting risks of stillbirth [SEP] and ovid between 2015 [SEP]', '[CLS] studies from a previous meta - analysis [SEP] and 2018 was performed of cohort studies reporting risks of stillbirth [SEP]', '[CLS] using a similar search strategy ( from inception to 2015 ) were combined. [SEP] studies from a previous meta - analysis [SEP]', '[CLS] methods of study selection : overall, of 57 eligible studies, 20 cohort studies [SEP] using a similar search strategy ( from inception to 2015 ) were combined. [SEP]', '[CLS] that contributed original data reporting on 7, 474 dichorionic [SEP] methods of study selection : overall, of 57 eligible studies, 20 cohort studies [SEP]', '[CLS] tabulation, integration, [SEP] that contributed original data reporting on 7, 474 dichorionic [SEP]', '[CLS] and results : [SEP] tabulation, integration, [SEP]', '[CLS] we performed an individual participant data meta - analysis [SEP] and results : [SEP]', '[CLS] to calculate the risk of perinatal death ( risk difference between prospective stillbirth [SEP] we performed an individual participant data meta - analysis [SEP]', '[CLS] analyses were stratified by chorionicity, [SEP] to calculate the risk of perinatal death ( risk difference between prospective stillbirth [SEP]', '[CLS] levels of growth discordance, [SEP] analyses were stratified by chorionicity, [SEP]', '[CLS] for both dichorionic [SEP] levels of growth discordance, [SEP]', '[CLS] the absolute risks of stillbirth [SEP] for both dichorionic [SEP]', '[CLS] and neonatal death were higher when one [SEP] the absolute risks of stillbirth [SEP]', '[CLS] or both twins were sga [SEP] and neonatal death were higher when one [SEP]', '[CLS] and increased with greater levels [SEP] or both twins were sga [SEP]', '[CLS] regardless of level of growth discordance [SEP] and increased with greater levels [SEP]', '[CLS] and birth weight, perinatal risk balanced between 36 0 / 7 - 6 / 7 [SEP] regardless of level of growth discordance [SEP]', '[CLS] and 37 0 / 7 - 6 / 7 weeks of gestation in both dichorionic [SEP] and birth weight, perinatal risk balanced between 36 0 / 7 - 6 / 7 [SEP]', '[CLS] and monochorionic twin pregnancies, [SEP] and 37 0 / 7 - 6 / 7 weeks of gestation in both dichorionic [SEP]', '[CLS] than neonatal death from 37 0 / 7 - 6 / 7 weeks onward. [SEP] and monochorionic twin pregnancies, [SEP]', '[CLS] conclusion : growth discordance [SEP] than neonatal death from 37 0 / 7 - 6 / 7 weeks onward. [SEP]', '[CLS] or sga [SEP] conclusion : growth discordance [SEP]', '[CLS] however, balancing these two risks, [SEP] or sga [SEP]', '[CLS] we did not find evidence [SEP] however, balancing these two risks, [SEP]', '[CLS] that the optimal timing of delivery is changed [SEP] we did not find evidence [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] purpose : pancreatic cancer ( pc ) is a common, [SEP] abstract [SEP]', '[CLS] autophagy is involved in the occurrence [SEP] purpose : pancreatic cancer ( pc ) is a common, [SEP]', '[CLS] this study aims to explore the feasibility [SEP] autophagy is involved in the occurrence [SEP]', '[CLS] of using an autophagy - related long noncoding rna [SEP] this study aims to explore the feasibility [SEP]', '[CLS] ( lncrna ) [SEP] of using an autophagy - related long noncoding rna [SEP]', '[CLS] methods : we obtained rna sequencing [SEP] ( lncrna ) [SEP]', '[CLS] and clinical data of patients from the tcga website. [SEP] methods : we obtained rna sequencing [SEP]', '[CLS] the prognostic model, [SEP] and clinical data of patients from the tcga website. [SEP]', '[CLS] generated through univariate [SEP] the prognostic model, [SEP]', '[CLS] and multivariate cox regression analyses, [SEP] generated through univariate [SEP]', '[CLS] receiver operating characteristic [SEP] and multivariate cox regression analyses, [SEP]', '[CLS] ( roc ) curves [SEP] receiver operating characteristic [SEP]', '[CLS] and forest plots were generated for univariate [SEP] ( roc ) curves [SEP]', '[CLS] and multivariate cox regression analyses, [SEP] and forest plots were generated for univariate [SEP]', '[CLS] gene set enrichment analysis ( gsea ) was used to screen enriched gene sets. [SEP] and multivariate cox regression analyses, [SEP]', '[CLS] results : twenty - eight autophagy - related lncrnas were filtered out through univariate cox regression analysis [SEP] gene set enrichment analysis ( gsea ) was used to screen enriched gene sets. [SEP]', '[CLS] ten autophagy - related lncrnas, [SEP] results : twenty - eight autophagy - related lncrnas were filtered out through univariate cox regression analysis [SEP]', '[CLS] including 4 poor prognosis factors [SEP] ten autophagy - related lncrnas, [SEP]', '[CLS] were further screened via multivariate cox regression analysis. [SEP] including 4 poor prognosis factors [SEP]', '[CLS] gsea results demonstrated [SEP] were further screened via multivariate cox regression analysis. [SEP]', '[CLS] conclusion : a signature based on ten autophagy - related lncrnas was identified. [SEP] gsea results demonstrated [SEP]', '[CLS] this signature could be potentially used for evaluating clinical prognosis [SEP] conclusion : a signature based on ten autophagy - related lncrnas was identified. [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] purpose : pancreatic cancer ( pc ) is a common, [SEP] abstract [SEP]', '[CLS] autophagy is involved in the occurrence [SEP] purpose : pancreatic cancer ( pc ) is a common, [SEP]', '[CLS] this study aims to explore the feasibility [SEP] autophagy is involved in the occurrence [SEP]', '[CLS] of using an autophagy - related long noncoding rna [SEP] this study aims to explore the feasibility [SEP]', '[CLS] ( lncrna ) [SEP] of using an autophagy - related long noncoding rna [SEP]', '[CLS] methods : we obtained rna sequencing [SEP] ( lncrna ) [SEP]', '[CLS] and clinical data of patients from the tcga website. [SEP] methods : we obtained rna sequencing [SEP]', '[CLS] the prognostic model, [SEP] and clinical data of patients from the tcga website. [SEP]', '[CLS] generated through univariate [SEP] the prognostic model, [SEP]', '[CLS] and multivariate cox regression analyses, [SEP] generated through univariate [SEP]', '[CLS] receiver operating characteristic [SEP] and multivariate cox regression analyses, [SEP]', '[CLS] ( roc ) curves [SEP] receiver operating characteristic [SEP]', '[CLS] and forest plots were generated for univariate [SEP] ( roc ) curves [SEP]', '[CLS] and multivariate cox regression analyses, [SEP] and forest plots were generated for univariate [SEP]', '[CLS] gene set enrichment analysis ( gsea ) was used to screen enriched gene sets. [SEP] and multivariate cox regression analyses, [SEP]', '[CLS] results : twenty - eight autophagy - related lncrnas were filtered out through univariate cox regression analysis [SEP] gene set enrichment analysis ( gsea ) was used to screen enriched gene sets. [SEP]', '[CLS] ten autophagy - related lncrnas, [SEP] results : twenty - eight autophagy - related lncrnas were filtered out through univariate cox regression analysis [SEP]', '[CLS] including 4 poor prognosis factors [SEP] ten autophagy - related lncrnas, [SEP]', '[CLS] were further screened via multivariate cox regression analysis. [SEP] including 4 poor prognosis factors [SEP]', '[CLS] gsea results demonstrated [SEP] were further screened via multivariate cox regression analysis. [SEP]', '[CLS] conclusion : a signature based on ten autophagy - related lncrnas was identified. [SEP] gsea results demonstrated [SEP]', '[CLS] this signature could be potentially used for evaluating clinical prognosis [SEP] conclusion : a signature based on ten autophagy - related lncrnas was identified. [SEP]', '[CLS] rare alleles and signatures of selection on the immunodominant domains of pfs230 [SEP] rare alleles and signatures of selection on the immunodominant domains of pfs230 [SEP]', '[CLS] and pfs48 / 45 in malaria parasites from western kenya [SEP] rare alleles and signatures of selection on the immunodominant domains of pfs230 [SEP]', '[CLS] abstract [SEP] and pfs48 / 45 in malaria parasites from western kenya [SEP]', '[CLS] background : malaria elimination [SEP] abstract [SEP]', '[CLS] and eradication efforts can be advanced [SEP] background : malaria elimination [SEP]', '[CLS] by including transmission - blocking or reducing vaccines [SEP] and eradication efforts can be advanced [SEP]', '[CLS] key transmission - blocking vaccine candidates, [SEP] by including transmission - blocking or reducing vaccines [SEP]', '[CLS] such as pfs230 domain one and pfs48 / 45 domain 3, [SEP] key transmission - blocking vaccine candidates, [SEP]', '[CLS] we evaluated genetic polymorphism [SEP] such as pfs230 domain one and pfs48 / 45 domain 3, [SEP]', '[CLS] and temporal stability of pfs230 domain one [SEP] we evaluated genetic polymorphism [SEP]', '[CLS] methods : dry blood spots on filter paper were collected from febrile malaria patients reporting [SEP] and temporal stability of pfs230 domain one [SEP]', '[CLS] to community health facilities in endemic areas of homa bay [SEP] methods : dry blood spots on filter paper were collected from febrile malaria patients reporting [SEP]', '[CLS] and an epidemic - prone area of kisii county in 2018 and 2019. [SEP] to community health facilities in endemic areas of homa bay [SEP]', '[CLS] plasmodium speciation was performed using eluted dna and real - time pcr. [SEP] and an epidemic - prone area of kisii county in 2018 and 2019. [SEP]', '[CLS] we sequenced pfs230 domain one on 156 clinical isolates [SEP] plasmodium speciation was performed using eluted dna and real - time pcr. [SEP]', '[CLS] and pfs48 / 45 domain three on 118 clinical isolates to infer the levels of genetic variability, [SEP] we sequenced pfs230 domain one on 156 clinical isolates [SEP]', '[CLS] results : pfs230 domain one had low nucleotide diversity [SEP] and pfs48 / 45 domain three on 118 clinical isolates to infer the levels of genetic variability, [SEP]', '[CLS] ( π = 0. 15 × 10 - 2 ) [SEP] results : pfs230 domain one had low nucleotide diversity [SEP]', '[CLS] six polymorphic sites with nonsynonymous mutations [SEP] ( π = 0. 15 × 10 - 2 ) [SEP]', '[CLS] i539t was a novel variant, whereas g605s was nearing fixation. [SEP] six polymorphic sites with nonsynonymous mutations [SEP]', '[CLS] high conservation index, and three segregating sites, [SEP] i539t was a novel variant, whereas g605s was nearing fixation. [SEP]', '[CLS] resulting in nonsynonymous mutation [SEP] high conservation index, and three segregating sites, [SEP]', '[CLS] some loci of pfs230 d1 were in positive [SEP] resulting in nonsynonymous mutation [SEP]', '[CLS] or positive selection signatures, [SEP] some loci of pfs230 d1 were in positive [SEP]', '[CLS] had a history of recombination. [SEP] or positive selection signatures, [SEP]', '[CLS] mutated loci pairs in pfs48 / 45 domain three had negative linkage disequilibrium, [SEP] had a history of recombination. [SEP]', '[CLS] and some had negative [SEP] mutated loci pairs in pfs48 / 45 domain three had negative linkage disequilibrium, [SEP]', '[CLS] conclusion : the two transmission blocking vaccine candidates have low nucleotide diversity, [SEP] and some had negative [SEP]', '[CLS] a small number of zone - specific variants, [SEP] conclusion : the two transmission blocking vaccine candidates have low nucleotide diversity, [SEP]', '[CLS] high nucleotide conservation index, [SEP] a small number of zone - specific variants, [SEP]', '[CLS] with the near fixation a polymorphic site [SEP] high nucleotide conservation index, [SEP]', '[CLS] it will be necessary [SEP] with the near fixation a polymorphic site [SEP]', '[CLS] to continue monitoring sequence modifications of these domains when designing tbvs [SEP] it will be necessary [SEP]', '[CLS] rare alleles and signatures of selection on the immunodominant domains of pfs230 [SEP] rare alleles and signatures of selection on the immunodominant domains of pfs230 [SEP]', '[CLS] and pfs48 / 45 in malaria parasites from western kenya [SEP] rare alleles and signatures of selection on the immunodominant domains of pfs230 [SEP]', '[CLS] abstract [SEP] and pfs48 / 45 in malaria parasites from western kenya [SEP]', '[CLS] background : malaria elimination [SEP] abstract [SEP]', '[CLS] and eradication efforts can be advanced [SEP] background : malaria elimination [SEP]', '[CLS] by including transmission - blocking or reducing vaccines [SEP] and eradication efforts can be advanced [SEP]', '[CLS] key transmission - blocking vaccine candidates, [SEP] by including transmission - blocking or reducing vaccines [SEP]', '[CLS] such as pfs230 domain one and pfs48 / 45 domain 3, [SEP] key transmission - blocking vaccine candidates, [SEP]', '[CLS] we evaluated genetic polymorphism [SEP] such as pfs230 domain one and pfs48 / 45 domain 3, [SEP]', '[CLS] and temporal stability of pfs230 domain one [SEP] we evaluated genetic polymorphism [SEP]', '[CLS] methods : dry blood spots on filter paper were collected from febrile malaria patients reporting [SEP] and temporal stability of pfs230 domain one [SEP]', '[CLS] to community health facilities in endemic areas of homa bay [SEP] methods : dry blood spots on filter paper were collected from febrile malaria patients reporting [SEP]', '[CLS] and an epidemic - prone area of kisii county in 2018 and 2019. [SEP] to community health facilities in endemic areas of homa bay [SEP]', '[CLS] plasmodium speciation was performed using eluted dna and real - time pcr. [SEP] and an epidemic - prone area of kisii county in 2018 and 2019. [SEP]', '[CLS] we sequenced pfs230 domain one on 156 clinical isolates [SEP] plasmodium speciation was performed using eluted dna and real - time pcr. [SEP]', '[CLS] and pfs48 / 45 domain three on 118 clinical isolates to infer the levels of genetic variability, [SEP] we sequenced pfs230 domain one on 156 clinical isolates [SEP]', '[CLS] results : pfs230 domain one had low nucleotide diversity [SEP] and pfs48 / 45 domain three on 118 clinical isolates to infer the levels of genetic variability, [SEP]', '[CLS] ( π = 0. 15 × 10 - 2 ) [SEP] results : pfs230 domain one had low nucleotide diversity [SEP]', '[CLS] six polymorphic sites with nonsynonymous mutations [SEP] ( π = 0. 15 × 10 - 2 ) [SEP]', '[CLS] i539t was a novel variant, whereas g605s was nearing fixation. [SEP] six polymorphic sites with nonsynonymous mutations [SEP]', '[CLS] high conservation index, and three segregating sites, [SEP] i539t was a novel variant, whereas g605s was nearing fixation. [SEP]', '[CLS] resulting in nonsynonymous mutation [SEP] high conservation index, and three segregating sites, [SEP]', '[CLS] some loci of pfs230 d1 were in positive [SEP] resulting in nonsynonymous mutation [SEP]', '[CLS] or positive selection signatures, [SEP] some loci of pfs230 d1 were in positive [SEP]', '[CLS] had a history of recombination. [SEP] or positive selection signatures, [SEP]', '[CLS] mutated loci pairs in pfs48 / 45 domain three had negative linkage disequilibrium, [SEP] had a history of recombination. [SEP]', '[CLS] and some had negative [SEP] mutated loci pairs in pfs48 / 45 domain three had negative linkage disequilibrium, [SEP]', '[CLS] conclusion : the two transmission blocking vaccine candidates have low nucleotide diversity, [SEP] and some had negative [SEP]', '[CLS] a small number of zone - specific variants, [SEP] conclusion : the two transmission blocking vaccine candidates have low nucleotide diversity, [SEP]', '[CLS] high nucleotide conservation index, [SEP] a small number of zone - specific variants, [SEP]', '[CLS] with the near fixation a polymorphic site [SEP] high nucleotide conservation index, [SEP]', '[CLS] it will be necessary [SEP] with the near fixation a polymorphic site [SEP]', '[CLS] to continue monitoring sequence modifications of these domains when designing tbvs [SEP] it will be necessary [SEP]', '[CLS] salivary progesterone levels in female patients with a history of idiopathic scoliosis : [SEP] salivary progesterone levels in female patients with a history of idiopathic scoliosis : [SEP]', '[CLS] abstract [SEP] salivary progesterone levels in female patients with a history of idiopathic scoliosis : [SEP]', '[CLS] however, in addition to the spinal curvature, [SEP] abstract [SEP]', '[CLS] it has also been reported that patients with idiopathic scoliosis can display other abnormal physiologic parameters, [SEP] however, in addition to the spinal curvature, [SEP]', '[CLS] such as hormone imbalances, [SEP] it has also been reported that patients with idiopathic scoliosis can display other abnormal physiologic parameters, [SEP]', '[CLS] genetic variants, [SEP] such as hormone imbalances, [SEP]', '[CLS] the present study evaluates the salivary progesterone levels [SEP] genetic variants, [SEP]', '[CLS] a group of female patients with a history of idiopathic scoliosis is compared [SEP] the present study evaluates the salivary progesterone levels [SEP]', '[CLS] salivary progesterone levels were 49 % higher in non - scoliosis patients [SEP] a group of female patients with a history of idiopathic scoliosis is compared [SEP]', '[CLS] compared to the scoliosis group [SEP] salivary progesterone levels were 49 % higher in non - scoliosis patients [SEP]', '[CLS] when compared by the menstrual cycling status, [SEP] compared to the scoliosis group [SEP]', '[CLS] while the progesterone level among non - cycling, [SEP] when compared by the menstrual cycling status, [SEP]', '[CLS] these results suggest a potential relationship between salivary progesterone [SEP] while the progesterone level among non - cycling, [SEP]', '[CLS] salivary progesterone levels in female patients with a history of idiopathic scoliosis : [SEP] salivary progesterone levels in female patients with a history of idiopathic scoliosis : [SEP]', '[CLS] abstract [SEP] salivary progesterone levels in female patients with a history of idiopathic scoliosis : [SEP]', '[CLS] however, in addition to the spinal curvature, [SEP] abstract [SEP]', '[CLS] it has also been reported that patients with idiopathic scoliosis can display other abnormal physiologic parameters, [SEP] however, in addition to the spinal curvature, [SEP]', '[CLS] such as hormone imbalances, [SEP] it has also been reported that patients with idiopathic scoliosis can display other abnormal physiologic parameters, [SEP]', '[CLS] genetic variants, [SEP] such as hormone imbalances, [SEP]', '[CLS] the present study evaluates the salivary progesterone levels [SEP] genetic variants, [SEP]', '[CLS] a group of female patients with a history of idiopathic scoliosis is compared [SEP] the present study evaluates the salivary progesterone levels [SEP]', '[CLS] salivary progesterone levels were 49 % higher in non - scoliosis patients [SEP] a group of female patients with a history of idiopathic scoliosis is compared [SEP]', '[CLS] compared to the scoliosis group [SEP] salivary progesterone levels were 49 % higher in non - scoliosis patients [SEP]', '[CLS] when compared by the menstrual cycling status, [SEP] compared to the scoliosis group [SEP]', '[CLS] while the progesterone level among non - cycling, [SEP] when compared by the menstrual cycling status, [SEP]', '[CLS] these results suggest a potential relationship between salivary progesterone [SEP] while the progesterone level among non - cycling, [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] introduction : epidermoid cysts [SEP] abstract [SEP]', '[CLS] ( ec ) [SEP] introduction : epidermoid cysts [SEP]', '[CLS] a primary brain lymphoma development related [SEP] ( ec ) [SEP]', '[CLS] to a ec is presented, [SEP] a primary brain lymphoma development related [SEP]', '[CLS] case presentation : [SEP] to a ec is presented, [SEP]', '[CLS] a woman 40 years old, [SEP] case presentation : [SEP]', '[CLS] harbouring a ec for more than 20 years, [SEP] a woman 40 years old, [SEP]', '[CLS] surgery was performed [SEP] harbouring a ec for more than 20 years, [SEP]', '[CLS] discussion : [SEP] surgery was performed [SEP]', '[CLS] usually to squamous cells carcinoma, [SEP] discussion : [SEP]', '[CLS] inflammatory mechanisms have been advocated [SEP] usually to squamous cells carcinoma, [SEP]', '[CLS] chronic inflammation [SEP] inflammatory mechanisms have been advocated [SEP]', '[CLS] and lymphoma genesis are related, [SEP] chronic inflammation [SEP]', '[CLS] conclusions : even being benign lesions, [SEP] and lymphoma genesis are related, [SEP]', '[CLS] ec can develop malignant tumours due [SEP] conclusions : even being benign lesions, [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] introduction : epidermoid cysts [SEP] abstract [SEP]', '[CLS] ( ec ) [SEP] introduction : epidermoid cysts [SEP]', '[CLS] a primary brain lymphoma development related [SEP] ( ec ) [SEP]', '[CLS] to a ec is presented, [SEP] a primary brain lymphoma development related [SEP]', '[CLS] case presentation : [SEP] to a ec is presented, [SEP]', '[CLS] a woman 40 years old, [SEP] case presentation : [SEP]', '[CLS] harbouring a ec for more than 20 years, [SEP] a woman 40 years old, [SEP]', '[CLS] surgery was performed [SEP] harbouring a ec for more than 20 years, [SEP]', '[CLS] discussion : [SEP] surgery was performed [SEP]', '[CLS] usually to squamous cells carcinoma, [SEP] discussion : [SEP]', '[CLS] inflammatory mechanisms have been advocated [SEP] usually to squamous cells carcinoma, [SEP]', '[CLS] chronic inflammation [SEP] inflammatory mechanisms have been advocated [SEP]', '[CLS] and lymphoma genesis are related, [SEP] chronic inflammation [SEP]', '[CLS] conclusions : even being benign lesions, [SEP] and lymphoma genesis are related, [SEP]', '[CLS] ec can develop malignant tumours due [SEP] conclusions : even being benign lesions, [SEP]', '[CLS] cerebral blood flow [SEP] cerebral blood flow [SEP]', '[CLS] and other predictors of responsiveness [SEP] cerebral blood flow [SEP]', '[CLS] abstract [SEP] and other predictors of responsiveness [SEP]', '[CLS] introduction : monoclonal antibodies [SEP] abstract [SEP]', '[CLS] ( mabs ) [SEP] introduction : monoclonal antibodies [SEP]', '[CLS] the aim of this study was [SEP] ( mabs ) [SEP]', '[CLS] to check if baseline clinical parameters [SEP] the aim of this study was [SEP]', '[CLS] ( cbf ) [SEP] to check if baseline clinical parameters [SEP]', '[CLS] measured by transcranial doppler [SEP] ( cbf ) [SEP]', '[CLS] ( tcd ) [SEP] measured by transcranial doppler [SEP]', '[CLS] may help predict mabs efficacy. [SEP] ( tcd ) [SEP]', '[CLS] methods : electronic charts of migraineurs treated with erenumab [SEP] may help predict mabs efficacy. [SEP]', '[CLS] with baseline tcd evaluations were collected, [SEP] methods : electronic charts of migraineurs treated with erenumab [SEP]', '[CLS] including data on migraine type, [SEP] with baseline tcd evaluations were collected, [SEP]', '[CLS] pain localization, [SEP] including data on migraine type, [SEP]', '[CLS] monthly migraine days [SEP] pain localization, [SEP]', '[CLS] ( mmd ), [SEP] monthly migraine days [SEP]', '[CLS] medication overuse headache [SEP] ( mmd ), [SEP]', '[CLS] mean blood flow velocity [SEP] medication overuse headache [SEP]', '[CLS] ( vm ), [SEP] mean blood flow velocity [SEP]', '[CLS] results : a total of 123 patients were enrolled, [SEP] ( vm ), [SEP]', '[CLS] mean age 38, [SEP] results : a total of 123 patients were enrolled, [SEP]', '[CLS] 75 years, [SEP] mean age 38, [SEP]', '[CLS] 61 with moh, [SEP] 75 years, [SEP]', '[CLS] 72 were good responders ( gr ), [SEP] 61 with moh, [SEP]', '[CLS] 43 ≥75 % reduction in mmd. [SEP] 72 were good responders ( gr ), [SEP]', '[CLS] baseline vm values in mcas were significantly lower in gr [SEP] 43 ≥75 % reduction in mmd. [SEP]', '[CLS] mabs responsiveness ≥50 % was positively associated with unilateral pain localization [SEP] baseline vm values in mcas were significantly lower in gr [SEP]', '[CLS] ( or : 6. 53, 95 % ci : 2. 01 - 23. 93 ; p = 0. 003 ) [SEP] mabs responsiveness ≥50 % was positively associated with unilateral pain localization [SEP]', '[CLS] whereas negatively associated with vm in right mca [SEP] ( or : 6. 53, 95 % ci : 2. 01 - 23. 93 ; p = 0. 003 ) [SEP]', '[CLS] ( or : 0. 96, 95 % ci : 0. 92 - 0. 99 ; p = 0. 012 ), [SEP] whereas negatively associated with vm in right mca [SEP]', '[CLS] and having no relatives with migraine [SEP] ( or : 0. 96, 95 % ci : 0. 92 - 0. 99 ; p = 0. 012 ), [SEP]', '[CLS] conclusions : baseline vm in mca is lower in mabs gr [SEP] and having no relatives with migraine [SEP]', '[CLS] as compared with non - responders [SEP] conclusions : baseline vm in mca is lower in mabs gr [SEP]', '[CLS] simple clinical features [SEP] as compared with non - responders [SEP]', '[CLS] and baseline cbf in anterior circulation might help [SEP] simple clinical features [SEP]', '[CLS] cerebral blood flow [SEP] cerebral blood flow [SEP]', '[CLS] and other predictors of responsiveness [SEP] cerebral blood flow [SEP]', '[CLS] abstract [SEP] and other predictors of responsiveness [SEP]', '[CLS] introduction : monoclonal antibodies [SEP] abstract [SEP]', '[CLS] ( mabs ) [SEP] introduction : monoclonal antibodies [SEP]', '[CLS] the aim of this study was [SEP] ( mabs ) [SEP]', '[CLS] to check if baseline clinical parameters [SEP] the aim of this study was [SEP]', '[CLS] ( cbf ) [SEP] to check if baseline clinical parameters [SEP]', '[CLS] measured by transcranial doppler [SEP] ( cbf ) [SEP]', '[CLS] ( tcd ) [SEP] measured by transcranial doppler [SEP]', '[CLS] may help predict mabs efficacy. [SEP] ( tcd ) [SEP]', '[CLS] methods : electronic charts of migraineurs treated with erenumab [SEP] may help predict mabs efficacy. [SEP]', '[CLS] with baseline tcd evaluations were collected, [SEP] methods : electronic charts of migraineurs treated with erenumab [SEP]', '[CLS] including data on migraine type, [SEP] with baseline tcd evaluations were collected, [SEP]', '[CLS] pain localization, [SEP] including data on migraine type, [SEP]', '[CLS] monthly migraine days [SEP] pain localization, [SEP]', '[CLS] ( mmd ), [SEP] monthly migraine days [SEP]', '[CLS] medication overuse headache [SEP] ( mmd ), [SEP]', '[CLS] mean blood flow velocity [SEP] medication overuse headache [SEP]', '[CLS] ( vm ), [SEP] mean blood flow velocity [SEP]', '[CLS] results : a total of 123 patients were enrolled, [SEP] ( vm ), [SEP]', '[CLS] mean age 38, [SEP] results : a total of 123 patients were enrolled, [SEP]', '[CLS] 75 years, [SEP] mean age 38, [SEP]', '[CLS] 61 with moh, [SEP] 75 years, [SEP]', '[CLS] 72 were good responders ( gr ), [SEP] 61 with moh, [SEP]', '[CLS] 43 ≥75 % reduction in mmd. [SEP] 72 were good responders ( gr ), [SEP]', '[CLS] baseline vm values in mcas were significantly lower in gr [SEP] 43 ≥75 % reduction in mmd. [SEP]', '[CLS] mabs responsiveness ≥50 % was positively associated with unilateral pain localization [SEP] baseline vm values in mcas were significantly lower in gr [SEP]', '[CLS] ( or : 6. 53, 95 % ci : 2. 01 - 23. 93 ; p = 0. 003 ) [SEP] mabs responsiveness ≥50 % was positively associated with unilateral pain localization [SEP]', '[CLS] whereas negatively associated with vm in right mca [SEP] ( or : 6. 53, 95 % ci : 2. 01 - 23. 93 ; p = 0. 003 ) [SEP]', '[CLS] ( or : 0. 96, 95 % ci : 0. 92 - 0. 99 ; p = 0. 012 ), [SEP] whereas negatively associated with vm in right mca [SEP]', '[CLS] and having no relatives with migraine [SEP] ( or : 0. 96, 95 % ci : 0. 92 - 0. 99 ; p = 0. 012 ), [SEP]', '[CLS] conclusions : baseline vm in mca is lower in mabs gr [SEP] and having no relatives with migraine [SEP]', '[CLS] as compared with non - responders [SEP] conclusions : baseline vm in mca is lower in mabs gr [SEP]', '[CLS] simple clinical features [SEP] as compared with non - responders [SEP]', '[CLS] and baseline cbf in anterior circulation might help [SEP] simple clinical features [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] objective : to explore the application value of pelvic magnetic resonance imaging [SEP] abstract [SEP]', '[CLS] ( mri ) scan combined with serum pyruvate kinase isozyme m2 ( pkm2 ), [SEP] objective : to explore the application value of pelvic magnetic resonance imaging [SEP]', '[CLS] and soluble leptin receptor [SEP] ( mri ) scan combined with serum pyruvate kinase isozyme m2 ( pkm2 ), [SEP]', '[CLS] detection in diagnosing endometrial carcinoma [SEP] and soluble leptin receptor [SEP]', '[CLS] methods : the clinical data of 45 patients with pathologically confirmed ec treated in our hospital from may 2019 to may 2020 were retrospectively analyzed. [SEP] detection in diagnosing endometrial carcinoma [SEP]', '[CLS] all patients received pelvic mri scan, [SEP] methods : the clinical data of 45 patients with pathologically confirmed ec treated in our hospital from may 2019 to may 2020 were retrospectively analyzed. [SEP]', '[CLS] serum pkm2, [SEP] all patients received pelvic mri scan, [SEP]', '[CLS] ngal [SEP] serum pkm2, [SEP]', '[CLS] detection was performed, [SEP] ngal [SEP]', '[CLS] and the combination of the two was performed [SEP] detection was performed, [SEP]', '[CLS] results : compared with the joint detection, [SEP] and the combination of the two was performed [SEP]', '[CLS] the number of true positive cases, [SEP] results : compared with the joint detection, [SEP]', '[CLS] sensitivity, specificity, [SEP] the number of true positive cases, [SEP]', '[CLS] and accuracy rate obtained by a single application of pelvic mri or serum pkm2, [SEP] sensitivity, specificity, [SEP]', '[CLS] ngal, [SEP] and accuracy rate obtained by a single application of pelvic mri or serum pkm2, [SEP]', '[CLS] the area under the roc curve of the joint detection was obviously larger than that of single detection ; [SEP] ngal, [SEP]', '[CLS] the results of the joint detection were better than those of single detection [SEP] the area under the roc curve of the joint detection was obviously larger than that of single detection ; [SEP]', '[CLS] the combined diagnosis obtained the highest sensitivity. [SEP] the results of the joint detection were better than those of single detection [SEP]', '[CLS] conclusion : combining pelvic mri with serum pkm2, [SEP] the combined diagnosis obtained the highest sensitivity. [SEP]', '[CLS] ngal, [SEP] conclusion : combining pelvic mri with serum pkm2, [SEP]', '[CLS] and sob - 4 detection can effectively promote the diagnostic accuracy for ec, [SEP] ngal, [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] objective : to explore the application value of pelvic magnetic resonance imaging [SEP] abstract [SEP]', '[CLS] ( mri ) scan combined with serum pyruvate kinase isozyme m2 ( pkm2 ), [SEP] objective : to explore the application value of pelvic magnetic resonance imaging [SEP]', '[CLS] and soluble leptin receptor [SEP] ( mri ) scan combined with serum pyruvate kinase isozyme m2 ( pkm2 ), [SEP]', '[CLS] detection in diagnosing endometrial carcinoma [SEP] and soluble leptin receptor [SEP]', '[CLS] methods : the clinical data of 45 patients with pathologically confirmed ec treated in our hospital from may 2019 to may 2020 were retrospectively analyzed. [SEP] detection in diagnosing endometrial carcinoma [SEP]', '[CLS] all patients received pelvic mri scan, [SEP] methods : the clinical data of 45 patients with pathologically confirmed ec treated in our hospital from may 2019 to may 2020 were retrospectively analyzed. [SEP]', '[CLS] serum pkm2, [SEP] all patients received pelvic mri scan, [SEP]', '[CLS] ngal [SEP] serum pkm2, [SEP]', '[CLS] detection was performed, [SEP] ngal [SEP]', '[CLS] and the combination of the two was performed [SEP] detection was performed, [SEP]', '[CLS] results : compared with the joint detection, [SEP] and the combination of the two was performed [SEP]', '[CLS] the number of true positive cases, [SEP] results : compared with the joint detection, [SEP]', '[CLS] sensitivity, specificity, [SEP] the number of true positive cases, [SEP]', '[CLS] and accuracy rate obtained by a single application of pelvic mri or serum pkm2, [SEP] sensitivity, specificity, [SEP]', '[CLS] ngal, [SEP] and accuracy rate obtained by a single application of pelvic mri or serum pkm2, [SEP]', '[CLS] the area under the roc curve of the joint detection was obviously larger than that of single detection ; [SEP] ngal, [SEP]', '[CLS] the results of the joint detection were better than those of single detection [SEP] the area under the roc curve of the joint detection was obviously larger than that of single detection ; [SEP]', '[CLS] the combined diagnosis obtained the highest sensitivity. [SEP] the results of the joint detection were better than those of single detection [SEP]', '[CLS] conclusion : combining pelvic mri with serum pkm2, [SEP] the combined diagnosis obtained the highest sensitivity. [SEP]', '[CLS] ngal, [SEP] conclusion : combining pelvic mri with serum pkm2, [SEP]', '[CLS] and sob - 4 detection can effectively promote the diagnostic accuracy for ec, [SEP] ngal, [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] objective : to explore the application value of pelvic magnetic resonance imaging [SEP] abstract [SEP]', '[CLS] ( mri ) scan combined with serum pyruvate kinase isozyme m2 ( pkm2 ), [SEP] objective : to explore the application value of pelvic magnetic resonance imaging [SEP]', '[CLS] and soluble leptin receptor [SEP] ( mri ) scan combined with serum pyruvate kinase isozyme m2 ( pkm2 ), [SEP]', '[CLS] detection in diagnosing endometrial carcinoma [SEP] and soluble leptin receptor [SEP]', '[CLS] methods : the clinical data of 45 patients with pathologically confirmed ec treated in our hospital from may 2019 to may 2020 were retrospectively analyzed. [SEP] detection in diagnosing endometrial carcinoma [SEP]', '[CLS] all patients received pelvic mri scan, [SEP] methods : the clinical data of 45 patients with pathologically confirmed ec treated in our hospital from may 2019 to may 2020 were retrospectively analyzed. [SEP]', '[CLS] serum pkm2, [SEP] all patients received pelvic mri scan, [SEP]', '[CLS] ngal [SEP] serum pkm2, [SEP]', '[CLS] detection was performed, [SEP] ngal [SEP]', '[CLS] and the combination of the two was performed [SEP] detection was performed, [SEP]', '[CLS] results : compared with the joint detection, [SEP] and the combination of the two was performed [SEP]', '[CLS] the number of true positive cases, [SEP] results : compared with the joint detection, [SEP]', '[CLS] sensitivity, specificity, [SEP] the number of true positive cases, [SEP]', '[CLS] and accuracy rate obtained by a single application of pelvic mri or serum pkm2, [SEP] sensitivity, specificity, [SEP]', '[CLS] ngal, [SEP] and accuracy rate obtained by a single application of pelvic mri or serum pkm2, [SEP]', '[CLS] the area under the roc curve of the joint detection was obviously larger than that of single detection ; [SEP] ngal, [SEP]', '[CLS] the results of the joint detection were better than those of single detection [SEP] the area under the roc curve of the joint detection was obviously larger than that of single detection ; [SEP]', '[CLS] the combined diagnosis obtained the highest sensitivity. [SEP] the results of the joint detection were better than those of single detection [SEP]', '[CLS] conclusion : combining pelvic mri with serum pkm2, [SEP] the combined diagnosis obtained the highest sensitivity. [SEP]', '[CLS] ngal, [SEP] conclusion : combining pelvic mri with serum pkm2, [SEP]', '[CLS] and sob - 4 detection can effectively promote the diagnostic accuracy for ec, [SEP] ngal, [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] objective : to explore the application value of pelvic magnetic resonance imaging [SEP] abstract [SEP]', '[CLS] ( mri ) scan combined with serum pyruvate kinase isozyme m2 ( pkm2 ), [SEP] objective : to explore the application value of pelvic magnetic resonance imaging [SEP]', '[CLS] and soluble leptin receptor [SEP] ( mri ) scan combined with serum pyruvate kinase isozyme m2 ( pkm2 ), [SEP]', '[CLS] detection in diagnosing endometrial carcinoma [SEP] and soluble leptin receptor [SEP]', '[CLS] methods : the clinical data of 45 patients with pathologically confirmed ec treated in our hospital from may 2019 to may 2020 were retrospectively analyzed. [SEP] detection in diagnosing endometrial carcinoma [SEP]', '[CLS] all patients received pelvic mri scan, [SEP] methods : the clinical data of 45 patients with pathologically confirmed ec treated in our hospital from may 2019 to may 2020 were retrospectively analyzed. [SEP]', '[CLS] serum pkm2, [SEP] all patients received pelvic mri scan, [SEP]', '[CLS] ngal [SEP] serum pkm2, [SEP]', '[CLS] detection was performed, [SEP] ngal [SEP]', '[CLS] and the combination of the two was performed [SEP] detection was performed, [SEP]', '[CLS] results : compared with the joint detection, [SEP] and the combination of the two was performed [SEP]', '[CLS] the number of true positive cases, [SEP] results : compared with the joint detection, [SEP]', '[CLS] sensitivity, specificity, [SEP] the number of true positive cases, [SEP]', '[CLS] and accuracy rate obtained by a single application of pelvic mri or serum pkm2, [SEP] sensitivity, specificity, [SEP]', '[CLS] ngal, [SEP] and accuracy rate obtained by a single application of pelvic mri or serum pkm2, [SEP]', '[CLS] the area under the roc curve of the joint detection was obviously larger than that of single detection ; [SEP] ngal, [SEP]', '[CLS] the results of the joint detection were better than those of single detection [SEP] the area under the roc curve of the joint detection was obviously larger than that of single detection ; [SEP]', '[CLS] the combined diagnosis obtained the highest sensitivity. [SEP] the results of the joint detection were better than those of single detection [SEP]', '[CLS] conclusion : combining pelvic mri with serum pkm2, [SEP] the combined diagnosis obtained the highest sensitivity. [SEP]', '[CLS] ngal, [SEP] conclusion : combining pelvic mri with serum pkm2, [SEP]', '[CLS] and sob - 4 detection can effectively promote the diagnostic accuracy for ec, [SEP] ngal, [SEP]', '[CLS] seven cases of acute sphenoid sinusitis were reviewed. [SEP] seven cases of acute sphenoid sinusitis were reviewed. [SEP]', '[CLS] two happened after deep sea diving. [SEP] seven cases of acute sphenoid sinusitis were reviewed. [SEP]', '[CLS] headache of variable location [SEP] two happened after deep sea diving. [SEP]', '[CLS] purulent discharge in the cavum was present in three. [SEP] headache of variable location [SEP]', '[CLS] correct diagnosis was delayed from six [SEP] purulent discharge in the cavum was present in three. [SEP]', '[CLS] to twenty - four days after beginning of symptoms, [SEP] correct diagnosis was delayed from six [SEP]', '[CLS] when a neurologic deficit became apparent : [SEP] to twenty - four days after beginning of symptoms, [SEP]', '[CLS] mainly paralysis of the sixth [SEP] when a neurologic deficit became apparent : [SEP]', '[CLS] and third cranial nerves. [SEP] mainly paralysis of the sixth [SEP]', '[CLS] computed axial tomography was the most useful radiologic procedure [SEP] and third cranial nerves. [SEP]', '[CLS] two patients died, [SEP] computed axial tomography was the most useful radiologic procedure [SEP]', '[CLS] three had complete recovery, [SEP] two patients died, [SEP]', '[CLS] one had persistent epileptic focus after brain abscess. [SEP] three had complete recovery, [SEP]', '[CLS] last patient had paraparesis [SEP] one had persistent epileptic focus after brain abscess. [SEP]', '[CLS] and incontinence of urine after study of cerebrospinal fluid [SEP] last patient had paraparesis [SEP]', '[CLS] cannulation of the sphenoid sinus was performed [SEP] and incontinence of urine after study of cerebrospinal fluid [SEP]', '[CLS] gram negative microorganisms [SEP] cannulation of the sphenoid sinus was performed [SEP]', '[CLS] and staphylococcus aureus were isolated from the sphenoid sinus of only three patients. [SEP] gram negative microorganisms [SEP]', '[CLS] blood cultures were negatives in every cases [SEP] and staphylococcus aureus were isolated from the sphenoid sinus of only three patients. [SEP]', '[CLS] and c. s. f. in six. [SEP] blood cultures were negatives in every cases [SEP]', '[CLS] first choice antibiotics was an association of aminoglucosides [SEP] and c. s. f. in six. [SEP]', '[CLS] and broad - spectrum penicillin in six cases. [SEP] first choice antibiotics was an association of aminoglucosides [SEP]', '[CLS] seven cases of acute sphenoid sinusitis were reviewed. [SEP] seven cases of acute sphenoid sinusitis were reviewed. [SEP]', '[CLS] two happened after deep sea diving. [SEP] seven cases of acute sphenoid sinusitis were reviewed. [SEP]', '[CLS] headache of variable location [SEP] two happened after deep sea diving. [SEP]', '[CLS] purulent discharge in the cavum was present in three. [SEP] headache of variable location [SEP]', '[CLS] correct diagnosis was delayed from six [SEP] purulent discharge in the cavum was present in three. [SEP]', '[CLS] to twenty - four days after beginning of symptoms, [SEP] correct diagnosis was delayed from six [SEP]', '[CLS] when a neurologic deficit became apparent : [SEP] to twenty - four days after beginning of symptoms, [SEP]', '[CLS] mainly paralysis of the sixth [SEP] when a neurologic deficit became apparent : [SEP]', '[CLS] and third cranial nerves. [SEP] mainly paralysis of the sixth [SEP]', '[CLS] computed axial tomography was the most useful radiologic procedure [SEP] and third cranial nerves. [SEP]', '[CLS] two patients died, [SEP] computed axial tomography was the most useful radiologic procedure [SEP]', '[CLS] three had complete recovery, [SEP] two patients died, [SEP]', '[CLS] one had persistent epileptic focus after brain abscess. [SEP] three had complete recovery, [SEP]', '[CLS] last patient had paraparesis [SEP] one had persistent epileptic focus after brain abscess. [SEP]', '[CLS] and incontinence of urine after study of cerebrospinal fluid [SEP] last patient had paraparesis [SEP]', '[CLS] cannulation of the sphenoid sinus was performed [SEP] and incontinence of urine after study of cerebrospinal fluid [SEP]', '[CLS] gram negative microorganisms [SEP] cannulation of the sphenoid sinus was performed [SEP]', '[CLS] and staphylococcus aureus were isolated from the sphenoid sinus of only three patients. [SEP] gram negative microorganisms [SEP]', '[CLS] blood cultures were negatives in every cases [SEP] and staphylococcus aureus were isolated from the sphenoid sinus of only three patients. [SEP]', '[CLS] and c. s. f. in six. [SEP] blood cultures were negatives in every cases [SEP]', '[CLS] first choice antibiotics was an association of aminoglucosides [SEP] and c. s. f. in six. [SEP]', '[CLS] and broad - spectrum penicillin in six cases. [SEP] first choice antibiotics was an association of aminoglucosides [SEP]', '[CLS] disturbed noradrenaline [SEP] disturbed noradrenaline [SEP]', '[CLS] ( na ) [SEP] disturbed noradrenaline [SEP]', '[CLS] metabolism is thought to play a causal role in certain types of endogenic depression. [SEP] ( na ) [SEP]', '[CLS] this study was based on data from 88 patients with depression. [SEP] metabolism is thought to play a causal role in certain types of endogenic depression. [SEP]', '[CLS] the metabolism of na was investigated by measuring urinary mhpg [SEP] this study was based on data from 88 patients with depression. [SEP]', '[CLS] and plasma dopeg concentrations [SEP] the metabolism of na was investigated by measuring urinary mhpg [SEP]', '[CLS] and platelet nonoamine oxidase activity [SEP] and plasma dopeg concentrations [SEP]', '[CLS] to determine if there were any differences in subgroups of depression defined [SEP] and platelet nonoamine oxidase activity [SEP]', '[CLS] by the dsm3. [SEP] to determine if there were any differences in subgroups of depression defined [SEP]', '[CLS] there was no difference in plasma dopeg concentrations [SEP] by the dsm3. [SEP]', '[CLS] or of mao activity in the different subgroups of depression, [SEP] there was no difference in plasma dopeg concentrations [SEP]', '[CLS] especially between episodes of major [SEP] or of mao activity in the different subgroups of depression, [SEP]', '[CLS] and non - major depression. [SEP] especially between episodes of major [SEP]', '[CLS] on the other hand, [SEP] and non - major depression. [SEP]', '[CLS] depressed patients with an episode of major depression had [SEP] on the other hand, [SEP]', '[CLS] significantly higher urinary mhpg concentrations [SEP] depressed patients with an episode of major depression had [SEP]', '[CLS] than those with non - major depression. [SEP] significantly higher urinary mhpg concentrations [SEP]', '[CLS] disturbed noradrenaline [SEP] disturbed noradrenaline [SEP]', '[CLS] ( na ) [SEP] disturbed noradrenaline [SEP]', '[CLS] metabolism is thought to play a causal role in certain types of endogenic depression. [SEP] ( na ) [SEP]', '[CLS] this study was based on data from 88 patients with depression. [SEP] metabolism is thought to play a causal role in certain types of endogenic depression. [SEP]', '[CLS] the metabolism of na was investigated by measuring urinary mhpg [SEP] this study was based on data from 88 patients with depression. [SEP]', '[CLS] and plasma dopeg concentrations [SEP] the metabolism of na was investigated by measuring urinary mhpg [SEP]', '[CLS] and platelet nonoamine oxidase activity [SEP] and plasma dopeg concentrations [SEP]', '[CLS] to determine if there were any differences in subgroups of depression defined [SEP] and platelet nonoamine oxidase activity [SEP]', '[CLS] by the dsm3. [SEP] to determine if there were any differences in subgroups of depression defined [SEP]', '[CLS] there was no difference in plasma dopeg concentrations [SEP] by the dsm3. [SEP]', '[CLS] or of mao activity in the different subgroups of depression, [SEP] there was no difference in plasma dopeg concentrations [SEP]', '[CLS] especially between episodes of major [SEP] or of mao activity in the different subgroups of depression, [SEP]', '[CLS] and non - major depression. [SEP] especially between episodes of major [SEP]', '[CLS] on the other hand, [SEP] and non - major depression. [SEP]', '[CLS] depressed patients with an episode of major depression had [SEP] on the other hand, [SEP]', '[CLS] significantly higher urinary mhpg concentrations [SEP] depressed patients with an episode of major depression had [SEP]', '[CLS] than those with non - major depression. [SEP] significantly higher urinary mhpg concentrations [SEP]', '[CLS] apart from the rarity of the causal agent, [SEP] apart from the rarity of the causal agent, [SEP]', '[CLS] this case is of interest [SEP] apart from the rarity of the causal agent, [SEP]', '[CLS] because of the absence of cervico - facial involvement [SEP] this case is of interest [SEP]', '[CLS] and the presence of multiple cutaneous nodules [SEP] because of the absence of cervico - facial involvement [SEP]', '[CLS] at a distance from the visceral localisation. [SEP] and the presence of multiple cutaneous nodules [SEP]', '[CLS] apart from the rarity of the causal agent, [SEP] apart from the rarity of the causal agent, [SEP]', '[CLS] this case is of interest [SEP] apart from the rarity of the causal agent, [SEP]', '[CLS] because of the absence of cervico - facial involvement [SEP] this case is of interest [SEP]', '[CLS] and the presence of multiple cutaneous nodules [SEP] because of the absence of cervico - facial involvement [SEP]', '[CLS] at a distance from the visceral localisation. [SEP] and the presence of multiple cutaneous nodules [SEP]', '[CLS] we report a new case of granuloma inguinale in france. [SEP] we report a new case of granuloma inguinale in france. [SEP]', '[CLS] a 47 years old turkish man living in france for eight years [SEP] we report a new case of granuloma inguinale in france. [SEP]', '[CLS] had an ulcerovegetative lesion on the corona sulcus [SEP] a 47 years old turkish man living in france for eight years [SEP]', '[CLS] and inner aspect of the prepuce for three months. [SEP] had an ulcerovegetative lesion on the corona sulcus [SEP]', '[CLS] donovanosis was established by granulation time smears [SEP] and inner aspect of the prepuce for three months. [SEP]', '[CLS] complete resolution occurred [SEP] donovanosis was established by granulation time smears [SEP]', '[CLS] after treatment with ampicillin for three weeks [SEP] complete resolution occurred [SEP]', '[CLS] and later doxycycline for a month. [SEP] after treatment with ampicillin for three weeks [SEP]', '[CLS] granuloma inguinale is rare in france [SEP] and later doxycycline for a month. [SEP]', '[CLS] clinical manifestations [SEP] granuloma inguinale is rare in france [SEP]', '[CLS] and therapeutic indications are reviewed. [SEP] clinical manifestations [SEP]', '[CLS] we report a new case of granuloma inguinale in france. [SEP] we report a new case of granuloma inguinale in france. [SEP]', '[CLS] a 47 years old turkish man living in france for eight years [SEP] we report a new case of granuloma inguinale in france. [SEP]', '[CLS] had an ulcerovegetative lesion on the corona sulcus [SEP] a 47 years old turkish man living in france for eight years [SEP]', '[CLS] and inner aspect of the prepuce for three months. [SEP] had an ulcerovegetative lesion on the corona sulcus [SEP]', '[CLS] donovanosis was established by granulation time smears [SEP] and inner aspect of the prepuce for three months. [SEP]', '[CLS] complete resolution occurred [SEP] donovanosis was established by granulation time smears [SEP]', '[CLS] after treatment with ampicillin for three weeks [SEP] complete resolution occurred [SEP]', '[CLS] and later doxycycline for a month. [SEP] after treatment with ampicillin for three weeks [SEP]', '[CLS] granuloma inguinale is rare in france [SEP] and later doxycycline for a month. [SEP]', '[CLS] clinical manifestations [SEP] granuloma inguinale is rare in france [SEP]', '[CLS] and therapeutic indications are reviewed. [SEP] clinical manifestations [SEP]', '[CLS] vascular purpura and haemoptysis developed in a patient suffering from accelerated hypertension [SEP] vascular purpura and haemoptysis developed in a patient suffering from accelerated hypertension [SEP]', '[CLS] and rapidly progressive renal failure due to cholesterol embolism. [SEP] vascular purpura and haemoptysis developed in a patient suffering from accelerated hypertension [SEP]', '[CLS] alveolar haemorrhage was diagnosed on the association of clinical [SEP] and rapidly progressive renal failure due to cholesterol embolism. [SEP]', '[CLS] and endoscopic results, [SEP] alveolar haemorrhage was diagnosed on the association of clinical [SEP]', '[CLS] and on the outcome of the disease. [SEP] and endoscopic results, [SEP]', '[CLS] this syndrome is usually associated with other immunological disorders [SEP] and on the outcome of the disease. [SEP]', '[CLS] and raises the question of immunological phenomena being the cause of some of the clinical manifestations observed [SEP] this syndrome is usually associated with other immunological disorders [SEP]', '[CLS] after cholesterol embolism. [SEP] and raises the question of immunological phenomena being the cause of some of the clinical manifestations observed [SEP]', '[CLS] vascular purpura and haemoptysis developed in a patient suffering from accelerated hypertension [SEP] vascular purpura and haemoptysis developed in a patient suffering from accelerated hypertension [SEP]', '[CLS] and rapidly progressive renal failure due to cholesterol embolism. [SEP] vascular purpura and haemoptysis developed in a patient suffering from accelerated hypertension [SEP]', '[CLS] alveolar haemorrhage was diagnosed on the association of clinical [SEP] and rapidly progressive renal failure due to cholesterol embolism. [SEP]', '[CLS] and endoscopic results, [SEP] alveolar haemorrhage was diagnosed on the association of clinical [SEP]', '[CLS] and on the outcome of the disease. [SEP] and endoscopic results, [SEP]', '[CLS] this syndrome is usually associated with other immunological disorders [SEP] and on the outcome of the disease. [SEP]', '[CLS] and raises the question of immunological phenomena being the cause of some of the clinical manifestations observed [SEP] this syndrome is usually associated with other immunological disorders [SEP]', '[CLS] after cholesterol embolism. [SEP] and raises the question of immunological phenomena being the cause of some of the clinical manifestations observed [SEP]', '[CLS] this research attempts to examine a possible connection [SEP] this research attempts to examine a possible connection [SEP]', '[CLS] between neuroticism ( level of anxiety ) [SEP] this research attempts to examine a possible connection [SEP]', '[CLS] and the meaning of life. [SEP] between neuroticism ( level of anxiety ) [SEP]', '[CLS] a research population composed of 446 people of whom 140 were imprisoned criminals [SEP] and the meaning of life. [SEP]', '[CLS] and 306 were non - criminals, [SEP] a research population composed of 446 people of whom 140 were imprisoned criminals [SEP]', '[CLS] was examined. [SEP] and 306 were non - criminals, [SEP]', '[CLS] a significant negative correlation was found [SEP] was examined. [SEP]', '[CLS] between the meaning of life and neuroticism, [SEP] a significant negative correlation was found [SEP]', '[CLS] which means that the higher the neuroticism is the lower the meaning of life is. [SEP] between the meaning of life and neuroticism, [SEP]', '[CLS] this negative connection was stronger for the criminal group [SEP] which means that the higher the neuroticism is the lower the meaning of life is. [SEP]', '[CLS] which on the average had a lower score for the meaning of life. [SEP] this negative connection was stronger for the criminal group [SEP]', '[CLS] according to these results, [SEP] which on the average had a lower score for the meaning of life. [SEP]', '[CLS] the processes allowing a positive meaning of life can diminish the effects of psychogenic anxiety. [SEP] according to these results, [SEP]', '[CLS] no connection was found between extraversion [SEP] the processes allowing a positive meaning of life can diminish the effects of psychogenic anxiety. [SEP]', '[CLS] and the meaning of life. [SEP] no connection was found between extraversion [SEP]', '[CLS] likewise, socioeconomic factors did not influence the negative correlation [SEP] and the meaning of life. [SEP]', '[CLS] in specific cases we found [SEP] likewise, socioeconomic factors did not influence the negative correlation [SEP]', '[CLS] that criminal solutions could be a substitute for the meaning of life. [SEP] in specific cases we found [SEP]', '[CLS] this research attempts to examine a possible connection [SEP] this research attempts to examine a possible connection [SEP]', '[CLS] between neuroticism ( level of anxiety ) [SEP] this research attempts to examine a possible connection [SEP]', '[CLS] and the meaning of life. [SEP] between neuroticism ( level of anxiety ) [SEP]', '[CLS] a research population composed of 446 people of whom 140 were imprisoned criminals [SEP] and the meaning of life. [SEP]', '[CLS] and 306 were non - criminals, [SEP] a research population composed of 446 people of whom 140 were imprisoned criminals [SEP]', '[CLS] was examined. [SEP] and 306 were non - criminals, [SEP]', '[CLS] a significant negative correlation was found [SEP] was examined. [SEP]', '[CLS] between the meaning of life and neuroticism, [SEP] a significant negative correlation was found [SEP]', '[CLS] which means that the higher the neuroticism is the lower the meaning of life is. [SEP] between the meaning of life and neuroticism, [SEP]', '[CLS] this negative connection was stronger for the criminal group [SEP] which means that the higher the neuroticism is the lower the meaning of life is. [SEP]', '[CLS] which on the average had a lower score for the meaning of life. [SEP] this negative connection was stronger for the criminal group [SEP]', '[CLS] according to these results, [SEP] which on the average had a lower score for the meaning of life. [SEP]', '[CLS] the processes allowing a positive meaning of life can diminish the effects of psychogenic anxiety. [SEP] according to these results, [SEP]', '[CLS] no connection was found between extraversion [SEP] the processes allowing a positive meaning of life can diminish the effects of psychogenic anxiety. [SEP]', '[CLS] and the meaning of life. [SEP] no connection was found between extraversion [SEP]', '[CLS] likewise, socioeconomic factors did not influence the negative correlation [SEP] and the meaning of life. [SEP]', '[CLS] in specific cases we found [SEP] likewise, socioeconomic factors did not influence the negative correlation [SEP]', '[CLS] that criminal solutions could be a substitute for the meaning of life. [SEP] in specific cases we found [SEP]', '[CLS] the cytomorphologic findings of a vitrectomy specimen [SEP] the cytomorphologic findings of a vitrectomy specimen [SEP]', '[CLS] from the right eye of an 80 - year - old woman [SEP] the cytomorphologic findings of a vitrectomy specimen [SEP]', '[CLS] with an unsuspected primary choroidal malignant melanoma are described. [SEP] from the right eye of an 80 - year - old woman [SEP]', '[CLS] the patient had undergone a closed vitrectomy because of chronic vitreous hemorrhage. [SEP] with an unsuspected primary choroidal malignant melanoma are described. [SEP]', '[CLS] histocytology of the vitreous fluid specimens revealed melanoma cells of variable shape [SEP] the patient had undergone a closed vitrectomy because of chronic vitreous hemorrhage. [SEP]', '[CLS] and size ( from 30 - 150 microns ) [SEP] histocytology of the vitreous fluid specimens revealed melanoma cells of variable shape [SEP]', '[CLS] with eccentric nuclei. [SEP] and size ( from 30 - 150 microns ) [SEP]', '[CLS] many of these cells were binucleated [SEP] with eccentric nuclei. [SEP]', '[CLS] or multinucleated with small, uniform, evenly dispersed intracytoplasmic melanin granules. [SEP] many of these cells were binucleated [SEP]', '[CLS] the histocytologic findings together with the postoperative tumor characteristics by ultrasonography [SEP] or multinucleated with small, uniform, evenly dispersed intracytoplasmic melanin granules. [SEP]', '[CLS] and fluorescein angiography suggested the diagnosis of choroidal malignant melanoma. [SEP] the histocytologic findings together with the postoperative tumor characteristics by ultrasonography [SEP]', '[CLS] the cytomorphologic findings of a vitrectomy specimen [SEP] the cytomorphologic findings of a vitrectomy specimen [SEP]', '[CLS] from the right eye of an 80 - year - old woman [SEP] the cytomorphologic findings of a vitrectomy specimen [SEP]', '[CLS] with an unsuspected primary choroidal malignant melanoma are described. [SEP] from the right eye of an 80 - year - old woman [SEP]', '[CLS] the patient had undergone a closed vitrectomy because of chronic vitreous hemorrhage. [SEP] with an unsuspected primary choroidal malignant melanoma are described. [SEP]', '[CLS] histocytology of the vitreous fluid specimens revealed melanoma cells of variable shape [SEP] the patient had undergone a closed vitrectomy because of chronic vitreous hemorrhage. [SEP]', '[CLS] and size ( from 30 - 150 microns ) [SEP] histocytology of the vitreous fluid specimens revealed melanoma cells of variable shape [SEP]', '[CLS] with eccentric nuclei. [SEP] and size ( from 30 - 150 microns ) [SEP]', '[CLS] many of these cells were binucleated [SEP] with eccentric nuclei. [SEP]', '[CLS] or multinucleated with small, uniform, evenly dispersed intracytoplasmic melanin granules. [SEP] many of these cells were binucleated [SEP]', '[CLS] the histocytologic findings together with the postoperative tumor characteristics by ultrasonography [SEP] or multinucleated with small, uniform, evenly dispersed intracytoplasmic melanin granules. [SEP]', '[CLS] and fluorescein angiography suggested the diagnosis of choroidal malignant melanoma. [SEP] the histocytologic findings together with the postoperative tumor characteristics by ultrasonography [SEP]', '[CLS] sexual impotence developed in four patients [SEP] sexual impotence developed in four patients [SEP]', '[CLS] who were receiving carbonic anhydrase inhibitor therapy for glaucoma. [SEP] sexual impotence developed in four patients [SEP]', '[CLS] whereas decreased libido has been reported as a side effect of carbonic anhydrase inhibitor therapy, [SEP] who were receiving carbonic anhydrase inhibitor therapy for glaucoma. [SEP]', '[CLS] organic impotence has not been documented as an isolated finding. [SEP] whereas decreased libido has been reported as a side effect of carbonic anhydrase inhibitor therapy, [SEP]', '[CLS] sexual impotence developed in four patients [SEP] sexual impotence developed in four patients [SEP]', '[CLS] who were receiving carbonic anhydrase inhibitor therapy for glaucoma. [SEP] sexual impotence developed in four patients [SEP]', '[CLS] whereas decreased libido has been reported as a side effect of carbonic anhydrase inhibitor therapy, [SEP] who were receiving carbonic anhydrase inhibitor therapy for glaucoma. [SEP]', '[CLS] organic impotence has not been documented as an isolated finding. [SEP] whereas decreased libido has been reported as a side effect of carbonic anhydrase inhibitor therapy, [SEP]', '[CLS] multiple epiphyseal dysplasias may present as congenital [SEP] multiple epiphyseal dysplasias may present as congenital [SEP]', '[CLS] or late - onset ( tarda ) forms [SEP] multiple epiphyseal dysplasias may present as congenital [SEP]', '[CLS] or as the variant pseudo - achondroplastic type. [SEP] or late - onset ( tarda ) forms [SEP]', '[CLS] while the association between cataract [SEP] or as the variant pseudo - achondroplastic type. [SEP]', '[CLS] and skin anomalies is well known in the congenital type, [SEP] while the association between cataract [SEP]', '[CLS] it has not been reported for the late - onset type of multiple epiphyseal dysplasia. [SEP] and skin anomalies is well known in the congenital type, [SEP]', '[CLS] a 5 1 / 2 - year - old child with the pseudo - achondroplastic variant of multiple epiphyseal dysplasia has been observed [SEP] it has not been reported for the late - onset type of multiple epiphyseal dysplasia. [SEP]', '[CLS] to have the unusual association of cataracts [SEP] a 5 1 / 2 - year - old child with the pseudo - achondroplastic variant of multiple epiphyseal dysplasia has been observed [SEP]', '[CLS] and vitiligo, [SEP] to have the unusual association of cataracts [SEP]', '[CLS] a condition which lacked previous documentation. [SEP] and vitiligo, [SEP]', '[CLS] multiple epiphyseal dysplasias may present as congenital [SEP] multiple epiphyseal dysplasias may present as congenital [SEP]', '[CLS] or late - onset ( tarda ) forms [SEP] multiple epiphyseal dysplasias may present as congenital [SEP]', '[CLS] or as the variant pseudo - achondroplastic type. [SEP] or late - onset ( tarda ) forms [SEP]', '[CLS] while the association between cataract [SEP] or as the variant pseudo - achondroplastic type. [SEP]', '[CLS] and skin anomalies is well known in the congenital type, [SEP] while the association between cataract [SEP]', '[CLS] it has not been reported for the late - onset type of multiple epiphyseal dysplasia. [SEP] and skin anomalies is well known in the congenital type, [SEP]', '[CLS] a 5 1 / 2 - year - old child with the pseudo - achondroplastic variant of multiple epiphyseal dysplasia has been observed [SEP] it has not been reported for the late - onset type of multiple epiphyseal dysplasia. [SEP]', '[CLS] to have the unusual association of cataracts [SEP] a 5 1 / 2 - year - old child with the pseudo - achondroplastic variant of multiple epiphyseal dysplasia has been observed [SEP]', '[CLS] and vitiligo, [SEP] to have the unusual association of cataracts [SEP]', '[CLS] a condition which lacked previous documentation. [SEP] and vitiligo, [SEP]', '[CLS] cystic lesions constituted 37 % of all the conjunctival specimens received over a period of 6 1 / 2 years. [SEP] cystic lesions constituted 37 % of all the conjunctival specimens received over a period of 6 1 / 2 years. [SEP]', '[CLS] epithelial inclusion cysts were the most common, [SEP] cystic lesions constituted 37 % of all the conjunctival specimens received over a period of 6 1 / 2 years. [SEP]', '[CLS] followed by dermoid [SEP] epithelial inclusion cysts were the most common, [SEP]', '[CLS] and parasitic cysts. [SEP] followed by dermoid [SEP]', '[CLS] a clinicopathologic correlation, [SEP] and parasitic cysts. [SEP]', '[CLS] along with an attempt at classification [SEP] a clinicopathologic correlation, [SEP]', '[CLS] of cystic lesions of conjunctiva, [SEP] along with an attempt at classification [SEP]', '[CLS] is made in this article. [SEP] of cystic lesions of conjunctiva, [SEP]', '[CLS] cystic lesions constituted 37 % of all the conjunctival specimens received over a period of 6 1 / 2 years. [SEP] cystic lesions constituted 37 % of all the conjunctival specimens received over a period of 6 1 / 2 years. [SEP]', '[CLS] epithelial inclusion cysts were the most common, [SEP] cystic lesions constituted 37 % of all the conjunctival specimens received over a period of 6 1 / 2 years. [SEP]', '[CLS] followed by dermoid [SEP] epithelial inclusion cysts were the most common, [SEP]', '[CLS] and parasitic cysts. [SEP] followed by dermoid [SEP]', '[CLS] a clinicopathologic correlation, [SEP] and parasitic cysts. [SEP]', '[CLS] along with an attempt at classification [SEP] a clinicopathologic correlation, [SEP]', '[CLS] of cystic lesions of conjunctiva, [SEP] along with an attempt at classification [SEP]', '[CLS] is made in this article. [SEP] of cystic lesions of conjunctiva, [SEP]', '[CLS] ticks commonly infest humans but rarely bite the eyes. [SEP] ticks commonly infest humans but rarely bite the eyes. [SEP]', '[CLS] they are vectors of disease ; [SEP] ticks commonly infest humans but rarely bite the eyes. [SEP]', '[CLS] therefore, rapid and complete removal is important. [SEP] they are vectors of disease ; [SEP]', '[CLS] their attachment can be so secure [SEP] therefore, rapid and complete removal is important. [SEP]', '[CLS] that removal is difficult or hazardous. [SEP] their attachment can be so secure [SEP]', \"[CLS] we excised a larval form of amblyomma americanum from a woodsman's conjunctiva. [SEP] that removal is difficult or hazardous. [SEP]\", '[CLS] ticks commonly infest humans but rarely bite the eyes. [SEP] ticks commonly infest humans but rarely bite the eyes. [SEP]', '[CLS] they are vectors of disease ; [SEP] ticks commonly infest humans but rarely bite the eyes. [SEP]', '[CLS] therefore, rapid and complete removal is important. [SEP] they are vectors of disease ; [SEP]', '[CLS] their attachment can be so secure [SEP] therefore, rapid and complete removal is important. [SEP]', '[CLS] that removal is difficult or hazardous. [SEP] their attachment can be so secure [SEP]', \"[CLS] we excised a larval form of amblyomma americanum from a woodsman's conjunctiva. [SEP] that removal is difficult or hazardous. [SEP]\", '[CLS] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP]', '[CLS] orbital roof fractures may occur via a blow - in mechanism rather than by direct transmission of an outside force. [SEP] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP]', '[CLS] as is the case with those caused by floor fractures, [SEP] orbital roof fractures may occur via a blow - in mechanism rather than by direct transmission of an outside force. [SEP]', '[CLS] motility disturbances from roof fractures merit clinical observation [SEP] as is the case with those caused by floor fractures, [SEP]', '[CLS] before surgery is performed, [SEP] motility disturbances from roof fractures merit clinical observation [SEP]', '[CLS] since some motility defects will resolve spontaneously. [SEP] before surgery is performed, [SEP]', '[CLS] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP]', '[CLS] orbital roof fractures may occur via a blow - in mechanism rather than by direct transmission of an outside force. [SEP] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP]', '[CLS] as is the case with those caused by floor fractures, [SEP] orbital roof fractures may occur via a blow - in mechanism rather than by direct transmission of an outside force. [SEP]', '[CLS] motility disturbances from roof fractures merit clinical observation [SEP] as is the case with those caused by floor fractures, [SEP]', '[CLS] before surgery is performed, [SEP] motility disturbances from roof fractures merit clinical observation [SEP]', '[CLS] since some motility defects will resolve spontaneously. [SEP] before surgery is performed, [SEP]', '[CLS] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP]', '[CLS] orbital roof fractures may occur via a blow - in mechanism rather than by direct transmission of an outside force. [SEP] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP]', '[CLS] as is the case with those caused by floor fractures, [SEP] orbital roof fractures may occur via a blow - in mechanism rather than by direct transmission of an outside force. [SEP]', '[CLS] motility disturbances from roof fractures merit clinical observation [SEP] as is the case with those caused by floor fractures, [SEP]', '[CLS] before surgery is performed, [SEP] motility disturbances from roof fractures merit clinical observation [SEP]', '[CLS] since some motility defects will resolve spontaneously. [SEP] before surgery is performed, [SEP]', '[CLS] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP]', '[CLS] orbital roof fractures may occur via a blow - in mechanism rather than by direct transmission of an outside force. [SEP] a two - year - old boy with orbital roof fracture had resolution of motility restriction without traditional surgical exploration. [SEP]', '[CLS] as is the case with those caused by floor fractures, [SEP] orbital roof fractures may occur via a blow - in mechanism rather than by direct transmission of an outside force. [SEP]', '[CLS] motility disturbances from roof fractures merit clinical observation [SEP] as is the case with those caused by floor fractures, [SEP]', '[CLS] before surgery is performed, [SEP] motility disturbances from roof fractures merit clinical observation [SEP]', '[CLS] since some motility defects will resolve spontaneously. [SEP] before surgery is performed, [SEP]', '[CLS] in this case report the authors present a patient in [SEP] in this case report the authors present a patient in [SEP]', '[CLS] whom loss of vision developed in the left eye in the early stage of herpes zoster ophthalmicus. [SEP] in this case report the authors present a patient in [SEP]', '[CLS] he suffered from optic neuropathy [SEP] whom loss of vision developed in the left eye in the early stage of herpes zoster ophthalmicus. [SEP]', '[CLS] followed by central retinal artery occlusion. [SEP] he suffered from optic neuropathy [SEP]', '[CLS] ophthalmologists should be aware of this rare [SEP] followed by central retinal artery occlusion. [SEP]', '[CLS] and early complication of herpes zoster. [SEP] ophthalmologists should be aware of this rare [SEP]', '[CLS] in this case report the authors present a patient in [SEP] in this case report the authors present a patient in [SEP]', '[CLS] whom loss of vision developed in the left eye in the early stage of herpes zoster ophthalmicus. [SEP] in this case report the authors present a patient in [SEP]', '[CLS] he suffered from optic neuropathy [SEP] whom loss of vision developed in the left eye in the early stage of herpes zoster ophthalmicus. [SEP]', '[CLS] followed by central retinal artery occlusion. [SEP] he suffered from optic neuropathy [SEP]', '[CLS] ophthalmologists should be aware of this rare [SEP] followed by central retinal artery occlusion. [SEP]', '[CLS] and early complication of herpes zoster. [SEP] ophthalmologists should be aware of this rare [SEP]', '[CLS] a case of limbal vernal keratoconjunctivitis associated with a hypertrophic mass lesion measuring [SEP] a case of limbal vernal keratoconjunctivitis associated with a hypertrophic mass lesion measuring [SEP]', '[CLS] 8 x 5 x 3 mm is reported. [SEP] a case of limbal vernal keratoconjunctivitis associated with a hypertrophic mass lesion measuring [SEP]', '[CLS] the histopathology of this mass, [SEP] 8 x 5 x 3 mm is reported. [SEP]', '[CLS] which consisted of hyperplastic epithelium with eosinophilic infiltration [SEP] the histopathology of this mass, [SEP]', '[CLS] and thickened subepithelial stroma of irregular hyperplastic collagenous connective tissue interspersed with numerous eosinophils [SEP] which consisted of hyperplastic epithelium with eosinophilic infiltration [SEP]', '[CLS] and inflammatory cells, is presented. [SEP] and thickened subepithelial stroma of irregular hyperplastic collagenous connective tissue interspersed with numerous eosinophils [SEP]', '[CLS] a large limbal - mass lesion [SEP] and inflammatory cells, is presented. [SEP]', '[CLS] such as this has not been previously described in association with limbal vernal keratoconjunctivitis. [SEP] a large limbal - mass lesion [SEP]', '[CLS] a case of limbal vernal keratoconjunctivitis associated with a hypertrophic mass lesion measuring [SEP] a case of limbal vernal keratoconjunctivitis associated with a hypertrophic mass lesion measuring [SEP]', '[CLS] 8 x 5 x 3 mm is reported. [SEP] a case of limbal vernal keratoconjunctivitis associated with a hypertrophic mass lesion measuring [SEP]', '[CLS] the histopathology of this mass, [SEP] 8 x 5 x 3 mm is reported. [SEP]', '[CLS] which consisted of hyperplastic epithelium with eosinophilic infiltration [SEP] the histopathology of this mass, [SEP]', '[CLS] and thickened subepithelial stroma of irregular hyperplastic collagenous connective tissue interspersed with numerous eosinophils [SEP] which consisted of hyperplastic epithelium with eosinophilic infiltration [SEP]', '[CLS] and inflammatory cells, is presented. [SEP] and thickened subepithelial stroma of irregular hyperplastic collagenous connective tissue interspersed with numerous eosinophils [SEP]', '[CLS] a large limbal - mass lesion [SEP] and inflammatory cells, is presented. [SEP]', '[CLS] such as this has not been previously described in association with limbal vernal keratoconjunctivitis. [SEP] a large limbal - mass lesion [SEP]', '[CLS] in a previous work on a group of patients with diabetic retinopathy, [SEP] in a previous work on a group of patients with diabetic retinopathy, [SEP]', '[CLS] the effect of laser photocoagulation on the serum zinc [SEP] in a previous work on a group of patients with diabetic retinopathy, [SEP]', '[CLS] and copper levels was evaluated. [SEP] the effect of laser photocoagulation on the serum zinc [SEP]', '[CLS] in many of those patients, [SEP] and copper levels was evaluated. [SEP]', '[CLS] an increase in the serum zinc [SEP] in many of those patients, [SEP]', '[CLS] and copper levels was found one day after the laser treatment. [SEP] an increase in the serum zinc [SEP]', '[CLS] changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit. [SEP] and copper levels was found one day after the laser treatment. [SEP]', '[CLS] a significant rise was found one day after the treatment in the level of serum copper [SEP] changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit. [SEP]', '[CLS] ( p less than. 005 ), [SEP] a significant rise was found one day after the treatment in the level of serum copper [SEP]', '[CLS] and one week thereafter in the level of serum zinc [SEP] ( p less than. 005 ), [SEP]', '[CLS] ( p less than. 005 ). [SEP] and one week thereafter in the level of serum zinc [SEP]', '[CLS] the significance of these results and their connection to the retinal pigment epithelium [SEP] ( p less than. 005 ). [SEP]', '[CLS] and choroid is discussed. [SEP] the significance of these results and their connection to the retinal pigment epithelium [SEP]', '[CLS] in a previous work on a group of patients with diabetic retinopathy, [SEP] in a previous work on a group of patients with diabetic retinopathy, [SEP]', '[CLS] the effect of laser photocoagulation on the serum zinc [SEP] in a previous work on a group of patients with diabetic retinopathy, [SEP]', '[CLS] and copper levels was evaluated. [SEP] the effect of laser photocoagulation on the serum zinc [SEP]', '[CLS] in many of those patients, [SEP] and copper levels was evaluated. [SEP]', '[CLS] an increase in the serum zinc [SEP] in many of those patients, [SEP]', '[CLS] and copper levels was found one day after the laser treatment. [SEP] an increase in the serum zinc [SEP]', '[CLS] changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit. [SEP] and copper levels was found one day after the laser treatment. [SEP]', '[CLS] a significant rise was found one day after the treatment in the level of serum copper [SEP] changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit. [SEP]', '[CLS] ( p less than. 005 ), [SEP] a significant rise was found one day after the treatment in the level of serum copper [SEP]', '[CLS] and one week thereafter in the level of serum zinc [SEP] ( p less than. 005 ), [SEP]', '[CLS] ( p less than. 005 ). [SEP] and one week thereafter in the level of serum zinc [SEP]', '[CLS] the significance of these results and their connection to the retinal pigment epithelium [SEP] ( p less than. 005 ). [SEP]', '[CLS] and choroid is discussed. [SEP] the significance of these results and their connection to the retinal pigment epithelium [SEP]', '[CLS] a case of tuberculous uveitis [SEP] a case of tuberculous uveitis [SEP]', '[CLS] that presented 34 years after a primary articular focus is evaluated. [SEP] a case of tuberculous uveitis [SEP]', '[CLS] the pathogenesis of the infection [SEP] that presented 34 years after a primary articular focus is evaluated. [SEP]', '[CLS] and the role of the tuberculin skin test in the diagnosis are discussed. [SEP] the pathogenesis of the infection [SEP]', '[CLS] isoniazid was used with good therapeutic results. [SEP] and the role of the tuberculin skin test in the diagnosis are discussed. [SEP]', '[CLS] fluorescein angiography was found to be essential in the follow - up of the choroidal lesions. [SEP] isoniazid was used with good therapeutic results. [SEP]', '[CLS] subretinal neovascularization was managed with argon laser photocoagulation. [SEP] fluorescein angiography was found to be essential in the follow - up of the choroidal lesions. [SEP]', '[CLS] a case of tuberculous uveitis [SEP] a case of tuberculous uveitis [SEP]', '[CLS] that presented 34 years after a primary articular focus is evaluated. [SEP] a case of tuberculous uveitis [SEP]', '[CLS] the pathogenesis of the infection [SEP] that presented 34 years after a primary articular focus is evaluated. [SEP]', '[CLS] and the role of the tuberculin skin test in the diagnosis are discussed. [SEP] the pathogenesis of the infection [SEP]', '[CLS] isoniazid was used with good therapeutic results. [SEP] and the role of the tuberculin skin test in the diagnosis are discussed. [SEP]', '[CLS] fluorescein angiography was found to be essential in the follow - up of the choroidal lesions. [SEP] isoniazid was used with good therapeutic results. [SEP]', '[CLS] subretinal neovascularization was managed with argon laser photocoagulation. [SEP] fluorescein angiography was found to be essential in the follow - up of the choroidal lesions. [SEP]', '[CLS] in a previous work on a group of patients with diabetic retinopathy, [SEP] in a previous work on a group of patients with diabetic retinopathy, [SEP]', '[CLS] the effect of laser photocoagulation on the serum zinc [SEP] in a previous work on a group of patients with diabetic retinopathy, [SEP]', '[CLS] and copper levels was evaluated. [SEP] the effect of laser photocoagulation on the serum zinc [SEP]', '[CLS] in many of those patients, [SEP] and copper levels was evaluated. [SEP]', '[CLS] an increase in the serum zinc [SEP] in many of those patients, [SEP]', '[CLS] and copper levels was found one day after the laser treatment. [SEP] an increase in the serum zinc [SEP]', '[CLS] changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit. [SEP] and copper levels was found one day after the laser treatment. [SEP]', '[CLS] a significant rise was found one day after the treatment in the level of serum copper [SEP] changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit. [SEP]', '[CLS] ( p less than. 005 ), [SEP] a significant rise was found one day after the treatment in the level of serum copper [SEP]', '[CLS] and one week thereafter in the level of serum zinc [SEP] ( p less than. 005 ), [SEP]', '[CLS] ( p less than. 005 ). [SEP] and one week thereafter in the level of serum zinc [SEP]', '[CLS] the significance of these results and their connection to the retinal pigment epithelium [SEP] ( p less than. 005 ). [SEP]', '[CLS] and choroid is discussed. [SEP] the significance of these results and their connection to the retinal pigment epithelium [SEP]', '[CLS] in a previous work on a group of patients with diabetic retinopathy, [SEP] in a previous work on a group of patients with diabetic retinopathy, [SEP]', '[CLS] the effect of laser photocoagulation on the serum zinc [SEP] in a previous work on a group of patients with diabetic retinopathy, [SEP]', '[CLS] and copper levels was evaluated. [SEP] the effect of laser photocoagulation on the serum zinc [SEP]', '[CLS] in many of those patients, [SEP] and copper levels was evaluated. [SEP]', '[CLS] an increase in the serum zinc [SEP] in many of those patients, [SEP]', '[CLS] and copper levels was found one day after the laser treatment. [SEP] an increase in the serum zinc [SEP]', '[CLS] changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit. [SEP] and copper levels was found one day after the laser treatment. [SEP]', '[CLS] a significant rise was found one day after the treatment in the level of serum copper [SEP] changes in the serum concentration of these metals after xenon arc photocoagulation were assessed in the rabbit. [SEP]', '[CLS] ( p less than. 005 ), [SEP] a significant rise was found one day after the treatment in the level of serum copper [SEP]', '[CLS] and one week thereafter in the level of serum zinc [SEP] ( p less than. 005 ), [SEP]', '[CLS] ( p less than. 005 ). [SEP] and one week thereafter in the level of serum zinc [SEP]', '[CLS] the significance of these results and their connection to the retinal pigment epithelium [SEP] ( p less than. 005 ). [SEP]', '[CLS] and choroid is discussed. [SEP] the significance of these results and their connection to the retinal pigment epithelium [SEP]', '[CLS] the best method for preventing medical malpractice litigation is the nurturing [SEP] the best method for preventing medical malpractice litigation is the nurturing [SEP]', '[CLS] and maintenance of the physician - patient relationship. [SEP] the best method for preventing medical malpractice litigation is the nurturing [SEP]', '[CLS] this relationship can be developed by means of care [SEP] and maintenance of the physician - patient relationship. [SEP]', '[CLS] and compassion for the patient as a person [SEP] this relationship can be developed by means of care [SEP]', '[CLS] and patient education through a free [SEP] and compassion for the patient as a person [SEP]', '[CLS] and open discussion of informed consent. [SEP] and patient education through a free [SEP]', '[CLS] the relationship can be maintained by recognizing \" poor result / litigious patient \" situations with one\\'s own patients [SEP] and open discussion of informed consent. [SEP]', '[CLS] as well as with patients of other physicians. [SEP] the relationship can be maintained by recognizing \" poor result / litigious patient \" situations with one\\'s own patients [SEP]', '[CLS] the best method for preventing medical malpractice litigation is the nurturing [SEP] the best method for preventing medical malpractice litigation is the nurturing [SEP]', '[CLS] and maintenance of the physician - patient relationship. [SEP] the best method for preventing medical malpractice litigation is the nurturing [SEP]', '[CLS] this relationship can be developed by means of care [SEP] and maintenance of the physician - patient relationship. [SEP]', '[CLS] and compassion for the patient as a person [SEP] this relationship can be developed by means of care [SEP]', '[CLS] and patient education through a free [SEP] and compassion for the patient as a person [SEP]', '[CLS] and open discussion of informed consent. [SEP] and patient education through a free [SEP]', '[CLS] the relationship can be maintained by recognizing \" poor result / litigious patient \" situations with one\\'s own patients [SEP] and open discussion of informed consent. [SEP]', '[CLS] as well as with patients of other physicians. [SEP] the relationship can be maintained by recognizing \" poor result / litigious patient \" situations with one\\'s own patients [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] orbital cellulitis as a complication of orbital fracture is discussed. [SEP] abstract [SEP]', '[CLS] three cases of orbital cellulitis [SEP] orbital cellulitis as a complication of orbital fracture is discussed. [SEP]', '[CLS] that followed trauma to the orbit are presented. [SEP] three cases of orbital cellulitis [SEP]', '[CLS] a description of the proposed mechanism by which an orbital fracture predisposes to cellulitis follows. [SEP] that followed trauma to the orbit are presented. [SEP]', '[CLS] suggestions are made for prophylactic management. [SEP] a description of the proposed mechanism by which an orbital fracture predisposes to cellulitis follows. [SEP]', '[CLS] abstract [SEP] abstract [SEP]', '[CLS] orbital cellulitis as a complication of orbital fracture is discussed. [SEP] abstract [SEP]', '[CLS] three cases of orbital cellulitis [SEP] orbital cellulitis as a complication of orbital fracture is discussed. [SEP]', '[CLS] that followed trauma to the orbit are presented. [SEP] three cases of orbital cellulitis [SEP]', '[CLS] a description of the proposed mechanism by which an orbital fracture predisposes to cellulitis follows. [SEP] that followed trauma to the orbit are presented. [SEP]', '[CLS] suggestions are made for prophylactic management. [SEP] a description of the proposed mechanism by which an orbital fracture predisposes to cellulitis follows. [SEP]', '[CLS] always frequent and often misleading, [SEP] always frequent and often misleading, [SEP]', '[CLS] chronic sinusitis are multifactorial. [SEP] always frequent and often misleading, [SEP]', '[CLS] consequently, the role of teeth apical infection is largely underestimated. [SEP] chronic sinusitis are multifactorial. [SEP]', '[CLS] the difficulties of interpretation of the sinus radiographic imaging are well established, [SEP] consequently, the role of teeth apical infection is largely underestimated. [SEP]', '[CLS] particularly in the study of the dental with sinus border. [SEP] the difficulties of interpretation of the sinus radiographic imaging are well established, [SEP]', '[CLS] the clinical circumstances in which the responsibility of teeth can be involved, [SEP] particularly in the study of the dental with sinus border. [SEP]', '[CLS] are studied. [SEP] the clinical circumstances in which the responsibility of teeth can be involved, [SEP]', '[CLS] diagnosis insufficiency are frequent, [SEP] are studied. [SEP]', '[CLS] and inadequate procedures can be undertaken. [SEP] diagnosis insufficiency are frequent, [SEP]', '[CLS] the particular conditions to obtain good polycyclicidal tomographs pictures are described [SEP] and inadequate procedures can be undertaken. [SEP]', '[CLS] and demonstrated. [SEP] the particular conditions to obtain good polycyclicidal tomographs pictures are described [SEP]', '[CLS] in special cases, [SEP] and demonstrated. [SEP]', '[CLS] with the eventual contribution of the bacterial - sensitivity evaluation, [SEP] in special cases, [SEP]', '[CLS] these sinusal - dental tomographs allow a relevant diagnosis. [SEP] with the eventual contribution of the bacterial - sensitivity evaluation, [SEP]', '[CLS] always frequent and often misleading, [SEP] always frequent and often misleading, [SEP]', '[CLS] chronic sinusitis are multifactorial. [SEP] always frequent and often misleading, [SEP]', '[CLS] consequently, the role of teeth apical infection is largely underestimated. [SEP] chronic sinusitis are multifactorial. [SEP]', '[CLS] the difficulties of interpretation of the sinus radiographic imaging are well established, [SEP] consequently, the role of teeth apical infection is largely underestimated. [SEP]', '[CLS] particularly in the study of the dental with sinus border. [SEP] the difficulties of interpretation of the sinus radiographic imaging are well established, [SEP]', '[CLS] the clinical circumstances in which the responsibility of teeth can be involved, [SEP] particularly in the study of the dental with sinus border. [SEP]', '[CLS] are studied. [SEP] the clinical circumstances in which the responsibility of teeth can be involved, [SEP]', '[CLS] diagnosis insufficiency are frequent, [SEP] are studied. [SEP]', '[CLS] and inadequate procedures can be undertaken. [SEP] diagnosis insufficiency are frequent, [SEP]', '[CLS] the particular conditions to obtain good polycyclicidal tomographs pictures are described [SEP] and inadequate procedures can be undertaken. [SEP]', '[CLS] and demonstrated. [SEP] the particular conditions to obtain good polycyclicidal tomographs pictures are described [SEP]', '[CLS] in special cases, [SEP] and demonstrated. [SEP]', '[CLS] with the eventual contribution of the bacterial - sensitivity evaluation, [SEP] in special cases, [SEP]', '[CLS] these sinusal - dental tomographs allow a relevant diagnosis. [SEP] with the eventual contribution of the bacterial - sensitivity evaluation, [SEP]', '[CLS] the authors used scanning electronic microscopy to investigate [SEP] the authors used scanning electronic microscopy to investigate [SEP]', '[CLS] and define morphological characteristics of the musculus stapedius tendon : [SEP] the authors used scanning electronic microscopy to investigate [SEP]', '[CLS] the opening through which the tendon protrudes at the eminentia pyramidalis [SEP] and define morphological characteristics of the musculus stapedius tendon : [SEP]', '[CLS] and it can be found at its base. [SEP] the opening through which the tendon protrudes at the eminentia pyramidalis [SEP]', '[CLS] fibers of the tendon may directly be attached to the bony surface of the stages [SEP] and it can be found at its base. [SEP]', '[CLS] or its prolongation of various forms and dimensions. [SEP] fibers of the tendon may directly be attached to the bony surface of the stages [SEP]', '[CLS] in some cases the tendon fibrous bundles are more or less ossified. [SEP] or its prolongation of various forms and dimensions. [SEP]', '[CLS] in one patient who was operate we found a congenital malformation manifested as absence of the tendon [SEP] in some cases the tendon fibrous bundles are more or less ossified. [SEP]', '[CLS] and deformation of the stapes. [SEP] in one patient who was operate we found a congenital malformation manifested as absence of the tendon [SEP]', '[CLS] the tendon absence is found in 0. 5 % of cases. [SEP] and deformation of the stapes. [SEP]', '[CLS] the authors used scanning electronic microscopy to investigate [SEP] the authors used scanning electronic microscopy to investigate [SEP]', '[CLS] and define morphological characteristics of the musculus stapedius tendon : [SEP] the authors used scanning electronic microscopy to investigate [SEP]', '[CLS] the opening through which the tendon protrudes at the eminentia pyramidalis [SEP] and define morphological characteristics of the musculus stapedius tendon : [SEP]', '[CLS] and it can be found at its base. [SEP] the opening through which the tendon protrudes at the eminentia pyramidalis [SEP]', '[CLS] fibers of the tendon may directly be attached to the bony surface of the stages [SEP] and it can be found at its base. [SEP]', '[CLS] or its prolongation of various forms and dimensions. [SEP] fibers of the tendon may directly be attached to the bony surface of the stages [SEP]', '[CLS] in some cases the tendon fibrous bundles are more or less ossified. [SEP] or its prolongation of various forms and dimensions. [SEP]', '[CLS] in one patient who was operate we found a congenital malformation manifested as absence of the tendon [SEP] in some cases the tendon fibrous bundles are more or less ossified. [SEP]', '[CLS] and deformation of the stapes. [SEP] in one patient who was operate we found a congenital malformation manifested as absence of the tendon [SEP]', '[CLS] the tendon absence is found in 0. 5 % of cases. [SEP] and deformation of the stapes. [SEP]', '[CLS] the insertion of autogenous fascia lata to suspend the paralyzed cheek [SEP] the insertion of autogenous fascia lata to suspend the paralyzed cheek [SEP]', '[CLS] and lips remains one of the most widely practiced procedures in correction of long term paralysis of the lower facial muscles. [SEP] the insertion of autogenous fascia lata to suspend the paralyzed cheek [SEP]', '[CLS] for 34 months we used thin layers [SEP] and lips remains one of the most widely practiced procedures in correction of long term paralysis of the lower facial muscles. [SEP]', '[CLS] ( 1 mm ) [SEP] for 34 months we used thin layers [SEP]', '[CLS] of expanded polytetrafluoroethylene [SEP] ( 1 mm ) [SEP]', '[CLS] ( e - ptfe gore - tex soft tissue patch ) [SEP] of expanded polytetrafluoroethylene [SEP]', '[CLS] to replace autogenous fascia lata, [SEP] ( e - ptfe gore - tex soft tissue patch ) [SEP]', '[CLS] in 20 patients, [SEP] to replace autogenous fascia lata, [SEP]', '[CLS] with satisfactory results. [SEP] in 20 patients, [SEP]', '[CLS] 13 patients were treated by classical techniques of insertion - suspensions of the paralyzed side with a perioral loop [SEP] with satisfactory results. [SEP]', '[CLS] and slings of ptfe suspended to the zygomatic arch [SEP] 13 patients were treated by classical techniques of insertion - suspensions of the paralyzed side with a perioral loop [SEP]', '[CLS] and the infraorbital rim, [SEP] and slings of ptfe suspended to the zygomatic arch [SEP]', '[CLS] by way of nasolabial angle or rhytidectomy incisions. [SEP] and the infraorbital rim, [SEP]', \"[CLS] in 7 patients, an eyelid suspension was performed with ptfe by arion's technique, [SEP] by way of nasolabial angle or rhytidectomy incisions. [SEP]\", \"[CLS] but by replacing the classical silicon thread by e - ptfe [SEP] in 7 patients, an eyelid suspension was performed with ptfe by arion's technique, [SEP]\", '[CLS] and transposing the medial part of the temporalis muscle on the external canthus, [SEP] but by replacing the classical silicon thread by e - ptfe [SEP]', '[CLS] and fixing the lateral end of the sling to the muscle. [SEP] and transposing the medial part of the temporalis muscle on the external canthus, [SEP]', '[CLS] this technique assures a good corneal coverage with healing of the previous ulceration [SEP] and fixing the lateral end of the sling to the muscle. [SEP]', '[CLS] and allows a voluntary occlusion of the eyelids. [SEP] this technique assures a good corneal coverage with healing of the previous ulceration [SEP]', '[CLS] the ptfe soft tissue is notably successful as an implant. [SEP] and allows a voluntary occlusion of the eyelids. [SEP]', '[CLS] this biocompatible material shows excellent tissue tolerance, [SEP] the ptfe soft tissue is notably successful as an implant. [SEP]', '[CLS] the porous microstructure encouraging tissue attachment [SEP] this biocompatible material shows excellent tissue tolerance, [SEP]', '[CLS] and infiltration. [SEP] the porous microstructure encouraging tissue attachment [SEP]', '[CLS] e - ptfe holds its shape, [SEP] and infiltration. [SEP]', '[CLS] resists to infection, [SEP] e - ptfe holds its shape, [SEP]', '[CLS] and permits a notably reduction of the hospitalization [SEP] resists to infection, [SEP]', '[CLS] because it avoids a second surgical site. [SEP] and permits a notably reduction of the hospitalization [SEP]', '[CLS] we have experience of the material as a reconstructive substance in 6 cases of either bone [SEP] because it avoids a second surgical site. [SEP]', '[CLS] or subcutaneous deficits, [SEP] we have experience of the material as a reconstructive substance in 6 cases of either bone [SEP]', '[CLS] where no other satisfactory solution was available. [SEP] or subcutaneous deficits, [SEP]', '[CLS] the insertion of autogenous fascia lata to suspend the paralyzed cheek [SEP] the insertion of autogenous fascia lata to suspend the paralyzed cheek [SEP]', '[CLS] and lips remains one of the most widely practiced procedures in correction of long term paralysis of the lower facial muscles. [SEP] the insertion of autogenous fascia lata to suspend the paralyzed cheek [SEP]', '[CLS] for 34 months we used thin layers [SEP] and lips remains one of the most widely practiced procedures in correction of long term paralysis of the lower facial muscles. [SEP]', '[CLS] ( 1 mm ) [SEP] for 34 months we used thin layers [SEP]', '[CLS] of expanded polytetrafluoroethylene [SEP] ( 1 mm ) [SEP]', '[CLS] ( e - ptfe gore - tex soft tissue patch ) [SEP] of expanded polytetrafluoroethylene [SEP]', '[CLS] to replace autogenous fascia lata, [SEP] ( e - ptfe gore - tex soft tissue patch ) [SEP]', '[CLS] in 20 patients, [SEP] to replace autogenous fascia lata, [SEP]', '[CLS] with satisfactory results. [SEP] in 20 patients, [SEP]', '[CLS] 13 patients were treated by classical techniques of insertion - suspensions of the paralyzed side with a perioral loop [SEP] with satisfactory results. [SEP]', '[CLS] and slings of ptfe suspended to the zygomatic arch [SEP] 13 patients were treated by classical techniques of insertion - suspensions of the paralyzed side with a perioral loop [SEP]', '[CLS] and the infraorbital rim, [SEP] and slings of ptfe suspended to the zygomatic arch [SEP]', '[CLS] by way of nasolabial angle or rhytidectomy incisions. [SEP] and the infraorbital rim, [SEP]', \"[CLS] in 7 patients, an eyelid suspension was performed with ptfe by arion's technique, [SEP] by way of nasolabial angle or rhytidectomy incisions. [SEP]\", \"[CLS] but by replacing the classical silicon thread by e - ptfe [SEP] in 7 patients, an eyelid suspension was performed with ptfe by arion's technique, [SEP]\", '[CLS] and transposing the medial part of the temporalis muscle on the external canthus, [SEP] but by replacing the classical silicon thread by e - ptfe [SEP]', '[CLS] and fixing the lateral end of the sling to the muscle. [SEP] and transposing the medial part of the temporalis muscle on the external canthus, [SEP]', '[CLS] this technique assures a good corneal coverage with healing of the previous ulceration [SEP] and fixing the lateral end of the sling to the muscle. [SEP]', '[CLS] and allows a voluntary occlusion of the eyelids. [SEP] this technique assures a good corneal coverage with healing of the previous ulceration [SEP]', '[CLS] the ptfe soft tissue is notably successful as an implant. [SEP] and allows a voluntary occlusion of the eyelids. [SEP]', '[CLS] this biocompatible material shows excellent tissue tolerance, [SEP] the ptfe soft tissue is notably successful as an implant. [SEP]', '[CLS] the porous microstructure encouraging tissue attachment [SEP] this biocompatible material shows excellent tissue tolerance, [SEP]', '[CLS] and infiltration. [SEP] the porous microstructure encouraging tissue attachment [SEP]', '[CLS] e - ptfe holds its shape, [SEP] and infiltration. [SEP]', '[CLS] resists to infection, [SEP] e - ptfe holds its shape, [SEP]', '[CLS] and permits a notably reduction of the hospitalization [SEP] resists to infection, [SEP]', '[CLS] because it avoids a second surgical site. [SEP] and permits a notably reduction of the hospitalization [SEP]', '[CLS] we have experience of the material as a reconstructive substance in 6 cases of either bone [SEP] because it avoids a second surgical site. [SEP]', '[CLS] or subcutaneous deficits, [SEP] we have experience of the material as a reconstructive substance in 6 cases of either bone [SEP]', '[CLS] where no other satisfactory solution was available. [SEP] or subcutaneous deficits, [SEP]', '[CLS] of the many proposed classifications for staging maxillary sinus cancer, [SEP] of the many proposed classifications for staging maxillary sinus cancer, [SEP]', '[CLS] none has been adopted universally and none is known to be superior to the others. [SEP] of the many proposed classifications for staging maxillary sinus cancer, [SEP]', '[CLS] this study identified the best of six currently used classifications using data from 53 previously untreated patients with squamous cell carcinoma of the maxillary sinus. [SEP] none has been adopted universally and none is known to be superior to the others. [SEP]', \"[CLS] analysis of each classification's ability to stage the majority of patients, [SEP] this study identified the best of six currently used classifications using data from 53 previously untreated patients with squamous cell carcinoma of the maxillary sinus. [SEP]\", \"[CLS] produce a balanced distribution of t stages, [SEP] analysis of each classification's ability to stage the majority of patients, [SEP]\", '[CLS] and correlate t stage with treatment [SEP] produce a balanced distribution of t stages, [SEP]', \"[CLS] and prognosis revealed harrison's classification to be the best. [SEP] and correlate t stage with treatment [SEP]\", \"[CLS] harrison's classification should be adopted worldwide as the classification of choice for staging squamous cell carcinoma of the maxillary sinus. [SEP] and prognosis revealed harrison's classification to be the best. [SEP]\", '[CLS] of the many proposed classifications for staging maxillary sinus cancer, [SEP] of the many proposed classifications for staging maxillary sinus cancer, [SEP]', '[CLS] none has been adopted universally and none is known to be superior to the others. [SEP] of the many proposed classifications for staging maxillary sinus cancer, [SEP]', '[CLS] this study identified the best of six currently used classifications using data from 53 previously untreated patients with squamous cell carcinoma of the maxillary sinus. [SEP] none has been adopted universally and none is known to be superior to the others. [SEP]', \"[CLS] analysis of each classification's ability to stage the majority of patients, [SEP] this study identified the best of six currently used classifications using data from 53 previously untreated patients with squamous cell carcinoma of the maxillary sinus. [SEP]\", \"[CLS] produce a balanced distribution of t stages, [SEP] analysis of each classification's ability to stage the majority of patients, [SEP]\", '[CLS] and correlate t stage with treatment [SEP] produce a balanced distribution of t stages, [SEP]', \"[CLS] and prognosis revealed harrison's classification to be the best. [SEP] and correlate t stage with treatment [SEP]\", \"[CLS] harrison's classification should be adopted worldwide as the classification of choice for staging squamous cell carcinoma of the maxillary sinus. [SEP] and prognosis revealed harrison's classification to be the best. [SEP]\", \"[CLS] twenty - four patients with non hodgkin's lymphoma of nasopharynx were treated in the institut gustave - roussy, [SEP] twenty - four patients with non hodgkin's lymphoma of nasopharynx were treated in the institut gustave - roussy, [SEP]\", \"[CLS] france, [SEP] twenty - four patients with non hodgkin's lymphoma of nasopharynx were treated in the institut gustave - roussy, [SEP]\", '[CLS] between 1976 and 1983. [SEP] france, [SEP]', '[CLS] mean age of patients was 53 years [SEP] between 1976 and 1983. [SEP]', '[CLS] and the male / female sex ratio was 2. [SEP] mean age of patients was 53 years [SEP]', '[CLS] symptomatology was immediately suggestive of a lesion in the nasopharynx in 21 of the 24 patients, [SEP] and the male / female sex ratio was 2. [SEP]', '[CLS] and histology gave an unfavorable prognosis [SEP] symptomatology was immediately suggestive of a lesion in the nasopharynx in 21 of the 24 patients, [SEP]', '[CLS] ( diffuse large cell tumors ) [SEP] and histology gave an unfavorable prognosis [SEP]', '[CLS] in 71 % of cases. [SEP] ( diffuse large cell tumors ) [SEP]', '[CLS] complete investigations showed the lymphoma to be relatively limited in extent in the majority of cases [SEP] in 71 % of cases. [SEP]', '[CLS] ( 68 % of stages i and ii ). [SEP] complete investigations showed the lymphoma to be relatively limited in extent in the majority of cases [SEP]', '[CLS] these findings are in agreement with published reports. [SEP] ( 68 % of stages i and ii ). [SEP]', '[CLS] the 4 patient with stage i disease were treated by irradiation alone [SEP] these findings are in agreement with published reports. [SEP]', '[CLS] and are all alive without recurrence after 43 months. [SEP] the 4 patient with stage i disease were treated by irradiation alone [SEP]', '[CLS] and at 42 - month follow up 8 were alive without recurrence [SEP] and are all alive without recurrence after 43 months. [SEP]', '[CLS] ( 1 patient died of intercurrent illness ). [SEP] and at 42 - month follow up 8 were alive without recurrence [SEP]', '[CLS] results of conventional chemotherapy [SEP] ( 1 patient died of intercurrent illness ). [SEP]', '[CLS] ( chvmp ) [SEP] results of conventional chemotherapy [SEP]', '[CLS] in patients with stages iii [SEP] ( chvmp ) [SEP]', '[CLS] and iv were disappointing, [SEP] in patients with stages iii [SEP]', '[CLS] however : only one patient survived for 60 months [SEP] and iv were disappointing, [SEP]', '[CLS] ( after a cerebral recurrence treated by irradiation ). [SEP] however : only one patient survived for 60 months [SEP]', '[CLS] although current therapy is effective for stage i [SEP] ( after a cerebral recurrence treated by irradiation ). [SEP]', '[CLS] and ii lesions, [SEP] although current therapy is effective for stage i [SEP]', '[CLS] patients with stages iii [SEP] and ii lesions, [SEP]', '[CLS] and iv [SEP] patients with stages iii [SEP]', '[CLS] require more aggressive therapy recently developed : [SEP] and iv [SEP]', '[CLS] heavy multiple chemotherapy, [SEP] require more aggressive therapy recently developed : [SEP]', '[CLS] and even intensive chemotherapy - total body irradiation [SEP] heavy multiple chemotherapy, [SEP]', '[CLS] and bone marrow graft. [SEP] and even intensive chemotherapy - total body irradiation [SEP]', \"[CLS] twenty - four patients with non hodgkin's lymphoma of nasopharynx were treated in the institut gustave - roussy, [SEP] twenty - four patients with non hodgkin's lymphoma of nasopharynx were treated in the institut gustave - roussy, [SEP]\", \"[CLS] france, [SEP] twenty - four patients with non hodgkin's lymphoma of nasopharynx were treated in the institut gustave - roussy, [SEP]\", '[CLS] between 1976 and 1983. [SEP] france, [SEP]', '[CLS] mean age of patients was 53 years [SEP] between 1976 and 1983. [SEP]', '[CLS] and the male / female sex ratio was 2. [SEP] mean age of patients was 53 years [SEP]', '[CLS] symptomatology was immediately suggestive of a lesion in the nasopharynx in 21 of the 24 patients, [SEP] and the male / female sex ratio was 2. [SEP]', '[CLS] and histology gave an unfavorable prognosis [SEP] symptomatology was immediately suggestive of a lesion in the nasopharynx in 21 of the 24 patients, [SEP]', '[CLS] ( diffuse large cell tumors ) [SEP] and histology gave an unfavorable prognosis [SEP]', '[CLS] in 71 % of cases. [SEP] ( diffuse large cell tumors ) [SEP]', '[CLS] complete investigations showed the lymphoma to be relatively limited in extent in the majority of cases [SEP] in 71 % of cases. [SEP]', '[CLS] ( 68 % of stages i and ii ). [SEP] complete investigations showed the lymphoma to be relatively limited in extent in the majority of cases [SEP]', '[CLS] these findings are in agreement with published reports. [SEP] ( 68 % of stages i and ii ). [SEP]', '[CLS] the 4 patient with stage i disease were treated by irradiation alone [SEP] these findings are in agreement with published reports. [SEP]', '[CLS] and are all alive without recurrence after 43 months. [SEP] the 4 patient with stage i disease were treated by irradiation alone [SEP]', '[CLS] and at 42 - month follow up 8 were alive without recurrence [SEP] and are all alive without recurrence after 43 months. [SEP]', '[CLS] ( 1 patient died of intercurrent illness ). [SEP] and at 42 - month follow up 8 were alive without recurrence [SEP]', '[CLS] results of conventional chemotherapy [SEP] ( 1 patient died of intercurrent illness ). [SEP]', '[CLS] ( chvmp ) [SEP] results of conventional chemotherapy [SEP]', '[CLS] in patients with stages iii [SEP] ( chvmp ) [SEP]', '[CLS] and iv were disappointing, [SEP] in patients with stages iii [SEP]', '[CLS] however : only one patient survived for 60 months [SEP] and iv were disappointing, [SEP]', '[CLS] ( after a cerebral recurrence treated by irradiation ). [SEP] however : only one patient survived for 60 months [SEP]', '[CLS] although current therapy is effective for stage i [SEP] ( after a cerebral recurrence treated by irradiation ). [SEP]', '[CLS] and ii lesions, [SEP] although current therapy is effective for stage i [SEP]', '[CLS] patients with stages iii [SEP] and ii lesions, [SEP]', '[CLS] and iv [SEP] patients with stages iii [SEP]', '[CLS] require more aggressive therapy recently developed : [SEP] and iv [SEP]', '[CLS] heavy multiple chemotherapy, [SEP] require more aggressive therapy recently developed : [SEP]', '[CLS] and even intensive chemotherapy - total body irradiation [SEP] heavy multiple chemotherapy, [SEP]', '[CLS] and bone marrow graft. [SEP] and even intensive chemotherapy - total body irradiation [SEP]', '[CLS] the auditory brain stem response to click stimuli was investigated in a group of 50 full - term healthy newborns, [SEP] the auditory brain stem response to click stimuli was investigated in a group of 50 full - term healthy newborns, [SEP]', '[CLS] as well as in a group of 20 older children and adults. [SEP] the auditory brain stem response to click stimuli was investigated in a group of 50 full - term healthy newborns, [SEP]', '[CLS] the stimulus parameters of click level [SEP] as well as in a group of 20 older children and adults. [SEP]', '[CLS] and click repetition rate were varied systematically [SEP] the stimulus parameters of click level [SEP]', '[CLS] to quantify the characteristics of the auditory brain stem response in the full - term newborn infant. [SEP] and click repetition rate were varied systematically [SEP]', '[CLS] the results reveal increased latencies for waves i, [SEP] to quantify the characteristics of the auditory brain stem response in the full - term newborn infant. [SEP]', '[CLS] iii, [SEP] the results reveal increased latencies for waves i, [SEP]', '[CLS] and v for all conditions among the newborns, [SEP] iii, [SEP]', '[CLS] relative to the older age group. [SEP] and v for all conditions among the newborns, [SEP]', '[CLS] the results suggest that the neurological system is the primary source of differences between newborns [SEP] relative to the older age group. [SEP]', '[CLS] and older subjects, [SEP] the results suggest that the neurological system is the primary source of differences between newborns [SEP]', '[CLS] but do not rule out the possibility that external ear, [SEP] and older subjects, [SEP]', '[CLS] middle ear, [SEP] but do not rule out the possibility that external ear, [SEP]', '[CLS] or cochlear mechanisms may also contribute to the differences observed. [SEP] middle ear, [SEP]', '[CLS] the auditory brain stem response to click stimuli was investigated in a group of 50 full - term healthy newborns, [SEP] the auditory brain stem response to click stimuli was investigated in a group of 50 full - term healthy newborns, [SEP]', '[CLS] as well as in a group of 20 older children and adults. [SEP] the auditory brain stem response to click stimuli was investigated in a group of 50 full - term healthy newborns, [SEP]', '[CLS] the stimulus parameters of click level [SEP] as well as in a group of 20 older children and adults. [SEP]', '[CLS] and click repetition rate were varied systematically [SEP] the stimulus parameters of click level [SEP]', '[CLS] to quantify the characteristics of the auditory brain stem response in the full - term newborn infant. [SEP] and click repetition rate were varied systematically [SEP]', '[CLS] the results reveal increased latencies for waves i, [SEP] to quantify the characteristics of the auditory brain stem response in the full - term newborn infant. [SEP]', '[CLS] iii, [SEP] the results reveal increased latencies for waves i, [SEP]', '[CLS] and v for all conditions among the newborns, [SEP] iii, [SEP]', '[CLS] relative to the older age group. [SEP] and v for all conditions among the newborns, [SEP]', '[CLS] the results suggest that the neurological system is the primary source of differences between newborns [SEP] relative to the older age group. [SEP]', '[CLS] and older subjects, [SEP] the results suggest that the neurological system is the primary source of differences between newborns [SEP]', '[CLS] but do not rule out the possibility that external ear, [SEP] and older subjects, [SEP]', '[CLS] middle ear, [SEP] but do not rule out the possibility that external ear, [SEP]', '[CLS] or cochlear mechanisms may also contribute to the differences observed. [SEP] middle ear, [SEP]', '[CLS] cervical spondylosis [SEP] cervical spondylosis [SEP]', '[CLS] and ankylosing hyperostosis of the cervical vertebrae are common findings. [SEP] cervical spondylosis [SEP]', '[CLS] although these hypertrophic changes can be completely asymptomatic, [SEP] and ankylosing hyperostosis of the cervical vertebrae are common findings. [SEP]', '[CLS] laryngotracheal symptoms due to cervical hyperostosis are less frequent [SEP] although these hypertrophic changes can be completely asymptomatic, [SEP]', '[CLS] and may be managed initially as tumors of the esophagus, [SEP] laryngotracheal symptoms due to cervical hyperostosis are less frequent [SEP]', '[CLS] trachea, [SEP] and may be managed initially as tumors of the esophagus, [SEP]', '[CLS] or thyroid gland. [SEP] trachea, [SEP]', '[CLS] the management of two severe cases of dyspnea due to cervical ankylosing hyperostosis are discussed. [SEP] or thyroid gland. [SEP]', '[CLS] cervical spondylosis [SEP] cervical spondylosis [SEP]', '[CLS] and ankylosing hyperostosis of the cervical vertebrae are common findings. [SEP] cervical spondylosis [SEP]', '[CLS] although these hypertrophic changes can be completely asymptomatic, [SEP] and ankylosing hyperostosis of the cervical vertebrae are common findings. [SEP]', '[CLS] laryngotracheal symptoms due to cervical hyperostosis are less frequent [SEP] although these hypertrophic changes can be completely asymptomatic, [SEP]', '[CLS] and may be managed initially as tumors of the esophagus, [SEP] laryngotracheal symptoms due to cervical hyperostosis are less frequent [SEP]', '[CLS] trachea, [SEP] and may be managed initially as tumors of the esophagus, [SEP]', '[CLS] or thyroid gland. [SEP] trachea, [SEP]', '[CLS] the management of two severe cases of dyspnea due to cervical ankylosing hyperostosis are discussed. [SEP] or thyroid gland. [SEP]', '[CLS] the value of a perceptual - physiologic approach to both the evaluation [SEP] the value of a perceptual - physiologic approach to both the evaluation [SEP]', '[CLS] and treatment of voice disorders is described [SEP] the value of a perceptual - physiologic approach to both the evaluation [SEP]', '[CLS] objective measures are obtained to facilitate the understanding of the underlying pathophysiology [SEP] and treatment of voice disorders is described [SEP]', '[CLS] and to document change over time. [SEP] objective measures are obtained to facilitate the understanding of the underlying pathophysiology [SEP]', '[CLS] evaluation data are recorded [SEP] and to document change over time. [SEP]', '[CLS] and interpreted collectively by a speech - otolaryngology team [SEP] evaluation data are recorded [SEP]', '[CLS] finally, the evaluation methods, particularly videolaryngoscopy, [SEP] and interpreted collectively by a speech - otolaryngology team [SEP]', '[CLS] the value of a perceptual - physiologic approach to both the evaluation [SEP] the value of a perceptual - physiologic approach to both the evaluation [SEP]', '[CLS] and treatment of voice disorders is described [SEP] the value of a perceptual - physiologic approach to both the evaluation [SEP]', '[CLS] objective measures are obtained to facilitate the understanding of the underlying pathophysiology [SEP] and treatment of voice disorders is described [SEP]', '[CLS] and to document change over time. [SEP] objective measures are obtained to facilitate the understanding of the underlying pathophysiology [SEP]', '[CLS] evaluation data are recorded [SEP] and to document change over time. [SEP]', '[CLS] and interpreted collectively by a speech - otolaryngology team [SEP] evaluation data are recorded [SEP]', '[CLS] finally, the evaluation methods, particularly videolaryngoscopy, [SEP] and interpreted collectively by a speech - otolaryngology team [SEP]', '[CLS] the postnatal development of the eustachian tube [SEP] the postnatal development of the eustachian tube [SEP]', '[CLS] and its surrounding structures [SEP] the postnatal development of the eustachian tube [SEP]', '[CLS] ( tubal cartilage, tubal gland, tensor veli palatini muscle, [SEP] and its surrounding structures [SEP]', '[CLS] and levator veli palatini muscle ) [SEP] ( tubal cartilage, tubal gland, tensor veli palatini muscle, [SEP]', '[CLS] was investigated in serial vertical histologic sections from 12 normal temporal bones of individuals [SEP] and levator veli palatini muscle ) [SEP]', '[CLS] after projecting tissue sections onto paper and tracing the structures, [SEP] was investigated in serial vertical histologic sections from 12 normal temporal bones of individuals [SEP]', '[CLS] several measurements were made in order to analyze this development. [SEP] after projecting tissue sections onto paper and tracing the structures, [SEP]', '[CLS] findings revealed that the eustachian tube [SEP] several measurements were made in order to analyze this development. [SEP]', '[CLS] and its accessory structures developed postnatally up to the age of 19 years. [SEP] findings revealed that the eustachian tube [SEP]', '[CLS] the lumen area in a 19 - year - old specimen was 4. 7 times [SEP] and its accessory structures developed postnatally up to the age of 19 years. [SEP]', '[CLS] that of the 1 - day - old infant. [SEP] the lumen area in a 19 - year - old specimen was 4. 7 times [SEP]', '[CLS] the development of the tube was greatest in the pharyngeal part. [SEP] that of the 1 - day - old infant. [SEP]', '[CLS] postnatal development of the eustachian tube appears [SEP] the development of the tube was greatest in the pharyngeal part. [SEP]', '[CLS] the cartilage area in the 19 - year - old specimen was 3. 6 times that [SEP] postnatal development of the eustachian tube appears [SEP]', '[CLS] the mucosal acinar cells were predominant in infants but no greater [SEP] the cartilage area in the 19 - year - old specimen was 3. 6 times that [SEP]', '[CLS] the areas of the tensor [SEP] the mucosal acinar cells were predominant in infants but no greater [SEP]', '[CLS] and levator veli palatini muscles in a 19 - year - old specimen were 5. 1 [SEP] the areas of the tensor [SEP]', '[CLS] respectively, [SEP] and levator veli palatini muscles in a 19 - year - old specimen were 5. 1 [SEP]', '[CLS] those of the 1 - day - old infant. [SEP] respectively, [SEP]', '[CLS] this preliminary study reports the postnatal development of the eustachian tube [SEP] those of the 1 - day - old infant. [SEP]', '[CLS] and its accessory structures, [SEP] this preliminary study reports the postnatal development of the eustachian tube [SEP]', '[CLS] however, because of the limited number of cases available, [SEP] and its accessory structures, [SEP]', '[CLS] further investigation of a greater number of cases should be performed so that the relationship between tubal development [SEP] however, because of the limited number of cases available, [SEP]', '[CLS] the postnatal development of the eustachian tube [SEP] the postnatal development of the eustachian tube [SEP]', '[CLS] and its surrounding structures [SEP] the postnatal development of the eustachian tube [SEP]', '[CLS] ( tubal cartilage, tubal gland, tensor veli palatini muscle, [SEP] and its surrounding structures [SEP]', '[CLS] and levator veli palatini muscle ) [SEP] ( tubal cartilage, tubal gland, tensor veli palatini muscle, [SEP]', '[CLS] was investigated in serial vertical histologic sections from 12 normal temporal bones of individuals [SEP] and levator veli palatini muscle ) [SEP]', '[CLS] after projecting tissue sections onto paper and tracing the structures, [SEP] was investigated in serial vertical histologic sections from 12 normal temporal bones of individuals [SEP]', '[CLS] several measurements were made in order to analyze this development. [SEP] after projecting tissue sections onto paper and tracing the structures, [SEP]', '[CLS] findings revealed that the eustachian tube [SEP] several measurements were made in order to analyze this development. [SEP]', '[CLS] and its accessory structures developed postnatally up to the age of 19 years. [SEP] findings revealed that the eustachian tube [SEP]', '[CLS] the lumen area in a 19 - year - old specimen was 4. 7 times [SEP] and its accessory structures developed postnatally up to the age of 19 years. [SEP]', '[CLS] that of the 1 - day - old infant. [SEP] the lumen area in a 19 - year - old specimen was 4. 7 times [SEP]', '[CLS] the development of the tube was greatest in the pharyngeal part. [SEP] that of the 1 - day - old infant. [SEP]', '[CLS] postnatal development of the eustachian tube appears [SEP] the development of the tube was greatest in the pharyngeal part. [SEP]', '[CLS] the cartilage area in the 19 - year - old specimen was 3. 6 times that [SEP] postnatal development of the eustachian tube appears [SEP]', '[CLS] the mucosal acinar cells were predominant in infants but no greater [SEP] the cartilage area in the 19 - year - old specimen was 3. 6 times that [SEP]', '[CLS] the areas of the tensor [SEP] the mucosal acinar cells were predominant in infants but no greater [SEP]', '[CLS] and levator veli palatini muscles in a 19 - year - old specimen were 5. 1 [SEP] the areas of the tensor [SEP]', '[CLS] respectively, [SEP] and levator veli palatini muscles in a 19 - year - old specimen were 5. 1 [SEP]', '[CLS] those of the 1 - day - old infant. [SEP] respectively, [SEP]', '[CLS] this preliminary study reports the postnatal development of the eustachian tube [SEP] those of the 1 - day - old infant. [SEP]', '[CLS] and its accessory structures, [SEP] this preliminary study reports the postnatal development of the eustachian tube [SEP]', '[CLS] however, because of the limited number of cases available, [SEP] and its accessory structures, [SEP]', '[CLS] further investigation of a greater number of cases should be performed so that the relationship between tubal development [SEP] however, because of the limited number of cases available, [SEP]', '[CLS] the eustachian tubes of 29 influenza [SEP] the eustachian tubes of 29 influenza [SEP]', '[CLS] a virus - infected chinchillas were examined for histopathologic signs at intervals up to 21 days after inoculation to elucidate the pathologic basis of negative middle ear pressure, [SEP] the eustachian tubes of 29 influenza [SEP]', '[CLS] in the animal model, [SEP] a virus - infected chinchillas were examined for histopathologic signs at intervals up to 21 days after inoculation to elucidate the pathologic basis of negative middle ear pressure, [SEP]', '[CLS] eardrum inflammation [SEP] in the animal model, [SEP]', '[CLS] and negative middle ear pressure mirrored epithelial damage in the eustachian tube [SEP] eardrum inflammation [SEP]', '[CLS] epithelial damage was greatest in the proximal two thirds of the tube near the nasopharynx, [SEP] and negative middle ear pressure mirrored epithelial damage in the eustachian tube [SEP]', '[CLS] whereas goblet cell metaplasia [SEP] epithelial damage was greatest in the proximal two thirds of the tube near the nasopharynx, [SEP]', '[CLS] and increased secretory activity was greatest in the distal, [SEP] whereas goblet cell metaplasia [SEP]', '[CLS] these results provide a morphologic correlate to the development of negative middle ear pressure, [SEP] and increased secretory activity was greatest in the distal, [SEP]', '[CLS] the eustachian tubes of 29 influenza [SEP] the eustachian tubes of 29 influenza [SEP]', '[CLS] a virus - infected chinchillas were examined for histopathologic signs at intervals up to 21 days after inoculation to elucidate the pathologic basis of negative middle ear pressure, [SEP] the eustachian tubes of 29 influenza [SEP]', '[CLS] in the animal model, [SEP] a virus - infected chinchillas were examined for histopathologic signs at intervals up to 21 days after inoculation to elucidate the pathologic basis of negative middle ear pressure, [SEP]', '[CLS] eardrum inflammation [SEP] in the animal model, [SEP]', '[CLS] and negative middle ear pressure mirrored epithelial damage in the eustachian tube [SEP] eardrum inflammation [SEP]', '[CLS] epithelial damage was greatest in the proximal two thirds of the tube near the nasopharynx, [SEP] and negative middle ear pressure mirrored epithelial damage in the eustachian tube [SEP]', '[CLS] whereas goblet cell metaplasia [SEP] epithelial damage was greatest in the proximal two thirds of the tube near the nasopharynx, [SEP]', '[CLS] and increased secretory activity was greatest in the distal, [SEP] whereas goblet cell metaplasia [SEP]', '[CLS] these results provide a morphologic correlate to the development of negative middle ear pressure, [SEP] and increased secretory activity was greatest in the distal, [SEP]', '[CLS] since 1980 we have used middle ear implants of aluminum oxide ceramic [SEP] since 1980 we have used middle ear implants of aluminum oxide ceramic [SEP]', '[CLS] we had opportunities to examine 108 ears [SEP] since 1980 we have used middle ear implants of aluminum oxide ceramic [SEP]', '[CLS] ( 91 patients ) ; [SEP] we had opportunities to examine 108 ears [SEP]', '[CLS] follow - up examinations showed that the rates of extrusion [SEP] ( 91 patients ) ; [SEP]', '[CLS] and liability to extrusion can be reduced by interposing a small disk of autologous cartilage [SEP] follow - up examinations showed that the rates of extrusion [SEP]', '[CLS] moreover, interposing cartilage trends to improve audiometric results. [SEP] and liability to extrusion can be reduced by interposing a small disk of autologous cartilage [SEP]', '[CLS] on the basis of our study, [SEP] moreover, interposing cartilage trends to improve audiometric results. [SEP]', '[CLS] we strongly recommend the interposition of a disk of autologous cartilage [SEP] on the basis of our study, [SEP]', '[CLS] and perichondrium between the middle ear implant [SEP] we strongly recommend the interposition of a disk of autologous cartilage [SEP]', '[CLS] since 1980 we have used middle ear implants of aluminum oxide ceramic [SEP] since 1980 we have used middle ear implants of aluminum oxide ceramic [SEP]', '[CLS] we had opportunities to examine 108 ears [SEP] since 1980 we have used middle ear implants of aluminum oxide ceramic [SEP]', '[CLS] ( 91 patients ) ; [SEP] we had opportunities to examine 108 ears [SEP]', '[CLS] follow - up examinations showed that the rates of extrusion [SEP] ( 91 patients ) ; [SEP]', '[CLS] and liability to extrusion can be reduced by interposing a small disk of autologous cartilage [SEP] follow - up examinations showed that the rates of extrusion [SEP]', '[CLS] moreover, interposing cartilage trends to improve audiometric results. [SEP] and liability to extrusion can be reduced by interposing a small disk of autologous cartilage [SEP]', '[CLS] on the basis of our study, [SEP] moreover, interposing cartilage trends to improve audiometric results. [SEP]', '[CLS] we strongly recommend the interposition of a disk of autologous cartilage [SEP] on the basis of our study, [SEP]', '[CLS] and perichondrium between the middle ear implant [SEP] we strongly recommend the interposition of a disk of autologous cartilage [SEP]', '[CLS] pneumosinus dilatans is an extremely rare condition [SEP] pneumosinus dilatans is an extremely rare condition [SEP]', '[CLS] in which facial deformity is caused by gross enlargement of apparently normal paranasal sinuses. [SEP] pneumosinus dilatans is an extremely rare condition [SEP]', '[CLS] pneumosinus dilatans is an extremely rare condition [SEP] pneumosinus dilatans is an extremely rare condition [SEP]', '[CLS] in which facial deformity is caused by gross enlargement of apparently normal paranasal sinuses. [SEP] pneumosinus dilatans is an extremely rare condition [SEP]', '[CLS] the internal jugular vein is an uncommon site of deep venous thrombosis. [SEP] the internal jugular vein is an uncommon site of deep venous thrombosis. [SEP]', '[CLS] infection, [SEP] the internal jugular vein is an uncommon site of deep venous thrombosis. [SEP]', '[CLS] prolonged central venous catheterization, [SEP] infection, [SEP]', '[CLS] we present an unusual case of spontaneous thrombosis of the internal jugular vein. [SEP] prolonged central venous catheterization, [SEP]', '[CLS] the evaluation of this case included a search for possible anatomic, [SEP] we present an unusual case of spontaneous thrombosis of the internal jugular vein. [SEP]', '[CLS] hematologic, [SEP] the evaluation of this case included a search for possible anatomic, [SEP]', '[CLS] and oncologic conditions which could predispose [SEP] hematologic, [SEP]', '[CLS] to hypercoagulability [SEP] and oncologic conditions which could predispose [SEP]', '[CLS] anticoagulation therapy resulted in complete resolution of thrombosis [SEP] to hypercoagulability [SEP]', '[CLS] the internal jugular vein is an uncommon site of deep venous thrombosis. [SEP] the internal jugular vein is an uncommon site of deep venous thrombosis. [SEP]', '[CLS] infection, [SEP] the internal jugular vein is an uncommon site of deep venous thrombosis. [SEP]', '[CLS] prolonged central venous catheterization, [SEP] infection, [SEP]', '[CLS] we present an unusual case of spontaneous thrombosis of the internal jugular vein. [SEP] prolonged central venous catheterization, [SEP]', '[CLS] the evaluation of this case included a search for possible anatomic, [SEP] we present an unusual case of spontaneous thrombosis of the internal jugular vein. [SEP]', '[CLS] hematologic, [SEP] the evaluation of this case included a search for possible anatomic, [SEP]', '[CLS] and oncologic conditions which could predispose [SEP] hematologic, [SEP]', '[CLS] to hypercoagulability [SEP] and oncologic conditions which could predispose [SEP]', '[CLS] anticoagulation therapy resulted in complete resolution of thrombosis [SEP] to hypercoagulability [SEP]', '[CLS] our previous investigation using normal guinea pigs disclosed that, [SEP] our previous investigation using normal guinea pigs disclosed that, [SEP]', '[CLS] like respiratory mucosa, [SEP] our previous investigation using normal guinea pigs disclosed that, [SEP]', '[CLS] the middle ear lining has ciliary activity, [SEP] like respiratory mucosa, [SEP]', '[CLS] in this experimental study, [SEP] the middle ear lining has ciliary activity, [SEP]', '[CLS] the effects of staphylococcus aureus on the middle ear lining was examined from functional [SEP] in this experimental study, [SEP]', '[CLS] in conclusion, [SEP] the effects of staphylococcus aureus on the middle ear lining was examined from functional [SEP]', '[CLS] the ciliary activity at the entrance to the eustachian tube [SEP] in conclusion, [SEP]', '[CLS] and that more distal to the tube present a similar pattern of reaction [SEP] the ciliary activity at the entrance to the eustachian tube [SEP]', '[CLS] the ciliary activity of the middle ear lining displays a varying pattern of reaction according to the locations within the tympanic cavity. [SEP] and that more distal to the tube present a similar pattern of reaction [SEP]', '[CLS] our previous investigation using normal guinea pigs disclosed that, [SEP] our previous investigation using normal guinea pigs disclosed that, [SEP]', '[CLS] like respiratory mucosa, [SEP] our previous investigation using normal guinea pigs disclosed that, [SEP]', '[CLS] the middle ear lining has ciliary activity, [SEP] like respiratory mucosa, [SEP]', '[CLS] in this experimental study, [SEP] the middle ear lining has ciliary activity, [SEP]', '[CLS] the effects of staphylococcus aureus on the middle ear lining was examined from functional [SEP] in this experimental study, [SEP]', '[CLS] in conclusion, [SEP] the effects of staphylococcus aureus on the middle ear lining was examined from functional [SEP]', '[CLS] the ciliary activity at the entrance to the eustachian tube [SEP] in conclusion, [SEP]', '[CLS] and that more distal to the tube present a similar pattern of reaction [SEP] the ciliary activity at the entrance to the eustachian tube [SEP]', '[CLS] the ciliary activity of the middle ear lining displays a varying pattern of reaction according to the locations within the tympanic cavity. [SEP] and that more distal to the tube present a similar pattern of reaction [SEP]', '[CLS] an otherwise healthy male patient had a bilateral, [SEP] an otherwise healthy male patient had a bilateral, [SEP]', '[CLS] slowly progressive hearing loss first noticed in early childhood [SEP] an otherwise healthy male patient had a bilateral, [SEP]', '[CLS] audiometric studies at the age of 32 showed a moderately severe, [SEP] slowly progressive hearing loss first noticed in early childhood [SEP]', '[CLS] bilateral, mixed type hearing loss. [SEP] audiometric studies at the age of 32 showed a moderately severe, [SEP]', '[CLS] studies of the temporal bones showed bilateral hypoplasia [SEP] bilateral, mixed type hearing loss. [SEP]', '[CLS] the history and findings are consistent with a genetically determined hypoplasia [SEP] studies of the temporal bones showed bilateral hypoplasia [SEP]', '[CLS] an otherwise healthy male patient had a bilateral, [SEP] an otherwise healthy male patient had a bilateral, [SEP]', '[CLS] slowly progressive hearing loss first noticed in early childhood [SEP] an otherwise healthy male patient had a bilateral, [SEP]', '[CLS] audiometric studies at the age of 32 showed a moderately severe, [SEP] slowly progressive hearing loss first noticed in early childhood [SEP]', '[CLS] bilateral, mixed type hearing loss. [SEP] audiometric studies at the age of 32 showed a moderately severe, [SEP]', '[CLS] studies of the temporal bones showed bilateral hypoplasia [SEP] bilateral, mixed type hearing loss. [SEP]', '[CLS] the history and findings are consistent with a genetically determined hypoplasia [SEP] studies of the temporal bones showed bilateral hypoplasia [SEP]', \"[CLS] epithelial malignancy in a warthin's tumor exists in three forms. [SEP] epithelial malignancy in a warthin's tumor exists in three forms. [SEP]\", \"[CLS] the most common is a coexistent, separate neoplasm. [SEP] epithelial malignancy in a warthin's tumor exists in three forms. [SEP]\", \"[CLS] metastasis to the lymphoid component of warthin's tumor follows, [SEP] the most common is a coexistent, separate neoplasm. [SEP]\", \"[CLS] and the least common is a primary carcinoma arising from the ductal component of the tumor. [SEP] metastasis to the lymphoid component of warthin's tumor follows, [SEP]\", \"[CLS] epithelial malignancy in a warthin's tumor exists in three forms. [SEP] epithelial malignancy in a warthin's tumor exists in three forms. [SEP]\", \"[CLS] the most common is a coexistent, separate neoplasm. [SEP] epithelial malignancy in a warthin's tumor exists in three forms. [SEP]\", \"[CLS] metastasis to the lymphoid component of warthin's tumor follows, [SEP] the most common is a coexistent, separate neoplasm. [SEP]\", \"[CLS] and the least common is a primary carcinoma arising from the ductal component of the tumor. [SEP] metastasis to the lymphoid component of warthin's tumor follows, [SEP]\", '[CLS] clinic and histologic analysis of 92 phototests of various photodermatosis [SEP] clinic and histologic analysis of 92 phototests of various photodermatosis [SEP]', '[CLS] ( 14 benign summer light eruption [SEP] clinic and histologic analysis of 92 phototests of various photodermatosis [SEP]', '[CLS] 47 polymorphous light eruption [SEP] ( 14 benign summer light eruption [SEP]', '[CLS] 20 persistent light reactor photodermatitis [SEP] 47 polymorphous light eruption [SEP]', '[CLS] 11 drug induced photodermatitis [SEP] 20 persistent light reactor photodermatitis [SEP]', '[CLS] clinically, [SEP] 11 drug induced photodermatitis [SEP]', '[CLS] a phototoxic aspect was observed in 30 cases, [SEP] clinically, [SEP]', '[CLS] it was photoallergic in 36 cases [SEP] a phototoxic aspect was observed in 30 cases, [SEP]', '[CLS] histologically, [SEP] it was photoallergic in 36 cases [SEP]', '[CLS] a phototoxic picture was present in 32 cases, [SEP] histologically, [SEP]', '[CLS] a photoallergic one in 48 cases, [SEP] a phototoxic picture was present in 32 cases, [SEP]', '[CLS] only one case of bsle was associated with photoallergic aspect, [SEP] a photoallergic one in 48 cases, [SEP]', '[CLS] photoallergic type was observed in 70 p. 100 of iterative exposure phototest [SEP] only one case of bsle was associated with photoallergic aspect, [SEP]', '[CLS] of pmle ; [SEP] photoallergic type was observed in 70 p. 100 of iterative exposure phototest [SEP]', '[CLS] clinical and histological aspects were concordant ; [SEP] of pmle ; [SEP]', '[CLS] in contrast phototoxic aspects were constantly observed when single exposure phototest [SEP] clinical and histological aspects were concordant ; [SEP]', '[CLS] in plrp histological aspect of iep was constantly of photoallergic type ; [SEP] in contrast phototoxic aspects were constantly observed when single exposure phototest [SEP]', '[CLS] whereas clinical aspect was uncertain in 50 p. 100 of the cases ; [SEP] in plrp histological aspect of iep was constantly of photoallergic type ; [SEP]', '[CLS] in dip histological aspect of iep was constantly photoallergic, [SEP] whereas clinical aspect was uncertain in 50 p. 100 of the cases ; [SEP]', '[CLS] whereas 5 of the 6 cases exhibited uncertain clinical aspect ; [SEP] in dip histological aspect of iep was constantly photoallergic, [SEP]', '[CLS] only one of the 5 sep gave a photoallergic reaction both clinically [SEP] whereas 5 of the 6 cases exhibited uncertain clinical aspect ; [SEP]', '[CLS] comparison of histological features revealed a more typical photoallergic picture in plrp [SEP] only one of the 5 sep gave a photoallergic reaction both clinically [SEP]', '[CLS] we conclude that : [SEP] comparison of histological features revealed a more typical photoallergic picture in plrp [SEP]', '[CLS] the reproducibility of clinical [SEP] we conclude that : [SEP]', '[CLS] and histological lesions is better with the iterative exposure phototest [SEP] the reproducibility of clinical [SEP]', '[CLS] histologic examination is more reliable [SEP] and histological lesions is better with the iterative exposure phototest [SEP]', '[CLS] than clinical findings in persistent light reactor photodermatitis [SEP] histologic examination is more reliable [SEP]', '[CLS] clinic and histologic analysis of 92 phototests of various photodermatosis [SEP] clinic and histologic analysis of 92 phototests of various photodermatosis [SEP]', '[CLS] ( 14 benign summer light eruption [SEP] clinic and histologic analysis of 92 phototests of various photodermatosis [SEP]', '[CLS] 47 polymorphous light eruption [SEP] ( 14 benign summer light eruption [SEP]', '[CLS] 20 persistent light reactor photodermatitis [SEP] 47 polymorphous light eruption [SEP]', '[CLS] 11 drug induced photodermatitis [SEP] 20 persistent light reactor photodermatitis [SEP]', '[CLS] clinically, [SEP] 11 drug induced photodermatitis [SEP]', '[CLS] a phototoxic aspect was observed in 30 cases, [SEP] clinically, [SEP]', '[CLS] it was photoallergic in 36 cases [SEP] a phototoxic aspect was observed in 30 cases, [SEP]', '[CLS] histologically, [SEP] it was photoallergic in 36 cases [SEP]', '[CLS] a phototoxic picture was present in 32 cases, [SEP] histologically, [SEP]', '[CLS] a photoallergic one in 48 cases, [SEP] a phototoxic picture was present in 32 cases, [SEP]', '[CLS] only one case of bsle was associated with photoallergic aspect, [SEP] a photoallergic one in 48 cases, [SEP]', '[CLS] photoallergic type was observed in 70 p. 100 of iterative exposure phototest [SEP] only one case of bsle was associated with photoallergic aspect, [SEP]', '[CLS] of pmle ; [SEP] photoallergic type was observed in 70 p. 100 of iterative exposure phototest [SEP]', '[CLS] clinical and histological aspects were concordant ; [SEP] of pmle ; [SEP]', '[CLS] in contrast phototoxic aspects were constantly observed when single exposure phototest [SEP] clinical and histological aspects were concordant ; [SEP]', '[CLS] in plrp histological aspect of iep was constantly of photoallergic type ; [SEP] in contrast phototoxic aspects were constantly observed when single exposure phototest [SEP]', '[CLS] whereas clinical aspect was uncertain in 50 p. 100 of the cases ; [SEP] in plrp histological aspect of iep was constantly of photoallergic type ; [SEP]', '[CLS] in dip histological aspect of iep was constantly photoallergic, [SEP] whereas clinical aspect was uncertain in 50 p. 100 of the cases ; [SEP]', '[CLS] whereas 5 of the 6 cases exhibited uncertain clinical aspect ; [SEP] in dip histological aspect of iep was constantly photoallergic, [SEP]', '[CLS] only one of the 5 sep gave a photoallergic reaction both clinically [SEP] whereas 5 of the 6 cases exhibited uncertain clinical aspect ; [SEP]', '[CLS] comparison of histological features revealed a more typical photoallergic picture in plrp [SEP] only one of the 5 sep gave a photoallergic reaction both clinically [SEP]', '[CLS] we conclude that : [SEP] comparison of histological features revealed a more typical photoallergic picture in plrp [SEP]', '[CLS] the reproducibility of clinical [SEP] we conclude that : [SEP]', '[CLS] and histological lesions is better with the iterative exposure phototest [SEP] the reproducibility of clinical [SEP]', '[CLS] histologic examination is more reliable [SEP] and histological lesions is better with the iterative exposure phototest [SEP]', '[CLS] than clinical findings in persistent light reactor photodermatitis [SEP] histologic examination is more reliable [SEP]', '[CLS] radioactive iodine [SEP] radioactive iodine [SEP]', '[CLS] ( i - 131 ) [SEP] radioactive iodine [SEP]', '[CLS] uptakes [SEP] ( i - 131 ) [SEP]', '[CLS] ( raiu ) [SEP] uptakes [SEP]', '[CLS] and serum t4 levels were observed in 72 patients referred for radioiodine therapy. [SEP] ( raiu ) [SEP]', '[CLS] of 65 patients whose results could be evaluated 47 had these tests while on carbiomazole [SEP] and serum t4 levels were observed in 72 patients referred for radioiodine therapy. [SEP]', '[CLS] ( cmz ). [SEP] of 65 patients whose results could be evaluated 47 had these tests while on carbiomazole [SEP]', '[CLS] in the remaining 18 patients of the 65, [SEP] ( cmz ). [SEP]', '[CLS] the differences in the results were again not significant. [SEP] in the remaining 18 patients of the 65, [SEP]', '[CLS] the incidence of hypothyroidism at 1 year was noted in the group receiving radioiodine therapy while on cmz as compared to those stopping the cmz for 1 - 12 days. [SEP] the differences in the results were again not significant. [SEP]', '[CLS] serum t4 levels were measured in 21 patients, [SEP] the incidence of hypothyroidism at 1 year was noted in the group receiving radioiodine therapy while on cmz as compared to those stopping the cmz for 1 - 12 days. [SEP]', '[CLS] radioactive iodine [SEP] radioactive iodine [SEP]', '[CLS] ( i - 131 ) [SEP] radioactive iodine [SEP]', '[CLS] uptakes [SEP] ( i - 131 ) [SEP]', '[CLS] ( raiu ) [SEP] uptakes [SEP]', '[CLS] and serum t4 levels were observed in 72 patients referred for radioiodine therapy. [SEP] ( raiu ) [SEP]', '[CLS] of 65 patients whose results could be evaluated 47 had these tests while on carbiomazole [SEP] and serum t4 levels were observed in 72 patients referred for radioiodine therapy. [SEP]', '[CLS] ( cmz ). [SEP] of 65 patients whose results could be evaluated 47 had these tests while on carbiomazole [SEP]', '[CLS] in the remaining 18 patients of the 65, [SEP] ( cmz ). [SEP]', '[CLS] the differences in the results were again not significant. [SEP] in the remaining 18 patients of the 65, [SEP]', '[CLS] the incidence of hypothyroidism at 1 year was noted in the group receiving radioiodine therapy while on cmz as compared to those stopping the cmz for 1 - 12 days. [SEP] the differences in the results were again not significant. [SEP]', '[CLS] serum t4 levels were measured in 21 patients, [SEP] the incidence of hypothyroidism at 1 year was noted in the group receiving radioiodine therapy while on cmz as compared to those stopping the cmz for 1 - 12 days. [SEP]', '[CLS] a prospective study of 17 patients with fracture neck of femur was undertaken to study the vascularity of the femoral head by bone scan using technetium - 99m mdp, [SEP] a prospective study of 17 patients with fracture neck of femur was undertaken to study the vascularity of the femoral head by bone scan using technetium - 99m mdp, [SEP]', '[CLS] as well as to study the viability of the femoral head by histopathology, [SEP] a prospective study of 17 patients with fracture neck of femur was undertaken to study the vascularity of the femoral head by bone scan using technetium - 99m mdp, [SEP]', '[CLS] and therefore determine correlation, [SEP] as well as to study the viability of the femoral head by histopathology, [SEP]', '[CLS] treatment was by primary endoprosthetic replacement [SEP] and therefore determine correlation, [SEP]', '[CLS] in 16 of the 17 patients the bone scan showed decreased vascularity of the head, [SEP] treatment was by primary endoprosthetic replacement [SEP]', '[CLS] it is likely that, [SEP] in 16 of the 17 patients the bone scan showed decreased vascularity of the head, [SEP]', '[CLS] since the majority of the patients were operated on within 4 days, [SEP] it is likely that, [SEP]', '[CLS] there was insufficient time for histopathological evidence of ischemia to develop, [SEP] since the majority of the patients were operated on within 4 days, [SEP]', '[CLS] a prospective study of 17 patients with fracture neck of femur was undertaken to study the vascularity of the femoral head by bone scan using technetium - 99m mdp, [SEP] a prospective study of 17 patients with fracture neck of femur was undertaken to study the vascularity of the femoral head by bone scan using technetium - 99m mdp, [SEP]', '[CLS] as well as to study the viability of the femoral head by histopathology, [SEP] a prospective study of 17 patients with fracture neck of femur was undertaken to study the vascularity of the femoral head by bone scan using technetium - 99m mdp, [SEP]', '[CLS] and therefore determine correlation, [SEP] as well as to study the viability of the femoral head by histopathology, [SEP]', '[CLS] treatment was by primary endoprosthetic replacement [SEP] and therefore determine correlation, [SEP]', '[CLS] in 16 of the 17 patients the bone scan showed decreased vascularity of the head, [SEP] treatment was by primary endoprosthetic replacement [SEP]', '[CLS] it is likely that, [SEP] in 16 of the 17 patients the bone scan showed decreased vascularity of the head, [SEP]', '[CLS] since the majority of the patients were operated on within 4 days, [SEP] it is likely that, [SEP]', '[CLS] there was insufficient time for histopathological evidence of ischemia to develop, [SEP] since the majority of the patients were operated on within 4 days, [SEP]', '[CLS] serum thyroglobulin antibodies [SEP] serum thyroglobulin antibodies [SEP]', '[CLS] ( tgab ) [SEP] serum thyroglobulin antibodies [SEP]', '[CLS] and serum thyroglobulin [SEP] ( tgab ) [SEP]', '[CLS] ( tg ) [SEP] and serum thyroglobulin [SEP]', '[CLS] the tgab and tg levels were measured in the patients when they were on replacement thyroxine therapy prior to the whole body radioiodine i - 131 scan ( wbs ), [SEP] ( tg ) [SEP]', '[CLS] the tgab level in normal controls was from 0 - 4. 8 micrograms / ml, [SEP] the tgab and tg levels were measured in the patients when they were on replacement thyroxine therapy prior to the whole body radioiodine i - 131 scan ( wbs ), [SEP]', '[CLS] with a mean of 0. 6 microgram / ml, [SEP] the tgab level in normal controls was from 0 - 4. 8 micrograms / ml, [SEP]', '[CLS] while the tg level in these controls was from 0. 0 - 48. 0 ng / ml, [SEP] with a mean of 0. 6 microgram / ml, [SEP]', '[CLS] in the patients with thyroid cancer, [SEP] while the tg level in these controls was from 0. 0 - 48. 0 ng / ml, [SEP]', '[CLS] their tg levels were normal except for 7 patients who had elevated values. [SEP] in the patients with thyroid cancer, [SEP]', '[CLS] 2 of these 7 had no evidence of disease on x - ray, [SEP] their tg levels were normal except for 7 patients who had elevated values. [SEP]', '[CLS] bone scan, [SEP] 2 of these 7 had no evidence of disease on x - ray, [SEP]', '[CLS] or wbs ( i - 131 ) [SEP] bone scan, [SEP]', '[CLS] however, 4 patients had metastatic disease with normal tg levels, [SEP] or wbs ( i - 131 ) [SEP]', '[CLS] and 2 of these patients had residual thyroid in the neck and raised tgab levels, [SEP] however, 4 patients had metastatic disease with normal tg levels, [SEP]', '[CLS] while in the remaining 2 patients, [SEP] and 2 of these patients had residual thyroid in the neck and raised tgab levels, [SEP]', '[CLS] serum tg is a useful tumour marker for recurrent or metastatic thyroid cancer, [SEP] while in the remaining 2 patients, [SEP]', '[CLS] serum thyroglobulin antibodies [SEP] serum thyroglobulin antibodies [SEP]', '[CLS] ( tgab ) [SEP] serum thyroglobulin antibodies [SEP]', '[CLS] and serum thyroglobulin [SEP] ( tgab ) [SEP]', '[CLS] ( tg ) [SEP] and serum thyroglobulin [SEP]', '[CLS] the tgab and tg levels were measured in the patients when they were on replacement thyroxine therapy prior to the whole body radioiodine i - 131 scan ( wbs ), [SEP] ( tg ) [SEP]', '[CLS] the tgab level in normal controls was from 0 - 4. 8 micrograms / ml, [SEP] the tgab and tg levels were measured in the patients when they were on replacement thyroxine therapy prior to the whole body radioiodine i - 131 scan ( wbs ), [SEP]', '[CLS] with a mean of 0. 6 microgram / ml, [SEP] the tgab level in normal controls was from 0 - 4. 8 micrograms / ml, [SEP]', '[CLS] while the tg level in these controls was from 0. 0 - 48. 0 ng / ml, [SEP] with a mean of 0. 6 microgram / ml, [SEP]', '[CLS] in the patients with thyroid cancer, [SEP] while the tg level in these controls was from 0. 0 - 48. 0 ng / ml, [SEP]', '[CLS] their tg levels were normal except for 7 patients who had elevated values. [SEP] in the patients with thyroid cancer, [SEP]', '[CLS] 2 of these 7 had no evidence of disease on x - ray, [SEP] their tg levels were normal except for 7 patients who had elevated values. [SEP]', '[CLS] bone scan, [SEP] 2 of these 7 had no evidence of disease on x - ray, [SEP]', '[CLS] or wbs ( i - 131 ) [SEP] bone scan, [SEP]', '[CLS] however, 4 patients had metastatic disease with normal tg levels, [SEP] or wbs ( i - 131 ) [SEP]', '[CLS] and 2 of these patients had residual thyroid in the neck and raised tgab levels, [SEP] however, 4 patients had metastatic disease with normal tg levels, [SEP]', '[CLS] while in the remaining 2 patients, [SEP] and 2 of these patients had residual thyroid in the neck and raised tgab levels, [SEP]', '[CLS] serum tg is a useful tumour marker for recurrent or metastatic thyroid cancer, [SEP] while in the remaining 2 patients, [SEP]', '[CLS] hepatic lithiasis is a major disease in asia, [SEP] hepatic lithiasis is a major disease in asia, [SEP]', '[CLS] especially east asia. [SEP] hepatic lithiasis is a major disease in asia, [SEP]', '[CLS] the radionuclide methods ranging from sequential scintigraphy [SEP] especially east asia. [SEP]', '[CLS] hepatic lithiasis is a major disease in asia, [SEP] hepatic lithiasis is a major disease in asia, [SEP]', '[CLS] especially east asia. [SEP] hepatic lithiasis is a major disease in asia, [SEP]', '[CLS] the radionuclide methods ranging from sequential scintigraphy [SEP] especially east asia. [SEP]', '[CLS] cancers of childhood tend to be aggressive and to be associated with poor prognoses. [SEP] cancers of childhood tend to be aggressive and to be associated with poor prognoses. [SEP]', '[CLS] early diagnosis is essential and simple methods of assessing response to therapy are critical. [SEP] cancers of childhood tend to be aggressive and to be associated with poor prognoses. [SEP]', '[CLS] nuclear medicine procedures have proved invaluable. [SEP] early diagnosis is essential and simple methods of assessing response to therapy are critical. [SEP]', '[CLS] achieving these objectives has become even more important by improvement in therapeutic regimes [SEP] nuclear medicine procedures have proved invaluable. [SEP]', '[CLS] which have reduced the pessimistic prognoses. [SEP] achieving these objectives has become even more important by improvement in therapeutic regimes [SEP]', '[CLS] recent advances in nuclear medicine equipment have markedly increased the sensitivity of the investigations [SEP] which have reduced the pessimistic prognoses. [SEP]', '[CLS] and the introduction of new radiopharmaceuticals has greatly raised the specificity, [SEP] recent advances in nuclear medicine equipment have markedly increased the sensitivity of the investigations [SEP]', '[CLS] recent research indicates the probability of localizing the radioactive labelled agents, [SEP] and the introduction of new radiopharmaceuticals has greatly raised the specificity, [SEP]', '[CLS] such as monoclonal antibodies, [SEP] recent research indicates the probability of localizing the radioactive labelled agents, [SEP]', '[CLS] cancers of childhood tend to be aggressive and to be associated with poor prognoses. [SEP] cancers of childhood tend to be aggressive and to be associated with poor prognoses. [SEP]', '[CLS] early diagnosis is essential and simple methods of assessing response to therapy are critical. [SEP] cancers of childhood tend to be aggressive and to be associated with poor prognoses. [SEP]', '[CLS] nuclear medicine procedures have proved invaluable. [SEP] early diagnosis is essential and simple methods of assessing response to therapy are critical. [SEP]', '[CLS] achieving these objectives has become even more important by improvement in therapeutic regimes [SEP] nuclear medicine procedures have proved invaluable. [SEP]', '[CLS] which have reduced the pessimistic prognoses. [SEP] achieving these objectives has become even more important by improvement in therapeutic regimes [SEP]', '[CLS] recent advances in nuclear medicine equipment have markedly increased the sensitivity of the investigations [SEP] which have reduced the pessimistic prognoses. [SEP]', '[CLS] and the introduction of new radiopharmaceuticals has greatly raised the specificity, [SEP] recent advances in nuclear medicine equipment have markedly increased the sensitivity of the investigations [SEP]', '[CLS] recent research indicates the probability of localizing the radioactive labelled agents, [SEP] and the introduction of new radiopharmaceuticals has greatly raised the specificity, [SEP]', '[CLS] such as monoclonal antibodies, [SEP] recent research indicates the probability of localizing the radioactive labelled agents, [SEP]', '[CLS] flow cytofluorometry and drug targeting with labelled neoglycoproteins are used as tools to probe for membrane lectins in two human adenocarcinoma cell lines. [SEP] flow cytofluorometry and drug targeting with labelled neoglycoproteins are used as tools to probe for membrane lectins in two human adenocarcinoma cell lines. [SEP]', '[CLS] both cell lines express activities for galactosides, [SEP] flow cytofluorometry and drug targeting with labelled neoglycoproteins are used as tools to probe for membrane lectins in two human adenocarcinoma cell lines. [SEP]', '[CLS] affinity chromatography on gels with immobilized sugar leads to purification of an alpha - galactoside - binding protein at an apparent molecular weight of 64 kda that also binds to lactose, [SEP] both cell lines express activities for galactosides, [SEP]', '[CLS] maltose and fucose [SEP] affinity chromatography on gels with immobilized sugar leads to purification of an alpha - galactoside - binding protein at an apparent molecular weight of 64 kda that also binds to lactose, [SEP]', '[CLS] and exhibits ca2 + - requirement for binding, [SEP] maltose and fucose [SEP]', '[CLS] a beta - galactoside - binding protein without ca2 + - requirement at an apparent molecular weight of 14 kda, [SEP] and exhibits ca2 + - requirement for binding, [SEP]', '[CLS] the description of membrane lectins, [SEP] a beta - galactoside - binding protein without ca2 + - requirement at an apparent molecular weight of 14 kda, [SEP]', '[CLS] documented here for the first human tumor cell lines, [SEP] the description of membrane lectins, [SEP]', '[CLS] flow cytofluorometry and drug targeting with labelled neoglycoproteins are used as tools to probe for membrane lectins in two human adenocarcinoma cell lines. [SEP] flow cytofluorometry and drug targeting with labelled neoglycoproteins are used as tools to probe for membrane lectins in two human adenocarcinoma cell lines. [SEP]', '[CLS] both cell lines express activities for galactosides, [SEP] flow cytofluorometry and drug targeting with labelled neoglycoproteins are used as tools to probe for membrane lectins in two human adenocarcinoma cell lines. [SEP]', '[CLS] affinity chromatography on gels with immobilized sugar leads to purification of an alpha - galactoside - binding protein at an apparent molecular weight of 64 kda that also binds to lactose, [SEP] both cell lines express activities for galactosides, [SEP]', '[CLS] maltose and fucose [SEP] affinity chromatography on gels with immobilized sugar leads to purification of an alpha - galactoside - binding protein at an apparent molecular weight of 64 kda that also binds to lactose, [SEP]', '[CLS] and exhibits ca2 + - requirement for binding, [SEP] maltose and fucose [SEP]', '[CLS] a beta - galactoside - binding protein without ca2 + - requirement at an apparent molecular weight of 14 kda, [SEP] and exhibits ca2 + - requirement for binding, [SEP]', '[CLS] the description of membrane lectins, [SEP] a beta - galactoside - binding protein without ca2 + - requirement at an apparent molecular weight of 14 kda, [SEP]', '[CLS] documented here for the first human tumor cell lines, [SEP] the description of membrane lectins, [SEP]', '[CLS] evaluations were made of the expression of blood group antigens, [SEP] evaluations were made of the expression of blood group antigens, [SEP]', '[CLS] dna content and s - phase rate in a prospective study of 97 patients with squamous cell carcinoma of the uterine cervix. [SEP] evaluations were made of the expression of blood group antigens, [SEP]', '[CLS] no correlation was found between loss of abh expression and ploidy level or the degree of proliferation, [SEP] dna content and s - phase rate in a prospective study of 97 patients with squamous cell carcinoma of the uterine cervix. [SEP]', '[CLS] in 11 ( 42 % ) of 26 tumors with two or three detectable stemlines, [SEP] no correlation was found between loss of abh expression and ploidy level or the degree of proliferation, [SEP]', '[CLS] distinct areas with disparate abh staining patterns were found, [SEP] in 11 ( 42 % ) of 26 tumors with two or three detectable stemlines, [SEP]', '[CLS] in tumors with one stemline significantly fewer such tumors were found [SEP] distinct areas with disparate abh staining patterns were found, [SEP]', '[CLS] twenty - four early recurrences [SEP] in tumors with one stemline significantly fewer such tumors were found [SEP]', '[CLS] ( within 20 - 36 months ) [SEP] twenty - four early recurrences [SEP]', '[CLS] high s - phase rates [SEP] ( within 20 - 36 months ) [SEP]', '[CLS] ( greater than 20 % ) [SEP] high s - phase rates [SEP]', '[CLS] were associated with a significantly increased recurrence rate [SEP] ( greater than 20 % ) [SEP]', '[CLS] ( p less than 0. 05 ). [SEP] were associated with a significantly increased recurrence rate [SEP]', '[CLS] evaluations were made of the expression of blood group antigens, [SEP] evaluations were made of the expression of blood group antigens, [SEP]', '[CLS] dna content and s - phase rate in a prospective study of 97 patients with squamous cell carcinoma of the uterine cervix. [SEP] evaluations were made of the expression of blood group antigens, [SEP]', '[CLS] no correlation was found between loss of abh expression and ploidy level or the degree of proliferation, [SEP] dna content and s - phase rate in a prospective study of 97 patients with squamous cell carcinoma of the uterine cervix. [SEP]', '[CLS] in 11 ( 42 % ) of 26 tumors with two or three detectable stemlines, [SEP] no correlation was found between loss of abh expression and ploidy level or the degree of proliferation, [SEP]', '[CLS] distinct areas with disparate abh staining patterns were found, [SEP] in 11 ( 42 % ) of 26 tumors with two or three detectable stemlines, [SEP]', '[CLS] in tumors with one stemline significantly fewer such tumors were found [SEP] distinct areas with disparate abh staining patterns were found, [SEP]', '[CLS] twenty - four early recurrences [SEP] in tumors with one stemline significantly fewer such tumors were found [SEP]', '[CLS] ( within 20 - 36 months ) [SEP] twenty - four early recurrences [SEP]', '[CLS] high s - phase rates [SEP] ( within 20 - 36 months ) [SEP]', '[CLS] ( greater than 20 % ) [SEP] high s - phase rates [SEP]', '[CLS] were associated with a significantly increased recurrence rate [SEP] ( greater than 20 % ) [SEP]', '[CLS] ( p less than 0. 05 ). [SEP] were associated with a significantly increased recurrence rate [SEP]', '[CLS] this study investigates the phenomenon of unscheduled dna synthesis [SEP] this study investigates the phenomenon of unscheduled dna synthesis [SEP]', '[CLS] ( uds ) [SEP] this study investigates the phenomenon of unscheduled dna synthesis [SEP]', '[CLS] in cells whose dna contains bromodeoxyuridine [SEP] ( uds ) [SEP]', '[CLS] ( brdu ) [SEP] in cells whose dna contains bromodeoxyuridine [SEP]', '[CLS] the technique utilized is capable of detecting two probes simultaneously from the dna of individual cells. [SEP] ( brdu ) [SEP]', '[CLS] however, after adjusting our experimental protocol to eliminate the artifacts produced by the incomplete suppression of dna synthesis by hu, [SEP] the technique utilized is capable of detecting two probes simultaneously from the dna of individual cells. [SEP]', '[CLS] this study investigates the phenomenon of unscheduled dna synthesis [SEP] this study investigates the phenomenon of unscheduled dna synthesis [SEP]', '[CLS] ( uds ) [SEP] this study investigates the phenomenon of unscheduled dna synthesis [SEP]', '[CLS] in cells whose dna contains bromodeoxyuridine [SEP] ( uds ) [SEP]', '[CLS] ( brdu ) [SEP] in cells whose dna contains bromodeoxyuridine [SEP]', '[CLS] the technique utilized is capable of detecting two probes simultaneously from the dna of individual cells. [SEP] ( brdu ) [SEP]', '[CLS] however, after adjusting our experimental protocol to eliminate the artifacts produced by the incomplete suppression of dna synthesis by hu, [SEP] the technique utilized is capable of detecting two probes simultaneously from the dna of individual cells. [SEP]', '[CLS] the effects of the 18 carbon chain length series of fatty acids on the viability of a 3t6 derived mouse fibroblast cell line were tested. [SEP] the effects of the 18 carbon chain length series of fatty acids on the viability of a 3t6 derived mouse fibroblast cell line were tested. [SEP]', '[CLS] an overall concentration dependent cytotoxic effect was demonstrated with the saturated fatty acid, [SEP] the effects of the 18 carbon chain length series of fatty acids on the viability of a 3t6 derived mouse fibroblast cell line were tested. [SEP]', '[CLS] and cis - and trans - mono - and dienoic fatty acids ; [SEP] an overall concentration dependent cytotoxic effect was demonstrated with the saturated fatty acid, [SEP]', '[CLS] however, within a narrow range of concentrations, [SEP] and cis - and trans - mono - and dienoic fatty acids ; [SEP]', '[CLS] as a result of the increase in cell viability over this concentration range, [SEP] however, within a narrow range of concentrations, [SEP]', '[CLS] the most dramatic cytotoxic effects were shown by cis - c18 : 3 omega 3 and cis - c18 : 3 [SEP] as a result of the increase in cell viability over this concentration range, [SEP]', '[CLS] omega 6 with id50 values of + / - 30 micrograms / ml. [SEP] the most dramatic cytotoxic effects were shown by cis - c18 : 3 omega 3 and cis - c18 : 3 [SEP]', '[CLS] cis - c18 : 4 omega 3 showed an initial cytotoxic effect at fatty acid concentrations of 0 - 20 micrograms / ml, [SEP] omega 6 with id50 values of + / - 30 micrograms / ml. [SEP]', '[CLS] furthermore, the effects observed do not appear to be dependent upon the 48 hour incubation period or the presence of fetal bovine serum, [SEP] cis - c18 : 4 omega 3 showed an initial cytotoxic effect at fatty acid concentrations of 0 - 20 micrograms / ml, [SEP]', '[CLS] the effects of the 18 carbon chain length series of fatty acids on the viability of a 3t6 derived mouse fibroblast cell line were tested. [SEP] the effects of the 18 carbon chain length series of fatty acids on the viability of a 3t6 derived mouse fibroblast cell line were tested. [SEP]', '[CLS] an overall concentration dependent cytotoxic effect was demonstrated with the saturated fatty acid, [SEP] the effects of the 18 carbon chain length series of fatty acids on the viability of a 3t6 derived mouse fibroblast cell line were tested. [SEP]', '[CLS] and cis - and trans - mono - and dienoic fatty acids ; [SEP] an overall concentration dependent cytotoxic effect was demonstrated with the saturated fatty acid, [SEP]', '[CLS] however, within a narrow range of concentrations, [SEP] and cis - and trans - mono - and dienoic fatty acids ; [SEP]', '[CLS] as a result of the increase in cell viability over this concentration range, [SEP] however, within a narrow range of concentrations, [SEP]', '[CLS] the most dramatic cytotoxic effects were shown by cis - c18 : 3 omega 3 and cis - c18 : 3 [SEP] as a result of the increase in cell viability over this concentration range, [SEP]', '[CLS] omega 6 with id50 values of + / - 30 micrograms / ml. [SEP] the most dramatic cytotoxic effects were shown by cis - c18 : 3 omega 3 and cis - c18 : 3 [SEP]', '[CLS] cis - c18 : 4 omega 3 showed an initial cytotoxic effect at fatty acid concentrations of 0 - 20 micrograms / ml, [SEP] omega 6 with id50 values of + / - 30 micrograms / ml. [SEP]', '[CLS] furthermore, the effects observed do not appear to be dependent upon the 48 hour incubation period or the presence of fetal bovine serum, [SEP] cis - c18 : 4 omega 3 showed an initial cytotoxic effect at fatty acid concentrations of 0 - 20 micrograms / ml, [SEP]', '[CLS] data obtained during post - surgical monitoring [SEP] data obtained during post - surgical monitoring [SEP]', '[CLS] ( 2 to 28 months ) [SEP] data obtained during post - surgical monitoring [SEP]', '[CLS] it was observed that : [SEP] ( 2 to 28 months ) [SEP]', '[CLS] whatever the grade of malignancy, [SEP] it was observed that : [SEP]', '[CLS] 100 % of the patients whose tumor recurred showed, [SEP] whatever the grade of malignancy, [SEP]', '[CLS] on at least one occasion, [SEP] 100 % of the patients whose tumor recurred showed, [SEP]', '[CLS] abnormal red blood cell [SEP] on at least one occasion, [SEP]', '[CLS] spermidine [SEP] abnormal red blood cell [SEP]', '[CLS] ( greater than 14 nmoles / 8. 10 ( 9 ) rbc ) [SEP] spermidine [SEP]', '[CLS] or spermine ( spm ) [SEP] ( greater than 14 nmoles / 8. 10 ( 9 ) rbc ) [SEP]', '[CLS] 30 % of the patients whose tumor recurred had abnormal rbc polyamine concentrations, [SEP] or spermine ( spm ) [SEP]', '[CLS] in patients with normal rbc polyamine values, [SEP] 30 % of the patients whose tumor recurred had abnormal rbc polyamine concentrations, [SEP]', '[CLS] during individual follow - up, [SEP] in patients with normal rbc polyamine values, [SEP]', '[CLS] rbc spd levels of each patient were generally significantly correlated to those of spm, [SEP] during individual follow - up, [SEP]', '[CLS] in such conditions we established the regression equation of the group of all recurring patients taken together. [SEP] rbc spd levels of each patient were generally significantly correlated to those of spm, [SEP]', '[CLS] we propose a graphic model, [SEP] in such conditions we established the regression equation of the group of all recurring patients taken together. [SEP]', '[CLS] including a space corresponding to rbc spd and spm levels observed in case of tumor recurrence, [SEP] we propose a graphic model, [SEP]', '[CLS] data obtained during post - surgical monitoring [SEP] data obtained during post - surgical monitoring [SEP]', '[CLS] ( 2 to 28 months ) [SEP] data obtained during post - surgical monitoring [SEP]', '[CLS] it was observed that : [SEP] ( 2 to 28 months ) [SEP]', '[CLS] whatever the grade of malignancy, [SEP] it was observed that : [SEP]', '[CLS] 100 % of the patients whose tumor recurred showed, [SEP] whatever the grade of malignancy, [SEP]', '[CLS] on at least one occasion, [SEP] 100 % of the patients whose tumor recurred showed, [SEP]', '[CLS] abnormal red blood cell [SEP] on at least one occasion, [SEP]', '[CLS] spermidine [SEP] abnormal red blood cell [SEP]', '[CLS] ( greater than 14 nmoles / 8. 10 ( 9 ) rbc ) [SEP] spermidine [SEP]', '[CLS] or spermine ( spm ) [SEP] ( greater than 14 nmoles / 8. 10 ( 9 ) rbc ) [SEP]', '[CLS] 30 % of the patients whose tumor recurred had abnormal rbc polyamine concentrations, [SEP] or spermine ( spm ) [SEP]', '[CLS] in patients with normal rbc polyamine values, [SEP] 30 % of the patients whose tumor recurred had abnormal rbc polyamine concentrations, [SEP]', '[CLS] during individual follow - up, [SEP] in patients with normal rbc polyamine values, [SEP]', '[CLS] rbc spd levels of each patient were generally significantly correlated to those of spm, [SEP] during individual follow - up, [SEP]', '[CLS] in such conditions we established the regression equation of the group of all recurring patients taken together. [SEP] rbc spd levels of each patient were generally significantly correlated to those of spm, [SEP]', '[CLS] we propose a graphic model, [SEP] in such conditions we established the regression equation of the group of all recurring patients taken together. [SEP]', '[CLS] including a space corresponding to rbc spd and spm levels observed in case of tumor recurrence, [SEP] we propose a graphic model, [SEP]', '[CLS] gestodene, a new synthetic progestogen, binds to progesterone receptor [SEP] gestodene, a new synthetic progestogen, binds to progesterone receptor [SEP]', '[CLS] ( pr ) [SEP] gestodene, a new synthetic progestogen, binds to progesterone receptor [SEP]', '[CLS] endometrium and both normal [SEP] ( pr ) [SEP]', '[CLS] pr in malignant breast could not be quantified on account of the binding of gestodene to oestrogen receptor [SEP] endometrium and both normal [SEP]', '[CLS] sucrose density ultracentrifugation using 3h - oestradiol and 3h - gestodene in normal and malignant breast demonstrated that the gestodene - er complex in addition to exhibiting the 4s and 5s peaks showed an additional peak [SEP] pr in malignant breast could not be quantified on account of the binding of gestodene to oestrogen receptor [SEP]', '[CLS] dissociation kinetics of gestodene from er which was either heat - activated or molybdate - stabilised were comparable to the triphenylethylene class of antioestrogens in that the rate of dissociation, [SEP] sucrose density ultracentrifugation using 3h - oestradiol and 3h - gestodene in normal and malignant breast demonstrated that the gestodene - er complex in addition to exhibiting the 4s and 5s peaks showed an additional peak [SEP]', '[CLS] unlike that of oestradiol from er, [SEP] dissociation kinetics of gestodene from er which was either heat - activated or molybdate - stabilised were comparable to the triphenylethylene class of antioestrogens in that the rate of dissociation, [SEP]', '[CLS] the binding of er - gestodene to dna - cellulose was also investigated and was found to be approximately 30 % less than that of er - oestradiol. [SEP] unlike that of oestradiol from er, [SEP]', '[CLS] gestodene, a new synthetic progestogen, binds to progesterone receptor [SEP] gestodene, a new synthetic progestogen, binds to progesterone receptor [SEP]', '[CLS] ( pr ) [SEP] gestodene, a new synthetic progestogen, binds to progesterone receptor [SEP]', '[CLS] endometrium and both normal [SEP] ( pr ) [SEP]', '[CLS] pr in malignant breast could not be quantified on account of the binding of gestodene to oestrogen receptor [SEP] endometrium and both normal [SEP]', '[CLS] sucrose density ultracentrifugation using 3h - oestradiol and 3h - gestodene in normal and malignant breast demonstrated that the gestodene - er complex in addition to exhibiting the 4s and 5s peaks showed an additional peak [SEP] pr in malignant breast could not be quantified on account of the binding of gestodene to oestrogen receptor [SEP]', '[CLS] dissociation kinetics of gestodene from er which was either heat - activated or molybdate - stabilised were comparable to the triphenylethylene class of antioestrogens in that the rate of dissociation, [SEP] sucrose density ultracentrifugation using 3h - oestradiol and 3h - gestodene in normal and malignant breast demonstrated that the gestodene - er complex in addition to exhibiting the 4s and 5s peaks showed an additional peak [SEP]', '[CLS] unlike that of oestradiol from er, [SEP] dissociation kinetics of gestodene from er which was either heat - activated or molybdate - stabilised were comparable to the triphenylethylene class of antioestrogens in that the rate of dissociation, [SEP]', '[CLS] the binding of er - gestodene to dna - cellulose was also investigated and was found to be approximately 30 % less than that of er - oestradiol. [SEP] unlike that of oestradiol from er, [SEP]', '[CLS] this study examines the suitability of the intracranial route of transplantation of human cell lines in nude mice for preclinical testing of cytostatic drugs. [SEP] this study examines the suitability of the intracranial route of transplantation of human cell lines in nude mice for preclinical testing of cytostatic drugs. [SEP]', '[CLS] animals in which intramuscularly or intracranially transplanted human tumours developed were treated with cytostatic drugs. [SEP] this study examines the suitability of the intracranial route of transplantation of human cell lines in nude mice for preclinical testing of cytostatic drugs. [SEP]', '[CLS] three different cell line generated tumours [SEP] animals in which intramuscularly or intracranially transplanted human tumours developed were treated with cytostatic drugs. [SEP]', '[CLS] ( t - all l735, hodgkin cell line l540 and burkitt lymphoma line bjab ) [SEP] three different cell line generated tumours [SEP]', '[CLS] were treated with four different drugs [SEP] ( t - all l735, hodgkin cell line l540 and burkitt lymphoma line bjab ) [SEP]', '[CLS] the data show a good correspondence between the results of treatment of intramuscularly and intracranially transplanted tumours. [SEP] were treated with four different drugs [SEP]', '[CLS] this study examines the suitability of the intracranial route of transplantation of human cell lines in nude mice for preclinical testing of cytostatic drugs. [SEP] this study examines the suitability of the intracranial route of transplantation of human cell lines in nude mice for preclinical testing of cytostatic drugs. [SEP]', '[CLS] animals in which intramuscularly or intracranially transplanted human tumours developed were treated with cytostatic drugs. [SEP] this study examines the suitability of the intracranial route of transplantation of human cell lines in nude mice for preclinical testing of cytostatic drugs. [SEP]', '[CLS] three different cell line generated tumours [SEP] animals in which intramuscularly or intracranially transplanted human tumours developed were treated with cytostatic drugs. [SEP]', '[CLS] ( t - all l735, hodgkin cell line l540 and burkitt lymphoma line bjab ) [SEP] three different cell line generated tumours [SEP]', '[CLS] were treated with four different drugs [SEP] ( t - all l735, hodgkin cell line l540 and burkitt lymphoma line bjab ) [SEP]', '[CLS] the data show a good correspondence between the results of treatment of intramuscularly and intracranially transplanted tumours. [SEP] were treated with four different drugs [SEP]', '[CLS] a method is described [SEP] a method is described [SEP]', '[CLS] that applies a low concentration of thymidine to achieve maximal hormonal synchronization in cultures of an established human breast cancer cell line, [SEP] a method is described [SEP]', '[CLS] the method consists of sequential treatments with tamoxifen, [SEP] that applies a low concentration of thymidine to achieve maximal hormonal synchronization in cultures of an established human breast cancer cell line, [SEP]', '[CLS] estradiol, [SEP] the method consists of sequential treatments with tamoxifen, [SEP]', '[CLS] the increase [SEP] estradiol, [SEP]', '[CLS] ( 3. 8 fold ) [SEP] the increase [SEP]', '[CLS] in the s - phase fraction achieved by this method is higher than that [SEP] ( 3. 8 fold ) [SEP]', '[CLS] ( 2. 6 fold ) [SEP] in the s - phase fraction achieved by this method is higher than that [SEP]', '[CLS] the mcf - 7 cells, [SEP] ( 2. 6 fold ) [SEP]', '[CLS] which have an enhanced labeling index [SEP] the mcf - 7 cells, [SEP]', '[CLS] ( s - phase fraction ) [SEP] which have an enhanced labeling index [SEP]', '[CLS] after being treated with the synchronizing method, [SEP] ( s - phase fraction ) [SEP]', '[CLS] have a markedly increased sensitivity to an s - phase - specific cytotoxic agent, [SEP] after being treated with the synchronizing method, [SEP]', '[CLS] this method may have potentially important implications for clinical chemotherapeutic strategy in the treatment of breast cancer. [SEP] have a markedly increased sensitivity to an s - phase - specific cytotoxic agent, [SEP]', '[CLS] a method is described [SEP] a method is described [SEP]', '[CLS] that applies a low concentration of thymidine to achieve maximal hormonal synchronization in cultures of an established human breast cancer cell line, [SEP] a method is described [SEP]', '[CLS] the method consists of sequential treatments with tamoxifen, [SEP] that applies a low concentration of thymidine to achieve maximal hormonal synchronization in cultures of an established human breast cancer cell line, [SEP]', '[CLS] estradiol, [SEP] the method consists of sequential treatments with tamoxifen, [SEP]', '[CLS] the increase [SEP] estradiol, [SEP]', '[CLS] ( 3. 8 fold ) [SEP] the increase [SEP]', '[CLS] in the s - phase fraction achieved by this method is higher than that [SEP] ( 3. 8 fold ) [SEP]', '[CLS] ( 2. 6 fold ) [SEP] in the s - phase fraction achieved by this method is higher than that [SEP]', '[CLS] the mcf - 7 cells, [SEP] ( 2. 6 fold ) [SEP]', '[CLS] which have an enhanced labeling index [SEP] the mcf - 7 cells, [SEP]', '[CLS] ( s - phase fraction ) [SEP] which have an enhanced labeling index [SEP]', '[CLS] after being treated with the synchronizing method, [SEP] ( s - phase fraction ) [SEP]', '[CLS] have a markedly increased sensitivity to an s - phase - specific cytotoxic agent, [SEP] after being treated with the synchronizing method, [SEP]', '[CLS] this method may have potentially important implications for clinical chemotherapeutic strategy in the treatment of breast cancer. [SEP] have a markedly increased sensitivity to an s - phase - specific cytotoxic agent, [SEP]', '[CLS] the glutathione transferase [SEP] the glutathione transferase [SEP]', '[CLS] ( ec 2. 5. 1. 18 ) [SEP] the glutathione transferase [SEP]', '[CLS] activity of multidrug - resistant [SEP] ( ec 2. 5. 1. 18 ) [SEP]', '[CLS] increase in activity was found in all multidrug - resistant sublines. [SEP] activity of multidrug - resistant [SEP]', '[CLS] electrophoretic separation of cytosolic proteins and immunodetection, [SEP] increase in activity was found in all multidrug - resistant sublines. [SEP]', '[CLS] using antisera against class alpha, [SEP] electrophoretic separation of cytosolic proteins and immunodetection, [SEP]', '[CLS] class mu [SEP] using antisera against class alpha, [SEP]', '[CLS] and class pi glutathione transferase, [SEP] class mu [SEP]', '[CLS] the role of glutathione transferase of class pi in the multidrug resistance [SEP] and class pi glutathione transferase, [SEP]', '[CLS] reversal of the resistance, [SEP] the role of glutathione transferase of class pi in the multidrug resistance [SEP]', '[CLS] using the calcium antagonist verapamil, [SEP] reversal of the resistance, [SEP]', '[CLS] the glutathione transferase [SEP] the glutathione transferase [SEP]', '[CLS] ( ec 2. 5. 1. 18 ) [SEP] the glutathione transferase [SEP]', '[CLS] activity of multidrug - resistant [SEP] ( ec 2. 5. 1. 18 ) [SEP]', '[CLS] increase in activity was found in all multidrug - resistant sublines. [SEP] activity of multidrug - resistant [SEP]', '[CLS] electrophoretic separation of cytosolic proteins and immunodetection, [SEP] increase in activity was found in all multidrug - resistant sublines. [SEP]', '[CLS] using antisera against class alpha, [SEP] electrophoretic separation of cytosolic proteins and immunodetection, [SEP]', '[CLS] class mu [SEP] using antisera against class alpha, [SEP]', '[CLS] and class pi glutathione transferase, [SEP] class mu [SEP]', '[CLS] the role of glutathione transferase of class pi in the multidrug resistance [SEP] and class pi glutathione transferase, [SEP]', '[CLS] reversal of the resistance, [SEP] the role of glutathione transferase of class pi in the multidrug resistance [SEP]', '[CLS] using the calcium antagonist verapamil, [SEP] reversal of the resistance, [SEP]', '[CLS] cytostatic treatment of liver metastases from colorectal cancer has been of limited value. [SEP] cytostatic treatment of liver metastases from colorectal cancer has been of limited value. [SEP]', '[CLS] it was the aim of this investigation to examine the effect of pala [SEP] cytostatic treatment of liver metastases from colorectal cancer has been of limited value. [SEP]', '[CLS] ( n - phosphonacetyl - l - aspartate ) [SEP] it was the aim of this investigation to examine the effect of pala [SEP]', '[CLS] and d - glucosamine on the level of uracil nucleotides in the liver [SEP] ( n - phosphonacetyl - l - aspartate ) [SEP]', '[CLS] and in an n - methyl - n - nitrosoguanidine - induced adenocarcinoma of the colon transplanted to the liver of rats, [SEP] and d - glucosamine on the level of uracil nucleotides in the liver [SEP]', '[CLS] and on the incorporation of 3h - furd into the acid - soluble fraction, [SEP] and in an n - methyl - n - nitrosoguanidine - induced adenocarcinoma of the colon transplanted to the liver of rats, [SEP]', '[CLS] combined treatment with pala and d - glucosamine reduced the utp pool in the liver and the tumor. [SEP] and on the incorporation of 3h - furd into the acid - soluble fraction, [SEP]', '[CLS] pretreatment with pala and d - glucosamine increased incorporation of 3h - furd into rna of the liver and kidney, [SEP] combined treatment with pala and d - glucosamine reduced the utp pool in the liver and the tumor. [SEP]', '[CLS] and into the dna fraction of the liver, [SEP] pretreatment with pala and d - glucosamine increased incorporation of 3h - furd into rna of the liver and kidney, [SEP]', '[CLS] cytostatic treatment of liver metastases from colorectal cancer has been of limited value. [SEP] cytostatic treatment of liver metastases from colorectal cancer has been of limited value. [SEP]', '[CLS] it was the aim of this investigation to examine the effect of pala [SEP] cytostatic treatment of liver metastases from colorectal cancer has been of limited value. [SEP]', '[CLS] ( n - phosphonacetyl - l - aspartate ) [SEP] it was the aim of this investigation to examine the effect of pala [SEP]', '[CLS] and d - glucosamine on the level of uracil nucleotides in the liver [SEP] ( n - phosphonacetyl - l - aspartate ) [SEP]', '[CLS] and in an n - methyl - n - nitrosoguanidine - induced adenocarcinoma of the colon transplanted to the liver of rats, [SEP] and d - glucosamine on the level of uracil nucleotides in the liver [SEP]', '[CLS] and on the incorporation of 3h - furd into the acid - soluble fraction, [SEP] and in an n - methyl - n - nitrosoguanidine - induced adenocarcinoma of the colon transplanted to the liver of rats, [SEP]', '[CLS] combined treatment with pala and d - glucosamine reduced the utp pool in the liver and the tumor. [SEP] and on the incorporation of 3h - furd into the acid - soluble fraction, [SEP]', '[CLS] pretreatment with pala and d - glucosamine increased incorporation of 3h - furd into rna of the liver and kidney, [SEP] combined treatment with pala and d - glucosamine reduced the utp pool in the liver and the tumor. [SEP]', '[CLS] and into the dna fraction of the liver, [SEP] pretreatment with pala and d - glucosamine increased incorporation of 3h - furd into rna of the liver and kidney, [SEP]', '[CLS] a 2, 5 - pyrrolidinedione linked nitrogen mustard derivative, [SEP] a 2, 5 - pyrrolidinedione linked nitrogen mustard derivative, [SEP]', \"[CLS] ( r, s ) 3 - [ n, n - bis ( 2 - chloroethyl ) ] - amino - 1 - ( 2'- methoxyphenyl ) pyrrolidine - 2, 5 - dione hydrochloride ( i ) [SEP] a 2, 5 - pyrrolidinedione linked nitrogen mustard derivative, [SEP]\", \"[CLS] since ( i ) is also active against l1210 and p388 leukaemias, [SEP] ( r, s ) 3 - [ n, n - bis ( 2 - chloroethyl ) ] - amino - 1 - ( 2'- methoxyphenyl ) pyrrolidine - 2, 5 - dione hydrochloride ( i ) [SEP]\", '[CLS] its toxicity in mice was evaluated. [SEP] since ( i ) is also active against l1210 and p388 leukaemias, [SEP]', \"[CLS] another derivative, ( r, s ) 3 - [ n, n - bis - ( 2 - chloroethyl ) ] amino - 1 - ( 4'- n - butoxyphenyl ) pyrrolidine - 2, 5 - dione dihydrate ( ii ), [SEP] its toxicity in mice was evaluated. [SEP]\", \"[CLS] which also possessed significant anticancer effect on p388 and l1210 leukaemias, [SEP] another derivative, ( r, s ) 3 - [ n, n - bis - ( 2 - chloroethyl ) ] amino - 1 - ( 4'- n - butoxyphenyl ) pyrrolidine - 2, 5 - dione dihydrate ( ii ), [SEP]\", '[CLS] a 2, 5 - pyrrolidinedione linked nitrogen mustard derivative, [SEP] a 2, 5 - pyrrolidinedione linked nitrogen mustard derivative, [SEP]', \"[CLS] ( r, s ) 3 - [ n, n - bis ( 2 - chloroethyl ) ] - amino - 1 - ( 2'- methoxyphenyl ) pyrrolidine - 2, 5 - dione hydrochloride ( i ) [SEP] a 2, 5 - pyrrolidinedione linked nitrogen mustard derivative, [SEP]\", \"[CLS] since ( i ) is also active against l1210 and p388 leukaemias, [SEP] ( r, s ) 3 - [ n, n - bis ( 2 - chloroethyl ) ] - amino - 1 - ( 2'- methoxyphenyl ) pyrrolidine - 2, 5 - dione hydrochloride ( i ) [SEP]\", '[CLS] its toxicity in mice was evaluated. [SEP] since ( i ) is also active against l1210 and p388 leukaemias, [SEP]', \"[CLS] another derivative, ( r, s ) 3 - [ n, n - bis - ( 2 - chloroethyl ) ] amino - 1 - ( 4'- n - butoxyphenyl ) pyrrolidine - 2, 5 - dione dihydrate ( ii ), [SEP] its toxicity in mice was evaluated. [SEP]\", \"[CLS] which also possessed significant anticancer effect on p388 and l1210 leukaemias, [SEP] another derivative, ( r, s ) 3 - [ n, n - bis - ( 2 - chloroethyl ) ] amino - 1 - ( 4'- n - butoxyphenyl ) pyrrolidine - 2, 5 - dione dihydrate ( ii ), [SEP]\", '[CLS] the fourth i. p. passage of the plasmacytoma [SEP] the fourth i. p. passage of the plasmacytoma [SEP]', '[CLS] \" pr \" [SEP] the fourth i. p. passage of the plasmacytoma [SEP]', '[CLS] the development of \" pr \" ascitic tumours was slower and the survival time of inoculated mice was longer than that of mopc - 315 inoculated mice. [SEP] \" pr \" [SEP]', '[CLS] moreover, [SEP] the development of \" pr \" ascitic tumours was slower and the survival time of inoculated mice was longer than that of mopc - 315 inoculated mice. [SEP]', '[CLS] the myeloma protein secreted by the \" pr \" tumour cells, [SEP] moreover, [SEP]', '[CLS] differed from mopc - 315 secreted myeloma protein in its electrophoretic mobility [SEP] the myeloma protein secreted by the \" pr \" tumour cells, [SEP]', '[CLS] ( fast gamma - 2 ) [SEP] differed from mopc - 315 secreted myeloma protein in its electrophoretic mobility [SEP]', '[CLS] chemotherapy by a single injection of melphalan 7. 5 mg / kg was effective in preventing the development of both mopc - 315 ascitic tumours and \" pr \" ascitic tumours. [SEP] ( fast gamma - 2 ) [SEP]', '[CLS] mice cured from mopc - 315 tumours, [SEP] chemotherapy by a single injection of melphalan 7. 5 mg / kg was effective in preventing the development of both mopc - 315 ascitic tumours and \" pr \" ascitic tumours. [SEP]', '[CLS] on the other hand, [SEP] mice cured from mopc - 315 tumours, [SEP]', '[CLS] mice cured from \" pr \" tumour by melphalan were highly susceptible to challenge [SEP] on the other hand, [SEP]', '[CLS] the fourth i. p. passage of the plasmacytoma [SEP] the fourth i. p. passage of the plasmacytoma [SEP]', '[CLS] \" pr \" [SEP] the fourth i. p. passage of the plasmacytoma [SEP]', '[CLS] the development of \" pr \" ascitic tumours was slower and the survival time of inoculated mice was longer than that of mopc - 315 inoculated mice. [SEP] \" pr \" [SEP]', '[CLS] moreover, [SEP] the development of \" pr \" ascitic tumours was slower and the survival time of inoculated mice was longer than that of mopc - 315 inoculated mice. [SEP]', '[CLS] the myeloma protein secreted by the \" pr \" tumour cells, [SEP] moreover, [SEP]', '[CLS] differed from mopc - 315 secreted myeloma protein in its electrophoretic mobility [SEP] the myeloma protein secreted by the \" pr \" tumour cells, [SEP]', '[CLS] ( fast gamma - 2 ) [SEP] differed from mopc - 315 secreted myeloma protein in its electrophoretic mobility [SEP]', '[CLS] chemotherapy by a single injection of melphalan 7. 5 mg / kg was effective in preventing the development of both mopc - 315 ascitic tumours and \" pr \" ascitic tumours. [SEP] ( fast gamma - 2 ) [SEP]', '[CLS] mice cured from mopc - 315 tumours, [SEP] chemotherapy by a single injection of melphalan 7. 5 mg / kg was effective in preventing the development of both mopc - 315 ascitic tumours and \" pr \" ascitic tumours. [SEP]', '[CLS] on the other hand, [SEP] mice cured from mopc - 315 tumours, [SEP]', '[CLS] mice cured from \" pr \" tumour by melphalan were highly susceptible to challenge [SEP] on the other hand, [SEP]', '[CLS] melphalan - induced dna cross - linking was compared in a human melanoma cell line [SEP] melphalan - induced dna cross - linking was compared in a human melanoma cell line [SEP]', '[CLS] ( rpmi 8322 ) [SEP] melphalan - induced dna cross - linking was compared in a human melanoma cell line [SEP]', '[CLS] and in phytohaemagglutinin [SEP] ( rpmi 8322 ) [SEP]', '[CLS] in both cell types a delayed induction of dna cross - links was observed, [SEP] and in phytohaemagglutinin [SEP]', '[CLS] significantly higher peak levels of dna cross - links were found in pha - stimulated lymphocytes, [SEP] in both cell types a delayed induction of dna cross - links was observed, [SEP]', '[CLS] the intracellular content of free melphalan was 1. 3 - fold higher in rpmi 8322 cells, [SEP] significantly higher peak levels of dna cross - links were found in pha - stimulated lymphocytes, [SEP]', '[CLS] rpmi 8322 cells had a 1. 8 - fold higher level of glutathione, [SEP] the intracellular content of free melphalan was 1. 3 - fold higher in rpmi 8322 cells, [SEP]', '[CLS] melphalan - induced dna cross - linking was compared in a human melanoma cell line [SEP] melphalan - induced dna cross - linking was compared in a human melanoma cell line [SEP]', '[CLS] ( rpmi 8322 ) [SEP] melphalan - induced dna cross - linking was compared in a human melanoma cell line [SEP]', '[CLS] and in phytohaemagglutinin [SEP] ( rpmi 8322 ) [SEP]', '[CLS] in both cell types a delayed induction of dna cross - links was observed, [SEP] and in phytohaemagglutinin [SEP]', '[CLS] significantly higher peak levels of dna cross - links were found in pha - stimulated lymphocytes, [SEP] in both cell types a delayed induction of dna cross - links was observed, [SEP]', '[CLS] the intracellular content of free melphalan was 1. 3 - fold higher in rpmi 8322 cells, [SEP] significantly higher peak levels of dna cross - links were found in pha - stimulated lymphocytes, [SEP]', '[CLS] rpmi 8322 cells had a 1. 8 - fold higher level of glutathione, [SEP] the intracellular content of free melphalan was 1. 3 - fold higher in rpmi 8322 cells, [SEP]', '[CLS] [ surgical treatment of functional tumors of the adrenal gland ] [SEP] [ surgical treatment of functional tumors of the adrenal gland ] [SEP]', '[CLS] - pmid : 3566191 [SEP] [ surgical treatment of functional tumors of the adrenal gland ] [SEP]', \"[CLS] the authors evaluated and operated 40 cases of cushing's syndrome, 8 cases of conn's syndrome and 19 cases of phaeochromocytoma. [SEP] - pmid : 3566191 [SEP]\", \"[CLS] cancer of the adrenal gland was observed in 17 % of cases of cushing's syndrome, [SEP] the authors evaluated and operated 40 cases of cushing's syndrome, 8 cases of conn's syndrome and 19 cases of phaeochromocytoma. [SEP]\", \"[CLS] there was a female predominance for both cushing's syndrome and conn's syndrome. [SEP] cancer of the adrenal gland was observed in 17 % of cases of cushing's syndrome, [SEP]\", \"[CLS] four malignant lesions were observed in the series of 19 tumours of the adrenal medulla [SEP] there was a female predominance for both cushing's syndrome and conn's syndrome. [SEP]\", '[CLS] ct scan was found to be the most accurate of modern diagnostic techniques [SEP] four malignant lesions were observed in the series of 19 tumours of the adrenal medulla [SEP]', '[CLS] and was able to define the site and the dimensions of the adrenal tumour. [SEP] ct scan was found to be the most accurate of modern diagnostic techniques [SEP]', '[CLS] arteriography was particularly useful in cases with large, [SEP] and was able to define the site and the dimensions of the adrenal tumour. [SEP]', '[CLS] highly vascular tumours. the preferred surgical approach was lumbotomy, [SEP] arteriography was particularly useful in cases with large, [SEP]', '[CLS] and, in cases of invasive tumour, [SEP] highly vascular tumours. the preferred surgical approach was lumbotomy, [SEP]', '[CLS] removal of the tumour by adrenalectomy led to resolution of the symptoms and, [SEP] and, in cases of invasive tumour, [SEP]', '[CLS] [ surgical treatment of functional tumors of the adrenal gland ] [SEP] [ surgical treatment of functional tumors of the adrenal gland ] [SEP]', '[CLS] - pmid : 3566191 [SEP] [ surgical treatment of functional tumors of the adrenal gland ] [SEP]', \"[CLS] the authors evaluated and operated 40 cases of cushing's syndrome, 8 cases of conn's syndrome and 19 cases of phaeochromocytoma. [SEP] - pmid : 3566191 [SEP]\", \"[CLS] cancer of the adrenal gland was observed in 17 % of cases of cushing's syndrome, [SEP] the authors evaluated and operated 40 cases of cushing's syndrome, 8 cases of conn's syndrome and 19 cases of phaeochromocytoma. [SEP]\", \"[CLS] there was a female predominance for both cushing's syndrome and conn's syndrome. [SEP] cancer of the adrenal gland was observed in 17 % of cases of cushing's syndrome, [SEP]\", \"[CLS] four malignant lesions were observed in the series of 19 tumours of the adrenal medulla [SEP] there was a female predominance for both cushing's syndrome and conn's syndrome. [SEP]\", '[CLS] ct scan was found to be the most accurate of modern diagnostic techniques [SEP] four malignant lesions were observed in the series of 19 tumours of the adrenal medulla [SEP]', '[CLS] and was able to define the site and the dimensions of the adrenal tumour. [SEP] ct scan was found to be the most accurate of modern diagnostic techniques [SEP]', '[CLS] arteriography was particularly useful in cases with large, [SEP] and was able to define the site and the dimensions of the adrenal tumour. [SEP]', '[CLS] highly vascular tumours. the preferred surgical approach was lumbotomy, [SEP] arteriography was particularly useful in cases with large, [SEP]', '[CLS] and, in cases of invasive tumour, [SEP] highly vascular tumours. the preferred surgical approach was lumbotomy, [SEP]', '[CLS] removal of the tumour by adrenalectomy led to resolution of the symptoms and, [SEP] and, in cases of invasive tumour, [SEP]', '[CLS] the authors report their experience of percutaneous nephrolithotomy over a period of three years. [SEP] the authors report their experience of percutaneous nephrolithotomy over a period of three years. [SEP]', '[CLS] with serious complications in 1 % of cases, [SEP] the authors report their experience of percutaneous nephrolithotomy over a period of three years. [SEP]', '[CLS] and an acceptable complete success rate of 83 %. [SEP] with serious complications in 1 % of cases, [SEP]', '[CLS] the indications for percutaneous nephrolithotomy have been modified since the opening of an extracorporeal lithotripsy unit. [SEP] and an acceptable complete success rate of 83 %. [SEP]', '[CLS] the volume, [SEP] the indications for percutaneous nephrolithotomy have been modified since the opening of an extracorporeal lithotripsy unit. [SEP]', '[CLS] chemical composition and site of the stone and the morphology of the upper urinary tract. [SEP] the volume, [SEP]', '[CLS] percutaneous nephrolithotomy is the treatment of choice for large stones [SEP] chemical composition and site of the stone and the morphology of the upper urinary tract. [SEP]', '[CLS] ( greater than 2. 5 cm ), [SEP] percutaneous nephrolithotomy is the treatment of choice for large stones [SEP]', '[CLS] hard stones or those formed proximally to a stenotic lesion of the urinary tract, [SEP] ( greater than 2. 5 cm ), [SEP]', '[CLS] which can then be dilated at the same procedure. [SEP] hard stones or those formed proximally to a stenotic lesion of the urinary tract, [SEP]', '[CLS] on the other hand, very large staghorn calculi with caliceal branches are best treated by open surgery. [SEP] which can then be dilated at the same procedure. [SEP]', '[CLS] the authors report their experience of percutaneous nephrolithotomy over a period of three years. [SEP] the authors report their experience of percutaneous nephrolithotomy over a period of three years. [SEP]', '[CLS] with serious complications in 1 % of cases, [SEP] the authors report their experience of percutaneous nephrolithotomy over a period of three years. [SEP]', '[CLS] and an acceptable complete success rate of 83 %. [SEP] with serious complications in 1 % of cases, [SEP]', '[CLS] the indications for percutaneous nephrolithotomy have been modified since the opening of an extracorporeal lithotripsy unit. [SEP] and an acceptable complete success rate of 83 %. [SEP]', '[CLS] the volume, [SEP] the indications for percutaneous nephrolithotomy have been modified since the opening of an extracorporeal lithotripsy unit. [SEP]', '[CLS] chemical composition and site of the stone and the morphology of the upper urinary tract. [SEP] the volume, [SEP]', '[CLS] percutaneous nephrolithotomy is the treatment of choice for large stones [SEP] chemical composition and site of the stone and the morphology of the upper urinary tract. [SEP]', '[CLS] ( greater than 2. 5 cm ), [SEP] percutaneous nephrolithotomy is the treatment of choice for large stones [SEP]', '[CLS] hard stones or those formed proximally to a stenotic lesion of the urinary tract, [SEP] ( greater than 2. 5 cm ), [SEP]', '[CLS] which can then be dilated at the same procedure. [SEP] hard stones or those formed proximally to a stenotic lesion of the urinary tract, [SEP]', '[CLS] on the other hand, very large staghorn calculi with caliceal branches are best treated by open surgery. [SEP] which can then be dilated at the same procedure. [SEP]', '[CLS] the authors describe their technique to preserve the erector nerves during a radical retropubic prostatectomy. [SEP] the authors describe their technique to preserve the erector nerves during a radical retropubic prostatectomy. [SEP]', \"[CLS] the 2nd part consists of discussion based on the author's experiences [SEP] the authors describe their technique to preserve the erector nerves during a radical retropubic prostatectomy. [SEP]\", \"[CLS] ( 20 cases of nerve preservation in a series of 115 radical prostatectomies ). the authors stress the insufficient oncological follow - up and the risk of an incorrect indication due to understaging of the cancer in pre - operative period. [SEP] the 2nd part consists of discussion based on the author's experiences [SEP]\", '[CLS] the authors describe their technique to preserve the erector nerves during a radical retropubic prostatectomy. [SEP] the authors describe their technique to preserve the erector nerves during a radical retropubic prostatectomy. [SEP]', \"[CLS] the 2nd part consists of discussion based on the author's experiences [SEP] the authors describe their technique to preserve the erector nerves during a radical retropubic prostatectomy. [SEP]\", \"[CLS] ( 20 cases of nerve preservation in a series of 115 radical prostatectomies ). the authors stress the insufficient oncological follow - up and the risk of an incorrect indication due to understaging of the cancer in pre - operative period. [SEP] the 2nd part consists of discussion based on the author's experiences [SEP]\", '[CLS] the authors report a case of differentiated myxoid liposarcoma with a survival of 27 years as a result of repeated resections. [SEP] the authors report a case of differentiated myxoid liposarcoma with a survival of 27 years as a result of repeated resections. [SEP]', '[CLS] well differentiated and / or myxoid forms of liposarcomas appear to have a more favourable long term prognosis. [SEP] the authors report a case of differentiated myxoid liposarcoma with a survival of 27 years as a result of repeated resections. [SEP]', '[CLS] this prognosis justifies repeated operations. [SEP] well differentiated and / or myxoid forms of liposarcomas appear to have a more favourable long term prognosis. [SEP]', '[CLS] the post - operative follow - up by computed tomography ensures earlier detection of recurrences. [SEP] this prognosis justifies repeated operations. [SEP]', '[CLS] the time of re - operation depends on the volume of the tumour, [SEP] the post - operative follow - up by computed tomography ensures earlier detection of recurrences. [SEP]', '[CLS] the rate of growth, [SEP] the time of re - operation depends on the volume of the tumour, [SEP]', '[CLS] the authors report a case of differentiated myxoid liposarcoma with a survival of 27 years as a result of repeated resections. [SEP] the authors report a case of differentiated myxoid liposarcoma with a survival of 27 years as a result of repeated resections. [SEP]', '[CLS] well differentiated and / or myxoid forms of liposarcomas appear to have a more favourable long term prognosis. [SEP] the authors report a case of differentiated myxoid liposarcoma with a survival of 27 years as a result of repeated resections. [SEP]', '[CLS] this prognosis justifies repeated operations. [SEP] well differentiated and / or myxoid forms of liposarcomas appear to have a more favourable long term prognosis. [SEP]', '[CLS] the post - operative follow - up by computed tomography ensures earlier detection of recurrences. [SEP] this prognosis justifies repeated operations. [SEP]', '[CLS] the time of re - operation depends on the volume of the tumour, [SEP] the post - operative follow - up by computed tomography ensures earlier detection of recurrences. [SEP]', '[CLS] the rate of growth, [SEP] the time of re - operation depends on the volume of the tumour, [SEP]', '[CLS] the author reports a case of rare malformation affecting all of the elements derived from the wolffian duct, [SEP] the author reports a case of rare malformation affecting all of the elements derived from the wolffian duct, [SEP]', '[CLS] the clinical picture, [SEP] the author reports a case of rare malformation affecting all of the elements derived from the wolffian duct, [SEP]', '[CLS] as in the majority of previously published cases, [SEP] the clinical picture, [SEP]', '[CLS] was initially suggestive of acute prostatitis, [SEP] as in the majority of previously published cases, [SEP]', '[CLS] but clinical, [SEP] was initially suggestive of acute prostatitis, [SEP]', '[CLS] the treatment is surgical and most authors prefer an intra - peritoneal transvesical approach. [SEP] but clinical, [SEP]', '[CLS] the author reports a case of rare malformation affecting all of the elements derived from the wolffian duct, [SEP] the author reports a case of rare malformation affecting all of the elements derived from the wolffian duct, [SEP]', '[CLS] the clinical picture, [SEP] the author reports a case of rare malformation affecting all of the elements derived from the wolffian duct, [SEP]', '[CLS] as in the majority of previously published cases, [SEP] the clinical picture, [SEP]', '[CLS] was initially suggestive of acute prostatitis, [SEP] as in the majority of previously published cases, [SEP]', '[CLS] but clinical, [SEP] was initially suggestive of acute prostatitis, [SEP]', '[CLS] the treatment is surgical and most authors prefer an intra - peritoneal transvesical approach. [SEP] but clinical, [SEP]', '[CLS] sub - trigonal phenol injection ensures a satisfactory response in about 50 % of patients with detrusor instability. [SEP] sub - trigonal phenol injection ensures a satisfactory response in about 50 % of patients with detrusor instability. [SEP]', '[CLS] this minimally invasive technique has been found, [SEP] sub - trigonal phenol injection ensures a satisfactory response in about 50 % of patients with detrusor instability. [SEP]', '[CLS] in our series of 11 patients, [SEP] this minimally invasive technique has been found, [SEP]', '[CLS] at least temporarily. [SEP] in our series of 11 patients, [SEP]', '[CLS] sub - trigonal phenol injection ensures a satisfactory response in about 50 % of patients with detrusor instability. [SEP] sub - trigonal phenol injection ensures a satisfactory response in about 50 % of patients with detrusor instability. [SEP]', '[CLS] this minimally invasive technique has been found, [SEP] sub - trigonal phenol injection ensures a satisfactory response in about 50 % of patients with detrusor instability. [SEP]', '[CLS] in our series of 11 patients, [SEP] this minimally invasive technique has been found, [SEP]', '[CLS] at least temporarily. [SEP] in our series of 11 patients, [SEP]', '[CLS] the authors report their experience of retropubic radical prostatectomy for prostatic cancer. [SEP] the authors report their experience of retropubic radical prostatectomy for prostatic cancer. [SEP]', '[CLS] their indications are essentially stage b [SEP] the authors report their experience of retropubic radical prostatectomy for prostatic cancer. [SEP]', '[CLS] nd stage c1 [SEP] their indications are essentially stage b [SEP]', '[CLS] the post - operative morbidity was low and no cases of incontinence were observed. [SEP] nd stage c1 [SEP]', \"[CLS] however, erection was preserved in two cases when walsh's technique was used. [SEP] the post - operative morbidity was low and no cases of incontinence were observed. [SEP]\", '[CLS] the authors report their experience of retropubic radical prostatectomy for prostatic cancer. [SEP] the authors report their experience of retropubic radical prostatectomy for prostatic cancer. [SEP]', '[CLS] their indications are essentially stage b [SEP] the authors report their experience of retropubic radical prostatectomy for prostatic cancer. [SEP]', '[CLS] nd stage c1 [SEP] their indications are essentially stage b [SEP]', '[CLS] the post - operative morbidity was low and no cases of incontinence were observed. [SEP] nd stage c1 [SEP]', \"[CLS] however, erection was preserved in two cases when walsh's technique was used. [SEP] the post - operative morbidity was low and no cases of incontinence were observed. [SEP]\", '[CLS] two populations of patients with invasive epithelial bladder carcinoma are considered. [SEP] two populations of patients with invasive epithelial bladder carcinoma are considered. [SEP]', '[CLS] received adjuvant postoperative chemotherapy with methotrexate, [SEP] two populations of patients with invasive epithelial bladder carcinoma are considered. [SEP]', '[CLS] adriamycin, [SEP] received adjuvant postoperative chemotherapy with methotrexate, [SEP]', '[CLS] 5 fluoro - uracil and cisplatinum. [SEP] adriamycin, [SEP]', '[CLS] group b includes 24 patients treated in the same institution but without chemotherapy. [SEP] 5 fluoro - uracil and cisplatinum. [SEP]', '[CLS] in group a, [SEP] group b includes 24 patients treated in the same institution but without chemotherapy. [SEP]', '[CLS] one death was not related to disease progression. [SEP] in group a, [SEP]', '[CLS] the 3 - year - disease - free survival rate is 67. 7 %. [SEP] one death was not related to disease progression. [SEP]', '[CLS] acceptability and tolerance of chemotherapy are satisfactory. [SEP] the 3 - year - disease - free survival rate is 67. 7 %. [SEP]', '[CLS] in group b, [SEP] acceptability and tolerance of chemotherapy are satisfactory. [SEP]', '[CLS] twenty patients died from distant metastases. [SEP] in group b, [SEP]', '[CLS] the 3 - year - disease - free survival rate is 18. 5 %. [SEP] twenty patients died from distant metastases. [SEP]', '[CLS] no treatment related deaths occurred in either group. [SEP] the 3 - year - disease - free survival rate is 18. 5 %. [SEP]', '[CLS] although biased selection in such conditions is inevitable, [SEP] no treatment related deaths occurred in either group. [SEP]', '[CLS] two populations of patients with invasive epithelial bladder carcinoma are considered. [SEP] two populations of patients with invasive epithelial bladder carcinoma are considered. [SEP]', '[CLS] received adjuvant postoperative chemotherapy with methotrexate, [SEP] two populations of patients with invasive epithelial bladder carcinoma are considered. [SEP]', '[CLS] adriamycin, [SEP] received adjuvant postoperative chemotherapy with methotrexate, [SEP]', '[CLS] 5 fluoro - uracil and cisplatinum. [SEP] adriamycin, [SEP]', '[CLS] group b includes 24 patients treated in the same institution but without chemotherapy. [SEP] 5 fluoro - uracil and cisplatinum. [SEP]', '[CLS] in group a, [SEP] group b includes 24 patients treated in the same institution but without chemotherapy. [SEP]', '[CLS] one death was not related to disease progression. [SEP] in group a, [SEP]', '[CLS] the 3 - year - disease - free survival rate is 67. 7 %. [SEP] one death was not related to disease progression. [SEP]', '[CLS] acceptability and tolerance of chemotherapy are satisfactory. [SEP] the 3 - year - disease - free survival rate is 67. 7 %. [SEP]', '[CLS] in group b, [SEP] acceptability and tolerance of chemotherapy are satisfactory. [SEP]', '[CLS] twenty patients died from distant metastases. [SEP] in group b, [SEP]', '[CLS] the 3 - year - disease - free survival rate is 18. 5 %. [SEP] twenty patients died from distant metastases. [SEP]', '[CLS] no treatment related deaths occurred in either group. [SEP] the 3 - year - disease - free survival rate is 18. 5 %. [SEP]', '[CLS] although biased selection in such conditions is inevitable, [SEP] no treatment related deaths occurred in either group. [SEP]', '[CLS] intraperitoneal infection of mice with m. anthritidis resulted in development of necrotic lesions in the omentum, [SEP] intraperitoneal infection of mice with m. anthritidis resulted in development of necrotic lesions in the omentum, [SEP]', '[CLS] the effect of 2, 4, 6 - trimethyloxypyrimidine on such affections was studied. [SEP] intraperitoneal infection of mice with m. anthritidis resulted in development of necrotic lesions in the omentum, [SEP]', '[CLS] intraperitoneal infection of mice with m. anthritidis resulted in development of necrotic lesions in the omentum, [SEP] intraperitoneal infection of mice with m. anthritidis resulted in development of necrotic lesions in the omentum, [SEP]', '[CLS] the effect of 2, 4, 6 - trimethyloxypyrimidine on such affections was studied. [SEP] intraperitoneal infection of mice with m. anthritidis resulted in development of necrotic lesions in the omentum, [SEP]', '[CLS] antimicrobial effect of lysozyme in combination with a wide set of antimicrobial drugs [SEP] antimicrobial effect of lysozyme in combination with a wide set of antimicrobial drugs [SEP]', '[CLS] was studied with respect to 74 bacterial cultures. [SEP] antimicrobial effect of lysozyme in combination with a wide set of antimicrobial drugs [SEP]', '[CLS] it was shown that synergism of the antimicrobial effect in the presence of lysozyme was variable for drugs differing in the mechanism of their action [SEP] was studied with respect to 74 bacterial cultures. [SEP]', '[CLS] and depended on the pathogen species. [SEP] it was shown that synergism of the antimicrobial effect in the presence of lysozyme was variable for drugs differing in the mechanism of their action [SEP]', '[CLS] ampiox, [SEP] and depended on the pathogen species. [SEP]', '[CLS] morphocycline, [SEP] ampiox, [SEP]', '[CLS] erythromycin and others. [SEP] morphocycline, [SEP]', '[CLS] the potentiation effect of lysozyme was less pronounced with respect to coli bacteria and pseudomonas. [SEP] erythromycin and others. [SEP]', '[CLS] tobramycin, [SEP] the potentiation effect of lysozyme was less pronounced with respect to coli bacteria and pseudomonas. [SEP]', '[CLS] sisomicin [SEP] tobramycin, [SEP]', '[CLS] and amikacin resulted in increasing antimicrobial effect with respect to practically all the microbial cultures tested. [SEP] sisomicin [SEP]', '[CLS] the clinical trials of the efficient combinations of the antibiotics [SEP] and amikacin resulted in increasing antimicrobial effect with respect to practically all the microbial cultures tested. [SEP]', '[CLS] thus, in children with acute pneumonia [SEP] the clinical trials of the efficient combinations of the antibiotics [SEP]', '[CLS] ( 92 observations ) [SEP] thus, in children with acute pneumonia [SEP]', '[CLS] it resulted in more rapid elimination of the temperature reaction, [SEP] ( 92 observations ) [SEP]', '[CLS] toxic and cardiorespiratiry syndromes, [SEP] it resulted in more rapid elimination of the temperature reaction, [SEP]', '[CLS] in treatment of 83 children with pyelonephritis complete clinico - laboratory remission was observed in 81 per cent of the cases against 56. 4 per cent in the patients treated with the antibiotics without lysozyme. [SEP] toxic and cardiorespiratiry syndromes, [SEP]', '[CLS] antimicrobial effect of lysozyme in combination with a wide set of antimicrobial drugs [SEP] antimicrobial effect of lysozyme in combination with a wide set of antimicrobial drugs [SEP]', '[CLS] was studied with respect to 74 bacterial cultures. [SEP] antimicrobial effect of lysozyme in combination with a wide set of antimicrobial drugs [SEP]', '[CLS] it was shown that synergism of the antimicrobial effect in the presence of lysozyme was variable for drugs differing in the mechanism of their action [SEP] was studied with respect to 74 bacterial cultures. [SEP]', '[CLS] and depended on the pathogen species. [SEP] it was shown that synergism of the antimicrobial effect in the presence of lysozyme was variable for drugs differing in the mechanism of their action [SEP]', '[CLS] ampiox, [SEP] and depended on the pathogen species. [SEP]', '[CLS] morphocycline, [SEP] ampiox, [SEP]', '[CLS] erythromycin and others. [SEP] morphocycline, [SEP]', '[CLS] the potentiation effect of lysozyme was less pronounced with respect to coli bacteria and pseudomonas. [SEP] erythromycin and others. [SEP]', '[CLS] tobramycin, [SEP] the potentiation effect of lysozyme was less pronounced with respect to coli bacteria and pseudomonas. [SEP]', '[CLS] sisomicin [SEP] tobramycin, [SEP]', '[CLS] and amikacin resulted in increasing antimicrobial effect with respect to practically all the microbial cultures tested. [SEP] sisomicin [SEP]', '[CLS] the clinical trials of the efficient combinations of the antibiotics [SEP] and amikacin resulted in increasing antimicrobial effect with respect to practically all the microbial cultures tested. [SEP]', '[CLS] thus, in children with acute pneumonia [SEP] the clinical trials of the efficient combinations of the antibiotics [SEP]', '[CLS] ( 92 observations ) [SEP] thus, in children with acute pneumonia [SEP]', '[CLS] it resulted in more rapid elimination of the temperature reaction, [SEP] ( 92 observations ) [SEP]', '[CLS] toxic and cardiorespiratiry syndromes, [SEP] it resulted in more rapid elimination of the temperature reaction, [SEP]', '[CLS] in treatment of 83 children with pyelonephritis complete clinico - laboratory remission was observed in 81 per cent of the cases against 56. 4 per cent in the patients treated with the antibiotics without lysozyme. [SEP] toxic and cardiorespiratiry syndromes, [SEP]', '[CLS] the etiologic profile and state of the immune system of the patients with different forms of sinusitis treated at the moscow research institute of the ear, [SEP] the etiologic profile and state of the immune system of the patients with different forms of sinusitis treated at the moscow research institute of the ear, [SEP]', '[CLS] the treatment of the patients with a combination of penicillin [SEP] the etiologic profile and state of the immune system of the patients with different forms of sinusitis treated at the moscow research institute of the ear, [SEP]', '[CLS] differences between the humoral factors of the nonspecific defence [SEP] the treatment of the patients with a combination of penicillin [SEP]', '[CLS] and between the indices of the cell reactivity of the patients before [SEP] differences between the humoral factors of the nonspecific defence [SEP]', '[CLS] the etiologic profile and state of the immune system of the patients with different forms of sinusitis treated at the moscow research institute of the ear, [SEP] the etiologic profile and state of the immune system of the patients with different forms of sinusitis treated at the moscow research institute of the ear, [SEP]', '[CLS] the treatment of the patients with a combination of penicillin [SEP] the etiologic profile and state of the immune system of the patients with different forms of sinusitis treated at the moscow research institute of the ear, [SEP]', '[CLS] differences between the humoral factors of the nonspecific defence [SEP] the treatment of the patients with a combination of penicillin [SEP]', '[CLS] and between the indices of the cell reactivity of the patients before [SEP] differences between the humoral factors of the nonspecific defence [SEP]']\n"
          ]
        }
      ],
      "source": [
        "def tokenize_function(example):\n",
        "    return scibert_tokenizer(example[\"#text\"], example[\"parent_text\"], truncation=True)\n",
        "\n",
        "#this is the sentence pair tokenization ^^ \n",
        "tokenized_datasets = raw_datasets.map(tokenize_function, batched=True)\n",
        "data_collator = DataCollatorWithPadding(tokenizer=scibert_tokenizer)\n",
        "\n",
        "#print(tokenized_datasets[\"input_ids\"])\n",
        "\n",
        "#input_ids = bio_tokenizer(sentence,truncation=True, add_special_tokens=True,max_length=512)['input_ids']\n",
        "\n",
        "wordpieces = [scibert_tokenizer.decode(input_id) for input_id in tokenized_datasets[\"input_ids\"]]\n",
        "print(wordpieces)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "yv4eKf8qsAJX"
      },
      "outputs": [],
      "source": [
        "tokenized_datasets.column_names\n",
        "tokenized_datasets.set_format(\"torch\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "Q7s2q0OZsC7N",
        "outputId": "10c24bef-0069-4de3-c8fc-3aebb145c3d3"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "Dataset({\n",
              "    features: ['labels', 'input_ids', 'token_type_ids', 'attention_mask'],\n",
              "    num_rows: 1844\n",
              "})"
            ]
          },
          "metadata": {},
          "execution_count": 13
        }
      ],
      "source": [
        "tokenized_datasets = tokenized_datasets.remove_columns([\"#text\", \"parent_text\", \"__index_level_0__\"])\n",
        "tokenized_datasets = tokenized_datasets.rename_column(\"int_relation\", \"labels\")\n",
        "tokenized_datasets.set_format(\"torch\")\n",
        "tokenized_datasets.column_names\n",
        "tokenized_datasets\n"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "CJlv2b_mTyKh",
        "outputId": "07e1d104-4f00-4c3b-ca4f-bc5784964a84"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "Dataset({\n",
              "    features: ['labels', 'input_ids', 'token_type_ids', 'attention_mask'],\n",
              "    num_rows: 1844\n",
              "})"
            ]
          },
          "metadata": {},
          "execution_count": 14
        }
      ],
      "source": [
        "tokenized_datasets"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "fTsVXgJ6sFhI"
      },
      "outputs": [],
      "source": [
        "tokenized_datasets = tokenized_datasets.train_test_split()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "YxSg7RfltLsv"
      },
      "outputs": [],
      "source": [
        "from torch.utils.data import DataLoader\n",
        "\n",
        "train_dataloader = DataLoader(\n",
        "    tokenized_datasets[\"train\"], shuffle=True, batch_size=64, collate_fn=data_collator\n",
        ")\n",
        "eval_dataloader = DataLoader(\n",
        "    tokenized_datasets[\"test\"], batch_size=64, collate_fn=data_collator\n",
        ")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "MsERQC86tRcy",
        "outputId": "218bd843-e38b-47e0-e37c-f6198e331fd6"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'labels': torch.Size([64]),\n",
              " 'input_ids': torch.Size([64, 50]),\n",
              " 'token_type_ids': torch.Size([64, 50]),\n",
              " 'attention_mask': torch.Size([64, 50])}"
            ]
          },
          "metadata": {},
          "execution_count": 17
        }
      ],
      "source": [
        "for batch in train_dataloader:\n",
        "    break\n",
        "{k: v.shape for k, v in batch.items()}"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "UPXezdx795w5",
        "outputId": "b3f93dff-49b7-4327-bac2-6242d473426c"
      },
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "<torch.utils.data.dataloader.DataLoader at 0x7f5d724d1d90>"
            ]
          },
          "metadata": {},
          "execution_count": 25
        }
      ],
      "source": [
        "train_dataloader"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "qxCV4lBBuvDO"
      },
      "outputs": [],
      "source": [
        "from transformers import AutoConfig\n",
        "config = AutoConfig #.from_pretrained(\"bert-base-cased\")\n",
        "config.classifier_dropout = 0.3\n",
        "\n",
        "config._num_labels=22\n",
        "config.num_hidden_layers=24\n",
        "config._name_or_path = \"allenai/scibert-base-uncased\"\n",
        "config.vocab_size = 30000"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 158,
          "referenced_widgets": [
            "dce837e4859c41a083dd63071a68c073",
            "e3d670b58d9f4971baa3e0e8a4a83ab1",
            "aab3732ea3754f8c8b1872c40b74f817",
            "c11754333c1a4ac98089ac32ff85a7f4",
            "8c97dd8da81e40d7b74d9995b75cd541",
            "f88b657362a94062b9ea95a5a9f96b67",
            "9aa537aab5af40359aeae44df5de79ca",
            "6c9ed62bdd5247abb9d16d91297558cc",
            "20737572eea442319ea8f8c4fea0efa9",
            "a77993573a5648899dc41c94f2854024",
            "f35f5710f7a04b92a3ec324ed53803ff"
          ]
        },
        "id": "2plrC1uEuqTH",
        "outputId": "db1a91e7-114c-4d09-f250-4786fd417a59"
      },
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Downloading:   0%|          | 0.00/442M [00:00<?, ?B/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "dce837e4859c41a083dd63071a68c073"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stderr",
          "text": [
            "Some weights of the model checkpoint at allenai/scibert_scivocab_uncased were not used when initializing BertForSequenceClassification: ['cls.predictions.bias', 'cls.predictions.transform.dense.weight', 'cls.predictions.transform.dense.bias', 'cls.predictions.transform.LayerNorm.weight', 'cls.predictions.transform.LayerNorm.bias', 'cls.predictions.decoder.weight', 'cls.predictions.decoder.bias', 'cls.seq_relationship.weight', 'cls.seq_relationship.bias']\n",
            "- This IS expected if you are initializing BertForSequenceClassification from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).\n",
            "- This IS NOT expected if you are initializing BertForSequenceClassification from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).\n",
            "Some weights of BertForSequenceClassification were not initialized from the model checkpoint at allenai/scibert_scivocab_uncased and are newly initialized: ['classifier.weight', 'classifier.bias']\n",
            "You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.\n"
          ]
        }
      ],
      "source": [
        "from transformers import AutoModelForSequenceClassification\n",
        "#mma_model = myModelAgain(\"allenai/scibert_scivocab_uncased\",config=confignum_classes=22)\n",
        "sci_bert_new_configs = AutoModelForSequenceClassification.from_pretrained(\"allenai/scibert_scivocab_uncased\",config=config)"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "Y15Ysc51yJtL"
      },
      "outputs": [],
      "source": [
        "from transformers import AutoModel"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "7EisTBy0yN6c"
      },
      "outputs": [],
      "source": [
        "device = torch.device(\"cuda\") if torch.cuda.is_available() else torch.device(\"cpu\")"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "T02d-Vrmx6oN"
      },
      "outputs": [],
      "source": [
        "\n",
        "import torch \n",
        "import torch.nn as nn\n",
        "from transformers import AutoModel\n",
        "\n",
        "#model = AutoModel.from_pretrained(\"allenai/scibert_scivocab_uncased\")\n",
        "\n",
        "\n",
        "#criterion = torch.nn.CrossEntropyLoss().to(device)\n",
        "\n",
        "class ScienceClassifier(nn.Module):\n",
        "  def __init__(self):\n",
        "    super(ScienceClassifier, self).__init__()\n",
        "    self.AutoModel = AutoModel.from_pretrained(\"allenai/scibert_scivocab_uncased\")\n",
        "    self.drop0 = nn.Dropout(0.25)\n",
        "    self.linear1 = nn.Linear(64*53, 512)\n",
        "    self.relu1 = nn.ReLU()\n",
        "    self.drop1 = nn.Dropout(0.25)\n",
        "    self.linear2 = nn.Linear(512, 22)\n",
        "    self.relu2 = nn.ReLU()\n",
        "\n",
        "  def forward(self, input_ids, attention_mask):\n",
        "    outputs = self.AutoModel(input_ids, attention_mask)\n",
        "    last_hidden_state = outputs[0]\n",
        "    pooled_output = torch.cat(tuple([last_hidden_state[:, i] for i in [-4, -3, -2, -1]]), dim=-1)\n",
        "    x = self.drop0(pooled_output)\n",
        "    x = self.relu1(self.linear1(x))\n",
        "    x = self.drop1(x)\n",
        "    x = self.relu2(self.linear2(x))\n",
        "    return x"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "2MWo9OLiVLeB"
      },
      "outputs": [],
      "source": [
        "#RuntimeError: size mismatch, m1: [16 x 3072], m2: [768 x 512] at /pytorch/aten/src/TH/generic/THTensorMath.cpp:41"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "P-rCE5y-R0Q5"
      },
      "outputs": [],
      "source": [
        "import torch.nn as nn\n",
        "class NewScienceClassifier(nn.Module):\n",
        "  def __init__(self):\n",
        "    super(NewScienceClassifier, self).__init__()\n",
        "    self.AutoModel = AutoModel.from_pretrained(\"allenai/scibert_scivocab_uncased\")\n",
        "    self.drop0 = nn.Dropout(0.25)\n",
        "    self.linear1 = nn.Linear(3072,512) #512 * 6 = 3072\n",
        "    \n",
        "    self.relu1 = nn.ReLU()\n",
        "    #self.drop1 = nn.Dropout(0.25)\n",
        "    self.drop1 = nn.Dropout(0.45)\n",
        "    self.linear2 = nn.Linear(512,22)\n",
        "    self.relu2 = nn.ReLU()\n",
        "\n",
        "\n",
        "  def forward(self, input_ids, attention_mask):\n",
        "    outputs = self.AutoModel(input_ids, attention_mask)\n",
        "    last_hidden_state = outputs[0]\n",
        "    pooled_output = torch.cat(tuple([last_hidden_state[:, i] for i in [-4, -3, -2, -1]]), dim=-1)\n",
        "    x = self.drop0(pooled_output)\n",
        "    x = self.relu1(self.linear1(x))\n",
        "    x = self.drop1(x)\n",
        "    x = self.relu2(self.linear2(x))\n",
        "    return x"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "zkd48PJ1y8WG"
      },
      "outputs": [],
      "source": [
        "model = ScienceClassifier()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "uaSUBaQbU9y4"
      },
      "outputs": [],
      "source": [
        "new_model = NewScienceClassifier()"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "seg-3tvXN7ii"
      },
      "outputs": [],
      "source": [
        "from tqdm.auto import tqdm \n",
        "from torch.optim import AdamW\n",
        "\n",
        "\n",
        "optim = AdamW(new_model.parameters(), lr=2e-5)\n",
        "criterion = nn.CrossEntropyLoss()\n",
        "\n",
        "def compute_accuracy(y_pred, y_target):\n",
        "    _, y_pred_indices = y_pred.max(dim=1)\n",
        "    n_correct = torch.eq(y_pred_indices, y_target).sum().item()\n",
        "    accuracy = (n_correct / len(y_pred_indices)) * 100\n",
        "    return accuracy"
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "gbcU_-jHajcm"
      },
      "outputs": [],
      "source": [
        "device = \"cuda\" if torch.cuda.is_available() else \"cpu\""
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "iRK529uNyy7m"
      },
      "outputs": [],
      "source": [
        "from tqdm.auto import tqdm \n",
        "from torch.optim import AdamW\n",
        "\n",
        "optim = AdamW(new_model.parameters(), lr=2e-5)\n",
        "criterion = nn.CrossEntropyLoss()\n",
        "num_training_steps = 60\n",
        "progress_bar = tqdm(range(num_training_steps))\n",
        "print(criterion)\n",
        "new_model.to(device)\n",
        "new_model.train()\n",
        "running_acc = 0.\n",
        "epoch = 1000\n",
        "for i in range(epoch):\n",
        "  for batch in train_dataloader:\n",
        "      optim.zero_grad()\n",
        "      #input_ids = input_ids.view(16,-1)\n",
        "      input_ids = batch['input_ids'].to(device)\n",
        "      #attention_mask = attention_mask.view(16,-1)\n",
        "      attention_mask = batch['attention_mask'].to(device)\n",
        "      \n",
        "      labels = batch['labels'].to(device)\n",
        "      #active_labels = labels.view(64, -1)\n",
        "      #print(labels)\n",
        "      #print(labels)\n",
        "      print(input_ids.shape)\n",
        "      outputs = new_model(input_ids, attention_mask)\n",
        "      print(outputs)\n",
        "      loss = criterion(outputs, labels) # ERROR\n",
        "      #print(loss)\n",
        "\n",
        "      loss.backward()\n",
        "      optim.step()\n",
        "      acc_t = compute_accuracy(outputs,labels)\n",
        "      print(\"acc_t: \",acc_t)\n",
        "      running_acc += (acc_t - running_acc) / (epoch + 1)\n",
        "      print(\"acc: \", running_acc)\n",
        "      #print(running_acc)\n",
        "      progress_bar.update(1)\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "from tqdm.auto import tqdm \n",
        "from torch.optim import AdamW\n",
        "\n",
        "optim = AdamW(new_model.parameters(), lr=2e-5)\n",
        "criterion = nn.CrossEntropyLoss()\n",
        "num_training_steps = 60\n",
        "progress_bar = tqdm(range(num_training_steps))\n",
        "print(criterion)\n",
        "new_model.to(device)\n",
        "new_model.train()\n",
        "running_acc = 0.\n",
        "epoch = 1000\n",
        "for i in range(epoch):\n",
        "  for batch in eval_dataloader:\n",
        "      optim.zero_grad()\n",
        "      #input_ids = input_ids.view(16,-1)\n",
        "      input_ids = batch['input_ids'].to(device)\n",
        "      #attention_mask = attention_mask.view(16,-1)\n",
        "      attention_mask = batch['attention_mask'].to(device)\n",
        "      \n",
        "      labels = batch['labels'].to(device)\n",
        "      #active_labels = labels.view(64, -1)\n",
        "      #print(labels)\n",
        "      #print(labels)\n",
        "      print(input_ids.shape)\n",
        "      outputs = new_model(input_ids, attention_mask)\n",
        "      print(outputs)\n",
        "      loss = criterion(outputs, labels) # ERROR\n",
        "      #print(loss)\n",
        "\n",
        "      loss.backward()\n",
        "      optim.step()\n",
        "      acc_t = compute_accuracy(outputs,labels)\n",
        "      print(\"acc_t: \",acc_t)\n",
        "      running_acc += (acc_t - running_acc) / (epoch + 1)\n",
        "      print(\"acc: \", running_acc)\n",
        "      #print(running_acc)\n",
        "      progress_bar.update(1)\n"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000,
          "referenced_widgets": [
            "156f3f87f3b247ecb7839a3123be81d0",
            "0a7bd295fd9342de983ceb6e35fb1658",
            "e70684f8312c49b3856ed0c76b30e9a8",
            "4772151442624801ac915791babda002",
            "5087b70e8f9840cf898a214f33c66da4",
            "30ba6122bd0c4d83aa1599ed3a499b58",
            "1b984b3ac62f4687b7540e750d966174",
            "c3caf825190e4931afa2375d9c587e5c",
            "fa88b46ec04d440cbd767f59ffd2e5a1",
            "9a33df6cc7e747cc86f297ef14416822",
            "440459c4178d4fada590659af5c73959"
          ]
        },
        "id": "nitjpEzg6If1",
        "outputId": "e13b271b-d466-4e7c-fa4f-33c7007b8f94"
      },
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "  0%|          | 0/60 [00:00<?, ?it/s]"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "156f3f87f3b247ecb7839a3123be81d0"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[1;30;43mStreaming output truncated to the last 5000 lines.\u001b[0m\n",
            "acc_t:  100.0\n",
            "acc:  99.74624070163985\n",
            "torch.Size([64, 48])\n",
            "tensor([[12.2105,  0.0000,  0.0000,  ...,  0.0803,  0.0000,  0.0000],\n",
            "        [ 1.6135,  0.0000,  0.0000,  ...,  0.0000,  0.7464,  0.0000],\n",
            "        [ 0.0000,  0.0000, 13.5383,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.8136,  0.0000,  1.4163,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 11.4077,  0.0000],\n",
            "        [ 0.6766,  0.0000,  0.0000,  ...,  0.0000,  0.0206,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74649420743242\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  2.8493,  ...,  0.0000,  0.0000, 13.5794],\n",
            "        [ 0.0000,  0.0000,  0.3249,  ...,  0.0000,  1.6899,  1.0990],\n",
            "        [ 0.0000,  0.0000,  0.6460,  ...,  0.0000,  0.0000, 13.2836],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.5501, 14.9105],\n",
            "        [ 0.0000,  0.0000,  0.2070,  ...,  0.0000,  0.0000, 14.3174],\n",
            "        [ 2.3360,  0.0000,  0.0000,  ...,  0.0000,  0.1886,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  96.875\n",
            "acc:  99.74362558185057\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.9522,  0.0000,  0.5347,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.7641,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.1556],\n",
            "        [ 0.3198,  0.0000,  1.2530,  ..., 11.4700,  0.0000,  1.0080],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.3503,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.0117, 14.2688]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74388170015041\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0247,  0.0000,  0.1887,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.2579,  0.0000,  0.2146,  ...,  0.9283,  0.0000,  2.0802],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0295,  ...,  2.0557,  0.0000,  2.7140],\n",
            "        [ 0.1923,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.9769],\n",
            "        [ 0.3856,  0.0000, 15.4426,  ...,  0.8260,  0.0000,  0.2083]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74413756258782\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.8632,  ...,  0.0000, 15.0537,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.6565, 13.3157],\n",
            "        [ 0.0000,  0.0000,  0.3966,  ...,  0.0000,  0.9148,  0.0000],\n",
            "        ...,\n",
            "        [ 0.6373,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.5705,  0.0000,  0.0000,  ...,  0.0000,  0.6280,  1.1481]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7443931694184\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 1.0671,  ..., 0.0000, 0.0000, 0.4255],\n",
            "        [3.2909, 0.0000, 0.0126,  ..., 0.3414, 0.0000, 1.4636],\n",
            "        [0.9824, 0.0000, 1.1062,  ..., 1.9660, 0.0000, 2.3127],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 1.1465,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.2345, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.7587]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7446485208975\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 8.2165e-04, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.6735e-02, 2.2273e-02, 1.6247e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6357e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 6.3850e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6356e+01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6167e-01, 1.6234e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 2.4932e-01, 4.7468e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 4.2066e-01, 2.1329e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.3415e+00, 0.0000e+00, 6.6686e-01, 1.5296e+01,\n",
            "         0.0000e+00, 8.0449e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.5929e-01,\n",
            "         0.0000e+00, 5.9083e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         7.8263e-01, 0.0000e+00, 5.1398e-01, 0.0000e+00],\n",
            "        [1.5545e+00, 0.0000e+00, 5.9391e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4393e+01, 0.0000e+00, 1.1143e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4401e+00, 0.0000e+00, 0.0000e+00, 9.0918e-02, 0.0000e+00, 0.0000e+00,\n",
            "         2.2377e+00, 1.5564e+00, 0.0000e+00, 7.0143e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.4275e-01, 0.0000e+00,\n",
            "         7.2466e-02, 0.0000e+00, 0.0000e+00, 1.4166e+00, 0.0000e+00, 1.4912e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.8045e-01,\n",
            "         1.5584e+01, 0.0000e+00, 6.6564e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.1536e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.6617e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.6310e+00, 1.1233e+00, 0.0000e+00, 0.0000e+00, 1.0499e+00,\n",
            "         5.5179e-02, 0.0000e+00, 0.0000e+00, 1.3763e+01],\n",
            "        [1.6955e+00, 0.0000e+00, 9.0580e-01, 6.7923e-01, 4.9867e-01, 0.0000e+00,\n",
            "         0.0000e+00, 1.9243e+00, 9.8760e-02, 1.3585e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.5213e-01, 0.0000e+00, 0.0000e+00, 1.8913e+00, 0.0000e+00, 0.0000e+00,\n",
            "         4.4022e-01, 0.0000e+00, 4.3966e-01, 0.0000e+00],\n",
            "        [1.4112e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.8098e-02, 0.0000e+00,\n",
            "         5.8973e-01, 2.3897e+00, 1.2721e+00, 1.0714e+00, 4.6910e-01, 1.3821e+00,\n",
            "         3.0850e-01, 0.0000e+00, 9.4759e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 8.9202e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 4.6966e-03, 0.0000e+00, 0.0000e+00, 1.3180e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.1468e-01, 4.0623e-01, 0.0000e+00, 0.0000e+00,\n",
            "         8.9067e-01, 0.0000e+00, 0.0000e+00, 1.7231e-01, 0.0000e+00, 2.2434e+00,\n",
            "         8.2086e-01, 0.0000e+00, 1.5079e+01, 1.4347e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.1603e+00, 0.0000e+00, 1.6424e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.4603e+00, 0.0000e+00, 0.0000e+00, 1.1239e+00, 0.0000e+00,\n",
            "         1.6361e+01, 0.0000e+00, 2.1222e-02, 0.0000e+00],\n",
            "        [2.4100e-02, 0.0000e+00, 0.0000e+00, 1.3107e+00, 0.0000e+00, 1.0475e+00,\n",
            "         2.8907e-01, 0.0000e+00, 1.3278e+00, 7.7898e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.6761e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.3256e-01,\n",
            "         1.6333e+01, 0.0000e+00, 3.1693e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.8424e-01, 0.0000e+00,\n",
            "         0.0000e+00, 1.3373e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 9.1515e-01, 5.4183e-01, 1.4254e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.4244e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [2.0664e+00, 0.0000e+00, 1.5398e+00, 0.0000e+00, 2.3704e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.3836e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 8.8362e-01, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74490361728022\n",
            "torch.Size([64, 57])\n",
            "tensor([[1.0946, 0.0000, 0.8619,  ..., 0.0000, 1.9830, 1.3137],\n",
            "        [1.0524, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 1.4014,  ..., 0.8063, 0.0000, 3.0100],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.3280],\n",
            "        [0.3249, 0.0000, 0.0000,  ..., 0.7401, 0.0000, 2.1721],\n",
            "        [1.3333, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7451584588214\n",
            "torch.Size([64, 48])\n",
            "tensor([[15.0405,  0.0000,  0.0000,  ...,  0.3944,  0.0000,  0.9968],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 14.1329,  ...,  0.8613,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.1330,  0.0000,  0.0000,  ...,  0.0000, 14.0894,  0.0000],\n",
            "        [ 2.7626,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74541304577562\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.0877,  ...,  0.0000,  0.0000, 13.7589],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0969,  0.1267],\n",
            "        [ 0.3420,  0.0000,  1.3197,  ...,  0.0000,  1.9952, 14.9280],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.3498,  ...,  0.0000,  0.2975, 14.7953],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 14.2230],\n",
            "        [ 1.2844,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.2827]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74566737839723\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  2.4252,  0.8584],\n",
            "        [ 1.9156,  0.0000,  0.0000,  ...,  0.2716,  0.0000,  3.1274],\n",
            "        [ 0.0000,  0.0000,  0.9627,  ...,  9.6093,  1.0371,  0.5317],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.1472,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.2368,  ...,  0.0000,  0.8122, 17.3318]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7459214569403\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.2632,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.3206,  0.0000,  2.2274,  ...,  0.0000,  0.4402,  0.0000],\n",
            "        [ 0.7051,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.3966],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.4370,  ...,  0.0000,  0.0000,  1.8990],\n",
            "        [ 0.7758,  0.0000,  0.3276,  ...,  0.0000,  0.0000,  0.6657],\n",
            "        [ 0.0000,  0.0000, 15.9553,  ...,  0.9381,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74617528165864\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.2189,  0.0000,  0.0000,  ...,  0.0000, 14.0766,  0.0000],\n",
            "        [ 0.8109,  0.0000,  1.3237,  ...,  0.0000,  1.1745, 14.3992],\n",
            "        [ 0.0638,  0.0000,  0.1140,  ...,  0.0000,  1.0147,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.5224,  ...,  0.0000,  0.3356,  0.6465],\n",
            "        [ 0.0000,  0.0000,  0.8075,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.9829,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74642885280583\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.7225,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [2.5780, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.0989],\n",
            "        [0.8004, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.8289],\n",
            "        ...,\n",
            "        [2.0206, 0.0000, 0.2051,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.7087]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7466821706352\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4856e+00, 0.0000e+00,\n",
            "         8.1707e-01, 0.0000e+00, 0.0000e+00, 2.2339e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4701e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 9.3532e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.4344e-01, 0.0000e+00, 1.5668e+01, 1.5923e+00,\n",
            "         0.0000e+00, 0.0000e+00, 8.4027e-01, 1.2929e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 4.3336e-01, 1.1410e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.4884e+00, 2.7731e+00],\n",
            "        [0.0000e+00, 2.2533e-01, 1.8420e+00, 0.0000e+00, 0.0000e+00, 1.2782e+01,\n",
            "         2.2389e-02, 1.1025e+00, 0.0000e+00, 9.3890e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.9463e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         5.0320e-01, 0.0000e+00, 0.0000e+00, 9.7296e-01],\n",
            "        [1.5948e+00, 0.0000e+00, 1.8851e+00, 0.0000e+00, 8.5241e-01, 0.0000e+00,\n",
            "         1.6534e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.0615e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4286e+00, 4.4367e-01, 0.0000e+00, 1.1618e+00],\n",
            "        [6.5794e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.5420e-01, 5.8430e-01,\n",
            "         7.1853e-01, 0.0000e+00, 7.2393e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.1877e+00, 0.0000e+00, 5.5535e-01, 0.0000e+00,\n",
            "         1.3550e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.3094e+00, 0.0000e+00, 0.0000e+00, 4.4275e-01, 0.0000e+00, 0.0000e+00,\n",
            "         4.0508e+00, 0.0000e+00, 8.9237e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4280e-02, 2.3162e+00, 3.2553e-01, 2.0615e-01, 0.0000e+00, 6.1815e-01,\n",
            "         1.1426e+00, 0.0000e+00, 3.9992e-01, 1.4532e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 3.6076e-01, 2.4640e-01, 0.0000e+00, 0.0000e+00,\n",
            "         2.1574e-01, 0.0000e+00, 0.0000e+00, 1.3472e+01, 0.0000e+00, 0.0000e+00,\n",
            "         5.4524e-01, 0.0000e+00, 0.0000e+00, 3.4598e-01, 9.5503e-01, 0.0000e+00,\n",
            "         1.1109e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.5577e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.3890e+00, 0.0000e+00,\n",
            "         1.6348e+00, 0.0000e+00, 1.6857e+00, 8.0459e-01, 0.0000e+00, 2.2993e+00,\n",
            "         0.0000e+00, 1.3469e-02, 3.5274e-01, 0.0000e+00, 0.0000e+00, 6.6465e-01,\n",
            "         0.0000e+00, 1.4118e+00, 0.0000e+00, 1.1472e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7270e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 4.6089e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.0230e-01,\n",
            "         0.0000e+00, 0.0000e+00, 1.4646e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3065e+00, 2.0801e+00,\n",
            "         1.7874e+00, 0.0000e+00, 7.7155e-01, 6.9673e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0219e+00, 0.0000e+00,\n",
            "         1.5104e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5820e+00, 0.0000e+00, 0.0000e+00, 7.9596e-01, 0.0000e+00, 0.0000e+00,\n",
            "         4.2241e-01, 7.2542e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6667e+00,\n",
            "         1.5038e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1624e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 4.4768e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.2483e-01, 0.0000e+00, 1.4957e+01, 0.0000e+00, 0.0000e+00,\n",
            "         9.6145e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [4.8755e-02, 0.0000e+00, 3.9426e-01, 0.0000e+00, 5.4120e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.7206e+01, 2.6820e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 6.9444e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 5.4688e-01, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7469352353998\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.4328,  ..., 0.0000, 0.0000, 0.7720],\n",
            "        [0.2306, 0.0000, 1.0521,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.1018, 1.5147],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.3522, 0.0000, 0.0000,  ..., 1.5863, 0.0000, 3.6182],\n",
            "        [0.0294, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.7143]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74718804735245\n",
            "torch.Size([64, 48])\n",
            "tensor([[15.3033,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.3391,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 13.1983,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.1366,  0.0000],\n",
            "        [ 2.0867,  0.0000,  0.9508,  ...,  0.0000,  1.9064,  0.1769]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74744060674571\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  0.2142,  ...,  1.0832,  0.0000, 13.1919],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.1690,  0.0000,  1.1283,  ...,  0.0000,  0.0000, 14.4364],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.3967,  ...,  0.0000,  0.3586, 14.3666],\n",
            "        [ 0.0000,  0.0000,  0.7702,  ...,  0.6476,  0.0000, 13.2617],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.6035]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.74613197477095\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.9527,  0.7354],\n",
            "        [ 1.3343,  0.0000,  0.0000,  ...,  1.4321,  0.0000,  0.4475],\n",
            "        [ 0.0000,  0.0000,  0.9094,  ..., 11.0514,  0.6907,  0.7895],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.4292,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.5176,  0.0000,  ...,  0.0000, 12.5956,  0.9756],\n",
            "        [ 0.0000,  0.0000,  0.8724,  ...,  0.0000,  0.0000, 15.8992]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74638558918177\n",
            "torch.Size([64, 59])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 5.0043e-01,  ..., 6.3990e-01, 8.0501e-03,\n",
            "         0.0000e+00],\n",
            "        [7.7164e-01, 0.0000e+00, 1.0863e+00,  ..., 7.4816e-01, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [8.8622e-01, 0.0000e+00, 0.0000e+00,  ..., 1.9377e+00, 0.0000e+00,\n",
            "         2.7186e+00],\n",
            "        ...,\n",
            "        [7.0960e-01, 0.0000e+00, 3.2299e-01,  ..., 1.0674e+00, 0.0000e+00,\n",
            "         5.5399e-01],\n",
            "        [1.6507e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.3872e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.3203e+01,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74663895023154\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.4387,  1.0806],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 14.2374],\n",
            "        [ 0.0000,  0.0000,  1.4922,  ...,  0.0000,  0.8533,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.4234,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.9189,  0.0000],\n",
            "        [ 1.0778,  0.0000,  0.0000,  ...,  0.0000,  0.8152,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74689205817337\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 1.9579,  ..., 0.0000, 0.7163, 0.0000],\n",
            "        [2.4280, 0.0000, 0.4873,  ..., 0.0000, 0.0000, 0.5011],\n",
            "        [0.3908, 0.0000, 0.0000,  ..., 0.2584, 0.0000, 1.0264],\n",
            "        ...,\n",
            "        [0.3796, 0.0000, 2.0797,  ..., 0.0000, 0.7045, 0.0000],\n",
            "        [0.6346, 0.0000, 0.1972,  ..., 0.5932, 0.0000, 0.0000],\n",
            "        [0.5886, 0.0000, 0.0000,  ..., 0.0000, 1.0032, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74714491326012\n",
            "torch.Size([13, 34])\n",
            "tensor([[ 0.9965,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.2522,  0.0000,\n",
            "          0.2791,  1.4304,  0.0000,  0.0000,  0.0000,  0.9337,  0.0000, 15.4995,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.2414],\n",
            "        [ 0.9083,  0.0000,  0.0000,  0.0000, 14.3757,  1.9596,  0.0000,  0.0826,\n",
            "          1.1219,  1.6695,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.8000,\n",
            "          0.0000,  0.0000,  0.0827,  0.0000,  0.6006,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.4721,  0.5374,  0.3969, 15.7185,  0.0000,  2.6349,\n",
            "          0.0000,  0.9276,  0.0000,  0.0000,  0.4713,  0.0000,  0.2834,  0.0000,\n",
            "          0.0000,  0.0000,  1.0191,  0.0000,  0.0000,  1.2200],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  1.0539,  0.0000, 15.8192,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  2.6560,  1.3622,  0.0000,  0.7248,\n",
            "          0.4399,  0.0000,  1.3806,  0.0000,  0.0000,  2.3296],\n",
            "        [ 2.1700,  0.0000,  0.0000,  0.0000,  1.6569,  0.0000,  0.5986,  0.0000,\n",
            "          1.1368,  1.0975,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000, 16.8084,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  3.8729,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.6162,  0.0000,  2.6757,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000, 16.3625],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.2118,  0.0000,  0.0000,  0.1497,\n",
            "          0.9944, 16.7143,  0.0000,  0.0000,  2.5218,  0.0000,  0.1628,  0.9312,\n",
            "          1.1444,  0.2154,  0.0000,  0.0000,  0.5855,  0.0000],\n",
            "        [13.0092,  0.0000,  0.4355,  0.0000,  1.0384,  0.0000,  0.0000,  0.0000,\n",
            "          0.2924,  0.0000,  0.0000,  0.0000,  1.1025,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  2.1800,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  1.5981,  0.8021,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  1.3266,  0.4365,  0.0487,  0.0000,  0.0000,\n",
            "          0.8652,  0.0000,  0.0000,  0.0000, 15.2880,  0.2943],\n",
            "        [ 0.2091,  0.0000,  0.0000,  0.0000,  4.0663,  0.0000,  0.8380,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.4626,\n",
            "          0.0000,  0.3082, 15.3527,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.8257,  0.0000,  0.0000,  0.0000,  1.4712,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.3376,\n",
            "          0.0877,  2.1216, 15.6208,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0494,  0.0000,  0.6112,  0.0000,  0.0000,  0.0000,\n",
            "          2.0143,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000, 13.5684,\n",
            "          2.0540,  0.0000,  0.5792,  0.0000,  0.6004,  0.0000],\n",
            "        [ 0.9707,  0.0000,  0.6623,  1.0787,  0.3375,  0.0000,  0.0000,  0.0000,\n",
            "         16.8754,  0.9745,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0388,  0.0000,  0.0000,  3.1094,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74739751574437\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 1.9924,  ..., 0.0000, 1.6217, 1.2315],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.7120, 0.8681],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.1055],\n",
            "        ...,\n",
            "        [0.9926, 0.0000, 0.0486,  ..., 0.0000, 0.2561, 0.3740],\n",
            "        [0.8953, 0.0000, 0.7969,  ..., 1.2120, 0.0000, 2.7505],\n",
            "        [0.0000, 0.0000, 0.3075,  ..., 0.0000, 0.0000, 1.2148]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74764986587849\n",
            "torch.Size([64, 48])\n",
            "tensor([[13.1018,  0.0000,  0.0969,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.4790,  0.3099,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 14.4679,  ...,  0.0000,  0.0000,  0.6838],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3468,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.7941,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74790196391457\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.8606,  ...,  0.0000,  0.0000, 15.6749],\n",
            "        [ 0.0000,  0.0000,  1.0648,  ...,  0.0000,  0.4155,  0.4517],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 14.0955],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 15.3993],\n",
            "        [ 0.1001,  0.0000,  2.1095,  ...,  0.0000,  0.0000, 14.3334],\n",
            "        [ 1.2525,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.74659287104353\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 1.9295,  0.0000,  0.0000,  ...,  0.0000,  1.0063,  0.6675],\n",
            "        [ 0.9400,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.6874],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ..., 12.6883,  0.2436,  1.0183],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.6353,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.7668,  0.0000,  0.2854,  ...,  0.0000, 14.4472,  0.0000],\n",
            "        [ 0.2288,  0.0000,  0.0000,  ...,  0.0000,  1.5950, 13.9994]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74684602501851\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  1.9230,  ...,  0.0000,  0.0597,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.8340,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  1.1998,  0.0000,  1.8365],\n",
            "        ...,\n",
            "        [ 0.2811,  0.0000,  0.5229,  ...,  1.2372,  0.0000,  2.3166],\n",
            "        [ 0.9151,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.3108],\n",
            "        [ 0.5808,  0.0000, 12.7453,  ...,  0.8997,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74709892609242\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.5259,  ...,  0.0000, 14.6076,  1.0820],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 13.1279],\n",
            "        [ 0.3190,  0.0000,  0.0000,  ...,  0.0000,  0.7506,  0.0000],\n",
            "        ...,\n",
            "        [ 2.4361,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.3827],\n",
            "        [ 0.5765,  0.0000,  0.9383,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 2.1050,  0.0000,  0.0000,  ...,  0.0000,  0.1203,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74735157451791\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 1.7487,  ..., 0.0000, 0.3963, 0.0000],\n",
            "        [1.9614, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 4.7957],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.9455],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.3135,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 1.4030,  ..., 0.0000, 0.4998, 0.0000],\n",
            "        [0.9414, 0.0000, 0.0000,  ..., 0.0000, 1.6946, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74760397054736\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 3.4648e-03, 7.7295e-01, 0.0000e+00, 0.0000e+00,\n",
            "         9.8431e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.9244e+00,\n",
            "         0.0000e+00, 6.8605e-01, 3.4252e-01, 1.2211e+01, 0.0000e+00, 0.0000e+00,\n",
            "         9.2272e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.3906e+00, 0.0000e+00, 1.6519e+01, 1.0006e+00,\n",
            "         0.0000e+00, 2.7186e-03, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.3070e+00, 1.7581e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.9409e-01, 4.3419e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 8.0336e-01, 0.0000e+00, 2.9333e-01, 1.4469e+01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 8.6809e-01, 0.0000e+00, 4.3318e-01,\n",
            "         0.0000e+00, 0.0000e+00, 1.5112e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         5.4236e-01, 0.0000e+00, 0.0000e+00, 1.1191e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2995e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.9811e+00, 1.9766e-01, 0.0000e+00, 1.9979e+00, 1.4386e+00, 0.0000e+00,\n",
            "         1.6586e+00, 1.8203e-01, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 3.3001e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.9430e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2339e+00, 1.4663e+00,\n",
            "         1.2736e+01, 2.1013e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.8136e-02, 1.1189e-01,\n",
            "         4.2388e+00, 0.0000e+00, 3.6157e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.4927e+00, 2.1966e-01, 1.1338e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5719e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 2.4348e+00, 0.0000e+00, 1.0933e+00,\n",
            "         0.0000e+00, 1.1143e+00, 3.6602e-01, 1.2613e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.9844e+00, 1.1423e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.3945e+01, 0.0000e+00, 1.6560e+00, 0.0000e+00, 5.9419e-01, 0.0000e+00,\n",
            "         6.1851e-01, 9.6201e-01, 1.2048e+00, 0.0000e+00, 0.0000e+00, 7.5359e-01,\n",
            "         4.9722e-01, 3.7200e-01, 0.0000e+00, 3.6691e-01, 0.0000e+00, 1.8647e+00,\n",
            "         2.1419e-01, 7.6556e-01, 0.0000e+00, 1.8627e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.1229e+00, 1.5266e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 8.9216e-01, 0.0000e+00, 1.7052e+00,\n",
            "         1.0210e+00, 0.0000e+00, 0.0000e+00, 1.3696e+00, 0.0000e+00, 1.6813e+00,\n",
            "         6.3902e-02, 0.0000e+00, 1.4967e+01, 2.6659e-02],\n",
            "        [0.0000e+00, 0.0000e+00, 1.8614e+00, 1.5760e-01, 0.0000e+00, 0.0000e+00,\n",
            "         3.1249e-01, 0.0000e+00, 0.0000e+00, 6.1672e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5474e+01, 0.0000e+00, 5.4246e-03, 0.0000e+00],\n",
            "        [1.4186e+00, 0.0000e+00, 8.8209e-02, 0.0000e+00, 0.0000e+00, 5.4900e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.7023e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.3298e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.9807e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.5862e-01,\n",
            "         0.0000e+00, 2.7286e-01, 0.0000e+00, 1.5841e+01, 8.5888e-01, 0.0000e+00,\n",
            "         2.5796e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.0726e+00, 0.0000e+00, 1.1335e+00, 5.5138e-01, 2.4447e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.3847e+01, 3.6273e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7170e-01,\n",
            "         1.1152e-01, 0.0000e+00, 9.1344e-01, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74785611443292\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 1.3044,  ..., 0.0000, 0.5569, 2.0098],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.2135, 0.0000, 0.2431,  ..., 0.1033, 0.0000, 1.5910],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.9916, 0.0000, 0.1025,  ..., 0.3995, 0.0000, 1.1503],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74810800642649\n",
            "torch.Size([64, 48])\n",
            "tensor([[13.6053,  0.0000,  0.8580,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.2827,  0.0000,  0.0000,  ...,  0.2662,  0.6091,  0.0000],\n",
            "        [ 0.2955,  0.0000, 14.8969,  ...,  1.4304,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.3506,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.2827,  0.1937],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.4359,  0.7687]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74835964677972\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  0.9785,  ...,  0.0000,  0.0000, 14.8528],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.5375,  0.0000],\n",
            "        [ 0.0675,  0.0000,  0.4246,  ...,  0.0000,  0.7477, 12.0226],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.0329,  ...,  0.0000,  0.0000, 15.0177],\n",
            "        [ 0.0000,  0.0000,  0.2702,  ...,  0.2483,  0.0000, 14.2391],\n",
            "        [ 2.1159,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.74705009668304\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.8658,  0.0000,  0.1826,  ...,  0.0000,  0.3759,  1.7590],\n",
            "        [ 1.2948,  0.0000,  0.0000,  ...,  0.3653,  0.0000,  2.6286],\n",
            "        [ 0.0000,  0.0000,  0.9349,  ..., 12.3539,  0.0000,  0.8247],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.2867,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.6759,  0.8394],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.0370, 14.8059]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74730279388915\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0332,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.4517],\n",
            "        [ 0.2837,  0.0000,  2.4866,  ...,  0.0000,  0.1545,  0.0000],\n",
            "        [ 0.7253,  0.0000,  0.0000,  ...,  0.7011,  0.0000,  2.4529],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.4749,  ...,  0.0000,  0.0000,  1.2373],\n",
            "        [ 1.3974,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.3947,  0.0000, 12.0436,  ...,  0.2760,  0.4530,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74755523865049\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.9043,  0.0000],\n",
            "        [ 0.0000,  0.0874,  0.0000,  ...,  0.0000,  0.0000, 12.9305],\n",
            "        [ 0.3144,  0.0000,  0.1612,  ...,  0.0000,  1.2864,  0.0000],\n",
            "        ...,\n",
            "        [ 1.1014,  0.0000,  0.0000,  ...,  0.0000,  0.7616,  1.1166],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.6510],\n",
            "        [ 1.0029,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74780743121927\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.4827,  ..., 0.0000, 1.0850, 0.4365],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.4055],\n",
            "        [0.7606, 0.0000, 0.0000,  ..., 0.4421, 0.0000, 3.7636],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.9825, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 3.4531, 1.1408]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74805937184742\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.7077e-01, 0.0000e+00,\n",
            "         1.3798e-01, 0.0000e+00, 2.7782e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5303e+01, 1.1853e+00, 0.0000e+00,\n",
            "         4.6659e-01, 0.0000e+00, 1.0105e-01, 1.6204e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7679e+01, 7.7733e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         9.3030e-01, 0.0000e+00, 8.0368e-01, 1.0540e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.9931e+00, 0.0000e+00, 6.2965e-01, 1.4128e+01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 9.8559e-01, 8.8442e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         9.6293e-01, 0.0000e+00, 0.0000e+00, 8.6403e-01],\n",
            "        [6.2163e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.3909e+01, 0.0000e+00, 5.8594e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2892e+00, 3.3274e-04, 1.0962e+00, 0.0000e+00, 1.0274e+00, 0.0000e+00,\n",
            "         1.9765e+00, 1.2006e-01, 0.0000e+00, 2.6064e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.2660e+00, 0.0000e+00, 1.0911e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.5889e-01, 0.0000e+00,\n",
            "         1.5496e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 3.1242e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.0263e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.0670e+00, 0.0000e+00, 0.0000e+00, 3.0232e-01, 0.0000e+00,\n",
            "         1.6369e+00, 0.0000e+00, 5.7847e-01, 1.6799e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3866e+00, 9.9021e-01, 9.9629e-01,\n",
            "         0.0000e+00, 1.1643e+00, 3.0462e+00, 1.5992e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.7991e+00, 0.0000e+00, 0.0000e+00, 1.9482e+00, 8.7537e-01, 4.7545e-01,\n",
            "         0.0000e+00, 0.0000e+00, 8.8973e-01, 0.0000e+00],\n",
            "        [1.6100e+01, 0.0000e+00, 1.5250e+00, 3.3055e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 5.2211e-01, 0.0000e+00, 7.0018e-01, 0.0000e+00, 2.0456e+00,\n",
            "         8.6338e-01, 0.0000e+00, 0.0000e+00, 3.8316e-01, 0.0000e+00, 1.7875e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.2547e-01, 2.6310e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4485e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4421e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6231e+00,\n",
            "         3.1943e-01, 0.0000e+00, 1.4868e+01, 3.8978e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.4092e+00, 0.0000e+00,\n",
            "         1.7634e+00, 0.0000e+00, 1.3682e+00, 0.0000e+00, 0.0000e+00, 3.5255e-01,\n",
            "         0.0000e+00, 4.1630e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.3397e-01,\n",
            "         1.5331e+01, 0.0000e+00, 1.1512e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 7.6620e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.2545e+00, 0.0000e+00, 0.0000e+00, 1.6806e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3670e+00,\n",
            "         1.5169e+01, 9.2726e-02, 5.3904e-02, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.4084e+00,\n",
            "         0.0000e+00, 2.0156e+00, 0.0000e+00, 1.2681e+01, 0.0000e+00, 1.3063e-01,\n",
            "         5.2426e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.5635e+00, 0.0000e+00, 1.6851e-01, 0.0000e+00, 1.6073e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.5473e+01, 6.3079e-01, 0.0000e+00, 6.5110e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 6.8268e-01, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74831106078663\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.1544, 2.1856],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 1.5698, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.9338],\n",
            "        ...,\n",
            "        [0.2021, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0736],\n",
            "        [1.0273, 0.4861, 0.0000,  ..., 0.0000, 0.0000, 2.1424],\n",
            "        [0.0000, 0.0000, 0.0528,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74856249828835\n",
            "torch.Size([64, 48])\n",
            "tensor([[14.3157,  0.0000,  0.0000,  ...,  1.0642,  0.0000,  0.0000],\n",
            "        [ 0.3699,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.1194],\n",
            "        [ 0.0000,  0.0000, 15.4040,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.3478,  0.0000,  0.0000,  ...,  0.0000,  0.7293,  0.5521],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.2778,  0.8141],\n",
            "        [ 0.7541,  0.0000,  0.0000,  ...,  0.0000,  0.8090,  0.3432]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74881368460375\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  3.4158,  ...,  0.0000,  0.0000, 15.3570],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.5585,  1.1329],\n",
            "        [ 1.8787,  0.0000,  0.0000,  ...,  0.0000,  1.4715, 14.1421],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.4191,  ...,  0.0000,  0.1465, 13.3067],\n",
            "        [ 0.0000,  0.0000,  1.1827,  ...,  0.0000,  0.9469, 15.6845],\n",
            "        [ 1.3967,  0.0000,  0.0000,  ...,  0.0000,  0.8287,  0.0466]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.74750368092282\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.1764,  0.0000,  0.0000,  ...,  0.0000,  0.1184,  0.0000],\n",
            "        [ 2.9547,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.9244],\n",
            "        [ 0.4106,  0.0000,  2.2868,  ..., 11.0440,  0.0000,  0.6417],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.7059,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.2664,  0.0000,  0.0000,  ...,  0.0000, 14.6963,  0.0000],\n",
            "        [ 0.0000,  0.4947,  0.0000,  ...,  0.0000,  0.6930, 14.3782]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74775592499782\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.4614],\n",
            "        [ 0.0000,  0.0000,  1.5386,  ...,  0.0000,  0.0000,  0.2532],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.3188,  0.0000,  0.8989],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  2.0120,  ...,  0.3103,  0.4513,  1.5779],\n",
            "        [ 1.4297,  0.4262,  0.0000,  ...,  0.0000,  0.0000,  2.3267],\n",
            "        [ 0.2826,  0.0000, 14.1058,  ...,  1.7316,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74800791708074\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.6582,  0.0000,  0.0000,  ...,  0.0000, 14.7465,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 15.6594],\n",
            "        [ 0.0000,  0.0000,  0.0277,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.6508,  0.0000,  0.0000,  ...,  0.0000,  1.5491,  0.8620],\n",
            "        [ 1.5998,  0.0000,  1.1974,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.1217,  1.2294,  0.3564]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74825965742332\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 1.7609,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [2.7329, 0.0000, 0.0000,  ..., 0.7510, 0.0000, 0.6686],\n",
            "        [1.3431, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.0683],\n",
            "        ...,\n",
            "        [0.6264, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.6854,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.6067, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74851114627704\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7066e-01, 0.0000e+00, 2.7075e-01,\n",
            "         0.0000e+00, 1.1092e-01, 0.0000e+00, 3.5423e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5160e+01, 0.0000e+00, 0.0000e+00,\n",
            "         7.4821e-01, 0.0000e+00, 0.0000e+00, 1.0141e-02],\n",
            "        [9.4763e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5760e+01, 1.3298e+00,\n",
            "         6.1582e-01, 0.0000e+00, 1.7234e+00, 9.5753e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 6.5653e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4231e+00, 0.0000e+00, 2.5663e+00, 9.0860e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 1.9879e+00, 0.0000e+00, 0.0000e+00, 1.4116e+01,\n",
            "         0.0000e+00, 4.9494e-01, 0.0000e+00, 9.5820e-02, 0.0000e+00, 1.2829e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.2927e+00, 1.1941e-01, 0.0000e+00, 0.0000e+00,\n",
            "         2.1667e+00, 0.0000e+00, 0.0000e+00, 1.6870e+00],\n",
            "        [1.2649e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6405e+00,\n",
            "         1.4569e+01, 0.0000e+00, 1.2192e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.8727e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4389e+00, 0.0000e+00, 0.0000e+00, 1.0732e+00],\n",
            "        [2.0637e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.8946e-01, 8.0194e-01,\n",
            "         3.2741e+00, 0.0000e+00, 9.2229e-01, 0.0000e+00, 0.0000e+00, 1.6166e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0229e-01,\n",
            "         1.7390e+01, 0.0000e+00, 5.1976e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.0741e-01,\n",
            "         1.6098e+00, 0.0000e+00, 9.9947e-01, 0.0000e+00, 0.0000e+00, 1.2721e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6392e-01,\n",
            "         1.8911e-01, 0.0000e+00, 7.3278e-01, 1.3700e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 3.3028e+00, 1.5284e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.1158e-01, 1.4289e+01, 0.0000e+00, 0.0000e+00,\n",
            "         3.9858e-01, 0.0000e+00, 0.0000e+00, 2.7698e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 4.4362e-01, 0.0000e+00],\n",
            "        [1.4407e+01, 0.0000e+00, 1.4008e+00, 9.8090e-02, 1.2974e+00, 0.0000e+00,\n",
            "         6.9005e-01, 2.0747e+00, 1.2233e+00, 9.0926e-01, 0.0000e+00, 3.3004e+00,\n",
            "         2.7764e-01, 8.8835e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         7.8451e-01, 2.1340e-02, 0.0000e+00, 0.0000e+00],\n",
            "        [3.6545e-01, 0.0000e+00, 1.5027e-01, 0.0000e+00, 3.3868e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.6673e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 3.7694e-01, 0.0000e+00, 1.4751e+00,\n",
            "         9.3768e-01, 0.0000e+00, 1.5743e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 6.2097e-01, 3.7663e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.4310e+00, 2.7575e-01, 0.0000e+00, 5.0925e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.9295e-01, 0.0000e+00,\n",
            "         1.4900e+01, 0.0000e+00, 7.3150e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.0150e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.1151e-01, 0.0000e+00, 3.3485e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2453e+00,\n",
            "         1.6582e+01, 0.0000e+00, 3.7024e-02, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 2.0225e-01, 1.3739e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0050e+00, 0.0000e+00, 4.1790e-01,\n",
            "         0.0000e+00, 9.1711e-01, 2.8453e-01, 1.5048e+01, 0.0000e+00, 0.0000e+00,\n",
            "         2.0600e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [2.0369e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.3840e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6450e+01, 2.9599e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.3945e-01, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74876238389315\n",
            "torch.Size([64, 57])\n",
            "tensor([[1.9801, 0.0000, 0.8372,  ..., 0.0000, 0.8845, 4.1930],\n",
            "        [0.6997, 0.0000, 0.4588,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.2898,  ..., 0.0000, 0.0000, 1.2627],\n",
            "        ...,\n",
            "        [0.4618, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.9150],\n",
            "        [0.7354, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.5577],\n",
            "        [0.0000, 0.0000, 0.6441,  ..., 0.0000, 0.0000, 0.3765]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74901337052263\n",
            "torch.Size([64, 48])\n",
            "tensor([[13.3318,  0.0000,  0.6726,  ...,  1.7606,  0.0000,  0.3511],\n",
            "        [ 0.9451,  0.0000,  0.0000,  ...,  0.0000,  0.2625,  0.0000],\n",
            "        [ 0.3071,  0.0000, 12.0807,  ...,  0.1518,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.9866,  0.0000,  0.8008,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.8959,  ...,  0.0000, 13.5333,  0.3388],\n",
            "        [ 0.8266,  0.0000,  0.0000,  ...,  0.0000,  0.4800,  1.0714]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74926410641622\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  3.2899,  ...,  0.0000,  0.0000, 14.4296],\n",
            "        [ 0.5346,  0.0000,  0.0000,  ...,  0.0000,  1.0597,  0.5282],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.1304, 15.0140],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.1475,  ...,  0.0000,  0.0000, 14.3017],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 14.6330],\n",
            "        [ 2.0562,  0.0000,  0.0000,  ...,  0.0000,  0.2676,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.74795365276346\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 1.5721,  0.0000,  0.0000,  ...,  0.0000,  2.1107,  0.0000],\n",
            "        [ 1.0887,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.4905],\n",
            "        [ 0.4000,  0.0000,  1.1440,  ..., 10.4480,  0.0000,  0.1370],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.0678,  ...,  0.0000,  0.0000,  0.9955],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.2588,  0.4662],\n",
            "        [ 0.1167,  0.0000,  2.1713,  ...,  0.0000,  0.0000, 14.7057]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74820544731614\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.6472,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.8150,  0.0000,  0.7421,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.9743,  0.0000,  0.0000,  ...,  0.7173,  0.1909,  2.4053],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.8434,  ...,  0.0000,  0.0000,  1.6721],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.0331],\n",
            "        [ 0.2120,  0.0000, 14.1796,  ...,  1.6272,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7484569903258\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  2.0125,  ...,  0.0000, 14.3406,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  2.5828, 15.2598],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.7798,  0.0000],\n",
            "        ...,\n",
            "        [ 2.5811,  0.0000,  0.0905,  ...,  0.0000,  0.3751,  1.0068],\n",
            "        [ 2.7411,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.9440,  0.0000,  0.0000,  ...,  0.0000,  0.7420,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74870828204376\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [3.8710, 0.0000, 0.0000,  ..., 2.5175, 0.0000, 3.5866],\n",
            "        [2.3111, 0.0000, 0.3205,  ..., 2.1365, 0.0000, 2.2918],\n",
            "        ...,\n",
            "        [0.2850, 0.0000, 0.6812,  ..., 0.0000, 0.0000, 0.2288],\n",
            "        [0.0000, 0.0000, 0.5508,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0130, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.7644]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74895932272105\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0325e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.6384e-01, 0.0000e+00, 0.0000e+00, 2.7982e-01,\n",
            "         0.0000e+00, 0.0000e+00, 1.0874e+00, 1.3239e+01, 7.7120e-01, 1.1921e-01,\n",
            "         0.0000e+00, 0.0000e+00, 3.3006e-01, 0.0000e+00],\n",
            "        [1.0071e+00, 0.0000e+00, 1.5063e+00, 0.0000e+00, 1.3869e+01, 2.0998e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 2.6874e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4851e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 2.0023e+00, 0.0000e+00, 0.0000e+00, 1.5439e+01,\n",
            "         0.0000e+00, 2.1762e+00, 0.0000e+00, 1.3637e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 7.1830e-01, 0.0000e+00, 0.0000e+00, 1.7077e-01,\n",
            "         5.4561e-01, 0.0000e+00, 1.3796e+00, 0.0000e+00],\n",
            "        [6.4871e-01, 0.0000e+00, 0.0000e+00, 7.8796e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.5439e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         6.1359e-01, 1.5572e+00, 8.1754e-01, 3.8132e-01, 7.0696e-01, 2.4379e-01,\n",
            "         1.8636e+00, 0.0000e+00, 0.0000e+00, 2.9082e+00],\n",
            "        [7.0996e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2196e-01, 0.0000e+00, 0.0000e+00, 2.0951e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3662e+00,\n",
            "         1.5488e+01, 0.0000e+00, 1.2075e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.0950e-01,\n",
            "         6.8467e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.7656e-01,\n",
            "         0.0000e+00, 1.6653e+00, 6.1218e-01, 0.0000e+00, 0.0000e+00, 2.2807e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4552e+01],\n",
            "        [7.8762e-02, 1.7570e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.7642e-01,\n",
            "         0.0000e+00, 5.4847e-02, 8.2375e-01, 1.2896e+01, 0.0000e+00, 0.0000e+00,\n",
            "         3.5875e-02, 0.0000e+00, 8.8383e-02, 9.6775e-02, 6.7285e-01, 1.6255e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.4566e+01, 0.0000e+00, 8.7843e-04, 0.0000e+00, 1.7927e-01, 0.0000e+00,\n",
            "         3.1307e+00, 0.0000e+00, 1.2320e-01, 0.0000e+00, 0.0000e+00, 1.7982e+00,\n",
            "         9.1643e-01, 2.3759e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.8210e+00, 6.5118e-01, 0.0000e+00, 1.7185e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 4.5696e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.9534e-01,\n",
            "         4.9904e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4339e+00,\n",
            "         2.9839e+00, 0.0000e+00, 1.4763e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 5.0744e-01, 0.0000e+00, 1.6490e+00, 0.0000e+00,\n",
            "         1.2586e+00, 0.0000e+00, 2.6309e+00, 0.0000e+00, 0.0000e+00, 1.0699e+00,\n",
            "         0.0000e+00, 1.0056e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.6181e+01, 0.0000e+00, 1.9668e+00, 9.4426e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         5.1607e-01, 0.0000e+00, 0.0000e+00, 1.0241e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.7607e-01, 1.5940e+00,\n",
            "         1.3992e+01, 0.0000e+00, 0.0000e+00, 1.0163e-01],\n",
            "        [3.8805e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 6.5600e-01, 1.0678e+00, 1.8307e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 7.2288e-01, 0.0000e+00, 1.2401e+01, 1.1726e-02, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.7996e-01, 0.0000e+00],\n",
            "        [5.0969e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.5254e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.5723e+01, 1.3589e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.0425e+00, 0.0000e+00, 1.5734e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74921011260844\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.8065,  ..., 0.0000, 0.2368, 2.1420],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.4729],\n",
            "        [0.8373, 0.0000, 0.5416,  ..., 0.0000, 0.0000, 2.4898],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 1.7543,  ..., 0.0000, 1.5705, 0.0000],\n",
            "        [2.2236, 0.0000, 0.3233,  ..., 0.1935, 0.0000, 2.1901],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74946065195648\n",
            "torch.Size([64, 48])\n",
            "tensor([[15.0111,  0.0000,  0.5484,  ...,  0.4994,  0.0587,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.7008,  0.0000, 12.7949,  ...,  1.4788,  0.0000,  1.2122],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.5296],\n",
            "        [ 0.0000,  0.0000,  0.1655,  ...,  0.0000, 14.0098,  0.1442],\n",
            "        [ 2.6033,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74971094101546\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.4845,  ...,  0.0000,  0.0000, 12.8159],\n",
            "        [ 0.4080,  0.0000,  0.0000,  ...,  0.0000,  1.8186,  0.0000],\n",
            "        [ 0.4022,  0.0000,  0.2259,  ...,  0.0000,  0.0000, 14.0749],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  3.2529,  ...,  0.0000,  0.0000, 15.7958],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 16.9269],\n",
            "        [ 0.4443,  0.0000,  0.0000,  ...,  0.0000,  0.1243,  1.8611]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.74840004097449\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.6169,  0.0000,  0.0000,  ...,  0.0000,  1.0277,  1.4922],\n",
            "        [ 1.6198,  0.0000,  0.0000,  ...,  0.4780,  0.0000,  1.7326],\n",
            "        [ 1.3882,  0.0000,  1.3009,  ..., 11.3704,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.9855,  0.3423],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.2257, 16.3500]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74865138958491\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 1.2828,  0.0000,  0.3820,  ...,  0.0000,  1.2465,  0.2863],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  2.1528,  0.0000],\n",
            "        [ 0.2764,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.2893],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.4550,  ...,  0.0000,  0.0000,  1.9331],\n",
            "        [ 1.0549,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.6994],\n",
            "        [ 0.0000,  0.0000, 13.0884,  ...,  0.0000,  1.0185,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74890248709781\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.5371,  ...,  0.0000, 16.0558,  0.4645],\n",
            "        [ 0.0000,  0.0000,  2.2443,  ...,  0.0311,  0.8203, 15.8091],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 2.1376,  0.0000,  0.4516,  ...,  0.0000,  1.5470,  0.0000],\n",
            "        [ 1.9334,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.8464],\n",
            "        [ 1.2125,  0.1706,  0.0000,  ...,  0.0000,  0.4087,  0.4215]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74915333376404\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.4359, 0.0000, 1.1068,  ..., 1.1240, 0.0000, 1.8079],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.4379, 0.0000, 1.5836],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.6231,  ..., 0.0177, 0.0000, 0.0000],\n",
            "        [0.0901, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.1142,  ..., 0.0000, 1.2093, 0.8751]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7494039298342\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5251e+00, 6.5794e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6523e+01, 1.5390e+00, 6.5024e-02,\n",
            "         0.0000e+00, 4.9861e-01, 0.0000e+00, 5.6573e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 1.5955e-01, 0.0000e+00, 1.5609e+01, 1.9702e-01,\n",
            "         5.7092e-01, 0.0000e+00, 1.6980e+00, 1.4308e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 8.5466e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.7770e+00, 0.0000e+00, 1.9423e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 2.1860e+00, 4.1329e-01, 9.9145e-02, 1.3051e+01,\n",
            "         0.0000e+00, 1.7857e+00, 0.0000e+00, 3.3398e-01, 0.0000e+00, 2.1835e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 8.6106e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [9.2783e-02, 0.0000e+00, 7.4607e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4474e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.4957e+00, 1.2868e+00, 3.3307e-01, 1.7634e+00, 1.2995e+00, 0.0000e+00,\n",
            "         1.0245e+00, 1.7487e+00, 0.0000e+00, 3.7655e+00],\n",
            "        [2.4089e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5435e-01, 0.0000e+00,\n",
            "         2.5017e+00, 0.0000e+00, 2.1740e+00, 3.8094e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 9.7142e-01, 0.0000e+00, 9.2715e-01, 1.4150e+00,\n",
            "         1.5574e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.6392e+00, 0.0000e+00, 1.2409e+00, 0.0000e+00, 0.0000e+00, 1.2433e-01,\n",
            "         4.3455e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2991e+00,\n",
            "         0.0000e+00, 2.6060e+00, 7.8890e-01, 0.0000e+00, 0.0000e+00, 4.8732e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7108e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 2.3033e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 5.7724e-01, 8.7025e-01, 1.7321e+01, 0.0000e+00, 0.0000e+00,\n",
            "         3.0968e-01, 0.0000e+00, 0.0000e+00, 2.3633e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.5827e+01, 0.0000e+00, 1.6012e+00, 0.0000e+00, 8.0062e-01, 0.0000e+00,\n",
            "         1.0293e+00, 0.0000e+00, 2.0544e+00, 5.2279e-01, 0.0000e+00, 3.0553e+00,\n",
            "         8.6313e-01, 6.5184e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.6631e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 4.1500e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 4.4112e-01, 6.8863e-01, 6.3375e-01,\n",
            "         0.0000e+00, 4.4190e-01, 1.1182e+00, 2.3006e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.7593e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.9614e-01, 0.0000e+00, 1.6504e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.5220e-02, 2.0656e-01, 1.4786e+00, 4.2296e-01,\n",
            "         1.1868e+00, 0.0000e+00, 0.0000e+00, 1.0123e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.8415e-01,\n",
            "         1.4443e+01, 0.0000e+00, 1.4713e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.8469e+00, 0.0000e+00, 0.0000e+00, 1.8801e+00,\n",
            "         1.9713e+00, 0.0000e+00, 4.3387e-01, 1.9571e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 7.1022e-01, 0.0000e+00, 0.0000e+00, 3.5832e-01, 5.7316e-01,\n",
            "         1.4355e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 7.2549e-02, 1.4502e+00, 0.0000e+00,\n",
            "         1.0364e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.9533e-01,\n",
            "         0.0000e+00, 6.4121e-01, 6.2770e-01, 1.5988e+01, 2.7916e-01, 0.0000e+00,\n",
            "         4.6429e-01, 3.7907e-02, 0.0000e+00, 6.4681e-01],\n",
            "        [1.0426e+00, 0.0000e+00, 0.0000e+00, 5.6696e-01, 3.8907e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.4583e+01, 1.1172e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 8.2714e-01, 0.0000e+00, 0.0000e+00, 1.8152e+00, 0.0000e+00,\n",
            "         1.3881e+00, 5.1175e-01, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74965427555864\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.1664, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.4176],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.4199, 1.0045, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.5698],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.8184,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.3118, 0.0000, 0.0000,  ..., 1.4561, 0.0000, 4.1492],\n",
            "        [0.4909, 0.0000, 0.5853,  ..., 0.0000, 0.0905, 0.6488]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74990437118745\n",
            "torch.Size([64, 48])\n",
            "tensor([[14.5190,  0.0000,  0.4313,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 2.9812,  0.0000,  0.0000,  ...,  0.0000,  2.1635,  1.2480],\n",
            "        [ 0.0000,  0.0000, 15.7491,  ...,  1.2547,  0.0000,  1.3780],\n",
            "        ...,\n",
            "        [ 2.2217,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.5078],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.6793,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3148,  0.3138]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75015421697049\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3868, 14.4298],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.1416],\n",
            "        [ 0.3125,  0.0000,  0.9176,  ...,  0.0000,  1.9053, 13.5179],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 15.3080],\n",
            "        [ 1.2225,  0.0000,  0.0981,  ...,  0.0000,  0.6699, 12.3599],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.5436,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.7488428740964\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.2923,  0.0000,  0.0000,  ...,  0.0000,  1.9500,  0.0000],\n",
            "        [ 1.8304,  0.0000,  0.0000,  ...,  0.0428,  0.0000,  2.7672],\n",
            "        [ 0.2664,  0.0000,  0.3119,  ..., 12.4056,  0.5295,  1.3038],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.3248,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.0852,  0.2533],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 13.8935]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74909378031609\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.9744,  0.0000,  0.6866,  ...,  0.0000,  0.6226,  0.0000],\n",
            "        [ 0.2901,  0.0000,  0.7688,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.0025,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.6512],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.0217,  ...,  0.0000,  0.0000,  0.7906],\n",
            "        [ 0.7863,  0.0000,  0.0000,  ...,  0.0000,  0.1063,  0.7838],\n",
            "        [ 2.5337,  0.0000, 14.4580,  ...,  1.6673,  0.1704,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7493444358802\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.3359,  ...,  0.0000, 14.8491,  0.3797],\n",
            "        [ 0.0000,  0.0754,  0.0000,  ...,  0.0000,  0.0000, 16.1674],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.8781,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 1.9532,  0.0000,  0.0000,  ...,  0.0000,  1.5662,  1.1650],\n",
            "        [ 0.7023,  0.0000,  1.0094,  ...,  0.0000,  0.5889,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74959484103917\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.4576, 0.0000, 0.8774,  ..., 0.0000, 0.0000, 0.4814],\n",
            "        [4.6216, 0.0000, 1.0306,  ..., 3.2747, 0.0000, 2.8585],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.8308],\n",
            "        ...,\n",
            "        [0.9564, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.8976, 0.0000, 1.2695,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.5667, 0.0000, 0.0000,  ..., 0.0000, 0.2190, 0.3218]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74984499604312\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0171e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.1052e-01,\n",
            "         0.0000e+00, 8.4105e-01, 3.9668e-01, 1.4659e+01, 0.0000e+00, 1.6079e+00,\n",
            "         0.0000e+00, 0.0000e+00, 8.0482e-02, 0.0000e+00],\n",
            "        [1.4981e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3754e+01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.3169e+00, 1.4934e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.2122e+00, 0.0000e+00, 0.0000e+00, 1.4912e+01,\n",
            "         4.5291e-02, 1.4800e+00, 0.0000e+00, 4.5105e-01, 0.0000e+00, 4.9178e-01,\n",
            "         0.0000e+00, 0.0000e+00, 4.8163e-01, 1.8602e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4793e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.0919e+00, 1.8514e+00, 1.0303e+00, 9.7989e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 8.3865e-01],\n",
            "        [1.9106e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.4359e-01, 0.0000e+00,\n",
            "         4.0436e-02, 6.0020e-01, 0.0000e+00, 7.0269e-01, 0.0000e+00, 1.5506e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.4391e-01, 0.0000e+00,\n",
            "         1.5858e+01, 0.0000e+00, 4.1797e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5212e+00, 0.0000e+00,\n",
            "         3.5212e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7316e-01,\n",
            "         0.0000e+00, 2.4502e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.8597e-02,\n",
            "         1.1645e+00, 0.0000e+00, 0.0000e+00, 1.6390e+01],\n",
            "        [9.0050e-02, 0.0000e+00, 0.0000e+00, 1.1242e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 7.8459e-01, 1.7192e+00, 1.3840e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5048e+00, 0.0000e+00, 7.8037e-01,\n",
            "         1.8374e+00, 0.0000e+00, 0.0000e+00, 6.1465e-02],\n",
            "        [1.3734e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.1358e-01, 5.0611e-01, 1.4120e+00, 3.8700e-01, 0.0000e+00, 5.1454e-02,\n",
            "         1.4772e+00, 3.8479e-01, 8.1488e-04, 0.0000e+00, 0.0000e+00, 1.3743e+00,\n",
            "         4.5766e-01, 0.0000e+00, 0.0000e+00, 1.0786e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.5205e-02, 1.5209e-01,\n",
            "         0.0000e+00, 0.0000e+00, 2.4868e-01, 1.2998e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.3632e-01,\n",
            "         1.3165e+00, 0.0000e+00, 1.5224e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.0871e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.0620e+00, 0.0000e+00, 8.9353e-01, 2.1283e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.1201e+00,\n",
            "         1.5392e+01, 0.0000e+00, 2.0348e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 9.8577e-03, 1.1732e-01, 0.0000e+00,\n",
            "         1.2262e+00, 0.0000e+00, 0.0000e+00, 9.5126e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 8.8771e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2397e+00,\n",
            "         1.6192e+01, 0.0000e+00, 0.0000e+00, 1.2751e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 8.8125e-02, 2.0598e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 4.4594e-02, 0.0000e+00, 1.3064e+01, 0.0000e+00, 0.0000e+00,\n",
            "         2.6607e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.9282e+00, 0.0000e+00, 1.1013e+00, 7.4202e-02, 1.8775e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.3803e+01, 1.0373e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.3992e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.0165e+00, 0.0000e+00, 3.8933e-02, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75009490114198\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 1.7906,  ..., 0.0000, 0.0000, 4.0811],\n",
            "        [0.0000, 0.0000, 1.1263,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.1537, 0.0000, 0.0000,  ..., 0.1076, 0.0000, 0.8723],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.9748,  ..., 1.2020, 0.0000, 1.3262],\n",
            "        [0.7729, 0.0000, 1.9369,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75034455658539\n",
            "torch.Size([64, 48])\n",
            "tensor([[13.9809,  0.0000,  2.3870,  ...,  1.1477,  0.0000,  0.4855],\n",
            "        [ 0.2183,  0.0000,  0.0000,  ...,  0.0000,  0.2557,  0.0000],\n",
            "        [ 0.3659,  0.0000, 12.5492,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.9657,  0.0000,  1.7118,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.5136,  0.0000,  0.0000,  ...,  0.0000, 14.0839,  2.2162],\n",
            "        [ 0.0687,  0.0000,  0.0000,  ...,  0.0000,  0.8025,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75059396262277\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  2.9719,  ...,  0.0000,  0.0000, 14.4579],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.8973, 15.6118],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  2.6561,  ...,  0.0000,  0.0000, 12.5834],\n",
            "        [ 0.0000,  0.0000,  0.8706,  ...,  0.0000,  0.0000, 14.5666],\n",
            "        [ 1.0103,  0.0000,  0.0000,  ...,  0.0000,  0.3326,  2.8536]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  96.875\n",
            "acc:  99.74772124138138\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 1.1425,  0.0000,  0.6856,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.4419,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  3.6653],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ..., 12.2959,  0.0000,  1.4434],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.6254,  ...,  0.0000,  0.0000,  0.2483],\n",
            "        [ 0.0000,  0.0000,  0.2947,  ...,  0.0000, 14.5573,  0.0627],\n",
            "        [ 0.0000,  0.0000,  1.3318,  ...,  0.0000,  0.0488, 14.0540]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74797326811327\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.5333,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  2.4376,  ...,  0.0000,  0.0796,  0.0000],\n",
            "        [ 0.0965,  0.0000,  0.0000,  ...,  1.1746,  0.0000,  1.3669],\n",
            "        ...,\n",
            "        [ 0.3194,  0.0000,  0.0000,  ...,  0.2705,  0.0000,  0.5516],\n",
            "        [ 2.7669,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.6316],\n",
            "        [ 0.0000,  0.0000, 12.2239,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7482250430702\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.9192,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 13.8543],\n",
            "        [ 0.0000,  0.0000,  0.3961,  ...,  0.0000,  0.2564,  0.0000],\n",
            "        ...,\n",
            "        [ 0.8138,  0.0000,  0.0000,  ...,  0.0000,  0.1351,  1.1623],\n",
            "        [ 0.7610,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.1050],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.7257,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7484765665037\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [3.1476, 0.0000, 0.6191,  ..., 0.0000, 0.0000, 1.1823],\n",
            "        [0.5087, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.6087],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.4337,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 2.3666, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74872783866503\n",
            "torch.Size([13, 34])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.8205,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.2532,  0.0000, 15.2308,\n",
            "          0.0000,  0.0000,  0.1174,  0.0000,  0.0000,  1.4898],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000, 15.7439,  2.2369,  0.0000,  0.1608,\n",
            "          1.4912,  1.2070,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  1.0635],\n",
            "        [ 0.0000,  0.0000,  1.8692,  0.3609,  0.1517, 15.1883,  0.0000,  0.9937,\n",
            "          0.0000,  0.0000,  0.0000,  1.4460,  0.0000,  0.0000,  1.7541,  0.6559,\n",
            "          0.0000,  0.0000,  0.8037,  0.0000,  0.0000,  0.0412],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.1801,  0.0000,  0.0000, 13.9047,  0.0000,\n",
            "          0.9404,  0.0000,  0.0000,  0.0000,  1.3211,  0.9084,  0.0000,  0.9043,\n",
            "          0.0000,  0.0000,  1.3908,  0.0000,  0.0000,  1.5638],\n",
            "        [ 0.0000,  0.0000,  0.0000,  1.0402,  2.0830,  0.0000,  1.8057,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.7462,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  3.0484, 16.6076,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.2045,  0.0000,  0.1222,  0.7767,  2.4672,  0.0000,\n",
            "          0.6082,  0.0000,  0.0000,  1.5410,  0.1957,  0.7293,  0.0000,  0.0000,\n",
            "          0.7813,  0.0000,  0.1050,  0.4325,  0.0000, 13.4600],\n",
            "        [ 0.0000,  0.0000,  0.2435,  1.0078,  1.0280,  0.1920,  0.0000,  0.0000,\n",
            "          0.8810, 13.9265,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  1.8254,\n",
            "          0.0000,  0.1588,  0.0000,  0.0000,  0.3384,  0.0000],\n",
            "        [17.0357,  0.0000,  0.9528,  0.0000,  1.2555,  0.0000,  0.3630,  0.0259,\n",
            "          1.1788,  1.4239,  0.0000,  1.3307,  1.3588,  0.3456,  1.5744,  0.1667,\n",
            "          0.0000,  0.7786,  0.0000,  0.0000,  0.0000,  0.0692],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.1939,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.4719,  0.0000,  0.2429,  0.2941,  1.4541,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.6066,  1.7963,  0.0000, 14.7965,  0.7063],\n",
            "        [ 0.3831,  0.0000,  0.1215,  0.0000,  0.0000,  0.0000,  0.4878,  0.0000,\n",
            "          1.5798,  0.1168,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0735, 16.6959,  0.0000,  0.0000,  0.4726],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.5900,  0.1289,  0.0000,  0.5177,  0.0000,\n",
            "          0.4251,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.7394, 14.7657,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.2995,  0.0000,  0.0890,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.1586,  0.0000,  0.0000,  0.8356, 14.0909,\n",
            "          0.0000,  0.0000,  0.0000,  1.9143,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  2.2032,  0.6458,  1.5958,  0.0000,  0.0000,  0.0000,\n",
            "         13.4316,  1.0930,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.4078,  0.0000,  0.6238,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74897885980523\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.9979,  ..., 0.7142, 0.0000, 2.3388],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.0564, 0.0000],\n",
            "        [0.0000, 0.0000, 2.2545,  ..., 1.6718, 0.0000, 4.0418],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.8374],\n",
            "        [0.0000, 0.0000, 0.0375,  ..., 2.8381, 0.0000, 3.8564],\n",
            "        [0.3614, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74922963017507\n",
            "torch.Size([64, 48])\n",
            "tensor([[15.6718,  0.0000,  0.7942,  ...,  2.0299,  0.0000,  1.1949],\n",
            "        [ 1.1662,  0.0000,  0.0000,  ...,  0.0000,  0.1665,  0.1145],\n",
            "        [ 0.0000,  0.0000, 11.1859,  ...,  0.0000,  0.0000,  0.1706],\n",
            "        ...,\n",
            "        [ 0.8360,  0.0000,  0.6560,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.1025,  0.1740],\n",
            "        [ 2.3628,  0.0000,  0.0000,  ...,  0.0000,  0.1751,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74948015002504\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.6437,  ...,  0.0000,  0.0000, 13.4242],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.4679,  1.9218],\n",
            "        [ 0.2853,  0.0000,  1.0406,  ...,  0.0000,  0.0000, 11.7882],\n",
            "        ...,\n",
            "        [ 0.0000,  1.8011,  2.0549,  ...,  0.0000,  0.0000, 14.0672],\n",
            "        [ 0.0000,  0.0000,  0.7101,  ...,  1.3024,  0.0000, 11.8220],\n",
            "        [ 2.5776,  0.0000,  0.0000,  ...,  0.0000,  1.7443,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74973041960543\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 1.6421,  0.0510,  0.0000,  ...,  0.0000,  1.4116,  0.0000],\n",
            "        [ 2.9765,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.6120],\n",
            "        [ 0.0000,  0.0000,  1.0273,  ..., 12.2084,  0.5646,  1.7724],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.0901],\n",
            "        [ 0.6844,  0.0000,  0.0806,  ...,  0.0000, 12.8224,  2.3796],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.1249, 12.4076]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74998043916627\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 1.0599,  0.0000,  0.0000,  ...,  0.0000,  0.7364,  0.0000],\n",
            "        [ 2.6497,  0.0000,  2.2139,  ...,  0.0000,  0.4104,  0.0000],\n",
            "        [ 0.0919,  0.0000,  2.1252,  ...,  0.0000,  0.0000,  3.9439],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  3.1651],\n",
            "        [ 0.5747,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.6162],\n",
            "        [ 0.0000,  0.0000, 15.8645,  ...,  1.2438,  0.3795,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7502302089573\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.6316,  0.0000,  0.0000,  ...,  0.0000, 15.6482,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.9640,  ...,  0.0000,  0.0000, 14.6547],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.1683,  0.0000],\n",
            "        ...,\n",
            "        [ 1.8034,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.6649,  0.0000],\n",
            "        [ 0.4968,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.3972]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75047972922808\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.9451,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.5290, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.9231],\n",
            "        [1.2891, 0.0000, 0.0000,  ..., 1.1418, 0.0000, 2.5259],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.3746,  ..., 0.0272, 0.5703, 0.0000],\n",
            "        [0.5091, 0.0000, 0.0000,  ..., 0.0000, 1.8007, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75072900022785\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3868e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.0715e-01, 1.2141e+00, 1.0440e+00, 0.0000e+00, 3.2300e-01,\n",
            "         0.0000e+00, 0.0000e+00, 2.4437e-01, 1.4567e+01, 0.0000e+00, 0.0000e+00,\n",
            "         6.6759e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.9486e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5700e+01, 5.8549e-01,\n",
            "         0.0000e+00, 0.0000e+00, 1.3262e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.7421e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 9.7891e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 5.2907e-01, 0.0000e+00, 5.0503e-01, 1.4652e+01,\n",
            "         4.9002e-01, 6.5764e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3653e+00,\n",
            "         0.0000e+00, 5.4448e-01, 1.5850e+00, 5.5069e-03, 3.8501e-01, 2.2242e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 7.0613e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.5132e-01, 0.0000e+00,\n",
            "         1.7364e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2402e+00, 0.0000e+00, 1.4318e+00, 5.5364e-01, 1.7948e+00, 0.0000e+00,\n",
            "         1.6837e+00, 9.6956e-02, 0.0000e+00, 2.7386e+00],\n",
            "        [9.4743e-01, 0.0000e+00, 0.0000e+00, 1.9107e-01, 2.5184e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.1108e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.6544e+01, 0.0000e+00, 1.3051e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1646e+00, 0.0000e+00,\n",
            "         3.2729e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.7415e+00, 7.6330e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         5.2946e-01, 0.0000e+00, 0.0000e+00, 1.6409e+01],\n",
            "        [0.0000e+00, 8.8009e-02, 5.2228e-01, 1.9267e-01, 1.8797e+00, 1.0996e+00,\n",
            "         0.0000e+00, 6.2852e-01, 0.0000e+00, 1.5146e+01, 0.0000e+00, 0.0000e+00,\n",
            "         7.0063e-02, 0.0000e+00, 9.0364e-02, 1.3113e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.5708e+01, 0.0000e+00, 3.9848e-01, 0.0000e+00, 3.0614e-01, 0.0000e+00,\n",
            "         6.8858e-01, 0.0000e+00, 0.0000e+00, 1.3881e+00, 0.0000e+00, 1.9919e+00,\n",
            "         6.2364e-01, 1.5448e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0327e+00,\n",
            "         0.0000e+00, 4.6846e-01, 9.4948e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 4.5025e-01, 1.1471e+00, 4.1429e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.4981e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.1532e-01, 2.3016e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.4813e+01, 0.0000e+00],\n",
            "        [1.7009e+00, 0.0000e+00, 1.4039e+00, 0.0000e+00, 3.5448e-01, 2.3671e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 4.0968e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.3187e+01, 0.0000e+00, 9.2671e-01, 0.0000e+00],\n",
            "        [6.8570e-01, 0.0000e+00, 3.9134e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.3093e+00, 0.0000e+00, 1.0776e+00, 5.3841e-01, 2.2748e-03, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0216e+00,\n",
            "         1.7737e+01, 0.0000e+00, 7.8931e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.0524e+00, 0.0000e+00, 1.3612e+00, 0.0000e+00, 8.5783e-01,\n",
            "         0.0000e+00, 0.0000e+00, 6.0961e-01, 1.3499e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 7.3632e-02, 8.5516e-01],\n",
            "        [1.6552e+00, 0.0000e+00, 0.0000e+00, 1.3939e+00, 3.1889e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6851e+01, 2.7400e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.5584e+00, 0.0000e+00, 8.2886e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.5163e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75097802220564\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.3660, 0.0000, 1.1157,  ..., 0.0000, 0.0000, 3.7340],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.3393, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.8306],\n",
            "        ...,\n",
            "        [0.1817, 0.0000, 0.5669,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.8808, 0.0000, 0.0000,  ..., 0.6528, 0.0000, 2.4749],\n",
            "        [0.0000, 0.0000, 1.0848,  ..., 0.0000, 0.4659, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75122679541023\n",
            "torch.Size([64, 48])\n",
            "tensor([[14.0380,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.4746,  0.0000,  0.0000,  ...,  0.0787,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 14.3913,  ...,  0.0000,  0.1141,  0.0000],\n",
            "        ...,\n",
            "        [ 0.5524,  0.0000,  0.0000,  ...,  0.0000,  0.8438,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.3474,  ...,  0.0000, 12.1312,  0.0000],\n",
            "        [ 0.2504,  0.0000,  0.6386,  ...,  0.0000,  0.0000,  2.5752]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75147532009014\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.0819,  ...,  0.2100,  1.7008, 13.2543],\n",
            "        [ 0.0914,  0.0000,  0.0000,  ...,  0.0000,  1.7221,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.0130,  ...,  0.0000,  0.9445, 15.0609],\n",
            "        ...,\n",
            "        [ 0.1986,  0.0000,  2.3990,  ...,  0.0000,  0.0000, 13.9102],\n",
            "        [ 0.0000,  0.0000,  0.4150,  ...,  0.0000,  0.2872, 14.5714],\n",
            "        [ 1.9813,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  96.875\n",
            "acc:  99.74860171837176\n",
            "torch.Size([64, 73])\n",
            "tensor([[1.1653e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 2.7654e-01,\n",
            "         1.5399e+00],\n",
            "        [1.2189e+00, 0.0000e+00, 0.0000e+00,  ..., 3.0676e-01, 0.0000e+00,\n",
            "         2.2975e+00],\n",
            "        [1.4080e+00, 0.0000e+00, 7.6476e-01,  ..., 1.0949e+01, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        ...,\n",
            "        [0.0000e+00, 0.0000e+00, 3.3559e-01,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         7.8571e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 3.2193e-01, 1.4521e+01,\n",
            "         1.3202e-02],\n",
            "        [4.3330e-01, 0.0000e+00, 1.1054e-01,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.2596e+01]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74885286550625\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.1789,  ...,  0.0000,  2.2818,  0.0000],\n",
            "        [ 0.0000,  0.0000,  2.1827,  ...,  0.0000,  1.1991,  0.0000],\n",
            "        [ 0.9912,  0.0000,  0.0000,  ...,  2.8510,  0.0000,  2.0347],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.0525,  ...,  0.0643,  0.0000,  0.0000],\n",
            "        [ 1.4556,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.4087],\n",
            "        [ 0.0000,  0.0000, 11.3900,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74910376174451\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.2321,  0.0000,  0.8046,  ...,  0.0000, 14.3416,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.7075,  ...,  0.0000,  0.0000, 13.6266],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.6759,  0.0000],\n",
            "        ...,\n",
            "        [ 1.1878,  0.0000,  0.0000,  ...,  0.0000,  0.5829,  0.3489],\n",
            "        [ 0.0000,  0.0000,  0.6695,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.8939,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74935440733717\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.2690,  ..., 0.0000, 0.2040, 0.0000],\n",
            "        [2.1775, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.2528],\n",
            "        [0.7545, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.4625],\n",
            "        ...,\n",
            "        [1.7733, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.9419, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.6173, 0.0000, 0.1202,  ..., 0.0000, 0.0099, 3.0555]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74960480253463\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2145e+00, 0.0000e+00,\n",
            "         6.5684e-01, 0.0000e+00, 0.0000e+00, 1.0767e-01, 0.0000e+00, 2.5755e-01,\n",
            "         0.0000e+00, 2.3527e-01, 0.0000e+00, 1.5813e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6674e+01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.1853e+00, 1.5678e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.2079e+00, 0.0000e+00, 2.6799e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.0215e+00, 0.0000e+00, 4.9763e-01, 2.1195e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 2.8521e-01, 8.6455e-01, 1.4658e+01,\n",
            "         0.0000e+00, 2.5455e+00, 0.0000e+00, 1.2082e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.4511e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.0046e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [3.1892e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4130e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 9.2009e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.1662e+00, 5.0953e-01, 0.0000e+00, 1.8210e+00],\n",
            "        [6.0856e-01, 2.8134e-01, 0.0000e+00, 1.1527e+00, 1.0103e+00, 7.5319e-01,\n",
            "         0.0000e+00, 0.0000e+00, 6.6594e-01, 0.0000e+00, 0.0000e+00, 5.4162e-02,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.5153e-01,\n",
            "         1.4541e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [6.1054e-01, 0.0000e+00, 3.5593e-01, 0.0000e+00, 9.8664e-02, 0.0000e+00,\n",
            "         1.4541e+00, 2.7873e-02, 5.3344e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.3985e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3693e+01],\n",
            "        [7.8378e-01, 0.0000e+00, 5.0780e-01, 2.3793e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 5.2743e-01, 0.0000e+00, 1.3298e+01, 0.0000e+00, 0.0000e+00,\n",
            "         2.7429e+00, 0.0000e+00, 6.9347e-01, 1.4155e+00, 0.0000e+00, 6.4828e-01,\n",
            "         6.4184e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.4594e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.0025e-01, 0.0000e+00,\n",
            "         2.6345e+00, 2.6121e-01, 0.0000e+00, 1.5437e+00, 0.0000e+00, 1.4390e+00,\n",
            "         5.4145e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.8358e+00, 0.0000e+00, 2.9045e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.8605e+00, 0.0000e+00,\n",
            "         9.9604e-03, 0.0000e+00, 0.0000e+00, 3.0976e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5773e+00, 0.0000e+00, 1.3982e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.3168e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 6.9072e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4377e+01, 0.0000e+00, 0.0000e+00, 4.9332e-01],\n",
            "        [1.4394e-01, 0.0000e+00, 1.8170e-01, 1.4062e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.4308e+00, 0.0000e+00, 0.0000e+00, 6.5970e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.2588e-01, 1.5315e+00,\n",
            "         1.6927e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 3.3095e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.5887e-01, 1.0160e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.3222e-01, 5.8339e-02, 1.4235e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.0650e+00, 0.0000e+00, 5.7522e-01, 0.0000e+00, 1.5193e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6829e+01, 1.2533e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.9133e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74985494758705\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 1.3374,  ..., 0.0000, 0.0000, 1.4227],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 2.2073, 0.0000],\n",
            "        [0.0000, 0.0000, 1.8234,  ..., 0.0000, 0.0000, 3.1083],\n",
            "        ...,\n",
            "        [1.2296, 0.0000, 3.0484,  ..., 0.0000, 1.9282, 0.0000],\n",
            "        [0.5682, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.9224],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0091, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7501048427443\n",
            "torch.Size([64, 48])\n",
            "tensor([[12.0343,  0.0000,  0.7092,  ...,  2.0400,  0.0000,  0.0000],\n",
            "        [ 1.1450,  0.0000,  1.6421,  ...,  0.0000,  2.1899,  0.0000],\n",
            "        [ 0.2890,  0.0000, 15.2368,  ...,  1.2048,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.4358, 15.6364,  0.8538],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0502,  1.2982,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75035448825605\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.9655,  ...,  0.0000,  0.0000, 13.1036],\n",
            "        [ 2.1131,  0.0000,  0.0000,  ...,  0.0000,  1.4149,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.2188,  ...,  1.7213,  1.8432, 13.7272],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  3.0802,  ...,  0.0000,  0.0000, 15.6196],\n",
            "        [ 0.0000,  0.0000,  1.5745,  ...,  0.0000,  0.0000, 14.3486],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.74904294531073\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 1.6743,  0.0000,  0.0000,  ...,  0.0000,  0.1651,  0.0000],\n",
            "        [ 4.6325,  0.0000,  0.0000,  ...,  1.3295,  0.0000,  0.8266],\n",
            "        [ 1.2202,  0.0000,  0.0000,  ...,  9.7677,  0.6795,  0.3187],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.4088,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.9985,  0.7093],\n",
            "        [ 0.0000,  0.0000,  1.8641,  ...,  0.0000,  0.0000, 13.8361]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74929365165907\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.1567,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  1.3790,  0.0000,  1.5915],\n",
            "        ...,\n",
            "        [ 1.7309,  0.0000,  2.0361,  ...,  0.9121,  0.0000,  0.8097],\n",
            "        [ 1.0656,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.7655],\n",
            "        [ 0.1907,  0.0000, 12.6203,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74954410755151\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.6623,  ...,  0.0000, 14.3031,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.7667,  ...,  0.0000,  1.7748, 12.9425],\n",
            "        [ 0.0000,  0.0000,  1.2328,  ...,  0.0000,  1.4714,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.5622,  0.1834],\n",
            "        [ 0.0000,  0.0000,  0.4701,  ...,  0.1408,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.3668]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74979431323828\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0218, 0.0000, 0.0000,  ..., 0.0000, 0.4588, 0.0000],\n",
            "        [1.1730, 0.0000, 0.0731,  ..., 0.0000, 0.0000, 2.7074],\n",
            "        [0.0103, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.3526],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.4242, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.7980, 0.5965]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75004426896932\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2610e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 7.9961e-01, 3.3097e-01, 0.0000e+00, 1.0138e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6459e+01, 0.0000e+00, 6.4991e-02,\n",
            "         0.0000e+00, 0.0000e+00, 1.2778e+00, 1.3176e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4242e+01, 2.4134e-01,\n",
            "         3.8209e-01, 0.0000e+00, 2.1098e+00, 1.8099e+00, 0.0000e+00, 1.7654e+00,\n",
            "         0.0000e+00, 1.6318e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5024e+00, 0.0000e+00, 1.5530e+00, 2.1084e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.0898e+00, 0.0000e+00, 8.1059e-01, 1.5386e+01,\n",
            "         0.0000e+00, 2.4566e-01, 0.0000e+00, 2.7641e-01, 0.0000e+00, 1.7875e+00,\n",
            "         0.0000e+00, 2.3385e-01, 1.5806e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         8.8491e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [7.0626e-01, 0.0000e+00, 0.0000e+00, 9.7717e-01, 0.0000e+00, 1.1431e+00,\n",
            "         1.3482e+01, 0.0000e+00, 4.5116e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         6.2120e-01, 1.4282e+00, 0.0000e+00, 1.5346e+00, 0.0000e+00, 1.8193e+00,\n",
            "         0.0000e+00, 1.3825e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.9712e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.2158e-01, 1.5611e+00,\n",
            "         6.0235e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.7007e-01,\n",
            "         0.0000e+00, 3.0659e-01, 0.0000e+00, 0.0000e+00, 6.0821e-01, 5.6431e-01,\n",
            "         1.5476e+01, 0.0000e+00, 1.4898e+00, 1.0885e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 2.0160e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.0964e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.0739e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.5047e-01, 1.4244e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 6.2386e-01, 1.5748e+00, 1.8779e-01, 0.0000e+00,\n",
            "         9.0878e-01, 3.6104e-01, 0.0000e+00, 1.4067e+01, 0.0000e+00, 0.0000e+00,\n",
            "         2.0943e+00, 0.0000e+00, 1.7079e-01, 1.1485e+00, 3.6980e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 8.4986e-01, 0.0000e+00],\n",
            "        [1.5382e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0405e+00, 0.0000e+00,\n",
            "         0.0000e+00, 9.8777e-01, 1.0444e+00, 0.0000e+00, 0.0000e+00, 3.4202e-01,\n",
            "         2.5165e-01, 9.0085e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.5045e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.5303e-01, 1.2600e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.7352e-02, 2.5562e-01,\n",
            "         1.3600e-01, 2.7181e-01, 1.2160e+01, 4.3974e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2110e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 6.1274e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 5.4654e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4733e+01, 0.0000e+00, 1.3063e-01, 0.0000e+00],\n",
            "        [2.7499e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2733e+00, 0.0000e+00, 8.1013e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4577e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 9.9604e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 4.5532e-02, 4.2362e-01, 8.6165e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4305e+01, 0.0000e+00, 0.0000e+00,\n",
            "         2.8567e+00, 0.0000e+00, 0.0000e+00, 1.0866e+00],\n",
            "        [1.8387e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7079e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.5715e+01, 1.0220e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75029397499432\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 1.4851,  ..., 0.1592, 0.5489, 4.4301],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.3533],\n",
            "        [0.0000, 0.0000, 0.1321,  ..., 0.0000, 0.0000, 2.2176],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0911,  ..., 0.0000, 0.6692, 0.0000],\n",
            "        [1.2147, 0.0000, 0.7169,  ..., 0.4208, 0.0000, 2.8769],\n",
            "        [0.0000, 0.0000, 0.7672,  ..., 0.0000, 1.0923, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75054343156276\n",
            "torch.Size([64, 48])\n",
            "tensor([[13.8273,  0.0000,  0.0000,  ...,  0.6228,  0.0000,  0.3981],\n",
            "        [ 0.5345,  0.0000,  0.0000,  ...,  0.0000,  0.3159,  0.0000],\n",
            "        [ 0.0000,  0.0000, 13.0059,  ...,  0.1098,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.1439,  0.0000,  0.1896,  ...,  0.5649,  0.4918,  1.5697],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.4917,  1.2639],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.8883,  0.0282]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75079263892384\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.2466, 11.2989],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.9024,  ...,  0.0000,  0.6095, 12.7012],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.2479, 11.8098],\n",
            "        [ 0.0000,  0.0000,  3.8231,  ...,  0.0000,  0.0000, 13.8614],\n",
            "        [ 0.8572,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.4396]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.74948065826557\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 2.3983,  0.0000,  0.0000,  ...,  0.0000,  0.6792,  0.0000],\n",
            "        [ 3.4314,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.2339],\n",
            "        [ 0.5126,  0.0000,  1.5960,  ..., 12.8248,  0.0000,  1.4000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.2989,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 16.0650,  1.5377],\n",
            "        [ 0.1113,  0.0000,  0.0000,  ...,  0.0000,  2.2182, 14.1240]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74973092733823\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.1602,  ...,  0.4123,  0.0000,  0.0000],\n",
            "        [ 0.6107,  0.0000,  1.0303,  ...,  0.2050,  0.1494,  0.0000],\n",
            "        [ 0.4024,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  3.7918],\n",
            "        ...,\n",
            "        [ 0.6855,  0.0000,  0.9597,  ...,  0.2204,  0.0000,  2.5999],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 11.9243,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.74998094639184\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.2699,  ...,  0.0000, 15.2711,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.2858,  ...,  0.0000,  0.0000, 14.7180],\n",
            "        [ 0.0000,  0.0000,  0.9196,  ...,  0.0000,  0.4802,  0.0000],\n",
            "        ...,\n",
            "        [ 0.4234,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.7407],\n",
            "        [ 0.4421,  0.0000,  0.8339,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.9083,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75023071567617\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.3364, 0.0000, 0.3146,  ..., 0.0000, 0.2374, 0.0000],\n",
            "        [2.3295, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 3.9777],\n",
            "        [1.4759, 0.0000, 0.5618,  ..., 1.9891, 0.0000, 1.0201],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.6796, 0.0000, 0.2849],\n",
            "        [1.2674, 0.0000, 0.0000,  ..., 0.0000, 0.4264, 0.0000],\n",
            "        [0.2421, 0.0000, 0.0000,  ..., 0.0000, 2.3204, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75048023544073\n",
            "torch.Size([13, 34])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  0.3132,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.4990,  0.7171,  0.0000,  0.0000,  0.0000,  0.7640,  0.2582, 15.3804,\n",
            "          0.3461,  0.0000,  0.0929,  0.0000,  0.0000,  0.7128],\n",
            "        [ 0.0437,  0.0000,  0.0000,  0.0000, 15.0405,  0.0912,  0.0000,  0.0000,\n",
            "          2.1133,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  1.8103,  0.0000,  0.0000,  1.1087],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000, 13.6576,  0.0000,  0.3567,\n",
            "          0.0000,  0.0000,  0.0000,  0.0259,  0.0000,  0.0000,  0.4034,  0.0000,\n",
            "          0.0000,  0.0000,  0.7376,  0.0000,  1.0371,  0.0000],\n",
            "        [ 0.9704,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000, 15.8544,  0.0000,\n",
            "          0.9350,  0.0000,  0.0000,  0.0000,  0.2687,  0.0000,  0.4611,  0.0000,\n",
            "          0.3401,  0.0000,  1.9277,  0.0000,  0.0000,  0.7326],\n",
            "        [ 1.8931,  0.0000,  0.0000,  0.0000,  1.6775,  0.8811,  0.5583,  0.0000,\n",
            "          0.0000,  1.0163,  0.0000,  0.5792,  0.8830,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000, 13.7166,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.1126,  0.3009,  0.2755,  0.0730,  2.9408,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  3.2995,  0.7842,  0.2286,\n",
            "          0.0000,  0.9882,  0.0000,  0.0000,  0.0000, 15.9863],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.4859,  0.0000,  0.0000,  0.1751,  0.2456,\n",
            "          1.9301, 14.0257,  0.0000,  0.0000,  1.2808,  0.0000,  0.0000,  1.9298,\n",
            "          0.0000,  0.0000,  0.5700,  0.0000,  0.0000,  0.8562],\n",
            "        [15.2084,  0.0000,  1.0242,  0.3741,  0.0000,  0.0000,  0.0000,  0.3505,\n",
            "          0.0000,  3.2927,  0.0000,  0.0000,  1.0446,  0.6756,  0.0000,  0.0000,\n",
            "          0.0000,  0.5356,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  2.1941,  0.1660,  0.0745,  0.0000,  0.0000,\n",
            "          0.0000,  0.9344,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0502,  0.0000, 14.6736,  0.7520],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  2.1442,  0.0000,  0.1343,  0.0000,\n",
            "          1.0229,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0417, 15.9430,  0.0000,  0.0000,  0.1474],\n",
            "        [ 0.6031,  0.0000,  0.3142,  0.0000,  0.0000,  0.1046,  1.9700,  0.0000,\n",
            "          0.0000,  2.7821,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.6390, 12.9164,  0.0000,  0.7810,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0209,  1.0582,  0.0000,  0.0000,  0.1239,\n",
            "          0.3912,  0.7428,  0.0000,  0.0000,  0.0000,  0.4581,  0.0000, 14.5245,\n",
            "          0.1811,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  1.0320,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "         14.1193,  0.3843,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.6751,  0.0000,  0.0000,  1.0534]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7507295059348\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.1167, 0.0000, 4.3614],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.1594, 0.0000],\n",
            "        [0.0000, 0.0000, 0.9284,  ..., 1.1136, 0.0498, 3.3538],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.4178, 0.0000],\n",
            "        [0.0000, 0.0000, 0.5942,  ..., 0.1655, 0.0000, 3.6033],\n",
            "        [0.0000, 0.0000, 0.9950,  ..., 0.0000, 0.2946, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7509785274074\n",
            "torch.Size([64, 48])\n",
            "tensor([[13.7844,  0.0000,  1.5621,  ...,  0.1325,  0.0000,  1.5193],\n",
            "        [ 0.3831,  0.0000,  0.0000,  ...,  0.0000,  1.3189,  0.0000],\n",
            "        [ 0.3147,  0.0000, 15.8925,  ...,  0.0000,  0.0000,  0.8992],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.2315,  ...,  0.0000,  0.0000,  0.7848],\n",
            "        [ 0.0000,  0.0000,  1.0855,  ...,  0.0000, 13.8778,  0.0000],\n",
            "        [ 0.8030,  0.0000,  0.0000,  ...,  0.0000,  1.2868,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75122730010729\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.2728,  2.3277,  ...,  0.0000,  0.6560, 13.5392],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.7117,  0.0000],\n",
            "        [ 0.0000,  0.5212,  1.3148,  ...,  0.0000,  0.6910, 14.0914],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.2126,  ...,  0.7252,  0.0000, 16.0177],\n",
            "        [ 0.0000,  0.4477,  0.3189,  ...,  0.0000,  0.0000, 15.9904],\n",
            "        [ 1.2146,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.74991488522207\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.9052,  1.1037],\n",
            "        [ 1.4091,  0.0000,  0.0000,  ...,  1.0964,  0.0000,  1.5508],\n",
            "        [ 0.8367,  0.0000,  0.0000,  ..., 12.0963,  0.0000,  0.8495],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.7271,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.8938,  0.4698],\n",
            "        [ 0.0000,  0.0000,  0.7019,  ...,  0.0000,  2.1187, 14.3265]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75016472050157\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.9664,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  2.6731,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.7414],\n",
            "        ...,\n",
            "        [ 0.8421,  0.0000,  1.9236,  ...,  1.6648,  0.0000,  2.3345],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.5775],\n",
            "        [ 0.0000,  0.0000, 13.4970,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75041430619538\n",
            "torch.Size([64, 75])\n",
            "tensor([[5.6833e-01, 0.0000e+00, 1.8037e+00,  ..., 0.0000e+00, 1.3246e+01,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 9.9076e-01,\n",
            "         1.2136e+01],\n",
            "        [4.2471e-03, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        ...,\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.7889e+00,\n",
            "         3.7807e-01],\n",
            "        [1.8094e-01, 0.0000e+00, 1.4831e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [1.2599e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 4.5247e-01,\n",
            "         0.0000e+00]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75066364255282\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.3011, 0.5829,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [2.5255, 0.0000, 0.0000,  ..., 1.4575, 0.0000, 2.3113],\n",
            "        [0.7186, 0.0000, 0.0000,  ..., 0.3493, 0.0000, 1.5277],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 1.4292,  ..., 0.0000, 0.0000, 0.1704],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.6343, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 2.7508, 0.0895]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.750912729823\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 8.6856e-01, 1.5781e+00, 4.4521e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 9.1120e-01, 0.0000e+00, 1.4949e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6411e+01, 0.0000e+00,\n",
            "         4.1257e-01, 0.0000e+00, 1.8535e+00, 7.9513e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         8.1407e-01, 0.0000e+00, 2.5393e-01, 1.7898e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.4911e+00, 1.6341e-01, 1.2309e+00, 1.2514e+01,\n",
            "         0.0000e+00, 1.8825e+00, 0.0000e+00, 8.9006e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 5.8763e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5764e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         7.0742e-01, 9.8683e-02, 7.3471e-01, 0.0000e+00, 5.1945e-01, 0.0000e+00,\n",
            "         2.7036e+00, 0.0000e+00, 0.0000e+00, 1.4521e+00],\n",
            "        [6.2176e-01, 0.0000e+00, 8.3673e-01, 0.0000e+00, 8.1591e-01, 0.0000e+00,\n",
            "         1.2008e+00, 0.0000e+00, 9.7824e-01, 1.3674e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.0832e-01,\n",
            "         1.5437e+01, 0.0000e+00, 1.1649e+00, 4.1965e-02],\n",
            "        [0.0000e+00, 0.0000e+00, 3.5891e-01, 0.0000e+00, 4.6640e-01, 9.0039e-01,\n",
            "         1.0106e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6675e+00,\n",
            "         0.0000e+00, 5.1847e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3460e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1009e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5812e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.5130e+00, 0.0000e+00, 6.1585e-01, 1.9471e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.9851e-04, 0.0000e+00],\n",
            "        [1.6012e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.4882e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 9.7519e-01, 1.2083e+00, 0.0000e+00, 2.3240e+00,\n",
            "         1.9277e+00, 2.3034e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.4987e-02, 0.0000e+00, 0.0000e+00],\n",
            "        [7.2881e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2856e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 9.4364e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.1479e-01,\n",
            "         7.1708e-01, 0.0000e+00, 1.3036e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.4628e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         7.3291e-01, 0.0000e+00, 9.7897e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.1484e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.1078e-01,\n",
            "         1.3729e+01, 0.0000e+00, 2.2914e+00, 6.1641e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         4.1261e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.6591e-01,\n",
            "         1.6099e+01, 0.0000e+00, 1.6859e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2845e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.9375e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3872e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 4.8267e-01, 0.0000e+00],\n",
            "        [1.3007e+00, 0.0000e+00, 0.0000e+00, 3.6244e-01, 1.9802e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6551e+01, 1.8292e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.2100e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75116156825474\n",
            "torch.Size([64, 57])\n",
            "tensor([[2.1253e-01, 0.0000e+00, 6.6352e-01,  ..., 0.0000e+00, 1.4866e-03,\n",
            "         3.0960e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.5252e-01,  ..., 0.0000e+00, 5.4962e-01,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         2.4158e+00],\n",
            "        ...,\n",
            "        [1.3480e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 2.7457e-01,\n",
            "         1.9385e-01],\n",
            "        [8.5477e-01, 0.0000e+00, 0.0000e+00,  ..., 3.6606e-01, 0.0000e+00,\n",
            "         3.9151e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 9.9031e-02,\n",
            "         7.4371e-01]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75141015809665\n",
            "torch.Size([64, 48])\n",
            "tensor([[15.2284,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 2.2803,  0.0000,  0.0000,  ...,  0.0000,  1.3316,  0.0000],\n",
            "        [ 0.0000,  0.0000, 15.4983,  ...,  0.6319,  0.0000,  0.5618],\n",
            "        ...,\n",
            "        [ 1.4197,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.8350,  0.3915],\n",
            "        [ 0.1263,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75165849959706\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.3887,  ...,  0.0000,  0.8201, 13.4361],\n",
            "        [ 0.2915,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.0687],\n",
            "        [ 1.3336,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 15.2603],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.7818,  ...,  1.6559,  0.2297, 14.3210],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3257, 13.6220],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.75034565394311\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.8897,  1.0607],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.3672,  0.0000,  1.0516],\n",
            "        [ 0.1286,  0.0000,  0.7737,  ..., 12.9572,  0.7488,  0.7330],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0555,  ...,  0.0000, 15.1128,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.9210,  ...,  0.0000,  0.0000, 13.5717]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75059505888423\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3096,  0.0000],\n",
            "        [ 0.8322,  0.0000,  0.6142,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.5582,  0.0000,  1.6833],\n",
            "        ...,\n",
            "        [ 0.3617,  0.0000,  0.0582,  ...,  0.5889,  0.0000,  2.2731],\n",
            "        [ 1.6786,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 13.4974,  ...,  0.9706,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75084421466956\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.1972,  ...,  0.0000, 14.1081,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0308,  ...,  0.0000,  0.0000, 15.4699],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.8135,  0.6028],\n",
            "        [ 0.3576,  0.0000,  0.6269,  ...,  0.0000,  0.6783,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.7801,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75109312154801\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 3.0972,  ..., 0.0000, 0.4456, 0.0000],\n",
            "        [1.9769, 0.0000, 1.1347,  ..., 0.9723, 0.0000, 2.1996],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 3.7618],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 1.0123,  ..., 0.0000, 0.8371, 0.0000],\n",
            "        [0.2108, 0.0000, 0.8276,  ..., 0.0000, 1.3267, 0.9541]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75134177976824\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1937e+00, 2.3677e-04, 0.0000e+00,\n",
            "         4.2166e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5741e+01, 0.0000e+00, 0.0000e+00,\n",
            "         4.0868e-01, 0.0000e+00, 1.4927e-01, 5.4062e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 1.5486e-01, 0.0000e+00, 1.4484e+01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 4.2967e-01, 3.1100e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.4477e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.6811e+00, 0.0000e+00, 1.6914e+00, 4.2898e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 1.4927e+00, 0.0000e+00, 0.0000e+00, 1.4367e+01,\n",
            "         2.8221e+00, 8.9068e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.9007e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.7393e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         7.4804e-01, 0.0000e+00, 1.0730e-01, 0.0000e+00],\n",
            "        [9.0410e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2742e+00, 0.0000e+00,\n",
            "         1.5595e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.8511e+00, 0.0000e+00, 1.5036e+00, 0.0000e+00, 2.4637e+00, 0.0000e+00,\n",
            "         2.0154e+00, 9.2790e-01, 0.0000e+00, 1.2345e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3067e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.0281e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.3408e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.7767e+01, 0.0000e+00, 4.3321e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 2.8529e-01, 1.0947e+00, 0.0000e+00,\n",
            "         2.2649e+00, 0.0000e+00, 1.3536e+00, 0.0000e+00, 0.0000e+00, 8.1884e-01,\n",
            "         0.0000e+00, 4.1094e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.5358e-02, 1.5683e-01, 1.5288e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3484e+00, 8.0616e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5108e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.5283e+00, 0.0000e+00, 1.7689e+00, 9.6198e-01, 5.9565e-01, 2.1170e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.4799e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.5454e-01, 0.0000e+00,\n",
            "         1.7321e-01, 5.9067e-01, 1.5008e+00, 7.6716e-01, 0.0000e+00, 2.0095e+00,\n",
            "         8.2042e-01, 1.4675e+00, 0.0000e+00, 0.0000e+00, 3.2091e-01, 1.4780e+00,\n",
            "         0.0000e+00, 2.5390e-02, 0.0000e+00, 5.6839e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.4535e-01,\n",
            "         0.0000e+00, 0.0000e+00, 1.7089e+00, 0.0000e+00, 0.0000e+00, 7.8394e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         7.2548e-01, 0.0000e+00, 1.4447e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2042e+00, 0.0000e+00,\n",
            "         1.2073e+00, 0.0000e+00, 1.5803e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.1760e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2991e+00,\n",
            "         1.5179e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [7.4407e-01, 0.0000e+00, 0.0000e+00, 3.3586e-02, 1.3292e+00, 0.0000e+00,\n",
            "         2.8275e+00, 0.0000e+00, 0.0000e+00, 1.3890e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 5.6601e-01, 4.6953e-01, 1.0303e+00,\n",
            "         1.3127e+01, 0.0000e+00, 2.4908e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1462e+00, 3.2065e-01, 0.0000e+00,\n",
            "         0.0000e+00, 1.4464e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.5276e-03,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5745e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5139e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.5015e+01, 1.7710e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2577e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75159018957866\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 1.2819,  ..., 0.0000, 0.6942, 3.5340],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.6698, 0.0000],\n",
            "        [0.5631, 0.0000, 1.6554,  ..., 0.8860, 0.0000, 2.2175],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.2747],\n",
            "        [0.3502, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 3.0920],\n",
            "        [0.0000, 0.0000, 0.3743,  ..., 0.0000, 0.1509, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75183835122743\n",
            "torch.Size([64, 48])\n",
            "tensor([[13.3687,  0.0000,  0.7768,  ...,  0.0000,  0.0000,  0.2741],\n",
            "        [ 1.1033,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 12.3244,  ...,  0.5071,  0.0000,  2.1959],\n",
            "        ...,\n",
            "        [ 0.4724,  0.0000,  1.1331,  ...,  0.0000,  1.2401,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.6218,  ...,  0.0000, 14.5448,  0.3982],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  2.1055,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75208626496247\n",
            "torch.Size([64, 54])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 1.5316e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.5294e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [1.3266e-02, 0.0000e+00, 1.6361e-01,  ..., 3.6269e-01, 6.5018e-01,\n",
            "         1.2492e+01],\n",
            "        ...,\n",
            "        [0.0000e+00, 0.0000e+00, 2.6004e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.7036e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 4.3284e-01,  ..., 0.0000e+00, 7.7860e-01,\n",
            "         1.2635e+01],\n",
            "        [1.3308e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 6.1180e-01,\n",
            "         1.1627e+00]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.7507729919705\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 1.7486,  0.0000,  0.0000,  ...,  0.0000,  0.9795,  0.0191],\n",
            "        [ 1.9523,  0.0000,  0.0000,  ...,  0.6163,  0.0000,  2.4843],\n",
            "        [ 0.0000,  0.0000,  0.8108,  ..., 10.1594,  0.0000,  0.1064],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.9403,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.3996,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  2.0648, 14.2543]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75102197000051\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  1.5284,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.8854,  0.0000,  1.1964,  ...,  0.0000,  0.7270,  0.0000],\n",
            "        [ 1.7841,  0.0000,  0.0000,  ...,  0.8550,  0.0000,  2.8375],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.6832,  ...,  0.0000,  0.0000,  1.1790],\n",
            "        [ 0.2975,  0.0000,  0.0000,  ...,  0.0000,  0.4174,  0.5997],\n",
            "        [ 0.0000,  0.0000, 15.8039,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7512706993012\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.7973,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0951, 13.8299],\n",
            "        [ 0.0000,  0.0000,  1.7381,  ...,  0.0000,  0.0000,  0.3782],\n",
            "        ...,\n",
            "        [ 1.2620,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.4781],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.2335,  0.0000],\n",
            "        [ 1.7251,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75151918012108\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.8426, 0.0000, 0.3039,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.9270, 0.3158, 0.0000,  ..., 0.4853, 0.0000, 1.6623],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.3969, 0.0000, 3.5210],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 2.2082,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.1778, 0.0000, 0.0000,  ..., 0.0000, 0.4639, 0.0000],\n",
            "        [0.2640, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75176741270838\n",
            "torch.Size([13, 34])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  0.0000,  1.3349,  0.0000,  0.6972,  0.2745,\n",
            "          0.0000,  0.1169,  0.0000,  0.0000,  0.0000,  0.0000,  0.2486, 14.5925,\n",
            "          0.5664,  0.0000,  0.0000,  0.0000,  0.0000,  0.1420],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000, 16.9693,  0.9113,  0.1265,  0.0000,\n",
            "          1.1017,  0.0000,  0.0000,  0.0000,  0.0000,  0.1215,  0.0000,  0.0000,\n",
            "          0.4677,  0.0000,  0.7530,  0.0000,  1.7333,  1.8324],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  2.1309, 13.5014,  0.0000,  0.2004,\n",
            "          0.0000,  0.0000,  0.0000,  0.7934,  0.0000,  0.0000,  2.2381,  0.1558,\n",
            "          0.2610,  0.0000,  0.1958,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000, 13.7244,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.1771,  0.7097,  0.0000,  0.0000,\n",
            "          0.4871,  0.0306,  1.4632,  0.0000,  0.0000,  1.1476],\n",
            "        [ 0.3685,  0.4369,  0.0000,  0.0000,  0.0000,  0.3159,  0.0000,  0.0000,\n",
            "          0.2406,  0.3878,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000, 14.7760,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.9656,  0.1711,  3.4102,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.4508,  0.0000,  1.2047,  0.4243,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000, 14.5103],\n",
            "        [ 0.3073,  0.0000,  0.0000,  1.9804,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          1.1338, 16.4683,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.3029,\n",
            "          0.0000,  2.1597,  0.0000,  0.0000,  0.1978,  0.0000],\n",
            "        [14.5724,  0.0000,  0.0000,  0.0000,  2.1527,  0.0000,  0.6311,  0.0000,\n",
            "          0.1920,  0.2284,  0.0000,  0.9532,  0.2870,  0.1481,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0440,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  1.7868,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.6439,  0.0000,  0.0000,  1.7666,  0.5628,  0.0000,  0.0000,\n",
            "          0.0000,  0.2612,  0.9877,  0.0000, 15.3768,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  1.2703,  1.5773,  2.8856,  0.2206,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.8280,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000, 15.2286,  0.0000,  0.4244,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  2.1879,  0.0000,  1.0072,  2.0192,  0.0000,\n",
            "          0.0000,  0.2357,  0.0000,  1.9984,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000, 15.8615,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  1.6647,  0.3463,  0.0000,  0.6362,\n",
            "          0.5898,  0.0000,  0.0000,  0.4886,  0.0000,  0.0000,  0.0636, 13.9152,\n",
            "          0.4888,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.9358,  0.0000,  0.8125,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "         14.4037,  0.6228,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.3249,  0.0000,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75201539731107\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.0682,  ..., 0.0000, 0.0000, 0.4921],\n",
            "        [0.3195, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0118,  ..., 0.0000, 0.0000, 1.2716],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0609,  ..., 0.0000, 0.0000, 2.3892],\n",
            "        [0.1868, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75226313417689\n",
            "torch.Size([64, 48])\n",
            "tensor([[1.4914e+01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         3.6153e-02],\n",
            "        [2.5536e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.4588e+00,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.3570e+01,  ..., 1.3809e+00, 0.0000e+00,\n",
            "         3.1502e-01],\n",
            "        ...,\n",
            "        [6.7778e-01, 0.0000e+00, 2.4359e-01,  ..., 0.0000e+00, 5.3433e-03,\n",
            "         1.4128e-02],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.5001e+01,\n",
            "         0.0000e+00],\n",
            "        [6.8739e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.9864e+00,\n",
            "         7.4379e-01]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75251062355333\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  2.4019,  ...,  0.5732,  0.9148, 17.1878],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3025,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.8356, 13.4109],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.0384,  ...,  0.0000,  1.0512, 14.6122],\n",
            "        [ 0.0000,  0.5909,  1.1490,  ...,  0.0000,  0.0000, 14.0434],\n",
            "        [ 1.2755,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.7511969266267\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.2237,  0.0000,  0.7967,  ...,  0.0000,  0.7057,  0.0000],\n",
            "        [ 2.6606,  0.0000,  0.0000,  ...,  1.5729,  0.0000,  0.5277],\n",
            "        [ 1.6386,  0.0000,  0.0490,  ..., 11.7400,  0.1326,  0.1790],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0235,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.8291,  0.7595],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  2.2850, 14.1892]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75144548114555\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.4299,  0.0000,  0.7257,  ...,  0.0000,  1.8884,  0.0000],\n",
            "        [ 1.2561,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.2327],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.2476,  ...,  0.3401,  0.0000,  2.4612],\n",
            "        [ 0.3612,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.4752],\n",
            "        [ 0.0000,  0.0000, 13.2391,  ...,  1.1498,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75169378735819\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.1485,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.0020,  ...,  0.8638,  0.0000, 12.9384],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.3708,  ...,  0.0000,  1.4327,  1.0676],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.9447,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75194184551268\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 2.4512,  ..., 0.0000, 0.5600, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.3314, 0.0000, 1.1063],\n",
            "        [0.2951, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.3532],\n",
            "        ...,\n",
            "        [0.4523, 0.0000, 0.2638,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0655, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.1775,  ..., 0.2333, 0.0000, 0.1279]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75218965585682\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 1.9948e-02, 0.0000e+00, 0.0000e+00,\n",
            "         5.4291e-01, 1.0637e+00, 3.2252e-01, 2.6172e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.8045e-02, 1.4035e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.9555e-01, 1.4107e+00, 0.0000e+00],\n",
            "        [6.3541e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6752e+01, 1.8213e+00,\n",
            "         0.0000e+00, 0.0000e+00, 6.7479e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         5.8985e-01, 0.0000e+00, 3.4323e-02, 4.6730e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 8.9983e-02, 0.0000e+00, 8.6507e-02, 1.3630e+01,\n",
            "         0.0000e+00, 1.9567e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.9083e-01,\n",
            "         0.0000e+00, 0.0000e+00, 3.2646e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         9.7713e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 4.1854e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.4271e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.1361e-01, 0.0000e+00,\n",
            "         2.0596e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.1694e+00, 0.0000e+00, 0.0000e+00, 1.1849e-01],\n",
            "        [2.4211e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.3613e-01, 9.1696e-01,\n",
            "         0.0000e+00, 0.0000e+00, 2.2498e-01, 1.3133e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6497e+00,\n",
            "         1.8298e+01, 0.0000e+00, 1.5692e+00, 3.9186e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.5184e-01,\n",
            "         2.3352e+00, 0.0000e+00, 4.2317e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.3179e+00, 0.0000e+00, 3.2347e-01, 0.0000e+00, 0.0000e+00,\n",
            "         3.8896e-01, 0.0000e+00, 0.0000e+00, 1.2150e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7633e+00, 1.4117e-01, 0.0000e+00,\n",
            "         0.0000e+00, 1.0996e-01, 0.0000e+00, 1.4289e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.0111e+00, 0.0000e+00, 0.0000e+00, 6.6207e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.1583e-02, 0.0000e+00],\n",
            "        [1.5464e+01, 0.0000e+00, 5.4961e-01, 0.0000e+00, 1.0201e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 9.7555e-01, 0.0000e+00, 0.0000e+00, 1.8188e+00,\n",
            "         1.9302e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.1382e+00, 0.0000e+00, 3.8765e-02, 1.5012e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.4129e+00, 9.6275e-01,\n",
            "         0.0000e+00, 0.0000e+00, 1.7393e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 4.8968e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.1593e-02,\n",
            "         5.0361e-02, 0.0000e+00, 1.4387e+01, 7.8430e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.1795e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.2121e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.7341e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.6306e+01, 0.0000e+00, 9.2281e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 8.6664e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2236e+00, 0.0000e+00, 6.1428e-01, 2.5338e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 8.3009e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5122e+00,\n",
            "         1.8363e+01, 0.0000e+00, 0.0000e+00, 2.1692e-01],\n",
            "        [7.4014e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.8162e-01, 0.0000e+00,\n",
            "         7.4939e-02, 1.2193e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3994e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.0749e+00, 0.0000e+00, 7.7034e-02, 0.0000e+00, 2.4109e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.4880e+01, 5.5349e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 5.0459e-01, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75243721863819\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 1.0755,  ..., 0.0000, 0.0000, 3.7276],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.3132, 0.0000, 3.4929],\n",
            "        ...,\n",
            "        [0.2188, 0.0000, 0.0000,  ..., 0.0000, 1.7856, 0.0000],\n",
            "        [0.1200, 0.0000, 0.5343,  ..., 0.3704, 0.0000, 1.4777],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0054, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75268453410409\n",
            "torch.Size([64, 48])\n",
            "tensor([[1.6392e+01, 0.0000e+00, 0.0000e+00,  ..., 1.4372e-02, 0.0000e+00,\n",
            "         8.1394e-01],\n",
            "        [5.3462e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [4.5452e-01, 0.0000e+00, 1.3024e+01,  ..., 0.0000e+00, 6.2482e-01,\n",
            "         9.3205e-01],\n",
            "        ...,\n",
            "        [4.6880e-01, 0.0000e+00, 1.6729e-01,  ..., 2.9194e-01, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.3434e+01,\n",
            "         3.6459e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.5182e-01,\n",
            "         0.0000e+00]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75293160250159\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 15.4660],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0333,  0.0000,  2.0640,  ...,  0.0000,  0.6388, 15.9508],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.5312,  ...,  0.0000,  0.0000, 15.9137],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.8759, 15.9162],\n",
            "        [ 0.7262,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  96.875\n",
            "acc:  99.75005654595563\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.8051,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.5400,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.4780],\n",
            "        [ 1.2728,  0.0000,  1.0653,  ..., 11.5767,  0.0000,  0.9530],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.5898,  0.6196],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.7564, 16.6065]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75030623971591\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.9946,  ...,  0.0000,  0.0000,  0.3416],\n",
            "        [ 0.0000,  0.0000,  0.6803,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.4413,  0.0000,  0.0299,  ...,  0.2980,  0.0000,  2.5440],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.2595,  0.0000,  1.0657],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.4805,  1.1460],\n",
            "        [ 1.7368,  0.0000, 14.6788,  ...,  0.9247,  0.0000,  0.4676]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75055568403188\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.2488,  0.0000],\n",
            "        [ 0.0000,  0.2297,  0.0000,  ...,  0.6958,  0.8895, 12.5984],\n",
            "        [ 0.0880,  0.0000,  0.0000,  ...,  0.0000,  0.0958,  0.4994],\n",
            "        ...,\n",
            "        [ 0.0913,  0.0000,  0.0000,  ...,  0.0000,  1.9476,  0.0724],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.8901,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75080487915272\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.8236,  ..., 0.0000, 0.0000, 0.3862],\n",
            "        [0.1342, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.2975],\n",
            "        [0.0000, 0.0000, 0.3270,  ..., 0.9990, 0.0000, 2.3446],\n",
            "        ...,\n",
            "        [0.5748, 0.0000, 2.2639,  ..., 0.0000, 1.5412, 0.0000],\n",
            "        [1.4556, 0.0000, 0.0000,  ..., 0.0000, 0.7561, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.5892]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7510538253274\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 8.2019e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.8167e-01, 8.4827e-01, 2.1403e-01, 0.0000e+00, 0.0000e+00, 2.3034e-01,\n",
            "         0.0000e+00, 5.7678e-01, 0.0000e+00, 1.4327e+01, 0.0000e+00, 7.4159e-02,\n",
            "         0.0000e+00, 0.0000e+00, 1.2775e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 6.9910e-02, 0.0000e+00, 1.7504e+01, 2.9091e+00,\n",
            "         2.0631e-01, 1.3331e-01, 2.2098e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.0391e-01, 1.0019e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2512e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 4.6501e-01, 0.0000e+00, 1.0131e+00, 1.4223e+01,\n",
            "         0.0000e+00, 5.6751e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 5.9251e-01, 6.7357e-02, 0.0000e+00, 0.0000e+00,\n",
            "         1.8424e+00, 0.0000e+00, 9.8879e-01, 1.1981e+00],\n",
            "        [5.1082e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.5060e-01, 8.7268e-01,\n",
            "         1.5083e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.0862e-02,\n",
            "         8.7634e-01, 0.0000e+00, 1.4848e-01, 0.0000e+00, 0.0000e+00, 1.8147e-02,\n",
            "         8.3705e-01, 0.0000e+00, 0.0000e+00, 2.2245e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 2.1929e-01, 0.0000e+00, 1.0834e+00, 0.0000e+00,\n",
            "         1.5507e+00, 0.0000e+00, 1.7036e+00, 1.4114e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.4606e-01,\n",
            "         1.2376e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.0108e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2339e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.0012e-01,\n",
            "         0.0000e+00, 2.3379e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6140e+01],\n",
            "        [3.0628e-01, 0.0000e+00, 5.2277e-01, 2.0165e+00, 8.7791e-01, 1.1464e+00,\n",
            "         0.0000e+00, 7.9184e-02, 2.1444e+00, 1.2776e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 9.4554e-01, 8.0048e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 8.6309e-01],\n",
            "        [1.3902e+01, 0.0000e+00, 6.6249e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.3495e+00, 7.5255e-01, 7.3194e-01, 0.0000e+00, 0.0000e+00, 8.8531e-01,\n",
            "         5.2156e-01, 1.0898e+00, 1.4035e+00, 0.0000e+00, 0.0000e+00, 2.4379e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 3.8375e-01, 1.1092e+00, 1.9575e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.0213e+00, 8.5616e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2041e-01, 0.0000e+00, 5.2660e-01,\n",
            "         0.0000e+00, 0.0000e+00, 1.3287e+01, 1.6431e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7085e+00, 8.3968e-02,\n",
            "         5.4158e-01, 0.0000e+00, 4.4304e-01, 5.4852e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 6.1083e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5112e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.0377e+00, 0.0000e+00, 6.8953e-01, 1.6261e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.2910e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.3849e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.8145e+00,\n",
            "         1.7504e+01, 0.0000e+00, 7.1577e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.4332e-01, 0.0000e+00,\n",
            "         4.8187e-01, 1.4317e-02, 0.0000e+00, 8.6901e-01, 0.0000e+00, 9.8109e-01,\n",
            "         0.0000e+00, 2.0318e-02, 2.8181e+00, 1.5552e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 4.2473e-01],\n",
            "        [2.8468e+00, 0.0000e+00, 1.5210e+00, 0.0000e+00, 5.6490e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.5294e+01, 3.8569e-01, 0.0000e+00, 2.4227e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7513025228046\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 3.2070],\n",
            "        [0.6706, 0.0000, 0.3089,  ..., 0.0000, 0.6009, 0.0000],\n",
            "        [1.3784, 0.0000, 0.7066,  ..., 0.0000, 0.0000, 2.8187],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.2284],\n",
            "        [0.0000, 0.0000, 2.0758,  ..., 2.7690, 0.0000, 2.2384],\n",
            "        [0.1555, 0.0000, 2.2031,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75155097183276\n",
            "torch.Size([64, 48])\n",
            "tensor([[14.4680,  0.0000,  0.0000,  ...,  0.3352,  0.0000,  1.4931],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.1140,  0.4334],\n",
            "        [ 0.0000,  0.0000, 15.2069,  ...,  1.0913,  0.0000,  0.3920],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  2.2261,  ...,  0.9927,  0.0000,  0.5976],\n",
            "        [ 0.0000,  0.0000,  0.2385,  ...,  0.0000, 15.4623,  1.1359],\n",
            "        [ 0.8383,  0.0000,  0.0000,  ...,  0.0000,  1.5067,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7517991726601\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 14.1908],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3039,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.3324,  ...,  0.0000,  0.0000, 13.9955],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.6677, 15.7437],\n",
            "        [ 2.7259,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 13.1292],\n",
            "        [ 1.1339,  0.0817,  0.0000,  ...,  0.0000,  0.0000,  1.1889]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75204712553456\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.5752,  0.0000,  0.7769,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0249,  0.0000,  0.0000,  ...,  0.0703,  0.0000,  1.8635],\n",
            "        [ 0.1683,  0.0000,  0.0000,  ..., 12.0778,  1.1314,  1.2230],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.7287,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.7867,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.4409, 13.8195]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75229483070386\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 1.4055,  0.0000,  0.3501,  ...,  0.0000,  0.0000,  0.2253],\n",
            "        [ 0.0000,  0.0000,  0.1722,  ...,  0.0000,  1.3932,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  2.0417,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.7099,  ...,  0.5972,  0.0000,  0.0000],\n",
            "        [ 1.0104,  0.0000,  1.1049,  ...,  0.0000,  0.0000,  2.6650],\n",
            "        [ 0.0000,  0.0000, 14.8330,  ...,  0.1947,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75254228841544\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0491,  ...,  0.0000, 14.5963,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0881,  0.0000, 12.3707],\n",
            "        [ 1.0875,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.0568,  1.0543],\n",
            "        [ 0.0695,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.1080,  0.0000,  0.0000,  ...,  2.3662,  0.8468,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75278949891653\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 1.9799,  ..., 0.0000, 0.5558, 0.0000],\n",
            "        [0.1835, 0.0000, 0.0000,  ..., 1.2804, 0.0000, 2.1854],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.1883],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.1197, 0.0000],\n",
            "        [1.1389, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.1152, 0.0000, 0.0000,  ..., 0.0000, 0.6661, 1.8203]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75303646245408\n",
            "torch.Size([13, 34])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  1.9533,  0.0000,  0.0000,  0.0000,  0.6129,\n",
            "          0.0000,  1.0555,  0.0000,  0.0000,  0.0000,  0.1422,  0.0000, 15.0593,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  1.4152,  0.3249],\n",
            "        [ 0.6652,  0.0000,  0.0000,  0.0000, 13.7538,  2.4348,  0.0000,  0.0000,\n",
            "          2.0601,  2.8557,  0.0000,  0.0000,  0.0000,  0.5814,  0.0000,  0.2388,\n",
            "          0.0000,  0.3189,  0.7095,  0.0000,  2.4539,  0.9439],\n",
            "        [ 0.0000,  0.0000,  0.9409,  0.0000,  0.6025, 15.6901,  0.0000,  1.3818,\n",
            "          0.0000,  0.0258,  0.0000,  0.5674,  0.0000,  0.0000,  1.7419,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.8400,  0.0000,  0.0000,  0.1548, 14.7214,  0.0000,\n",
            "          0.2327,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0523,  0.0000,\n",
            "          0.0000,  0.0000,  0.6906,  0.0000,  0.8905,  1.2261],\n",
            "        [ 1.3125,  0.0000,  0.9543,  0.0000,  0.9216,  0.0000,  0.0000,  0.0000,\n",
            "          0.3830,  0.0000,  0.0000,  1.4566,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  2.2708, 18.4791,  0.0000,  1.8527,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  1.7121,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.7509,  0.0638,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.2517,  0.0000, 12.7943],\n",
            "        [ 1.1842,  0.0000,  0.0000,  1.5981,  1.6696,  1.3871,  0.3902,  0.0000,\n",
            "          1.1421, 14.2912,  0.0000,  0.0000,  0.1964,  0.0000,  0.0000,  0.0634,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [15.0562,  0.0000,  0.0000,  0.0000,  2.1003,  0.0000,  1.4394,  0.1219,\n",
            "          0.0000,  0.1713,  0.0000,  0.5187,  0.0000,  0.0000,  0.1513,  0.0000,\n",
            "          0.0000,  1.3191,  0.0000,  0.0000,  0.0000,  0.0322],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.9702,  2.6465,  1.3537,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000, 14.8854,  0.2938],\n",
            "        [ 0.6665,  0.0000,  0.0000,  0.0000,  1.8387,  0.0000,  0.0734,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.2573,  2.6943, 15.7506,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.9062,  0.0000,  1.8242,  0.0000,  0.0000,  0.4628,  1.5525,  0.0000,\n",
            "          0.0000,  1.0212,  0.0000,  0.0000,  0.0000,  1.6399,  0.0000,  0.0000,\n",
            "          0.0000,  1.0961, 15.4407,  0.0000,  0.1357,  0.0918],\n",
            "        [ 0.9498,  0.0000,  0.0000,  0.0000,  0.6271,  0.0000,  0.0000,  0.0000,\n",
            "          0.3878,  0.0000,  0.0000,  0.0000,  0.0000,  1.1528,  0.0000, 15.4368,\n",
            "          1.4983,  0.0405,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.4689,  0.0000,  0.0250,  1.6557,  2.3972,  0.0000,  0.0000,  0.0000,\n",
            "         16.1512,  0.8512,  0.0000,  0.0516,  0.0000,  0.0000,  0.6981,  0.0000,\n",
            "          0.0000,  0.1549,  0.0000,  0.0000,  1.4014,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7532831792748\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.6625,  ..., 0.0000, 0.0000, 2.1206],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.7455, 0.0000],\n",
            "        [0.0000, 0.0000, 2.1966,  ..., 0.0000, 0.0000, 1.6908],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.4432],\n",
            "        [1.1098, 0.0000, 0.0000,  ..., 0.5345, 0.0000, 3.2547],\n",
            "        [0.0000, 0.0000, 1.8056,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75352964962518\n",
            "torch.Size([64, 48])\n",
            "tensor([[12.6868,  0.0000,  0.4911,  ...,  0.7551,  0.0000,  0.0000],\n",
            "        [ 0.2335,  0.0000,  0.0000,  ...,  0.0000,  1.9243,  0.1777],\n",
            "        [ 0.0000,  0.0000, 13.1950,  ...,  0.0000,  0.0000,  0.8918],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.2664,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.0815,  ...,  0.0000, 14.0057,  0.8994],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75377587375142\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.1659,  ...,  0.0000,  1.4390, 14.0238],\n",
            "        [ 0.0000,  0.2163,  0.0000,  ...,  0.0000,  0.7218,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.3264,  ...,  0.0000,  0.0000, 12.9523],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.8912,  ...,  0.0000,  0.0000, 13.9103],\n",
            "        [ 0.0000,  0.0000,  0.0535,  ...,  0.0000,  0.0000, 15.8995],\n",
            "        [ 0.4493,  0.0000,  0.0000,  ...,  0.0000,  0.8323,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.75246091283859\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 2.2055,  0.0000,  0.0000,  ...,  0.0000,  1.2687,  2.0169],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  1.3327,  0.0000,  2.1241],\n",
            "        [ 1.1460,  0.0000,  0.8179,  ..., 10.8068,  0.3678,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.9749,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.4847,  0.0000,  0.0000,  ...,  0.0000, 13.5944,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.6599,  ...,  0.0000,  0.0000, 14.8091]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75270820463395\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3933,  0.0000],\n",
            "        [ 0.0000,  0.0000,  3.5981,  ...,  1.7238,  0.0000,  0.0000],\n",
            "        [ 1.0954,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.8696],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.0574,  ...,  0.0000,  0.0000,  0.0191],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.1295,  1.9009],\n",
            "        [ 0.0000,  0.0000, 14.3641,  ...,  1.7521,  0.0000,  0.2084]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75295524938456\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 16.9087,  0.3458],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 13.0097],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.1618,  0.4211],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.1811,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.9934,  0.0000,  0.0000,  ...,  0.0000,  0.4417,  0.0000],\n",
            "        [ 0.6611,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75320204733721\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.9668, 0.0000, 0.5182,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.8357, 0.0000, 0.0000,  ..., 2.1164, 0.0000, 3.7581],\n",
            "        [0.1029, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.7599],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.6410,  ..., 0.0000, 0.0000, 2.3061],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.3934, 0.0000],\n",
            "        [0.0000, 0.0000, 0.2765,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75344859873847\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 2.0838e+00, 1.8424e+00, 0.0000e+00,\n",
            "         0.0000e+00, 9.9980e-02, 3.1681e-01, 1.5256e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4015e+01, 0.0000e+00, 5.7297e-01,\n",
            "         0.0000e+00, 0.0000e+00, 8.6718e-01, 0.0000e+00],\n",
            "        [9.5336e-01, 0.0000e+00, 7.6314e-02, 0.0000e+00, 1.4244e+01, 2.1471e+00,\n",
            "         4.6991e-02, 0.0000e+00, 1.7610e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 9.1549e-01, 0.0000e+00, 2.0821e-01, 4.0934e-01, 0.0000e+00,\n",
            "         3.2666e-02, 0.0000e+00, 1.2269e+00, 4.2726e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 4.1820e-01, 0.0000e+00, 1.6071e+01,\n",
            "         0.0000e+00, 6.1183e-01, 0.0000e+00, 9.5467e-01, 0.0000e+00, 7.1355e-01,\n",
            "         0.0000e+00, 0.0000e+00, 6.6234e-01, 0.0000e+00, 5.4782e-02, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 2.3233e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.7068e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.4962e+00, 8.2766e-01, 6.4173e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.8282e+00, 1.4925e+00, 0.0000e+00, 2.1320e+00],\n",
            "        [8.6288e-01, 0.0000e+00, 2.1570e-01, 2.0051e-01, 1.7742e+00, 0.0000e+00,\n",
            "         9.9621e-01, 0.0000e+00, 1.2361e-01, 1.6095e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.2330e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5679e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.2685e-01, 0.0000e+00,\n",
            "         2.2749e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.7924e+00, 0.0000e+00, 0.0000e+00, 4.7351e-01, 3.2349e-01,\n",
            "         6.2842e-01, 0.0000e+00, 0.0000e+00, 1.5349e+01],\n",
            "        [3.4312e-01, 0.0000e+00, 0.0000e+00, 1.1660e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.0892e-01, 8.8562e-01, 7.3672e-01, 1.4918e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.0662e-01, 1.6937e+00, 1.1118e-01, 2.4699e+00,\n",
            "         9.0868e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.5170e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.8915e-01, 0.0000e+00,\n",
            "         7.3852e-01, 1.7490e+00, 0.0000e+00, 1.8543e-01, 0.0000e+00, 1.6712e+00,\n",
            "         1.4316e+00, 0.0000e+00, 2.7824e-02, 0.0000e+00, 0.0000e+00, 4.6144e-01,\n",
            "         4.2492e-01, 0.0000e+00, 0.0000e+00, 7.3946e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 9.5083e-01, 1.7288e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6497e+00, 1.3114e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.9082e-01,\n",
            "         0.0000e+00, 0.0000e+00, 1.4141e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 2.9047e-01, 1.6863e+00, 4.3061e+00, 2.1979e+00,\n",
            "         1.5276e-01, 0.0000e+00, 0.0000e+00, 6.2360e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.0806e-01, 1.1233e+00,\n",
            "         1.4616e+01, 0.0000e+00, 6.3912e-01, 0.0000e+00],\n",
            "        [2.3181e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.9415e-01, 0.0000e+00, 1.1849e+00, 1.8530e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.9596e-02, 1.1023e+00, 0.0000e+00, 0.0000e+00, 8.2945e-01, 0.0000e+00,\n",
            "         1.4004e+01, 0.0000e+00, 0.0000e+00, 1.2477e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.0432e-04, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         6.6183e-01, 0.0000e+00, 7.0182e-01, 1.3921e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [9.7104e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2828e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.2179e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75369490383464\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.3551, 0.0000, 0.0000,  ..., 0.0000, 0.2649, 3.6947],\n",
            "        [0.6162, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.2385, 0.0000, 0.1841,  ..., 0.7556, 0.0000, 2.1593],\n",
            "        ...,\n",
            "        [0.9558, 0.0000, 0.0000,  ..., 0.0000, 1.0090, 2.1627],\n",
            "        [0.9175, 0.0000, 0.0000,  ..., 1.5305, 0.0000, 3.6035],\n",
            "        [0.8476, 0.0000, 0.0907,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75394096287177\n",
            "torch.Size([64, 48])\n",
            "tensor([[1.3644e+01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [1.4214e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 2.1661e+00,\n",
            "         1.3332e-02],\n",
            "        [0.0000e+00, 0.0000e+00, 1.4054e+01,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        ...,\n",
            "        [0.0000e+00, 0.0000e+00, 1.2403e-01,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 7.3223e-01,  ..., 0.0000e+00, 1.4672e+01,\n",
            "         1.3078e+00],\n",
            "        [4.6720e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.4879e+00,\n",
            "         0.0000e+00]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75418677609566\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.1325,  0.0000,  2.7295,  ...,  0.0000,  0.9662, 13.2842],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.8443,  0.0000],\n",
            "        [ 0.1233,  0.0000,  0.0812,  ...,  0.0000,  0.0000, 15.7055],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 16.0230],\n",
            "        [ 0.6350,  0.0000,  0.9040,  ...,  0.0000,  0.0000, 15.2623],\n",
            "        [ 0.7144,  0.0000,  0.0000,  ...,  0.1015,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.75287140469098\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.8158],\n",
            "        [ 1.2314,  0.0000,  0.0000,  ...,  0.2716,  0.0000,  3.5582],\n",
            "        [ 0.5824,  0.0000,  0.0000,  ..., 10.9409,  0.8679,  0.1990],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.7177,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 12.7037,  0.9528],\n",
            "        [ 0.0000,  0.0000,  1.1202,  ...,  0.0000,  2.0606, 16.1670]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75311828640457\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.8205,  0.0000,  1.6034,  ...,  0.0000,  1.0969,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.2933,  0.0000,  1.7214],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  2.2075,  ...,  0.5476,  0.0000,  1.6335],\n",
            "        [ 0.8401,  0.0000,  0.6965,  ...,  0.0000,  0.0000,  1.5278],\n",
            "        [ 0.0000,  0.0000, 11.6805,  ...,  0.2468,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75336492148308\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.5319,  0.5320],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.7011, 15.6099],\n",
            "        [ 0.3042,  0.0000,  1.0720,  ...,  0.0000,  0.1599,  0.0000],\n",
            "        ...,\n",
            "        [ 1.0491,  0.0000,  0.0000,  ...,  0.0000,  1.7842,  0.2423],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.5707,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75361131017291\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.6433,  ..., 0.0000, 0.0113, 0.0000],\n",
            "        [1.9743, 0.0000, 0.0000,  ..., 1.7326, 0.0367, 2.8788],\n",
            "        [0.2064, 0.0000, 0.0000,  ..., 1.9389, 0.0000, 2.8392],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0549],\n",
            "        [0.0000, 0.0000, 0.0403,  ..., 0.0000, 0.7443, 0.0000],\n",
            "        [0.5358, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7538574527202\n",
            "torch.Size([13, 34])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000, 13.1010,\n",
            "          0.0000,  0.5872,  0.0000,  0.0000,  1.2692,  2.7653],\n",
            "        [ 0.5991,  0.0000,  0.2534,  0.0000, 15.8767,  0.0000,  1.1069,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.2232,  0.0000,\n",
            "          0.2056,  0.8496,  0.0217,  0.0000,  0.2121,  1.0562],\n",
            "        [ 0.0000,  0.0000,  1.4514,  0.9031,  0.2592, 13.6734,  0.0000,  1.3366,\n",
            "          0.0000,  0.4342,  0.0000,  1.0615,  0.0000,  0.0000,  0.6925,  0.0000,\n",
            "          0.3604,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0407,  0.0000, 14.7363,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.3838,  2.2172,  0.3595,  1.9311,\n",
            "          0.4075,  0.0000,  2.0565,  0.2872,  0.0000,  1.1069],\n",
            "        [ 2.2377,  0.0000,  0.0000,  0.0000,  2.2712,  0.1993,  2.1007,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          1.2117,  0.8648, 13.6651,  0.0000,  0.4359,  0.8100],\n",
            "        [ 0.1332,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  1.0628,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  2.3330,  1.1364,  0.0000,\n",
            "          0.0244,  0.0000,  0.0000,  0.0000,  0.0000, 14.9464],\n",
            "        [ 0.0000,  0.0000,  0.0000,  1.8762,  0.2604,  0.2970,  0.0000,  1.7397,\n",
            "          0.5799, 13.9326,  0.0000,  0.0000,  0.3755,  0.0000,  0.0000,  0.5727,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  1.6551,  0.0000],\n",
            "        [14.1891,  0.0000,  0.0000,  0.0000,  2.2873,  0.0000,  0.0000,  1.1360,\n",
            "          1.3408,  0.0000,  0.0000,  1.9437,  0.9927,  1.2597,  0.0000,  0.0000,\n",
            "          0.0000,  0.2432,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0484,  0.4018,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0836,  0.0000,  0.0000,  1.0693,  0.0000,  1.6310,  0.0000,  0.0000,\n",
            "          0.0000,  0.4502,  0.3134,  0.0000, 14.9466,  0.6876],\n",
            "        [ 0.0000,  0.0000,  0.0000,  1.4915,  2.5957,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  1.1487,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0464,\n",
            "          0.0000,  0.7436, 19.1096,  0.0000,  0.1027,  0.0000],\n",
            "        [ 0.2027,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.6989,  0.0000,\n",
            "          0.0000,  0.5952,  0.0000,  0.0000,  0.0000,  0.1142,  0.0000,  0.7648,\n",
            "          0.0000,  0.4774, 16.5337,  0.0000,  0.0000,  0.1999],\n",
            "        [ 0.5955,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.6743,  0.2242,\n",
            "          0.0000,  1.1472,  0.0000,  0.2178,  0.0000,  0.0000,  0.0000, 14.7533,\n",
            "          1.1468,  0.0000,  1.0436,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.4073,  0.0000,  0.0912,  0.0000,  0.0000,  0.0000,\n",
            "         12.5262,  1.8284,  0.0000,  0.0000,  0.0918,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  1.7505,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75410334937082\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 1.3243,  ..., 0.0000, 0.0000, 1.3662],\n",
            "        [0.0202, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.2894,  ..., 0.0000, 0.0000, 0.6775],\n",
            "        ...,\n",
            "        [0.5187, 0.0000, 0.0000,  ..., 0.0000, 0.8019, 0.0000],\n",
            "        [0.8946, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.7026],\n",
            "        [0.0000, 0.0000, 0.7576,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75434900037045\n",
            "torch.Size([64, 48])\n",
            "tensor([[16.5083,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.5221,  0.0000,  0.0000,  ...,  0.0000,  0.1556,  0.2187],\n",
            "        [ 0.0000,  0.0000, 13.6780,  ...,  0.0000,  0.3512,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  2.1089,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 12.0067,  3.4986],\n",
            "        [ 0.5353,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75459440596448\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 1.2099,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 12.8307],\n",
            "        [ 0.0942,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.2863],\n",
            "        [ 0.0000,  0.0000,  1.8510,  ...,  0.0000,  1.5709, 13.3531],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.2406,  ...,  0.0000,  1.6676, 16.5186],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 13.7977],\n",
            "        [ 0.3191,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.2850]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.75327862733714\n",
            "torch.Size([64, 73])\n",
            "tensor([[3.9896e-01, 0.0000e+00, 5.4044e-01,  ..., 0.0000e+00, 1.6236e+00,\n",
            "         0.0000e+00],\n",
            "        [1.9019e+00, 0.0000e+00, 0.0000e+00,  ..., 8.5266e-01, 0.0000e+00,\n",
            "         6.2504e-01],\n",
            "        [2.6656e-01, 0.0000e+00, 1.7554e+00,  ..., 1.0155e+01, 9.1808e-01,\n",
            "         0.0000e+00],\n",
            "        ...,\n",
            "        [0.0000e+00, 0.0000e+00, 2.9131e-01,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.1487e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.4060e+01,\n",
            "         2.7199e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 1.4078e-02,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.4541e+01]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7535251022349\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0321,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  3.2297,  ...,  0.0000,  1.4443,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.7039,  0.3262,  2.3040],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  1.4732,  0.0000,  0.6477],\n",
            "        [ 0.7918,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 15.1874,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.753771330904\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.4726,  ...,  0.0000, 15.2296,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.4213,  ...,  0.0000,  0.2434, 14.2537],\n",
            "        [ 1.4213,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.0162],\n",
            "        ...,\n",
            "        [ 0.5916,  0.0000,  0.0000,  ...,  0.0000,  2.2842,  2.1366],\n",
            "        [ 0.0000,  0.0000,  0.9724,  ...,  0.0000,  0.0000,  0.0852],\n",
            "        [ 0.8946,  0.0000,  0.0000,  ...,  0.0000,  1.4904,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75401731359041\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.5972, 0.0000, 0.0000,  ..., 0.6272, 0.0000, 5.3276],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.9783],\n",
            "        ...,\n",
            "        [0.5271, 0.0000, 0.0000,  ..., 0.6279, 0.0000, 0.0000],\n",
            "        [3.1286, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.7040, 0.1865]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75426305053988\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 4.9098e-01, 0.0000e+00, 9.5687e-01,\n",
            "         0.0000e+00, 2.2284e-02, 1.2445e+00, 0.0000e+00, 0.0000e+00, 3.8447e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3338e+01, 0.0000e+00, 4.1160e-01,\n",
            "         1.0776e+00, 0.0000e+00, 0.0000e+00, 1.2929e+00],\n",
            "        [1.1655e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4825e+01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.3714e+00, 0.0000e+00, 0.0000e+00, 1.0534e-01,\n",
            "         0.0000e+00, 4.8661e-03, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         8.7197e-01, 0.0000e+00, 0.0000e+00, 4.0996e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.6461e+00, 0.0000e+00, 0.0000e+00, 1.4133e+01,\n",
            "         0.0000e+00, 1.1120e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.4237e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.6510e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 3.4006e-01],\n",
            "        [4.1039e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.6772e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         8.8013e-01, 1.1674e+00, 0.0000e+00, 0.0000e+00, 8.8785e-01, 2.8668e-01,\n",
            "         1.6183e+00, 6.5092e-01, 0.0000e+00, 2.2690e+00],\n",
            "        [8.6912e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6463e+00, 3.1045e-01,\n",
            "         2.5176e-01, 0.0000e+00, 1.0149e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.1731e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3177e-01,\n",
            "         1.4124e+01, 0.0000e+00, 3.2762e-03, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.4995e-01,\n",
            "         1.0267e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.2685e-01,\n",
            "         0.0000e+00, 2.3516e+00, 8.9459e-01, 0.0000e+00, 0.0000e+00, 1.0792e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5306e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 1.0218e+00, 2.6846e+00, 1.5675e+00, 0.0000e+00,\n",
            "         3.0211e-01, 1.1698e+00, 1.7711e+00, 1.3177e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.2270e-01, 5.2659e-01, 0.0000e+00, 7.0859e-01, 0.0000e+00, 0.0000e+00,\n",
            "         6.3835e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.5364e+01, 0.0000e+00, 1.5284e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.4288e+00, 0.0000e+00, 0.0000e+00, 4.5116e-01, 0.0000e+00, 2.0064e+00,\n",
            "         0.0000e+00, 1.7576e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.9239e-01, 2.1714e-01, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2518e-01, 9.5321e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5022e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.8822e+00, 0.0000e+00, 0.0000e+00, 1.9107e-01, 0.0000e+00, 0.0000e+00,\n",
            "         2.0144e-01, 0.0000e+00, 1.3143e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 2.3906e-01, 1.4686e+00, 8.1317e-01, 1.3378e+00,\n",
            "         0.0000e+00, 0.0000e+00, 8.7017e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.4419e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.3748e+01, 0.0000e+00, 1.8704e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4030e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 5.3426e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5516e+01, 0.0000e+00, 2.7696e-01, 0.0000e+00],\n",
            "        [7.5985e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 6.2161e-01, 0.0000e+00, 1.3692e+01, 1.3461e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.0887e-02, 3.5290e-01],\n",
            "        [3.1249e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.0647e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.3610e+01, 7.2716e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 8.1264e-01, 0.0000e+00, 5.8964e-02, 0.0000e+00, 0.0000e+00,\n",
            "         4.4218e-01, 0.0000e+00, 8.1138e-01, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75450854199788\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.7453,  ..., 0.0000, 1.0847, 1.3654],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.0961, 0.0000],\n",
            "        [0.3488, 0.0000, 0.0000,  ..., 0.5113, 0.0000, 2.3020],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 1.3299, 0.0000, 1.2857],\n",
            "        [0.0000, 0.0000, 0.2865,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75475378820967\n",
            "torch.Size([64, 48])\n",
            "tensor([[16.4495,  0.0000,  0.0000,  ...,  1.0442,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.1632,  ...,  0.0000,  2.3194,  0.0000],\n",
            "        [ 0.0000,  0.0000, 14.4428,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.2305,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.6753,  1.7714],\n",
            "        [ 0.1318,  0.0000,  0.0000,  ...,  0.0000,  0.3891,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75499878942026\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.0176,  ...,  0.0000,  0.3882, 14.8176],\n",
            "        [ 2.0248,  0.0000,  0.7723,  ...,  0.0000,  0.5222,  0.0000],\n",
            "        [ 0.3283,  0.0000,  2.1241,  ...,  0.0000,  0.0000, 12.3980],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.4153,  ...,  0.0000,  0.1994, 14.8762],\n",
            "        [ 0.0000,  0.0000,  0.2298,  ...,  0.0000,  0.0000, 13.5137],\n",
            "        [ 1.2465,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.75368260681344\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 1.1249,  0.0000,  1.4392,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.8818,  0.0000,  0.0000,  ...,  1.3576,  0.0000,  1.0251],\n",
            "        [ 0.4521,  0.0000,  0.0000,  ..., 10.3642,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.9701,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.7249,  ...,  0.0000, 14.9236,  1.7522],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.2465, 13.4720]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75392867813531\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 1.6644,  0.0000,  0.0000,  ...,  0.0000,  0.4812,  0.0000],\n",
            "        [ 1.0419,  0.0000,  0.8296,  ...,  0.0000,  0.6458,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.4811,  ...,  0.4966,  0.0000,  1.6908],\n",
            "        ...,\n",
            "        [ 0.2352,  0.0000,  0.0554,  ...,  0.0000,  0.0000,  4.1912],\n",
            "        [ 1.1478,  0.0000,  1.9606,  ...,  0.0000,  0.0000,  0.1981],\n",
            "        [ 0.0000,  0.0000, 14.8952,  ...,  0.0247,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75417450363167\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.8235,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.5907,  ...,  0.0000,  0.8790, 13.8507],\n",
            "        [ 0.0000,  0.0000,  3.0360,  ...,  0.0000,  0.0000,  1.0373],\n",
            "        ...,\n",
            "        [ 1.9994,  0.0000,  0.0000,  ...,  0.0000,  0.8385,  1.6584],\n",
            "        [ 0.0912,  0.0000,  0.4533,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.2958,  0.0000,  0.0000,  ...,  0.0000,  0.6639,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75442008354813\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.9730, 0.0000],\n",
            "        [0.4863, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.5881],\n",
            "        [2.2959, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.2162],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.8983,  ..., 0.0000, 0.1733, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.4030, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75466541813\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 3.1752e-01, 5.7275e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.8626e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.5603e-01, 0.0000e+00, 0.0000e+00, 1.6209e+01, 0.0000e+00, 0.0000e+00,\n",
            "         7.6066e-01, 0.0000e+00, 0.0000e+00, 4.8427e-01],\n",
            "        [1.0927e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5413e+01, 1.0459e+00,\n",
            "         1.7311e+00, 0.0000e+00, 0.0000e+00, 1.5546e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.3112e-01, 4.0976e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 7.0069e-01, 2.6499e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2423e+00, 7.7223e-01, 1.2046e+01,\n",
            "         5.6611e-02, 2.8509e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.5553e-01,\n",
            "         2.7131e-01, 0.0000e+00, 4.9530e-01, 0.0000e+00, 3.0797e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3371e+00],\n",
            "        [7.8995e-01, 0.0000e+00, 2.9313e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2932e+01, 0.0000e+00, 1.4246e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.0794e+00, 0.0000e+00, 9.6483e-01, 0.0000e+00, 5.4737e-01, 4.1605e-01,\n",
            "         6.5492e-01, 6.6383e-01, 0.0000e+00, 3.1107e+00],\n",
            "        [1.5423e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.4864e-01,\n",
            "         1.6097e+00, 9.4234e-01, 3.2518e-03, 1.0727e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.7253e-01, 7.9811e-02,\n",
            "         1.5876e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [5.3372e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3864e+00,\n",
            "         4.1081e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.1304e+00, 9.4590e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.6131e-01, 0.0000e+00, 0.0000e+00, 1.6313e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 6.4480e-01, 8.6838e-01, 6.2078e-01, 3.9341e-01,\n",
            "         4.4882e-01, 0.0000e+00, 4.0670e-01, 1.6628e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 6.2979e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.7224e+00, 0.0000e+00, 8.3784e-01, 0.0000e+00],\n",
            "        [1.4141e+01, 0.0000e+00, 1.3260e+00, 2.1156e-01, 4.9009e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.7298e+00, 1.6606e-01, 0.0000e+00, 2.2210e+00,\n",
            "         1.0260e+00, 6.1814e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2502e+00,\n",
            "         0.0000e+00, 3.9240e-01, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 2.9776e-01, 0.0000e+00, 5.0024e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 4.5451e-01, 7.0048e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.4200e+00, 3.7884e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.0172e-01,\n",
            "         0.0000e+00, 0.0000e+00, 1.3311e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.2303e-01, 1.3176e-01,\n",
            "         7.1084e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.0886e-01,\n",
            "         1.4629e+01, 0.0000e+00, 0.0000e+00, 6.4775e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.9753e+00, 7.0532e-02, 0.0000e+00,\n",
            "         3.4118e+00, 0.0000e+00, 3.6374e-01, 1.2140e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 9.0094e-01, 0.0000e+00, 3.5368e-02, 0.0000e+00, 1.1489e+00,\n",
            "         1.7243e+01, 0.0000e+00, 0.0000e+00, 1.4741e-02],\n",
            "        [3.0013e-01, 0.0000e+00, 0.0000e+00, 8.3982e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.5796e-01, 1.2252e+00, 0.0000e+00, 3.1948e-01, 0.0000e+00, 1.2890e+00,\n",
            "         0.0000e+00, 2.0392e-01, 2.3788e-01, 1.4515e+01, 4.1514e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.0975e+00, 0.0000e+00, 2.6922e-01, 5.7570e-01, 2.2351e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.5369e+01, 2.6720e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.1599e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75491050762238\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.4564, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.6710],\n",
            "        [0.2242, 0.0000, 0.5401,  ..., 0.0000, 2.1405, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 3.7516],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0497,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.6136, 0.0000, 0.1929,  ..., 2.5287, 0.0000, 2.6530],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75515535227011\n",
            "torch.Size([64, 48])\n",
            "tensor([[13.2871,  0.0000,  0.2530,  ...,  0.6239,  0.0000,  0.0000],\n",
            "        [ 0.4005,  0.0000,  0.0000,  ...,  0.0000,  3.7080,  0.0000],\n",
            "        [ 0.9852,  0.0000, 14.0061,  ...,  0.0000,  0.1347,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.5020],\n",
            "        [ 0.0000,  0.0000,  0.8912,  ...,  0.0000, 16.4634,  0.1696],\n",
            "        [ 1.5914,  0.0000,  0.0000,  ...,  0.0000,  0.3531,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75539995231779\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.7126,  ...,  0.0000,  0.0000, 14.0716],\n",
            "        [ 0.5596,  0.0000,  0.0000,  ...,  0.0000,  0.2455,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0721,  ...,  0.2897,  0.0491, 12.0141],\n",
            "        ...,\n",
            "        [ 0.8435,  0.0000,  2.7841,  ...,  0.0000,  1.7370, 13.8279],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 14.3295],\n",
            "        [ 2.7659,  0.0000,  0.0000,  ...,  0.0000,  1.4124,  0.2535]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75564430800978\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.2851,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 2.8687,  0.0000,  0.0000,  ...,  0.0856,  0.0000,  2.5095],\n",
            "        [ 0.0000,  0.0000,  0.4528,  ..., 10.3045,  0.7342,  0.7816],\n",
            "        ...,\n",
            "        [ 0.2551,  0.0000,  0.6761,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.1897,  0.0000,  0.0000,  ...,  0.0000, 13.4909,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0638,  ...,  0.0000,  0.3821, 14.7070]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75588841959019\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.2534,  0.0000,  0.0226,  ...,  0.2759,  0.2224,  0.4266],\n",
            "        [ 0.0000,  0.0000,  0.8338,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.1153,  0.0000,  0.0000,  ...,  0.8994,  0.0000,  2.2804],\n",
            "        ...,\n",
            "        [ 0.8566,  0.0000,  1.6150,  ...,  0.0000,  0.0000,  1.0048],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.6311],\n",
            "        [ 0.3278,  0.0000, 13.9323,  ...,  1.9207,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75613228730289\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  1.4897,  ...,  0.0000, 14.2056,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.8291,  ...,  0.0000,  0.0337, 14.9846],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.5238,  0.3342],\n",
            "        ...,\n",
            "        [ 0.7058,  0.0000,  0.0000,  ...,  0.0000,  0.2410,  0.5204],\n",
            "        [ 0.2661,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.4670],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.4909,  0.2467]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7563759113915\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.6383, 0.0000, 0.0000,  ..., 0.8183, 0.0000, 2.6551],\n",
            "        [0.1460, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.7488],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.3826, 0.0000],\n",
            "        [0.0000, 0.0000, 0.4597,  ..., 0.0000, 1.5309, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.7647, 0.3380]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7566192920994\n",
            "torch.Size([13, 34])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.4917,  0.0000,\n",
            "          0.0000,  1.0051,  0.0000,  0.2223,  0.0000,  0.6405,  0.8430, 13.5567,\n",
            "          0.9685,  0.0000,  0.0000,  0.0000,  0.0000,  1.2568],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.3647, 14.1023,  3.1023,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  1.6120],\n",
            "        [ 0.0000,  0.0000,  1.6810,  0.0000,  0.0000, 15.5668,  0.0000,  2.3901,\n",
            "          0.0000,  1.0882,  0.0000,  0.0000,  0.0000,  0.0000,  1.1041,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  2.2766],\n",
            "        [ 1.5150,  0.0000,  0.4112,  0.0000,  0.0000,  0.0000, 12.8228,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  1.7799,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  2.9757,  0.0000,  0.0000,  1.5602],\n",
            "        [ 0.1461,  0.0000,  1.2542,  0.3598,  0.2008,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  2.0276, 13.7006,  0.0000,  0.5899,  0.0000],\n",
            "        [ 0.1546,  0.0000,  1.7037,  0.0000,  0.0000,  1.4280,  2.6642,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  1.5722,  0.0000,  1.8820,  1.0780,  0.0000,\n",
            "          0.0000,  0.6428,  0.0000,  0.1832,  0.0000, 16.6357],\n",
            "        [ 1.6474,  0.0000,  0.0000,  1.6278,  1.7464,  0.0000,  0.0000,  2.1415,\n",
            "          1.2957, 15.4345,  0.0000,  0.0000,  0.2693,  0.0000,  0.0000,  1.2578,\n",
            "          1.3672,  0.7375,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [12.6596,  0.0000,  0.7377,  0.0000,  1.2390,  0.0000,  0.0832,  0.8688,\n",
            "          0.4046,  1.8070,  0.0000,  0.4871,  0.0000,  0.0000,  1.2299,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.6216,  0.0000,  0.3165],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          1.4907,  0.0000,  0.0000,  0.0000,  0.2149,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000, 14.8652,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  1.5405,  2.0507,  0.3946,  0.0000,\n",
            "          0.0000,  1.3224,  0.0000,  0.3936,  0.0000,  0.1131,  0.0000,  0.0000,\n",
            "          1.7481,  0.0000, 15.6967,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.3692,  0.0000,  0.0000,  0.0000,  0.8958,  0.0000,\n",
            "          0.2902,  0.0000,  0.0000,  0.3162,  0.0000,  0.0000,  0.0000,  0.5630,\n",
            "          0.0000,  0.0000, 15.5032,  0.0000,  0.0000,  1.4686],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  1.9082,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.7577,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000, 12.7353,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.2030,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  2.1099,  0.0000,  0.0000,  0.0000,\n",
            "         16.7261,  1.4192,  0.0000,  0.0000,  0.0000,  1.4991,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.4221,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75686242966974\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.4355, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.8621],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.7674, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 2.5398,  ..., 1.4511, 0.0000, 5.0743],\n",
            "        ...,\n",
            "        [0.3114, 0.0000, 1.2977,  ..., 0.0000, 1.2206, 0.6518],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 2.2646, 0.0000, 1.5692],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75710532434539\n",
            "torch.Size([64, 48])\n",
            "tensor([[16.5182,  0.0000,  1.6062,  ...,  2.0202,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.6515,  0.0000],\n",
            "        [ 0.0000,  0.0000, 14.2350,  ...,  0.4391,  0.1406,  1.2962],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.0874,  0.1743],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 12.4016,  0.0000],\n",
            "        [ 1.3852,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75734797636902\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.7737, 16.3894],\n",
            "        [ 0.0000,  0.0000,  0.1891,  ...,  0.0000,  0.6536,  0.0911],\n",
            "        [ 0.3258,  0.0000,  0.0000,  ...,  0.0000,  0.9700, 14.7473],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.0402,  ...,  0.0000,  1.3391, 14.0994],\n",
            "        [ 0.0000,  0.0000,  1.3131,  ...,  0.2820,  0.0000, 13.0780],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3124,  1.7803]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  96.875\n",
            "acc:  99.75446850786116\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 2.0824,  0.0000,  0.1396,  ...,  0.0000,  0.1580,  1.1948],\n",
            "        [ 1.7227,  0.0000,  0.0000,  ...,  1.4902,  0.0000,  1.4459],\n",
            "        [ 0.4167,  0.0000,  0.0000,  ..., 11.9771,  0.0000,  1.0056],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.8912,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.1047,  ...,  0.0000,  0.7369, 15.0453]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7547137940671\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.7630,  0.0000],\n",
            "        [ 0.8547,  0.0000,  1.0034,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.6655,  0.0000,  0.0000,  ...,  0.7199,  0.0000,  1.2366],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.1394,  ...,  0.0000,  0.0000,  1.9053],\n",
            "        [ 1.1989,  0.0000,  0.0000,  ...,  0.0000,  0.5952,  1.2965],\n",
            "        [ 0.0000,  0.0000, 12.9662,  ...,  0.5839,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75495883523188\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.7970,  ...,  0.0000, 13.9774,  0.6718],\n",
            "        [ 0.0000,  0.0000,  0.9757,  ...,  0.0000,  0.0000, 14.9626],\n",
            "        [ 0.1010,  0.0000,  0.3342,  ...,  0.0000,  0.1167,  0.0000],\n",
            "        ...,\n",
            "        [ 1.4160,  0.0000,  0.0000,  ...,  0.0000,  1.8502,  1.2921],\n",
            "        [ 0.0528,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.2943],\n",
            "        [ 0.3840,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.1683]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75520363160028\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.4971, 0.3770],\n",
            "        [1.1389, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.0103],\n",
            "        [0.6768, 0.4793, 0.0000,  ..., 0.0000, 0.0000, 2.7116],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.2733,  ..., 0.0000, 0.0000, 0.1733],\n",
            "        [0.3448, 0.0000, 0.4870,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.2498,  ..., 0.0000, 0.9765, 0.8467]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75544818341686\n",
            "torch.Size([13, 34])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  0.1798,  0.0000,  0.0000,  0.7107,  0.4872,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  1.4041,  1.2574, 15.5909,\n",
            "          0.0000,  0.0000,  0.0000,  0.5096,  0.0000,  1.6539],\n",
            "        [ 0.4543,  0.0000,  0.7526,  0.0000, 13.2102,  0.9328,  0.8277,  0.0000,\n",
            "          1.6432,  0.6229,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  1.2331,  0.0000,  0.6067,  0.9170],\n",
            "        [ 0.0000,  0.0000,  1.3581,  0.0000,  0.0000, 15.7293,  0.6016,  1.4428,\n",
            "          0.0000,  0.6913,  0.0000,  0.0000,  0.0000,  0.0000,  1.0480,  0.0000,\n",
            "          0.0000,  0.0000,  0.0897,  0.0000,  1.8330,  0.3742],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0481, 15.9741,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.9022,  0.4429,  0.0000,  2.3834,\n",
            "          1.8690,  0.1674,  2.2719,  0.2325,  0.0000,  2.1676],\n",
            "        [ 1.8780,  0.0000,  0.2056,  0.0000,  1.7663,  0.0000,  0.0838,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  1.2073, 15.8762,  0.0000,  2.2546,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  2.4950,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  1.2426,  0.0876,  0.0000,\n",
            "          1.5768,  0.1038,  0.0000,  0.4390,  0.0000, 15.1756],\n",
            "        [ 0.8694,  0.0000,  0.0000,  0.4528,  0.0000,  0.9450,  0.0000,  0.0000,\n",
            "          0.5422, 15.4125,  0.0000,  0.0000,  0.0000,  0.0000,  1.0120,  0.5631,\n",
            "          0.0000,  0.7239,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [14.6019,  0.0000,  0.6902,  0.0000,  1.0497,  0.0000,  1.6997,  0.0000,\n",
            "          1.1020,  0.2880,  0.0000,  2.0957,  1.1672,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.1470,  0.0000,  0.0000,  0.0000,  0.1184],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.2909,  0.7391,  0.0000,  0.0000,  1.7414,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.8495,  1.1249,  0.0000, 15.3579,  0.7609],\n",
            "        [ 0.0000,  0.0000,  0.5425,  0.0000,  2.5888,  1.0245,  0.4123,  0.0000,\n",
            "          0.8944,  0.0000,  0.0000,  0.4656,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.7843, 17.5520,  0.0000,  0.5155,  0.0000],\n",
            "        [ 0.3257,  0.0000,  0.0000,  1.7962,  0.0000,  0.0000,  1.6407,  0.0000,\n",
            "          0.0000,  0.4904,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  1.9270, 15.6453,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  1.3914,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0842,  0.0000,  1.3666,  0.0000,  0.9411,  0.0000, 16.8999,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0403,  0.0000],\n",
            "        [ 1.1740,  0.0000,  0.0000,  0.0000,  1.3579,  0.0000,  0.0000,  0.0000,\n",
            "         15.5981,  2.2662,  0.1790,  0.0000,  0.1688,  0.7288,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  2.2928,  0.0000,  0.2503,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75569249092594\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.9656, 0.0000, 0.1046,  ..., 0.0000, 0.2047, 1.1070],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.3554, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.6592],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.1379,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.4118, 0.0000, 1.2821,  ..., 0.0000, 0.0000, 3.5679],\n",
            "        [0.3141, 0.0000, 0.0000,  ..., 0.0000, 0.2514, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75593655437157\n",
            "torch.Size([64, 48])\n",
            "tensor([[15.5238,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.5001,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 11.7421,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.5157,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.7800,  0.0000],\n",
            "        [ 0.5494,  0.0000,  0.0000,  ...,  0.0000,  1.9273,  1.4114]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75618037399758\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.5636,  ...,  0.0000,  0.0000, 15.1948],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.6380,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.4290,  ...,  0.0000,  0.0000, 14.0078],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  2.1734,  ...,  0.0000,  1.5476, 14.6378],\n",
            "        [ 0.0000,  0.0000,  3.9294,  ...,  0.0000,  0.0000, 13.5044],\n",
            "        [ 1.9322,  0.0000,  0.0000,  ...,  0.8347,  0.0000,  0.0931]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.7548630109866\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.3795,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.4259],\n",
            "        [ 2.4125,  0.0000,  0.0000,  ...,  2.1273,  0.0000,  1.6958],\n",
            "        [ 0.8064,  0.0000,  1.2581,  ...,  9.8994,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.9508,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.8756,  0.4435],\n",
            "        [ 0.0000,  0.0000,  0.3842,  ...,  0.0000,  0.0000, 14.1479]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75510790308351\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.1400],\n",
            "        [ 0.4422,  0.8096,  0.4602,  ...,  0.0000,  1.3258,  0.0000],\n",
            "        [ 0.0181,  0.0000,  0.0000,  ...,  0.1970,  0.0000,  1.7673],\n",
            "        ...,\n",
            "        [ 0.1382,  0.0000,  0.7100,  ...,  0.7823,  0.0000,  1.6510],\n",
            "        [ 1.4398,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.0709],\n",
            "        [ 0.0000,  0.0000, 12.5018,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75535255053298\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.8139,  0.0000],\n",
            "        [ 0.6358,  0.0000,  0.0000,  ...,  0.0000,  0.7898, 14.6327],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 1.4073,  0.0000,  0.0000,  ...,  0.0000,  0.3491,  0.3049],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.2286,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.3690]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7555969535794\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.1739,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 1.9823, 0.0000, 1.7639],\n",
            "        [1.0282, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.6899],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.6018,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.2970, 1.9473]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75584111246694\n",
            "torch.Size([13, 34])\n",
            "tensor([[1.1333e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.3657e-01, 0.0000e+00,\n",
            "         0.0000e+00, 2.2622e+00, 3.8797e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.8629e-01, 1.4702e+01, 0.0000e+00, 6.0137e-01,\n",
            "         1.3524e-01, 0.0000e+00, 0.0000e+00, 2.1286e-01],\n",
            "        [9.4414e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3678e+01, 1.9042e+00,\n",
            "         0.0000e+00, 8.5080e-01, 5.9517e-01, 6.0807e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.8400e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         5.7938e-01, 0.0000e+00, 5.8890e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.6157e+00, 0.0000e+00, 6.0328e-01, 1.4267e+01,\n",
            "         0.0000e+00, 1.3617e+00, 0.0000e+00, 2.4867e-02, 0.0000e+00, 1.1351e+00,\n",
            "         0.0000e+00, 1.9299e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4364e+00, 0.0000e+00, 0.0000e+00, 1.2600e+00],\n",
            "        [6.9108e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.3121e+00, 2.6146e+00,\n",
            "         1.3303e+01, 0.0000e+00, 5.6516e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         8.5504e-01, 1.5166e+00, 1.0411e+00, 0.0000e+00, 5.4348e-01, 0.0000e+00,\n",
            "         1.6398e+00, 0.0000e+00, 0.0000e+00, 1.2011e+00],\n",
            "        [1.6089e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.3289e-01, 0.0000e+00,\n",
            "         2.2947e+00, 0.0000e+00, 8.2322e-01, 1.5294e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 4.7534e-01, 0.0000e+00, 0.0000e+00, 6.7111e-01, 7.3732e-01,\n",
            "         1.6295e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7185e+00, 0.0000e+00,\n",
            "         2.7105e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.1765e+00, 1.2098e+00, 0.0000e+00, 1.8806e+00, 0.0000e+00,\n",
            "         8.1683e-01, 0.0000e+00, 0.0000e+00, 1.5818e+01],\n",
            "        [7.7919e-01, 0.0000e+00, 6.6256e-01, 2.9390e+00, 9.7854e-02, 0.0000e+00,\n",
            "         0.0000e+00, 1.5425e+00, 8.3640e-01, 1.2697e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 6.9470e-01, 0.0000e+00, 8.7957e-01,\n",
            "         0.0000e+00, 6.0154e-01, 3.1541e-01, 0.0000e+00],\n",
            "        [1.3624e+01, 0.0000e+00, 1.4721e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.1219e+00, 1.1843e+00, 1.7379e+00, 0.0000e+00, 2.1630e+00,\n",
            "         0.0000e+00, 1.6139e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.3348e-01,\n",
            "         0.0000e+00, 1.7389e+00, 0.0000e+00, 1.3755e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 2.7344e-01, 0.0000e+00, 1.9262e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 7.0383e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 7.1959e-01, 0.0000e+00, 1.6818e+00,\n",
            "         5.9231e-01, 0.0000e+00, 1.6360e+01, 3.9413e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 9.3713e-01, 1.2813e+00, 0.0000e+00, 1.4639e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.8613e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2996e+01, 0.0000e+00, 2.2172e+00, 0.0000e+00],\n",
            "        [1.8784e+00, 0.0000e+00, 1.0628e+00, 0.0000e+00, 1.8300e-01, 0.0000e+00,\n",
            "         2.0978e+00, 0.0000e+00, 5.5318e-01, 2.3893e-03, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.9232e-01,\n",
            "         1.5292e+01, 0.0000e+00, 4.0127e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.7139e-01,\n",
            "         0.0000e+00, 5.7369e-01, 0.0000e+00, 1.3025e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.7580e-02, 0.0000e+00, 0.0000e+00],\n",
            "        [8.7431e-01, 0.0000e+00, 0.0000e+00, 3.7577e-01, 1.5134e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.4578e+01, 0.0000e+00, 0.0000e+00, 1.4990e-01,\n",
            "         0.0000e+00, 1.3120e+00, 0.0000e+00, 0.0000e+00, 8.8039e-01, 8.5601e-01,\n",
            "         1.1360e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7560850274395\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 1.1359,  ..., 0.0000, 0.2796, 3.0406],\n",
            "        [0.3268, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.3479],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 2.2825, 1.9276],\n",
            "        [0.4829, 0.0000, 0.0000,  ..., 0.8636, 0.0000, 2.3262],\n",
            "        [0.4028, 0.0000, 1.6827,  ..., 0.0000, 0.0000, 0.0041]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75632869874076\n",
            "torch.Size([64, 48])\n",
            "tensor([[18.5006,  0.0000,  0.0000,  ...,  0.0000,  0.1198,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 13.5617,  ...,  0.0000,  0.0000,  0.0549],\n",
            "        ...,\n",
            "        [ 0.1174,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.2816,  0.0000],\n",
            "        [ 0.6634,  0.0000,  0.0000,  ...,  0.0000,  0.0284,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75657212661415\n",
            "torch.Size([64, 54])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 4.5240e-01,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.4401e+01],\n",
            "        [1.3217e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [3.0150e-01, 0.0000e+00, 1.5260e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.5739e+01],\n",
            "        ...,\n",
            "        [0.0000e+00, 0.0000e+00, 1.6473e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.4807e+01],\n",
            "        [0.0000e+00, 1.0136e-02, 2.0099e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.2685e+01],\n",
            "        [9.2288e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         2.4513e-01]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.75525437224191\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.3207,  0.0000,  0.0000,  ...,  0.0000,  0.8048,  0.9254],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.5689],\n",
            "        [ 0.0708,  0.0000,  1.0801,  ..., 11.3874,  0.1027,  0.3970],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.6558,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.1626,  1.8099],\n",
            "        [ 0.0000,  0.0000,  0.2465,  ...,  0.0000,  0.0000, 14.7095]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75549887336854\n",
            "torch.Size([64, 59])\n",
            "tensor([[2.5300e-02, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.5674e-01,\n",
            "         1.1713e+00],\n",
            "        [9.9968e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.1024e-02,\n",
            "         7.8628e-01],\n",
            "        [2.2864e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.9502e+00],\n",
            "        ...,\n",
            "        [0.0000e+00, 0.0000e+00, 2.3305e-01,  ..., 2.2223e+00, 0.0000e+00,\n",
            "         1.5896e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.8728e+00],\n",
            "        [1.7658e+00, 0.0000e+00, 1.2578e+01,  ..., 4.4170e-01, 0.0000e+00,\n",
            "         0.0000e+00]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7557431302383\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 14.3442,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.0458,  ...,  0.0000,  0.0000, 15.2838],\n",
            "        [ 2.8113,  0.0000,  1.8494,  ...,  0.0000,  0.0000,  0.7528],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.9759,  1.0791],\n",
            "        [ 1.5476,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.2244],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.5010]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75598714309521\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.1594, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.9350],\n",
            "        [1.4795, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 3.6387],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.9617,  ..., 0.0000, 0.0561, 0.0000],\n",
            "        [1.5456, 0.0000, 1.5315,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.6404, 0.1867]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75623091218303\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 8.4060e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.9577e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.4116e+00, 0.0000e+00, 1.2751e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3855e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.8137e+01, 1.5317e+00,\n",
            "         8.7385e-01, 0.0000e+00, 4.8454e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.7701e-01, 3.8613e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 3.6176e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 2.5882e+00, 0.0000e+00, 0.0000e+00, 1.6144e+01,\n",
            "         0.0000e+00, 2.2131e-01, 0.0000e+00, 3.9707e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.3563e+00, 7.7850e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 9.8049e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5249e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.7119e+00, 5.3183e-01, 0.0000e+00, 0.0000e+00, 5.2773e-01, 1.2020e+00,\n",
            "         6.7656e-01, 0.0000e+00, 0.0000e+00, 2.8382e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 3.6995e-01, 1.3433e+00, 0.0000e+00,\n",
            "         9.1480e-01, 6.6055e-01, 0.0000e+00, 2.3741e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.3561e-01, 1.1708e-01,\n",
            "         1.2995e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4648e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 5.0014e+00, 1.2226e+00, 0.0000e+00, 0.0000e+00, 1.9662e-03,\n",
            "         6.7243e-01, 0.0000e+00, 0.0000e+00, 1.7405e+01],\n",
            "        [1.1060e+00, 0.0000e+00, 0.0000e+00, 1.0420e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.0566e+00, 1.3412e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5620e+00, 3.0946e-02, 1.4071e+00,\n",
            "         6.8577e-02, 0.0000e+00, 9.8087e-01, 0.0000e+00],\n",
            "        [1.4358e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6139e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.2491e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6667e+00,\n",
            "         3.3225e-01, 0.0000e+00, 6.6572e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.0366e-01, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.9908e+00, 1.9248e-01, 0.0000e+00,\n",
            "         0.0000e+00, 3.3003e-01, 3.2937e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.1392e+00, 0.0000e+00, 1.2512e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3744e+00, 0.0000e+00,\n",
            "         1.1098e-01, 4.0935e-01, 3.4930e-01, 2.8838e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.1010e-02, 0.0000e+00,\n",
            "         1.5328e+01, 0.0000e+00, 1.0369e+00, 0.0000e+00],\n",
            "        [1.2260e+00, 0.0000e+00, 5.8732e-01, 0.0000e+00, 0.0000e+00, 3.6249e-01,\n",
            "         1.8091e+00, 0.0000e+00, 7.6334e-01, 5.9983e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4436e+01, 0.0000e+00, 0.0000e+00, 6.7710e-02],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 2.1459e+00, 6.7582e-02, 0.0000e+00,\n",
            "         0.0000e+00, 1.1650e+00, 8.2889e-01, 2.2444e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.6266e-02, 0.0000e+00, 0.0000e+00, 1.4580e+01, 1.1182e-01, 4.9109e-02,\n",
            "         0.0000e+00, 0.0000e+00, 2.9425e-01, 0.0000e+00],\n",
            "        [1.3351e+00, 0.0000e+00, 0.0000e+00, 1.2918e-01, 6.9372e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.2470e+01, 2.9980e+00, 0.0000e+00, 0.0000e+00,\n",
            "         4.9191e-01, 0.0000e+00, 0.0000e+00, 1.6657e-01, 0.0000e+00, 6.8603e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75647443774528\n",
            "torch.Size([64, 57])\n",
            "tensor([[1.1228, 0.0000, 1.7257,  ..., 0.0000, 0.0000, 2.4501],\n",
            "        [0.3022, 0.0000, 0.0718,  ..., 0.0000, 1.1375, 0.0000],\n",
            "        [0.0644, 0.0000, 2.6148,  ..., 0.0084, 0.0000, 5.5984],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.2780,  ..., 0.0000, 0.0000, 0.0746],\n",
            "        [0.9723, 0.0000, 0.5475,  ..., 0.6098, 0.0000, 2.1803],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75671772002526\n",
            "torch.Size([64, 48])\n",
            "tensor([[14.7306,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.2082],\n",
            "        [ 0.2449,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 14.6245,  ...,  0.0000,  0.0000,  0.6569],\n",
            "        ...,\n",
            "        [ 0.7955,  0.0000,  0.0000,  ...,  0.0000,  0.1548,  0.4762],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  9.8710,  0.0357],\n",
            "        [ 1.8655,  0.0000,  0.0000,  ...,  0.0000,  1.0939,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.756960759266\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.4289,  ...,  0.0000,  0.9409, 13.1980],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  2.1728,  0.0000],\n",
            "        [ 0.1549,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 13.2469],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.5569,  ...,  1.0886,  0.0000, 13.4938],\n",
            "        [ 0.0000,  0.0000,  0.6016,  ...,  1.3991,  0.0000, 15.0062],\n",
            "        [ 2.2996,  0.0000,  0.0000,  ...,  0.0000,  0.4407,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75720355571028\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.4886,  0.0000,  1.9848,  ...,  0.0000,  0.7950,  0.0000],\n",
            "        [ 1.1484,  0.0000,  0.0000,  ...,  0.5424,  0.0000,  0.7171],\n",
            "        [ 0.0000,  0.0000,  2.0334,  ..., 10.2532,  0.0000,  1.5740],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.5382,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 11.6313,  1.7120],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 13.5604]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75744610960068\n",
            "torch.Size([64, 59])\n",
            "tensor([[1.3478e+00, 0.0000e+00, 1.4032e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 1.0731e+00, 0.0000e+00,\n",
            "         8.5368e-01],\n",
            "        ...,\n",
            "        [0.0000e+00, 0.0000e+00, 8.9744e-01,  ..., 0.0000e+00, 3.4756e-01,\n",
            "         0.0000e+00],\n",
            "        [5.1804e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 9.8744e-03,\n",
            "         9.4335e-01],\n",
            "        [1.2787e-01, 0.0000e+00, 1.3897e+01,  ..., 0.0000e+00, 4.8922e-01,\n",
            "         0.0000e+00]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7576884211795\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 16.7294,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 14.4176],\n",
            "        [ 0.0000,  0.0000,  0.7117,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0838,  0.0000,  0.0000,  ...,  0.0000,  0.8569,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.0035,  0.0000,  0.0000,  ...,  0.0000,  0.6409,  0.2792]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7579304906888\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 1.6671,  ..., 0.0000, 0.5252, 0.0000],\n",
            "        [2.4330, 0.0000, 0.0000,  ..., 0.6772, 0.0000, 3.8081],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 1.1180, 0.0000, 1.0251],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.7849, 0.0000, 0.0000,  ..., 0.0000, 0.5157, 0.0000],\n",
            "        [0.0000, 0.0000, 0.2671,  ..., 0.0000, 0.4381, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75817231837044\n",
            "torch.Size([13, 34])\n",
            "tensor([[ 0.0000,  0.0000,  0.3391,  0.0000,  0.6820,  0.0000,  0.8605,  0.2428,\n",
            "          0.0000,  1.4239,  0.0000,  0.0000,  0.0000,  0.0000,  0.4671, 15.1238,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.3007,  0.0000],\n",
            "        [ 1.2081,  0.1995,  0.0000,  0.2725, 14.9534,  0.0000,  0.0000,  0.0000,\n",
            "          1.8350,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.4106,  0.0000,  0.0000,  1.7097],\n",
            "        [ 0.0000,  0.0000,  0.4803,  0.4273,  0.0000, 13.4208,  0.0000,  1.4365,\n",
            "          0.0000,  0.0000,  0.0000,  0.1512,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.4158,  0.0000,  2.2708,  0.0000,  0.0000,  1.8371],\n",
            "        [ 0.5231,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000, 12.8088,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.8126,  1.9225,  0.0000,  2.5634,\n",
            "          1.8705,  0.0000,  0.7058,  0.0000,  0.0000,  1.2650],\n",
            "        [ 0.5120,  0.0000,  0.0000,  0.0000,  0.1389,  0.0000,  0.9547,  0.0000,\n",
            "          0.0000,  0.9308,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000, 15.7199,  0.0203,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  3.0665,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  2.0100,  1.9475,  0.0000,\n",
            "          0.0000,  0.8090,  0.0000,  0.0000,  0.0000, 15.5697],\n",
            "        [ 1.3409,  0.0000,  0.0000,  2.6368,  2.8629,  0.0000,  0.6156,  1.3907,\n",
            "          0.0483, 13.0025,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.3910,\n",
            "          0.0000,  2.1142,  0.3585,  0.0000,  0.0000,  0.0000],\n",
            "        [15.6436,  0.0000,  0.0000,  0.0000,  0.2995,  0.0000,  0.0000,  1.2836,\n",
            "          1.7679,  0.0000,  0.0000,  1.6879,  0.0000,  0.8758,  0.2462,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  3.6577,  0.0000,  0.0000],\n",
            "        [ 0.2601,  0.0000,  0.0000,  0.0000,  3.3148,  0.0000,  0.0000,  0.0000,\n",
            "          0.5068,  0.0000,  0.0000,  0.0000,  0.1850,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  1.3378,  0.6058,  0.0000, 14.7562,  0.0493],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.9401,  1.2891,  2.0947,  0.6314,\n",
            "          0.4852,  0.0000,  0.0000,  0.0000,  0.0000,  0.3927,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000, 16.8013,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.4266,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          1.0485,  0.0000,  0.0000,  0.0000,  0.0643,  0.4120,  0.0000,  0.0000,\n",
            "          0.0000,  0.3375, 12.6726,  0.0000,  1.9437,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.2266,  0.0000,  0.0000,  0.6527,\n",
            "          0.0000,  0.2307,  0.0000,  0.0000,  0.2974,  1.4183,  0.0000, 14.7202,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0222,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0587,  2.9324,  0.0000,  0.0000,  0.0000,\n",
            "         14.9103,  0.9058,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          1.1195,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75841390446598\n",
            "torch.Size([64, 57])\n",
            "tensor([[1.4866, 0.0000, 1.0828,  ..., 0.0000, 0.6552, 3.9884],\n",
            "        [1.0290, 0.0000, 0.0000,  ..., 0.0000, 0.6718, 0.0000],\n",
            "        [0.0000, 0.0000, 1.0749,  ..., 0.4501, 0.0000, 2.1675],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0442,  ..., 0.0000, 0.2004, 0.0000],\n",
            "        [1.4369, 0.0000, 0.3023,  ..., 0.1738, 0.0000, 2.6880],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75865524921676\n",
            "torch.Size([64, 48])\n",
            "tensor([[15.9884,  0.0000,  0.0992,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.0250,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 13.6816,  ...,  2.4607,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.8084],\n",
            "        [ 1.1452,  0.0000,  0.0000,  ...,  0.0000, 16.6503,  1.1810],\n",
            "        [ 1.2652,  0.0000,  0.0000,  ...,  0.0000,  1.3685,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7588963528639\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  2.2497,  ...,  0.5031,  0.0000, 13.1924],\n",
            "        [ 0.9604,  0.0000,  0.0229,  ...,  0.0000,  1.8433,  0.0000],\n",
            "        [ 0.5333,  0.0000,  0.1716,  ...,  0.0195,  0.0000, 15.7721],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.9741,  ...,  0.0000,  0.0000, 17.8323],\n",
            "        [ 0.0000,  0.0000,  1.4781,  ...,  0.3772,  0.0382, 13.6253],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.75757627658732\n",
            "torch.Size([64, 73])\n",
            "tensor([[1.5008e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.0766e-02],\n",
            "        [3.9986e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.6266e+00],\n",
            "        [4.3886e-01, 0.0000e+00, 1.4682e-02,  ..., 1.0895e+01, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        ...,\n",
            "        [0.0000e+00, 0.0000e+00, 2.3915e-01,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.3703e+01,\n",
            "         1.1314e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.8933e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.5210e+01]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75781845812918\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.6980,  ...,  0.7779,  0.5946,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.1486,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.9259,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.9103,  ...,  0.4847,  0.0000,  1.2085],\n",
            "        [ 0.7906,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.6841],\n",
            "        [ 0.0000,  0.0000, 15.0498,  ...,  0.7930,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75806039773144\n",
            "torch.Size([64, 75])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.5190e+01,\n",
            "         0.0000e+00],\n",
            "        [4.5754e-01, 0.0000e+00, 6.6654e-01,  ..., 0.0000e+00, 7.3304e-03,\n",
            "         1.2133e+01],\n",
            "        [1.6129e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.0046e+00,\n",
            "         0.0000e+00],\n",
            "        ...,\n",
            "        [9.1394e-01, 0.0000e+00, 2.6265e-01,  ..., 0.0000e+00, 2.3166e+00,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.2517e+00,\n",
            "         0.0000e+00],\n",
            "        [2.7740e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7583020956358\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 1.0171,  ..., 0.0000, 0.3479, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 4.1409],\n",
            "        [0.5482, 0.0000, 0.0000,  ..., 1.8062, 0.0000, 0.0232],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.5495, 0.0000],\n",
            "        [1.5942, 0.0000, 1.1911,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.2410, 0.0000, 1.1838,  ..., 0.0000, 1.5233, 0.6005]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75854355208371\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0847e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.1820e-02,\n",
            "         0.0000e+00, 1.7705e+00, 0.0000e+00, 1.5188e+01, 2.0092e-01, 0.0000e+00,\n",
            "         3.6609e-01, 0.0000e+00, 0.0000e+00, 9.2826e-01],\n",
            "        [8.1686e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5390e+01, 2.5411e-01,\n",
            "         8.1847e-01, 0.0000e+00, 8.1969e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 6.8834e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.6435e+00, 3.0104e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 6.2770e-01, 0.0000e+00, 2.4477e+00, 1.3433e+01,\n",
            "         1.3549e+00, 7.4094e-01, 0.0000e+00, 1.2420e+00, 0.0000e+00, 4.1164e-02,\n",
            "         0.0000e+00, 0.0000e+00, 2.0417e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.1313e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.6028e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.0715e+00, 1.6206e+00, 5.4999e-01, 3.5269e+00, 8.5807e-01, 0.0000e+00,\n",
            "         2.1260e+00, 9.2205e-01, 0.0000e+00, 1.5572e+00],\n",
            "        [5.1661e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.1916e+00, 0.0000e+00,\n",
            "         2.0804e+00, 0.0000e+00, 1.1671e+00, 1.1653e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.3158e+00,\n",
            "         2.0105e+01, 0.0000e+00, 8.0274e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4507e-02, 0.0000e+00,\n",
            "         3.3836e+00, 0.0000e+00, 3.1705e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.5852e+00, 7.6632e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5197e+01],\n",
            "        [2.0540e-01, 0.0000e+00, 5.1682e-01, 1.4092e+00, 8.8401e-01, 0.0000e+00,\n",
            "         0.0000e+00, 1.6142e+00, 5.1668e-01, 1.3456e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 5.7907e-02, 0.0000e+00, 2.7958e-01,\n",
            "         0.0000e+00, 0.0000e+00, 6.1929e-02, 0.0000e+00],\n",
            "        [1.6032e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.2362e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.3694e+00, 0.0000e+00, 0.0000e+00, 1.0326e+00,\n",
            "         1.0538e+00, 0.0000e+00, 2.6021e-01, 0.0000e+00, 2.9451e-01, 1.5835e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1130e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 5.6354e-01, 6.3373e-01, 6.2137e-01,\n",
            "         1.7275e+00, 0.0000e+00, 0.0000e+00, 5.8422e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.3934e+01, 4.3171e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.9349e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2796e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.7034e+01, 0.0000e+00, 1.1357e+00, 8.1691e-02],\n",
            "        [0.0000e+00, 0.0000e+00, 5.2317e-02, 0.0000e+00, 1.6728e-01, 8.9973e-01,\n",
            "         8.3175e-01, 0.0000e+00, 0.0000e+00, 1.1114e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.2028e-01, 6.1781e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3025e+00,\n",
            "         1.4694e+01, 0.0000e+00, 8.2899e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.3229e-01,\n",
            "         7.0902e-02, 1.6001e-01, 0.0000e+00, 1.8429e-01, 0.0000e+00, 1.0391e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3502e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 6.5790e-01, 0.0000e+00],\n",
            "        [1.4748e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.8915e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6439e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75878476731639\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.4921,  ..., 0.0000, 0.0000, 3.9487],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.6727, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.9409, 0.0000, 2.6759],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0488,  ..., 0.0000, 0.9978, 0.0000],\n",
            "        [1.2548, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.8112],\n",
            "        [0.7273, 0.0000, 2.1815,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75902574157482\n",
            "torch.Size([64, 48])\n",
            "tensor([[16.9577,  0.0000,  0.8453,  ...,  0.0000,  0.0000,  0.4998],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 13.7837,  ...,  0.5009,  0.0000,  0.4058],\n",
            "        ...,\n",
            "        [ 0.7420,  0.0000,  0.0899,  ...,  0.0000,  0.0000,  1.5960],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 12.2949,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75926647509972\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.5735, 12.2064],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.9681],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.2896, 12.7551],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  3.8493,  ...,  0.0000,  0.0000, 15.9314],\n",
            "        [ 0.0000,  0.0000,  1.5768,  ...,  0.0000,  0.0000, 14.7262],\n",
            "        [ 0.5467,  0.0000,  0.0000,  ...,  0.0000,  0.4457,  0.6053]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75950696813159\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 1.5149,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.6696,  0.0000,  0.0000,  ...,  1.0790,  0.0000,  0.0000],\n",
            "        [ 0.0633,  0.0000,  0.1854,  ...,  9.2481,  0.2759,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.9408,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 12.7933,  0.6663],\n",
            "        [ 0.0000,  0.0000,  0.3346,  ...,  0.0000,  0.0000, 16.7312]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75974722091068\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0832,  ...,  0.0000,  0.2742,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.6832,  ...,  0.0000,  2.2267,  0.0000],\n",
            "        [ 0.0329,  0.0000,  0.0000,  ...,  0.5380,  0.0000,  1.2059],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  3.8057],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.1256],\n",
            "        [ 0.0000,  0.0000, 15.2283,  ...,  0.3999,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75998723367701\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.1489,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.5612,  ...,  0.4818,  0.0000, 14.6304],\n",
            "        [ 0.8133,  0.0000,  1.5925,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.6113,  ...,  0.0000,  1.4837,  0.0000],\n",
            "        [ 0.9186,  0.0000,  0.7309,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.1296,  0.0000,  0.0000,  ...,  0.0000,  0.3200,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76022700667033\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.0603,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.7654, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 4.6605],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.6196],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.1121,  ..., 0.0000, 0.0000, 1.2368],\n",
            "        [0.7338, 0.0000, 0.0000,  ..., 0.0000, 0.1735, 0.0000],\n",
            "        [0.0670, 0.0000, 0.0000,  ..., 0.0000, 2.1316, 0.1862]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76046654013021\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4050e-01, 9.3647e-01, 0.0000e+00,\n",
            "         1.9442e-01, 1.0713e-01, 0.0000e+00, 3.1793e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.8225e-01, 0.0000e+00, 7.3233e-01, 1.5013e+01, 1.6667e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [2.1293e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5157e+01, 1.9004e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.5052e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.7085e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.9788e+00, 0.0000e+00, 0.0000e+00, 1.1485e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.7695e+00, 0.0000e+00, 0.0000e+00, 1.5160e+01,\n",
            "         7.9810e-01, 1.7640e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.1204e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.3089e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.1080e+00, 0.0000e+00, 2.0403e-01, 0.0000e+00],\n",
            "        [6.7130e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2271e+00, 0.0000e+00,\n",
            "         1.6406e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4563e+00, 0.0000e+00, 0.0000e+00, 1.9623e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.6574e+00, 1.0954e+00, 0.0000e+00, 3.3338e+00],\n",
            "        [1.0541e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.0417e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.9199e-01,\n",
            "         1.4301e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.3638e-01, 3.2648e-01,\n",
            "         2.7224e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.5172e-01,\n",
            "         0.0000e+00, 2.7564e+00, 1.3366e-01, 0.0000e+00, 3.4261e-01, 2.7368e-03,\n",
            "         0.0000e+00, 2.1192e-01, 0.0000e+00, 1.6380e+01],\n",
            "        [4.4880e-01, 0.0000e+00, 7.0077e-01, 0.0000e+00, 6.1186e-01, 6.6914e-02,\n",
            "         0.0000e+00, 7.0385e-01, 8.9222e-01, 1.4213e+01, 0.0000e+00, 0.0000e+00,\n",
            "         2.1406e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.4587e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.0161e-01, 0.0000e+00],\n",
            "        [1.4974e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.8493e+00, 0.0000e+00,\n",
            "         0.0000e+00, 5.0655e-01, 0.0000e+00, 2.4359e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.7851e+00, 5.4104e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.9960e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 8.1053e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 3.7888e-01, 0.0000e+00, 8.0058e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 5.0083e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.0861e+00, 7.2861e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.3682e-01,\n",
            "         0.0000e+00, 0.0000e+00, 1.6925e+01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.9920e-01, 0.0000e+00,\n",
            "         6.8282e-01, 0.0000e+00, 1.4259e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.7988e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5056e+01, 0.0000e+00, 1.4906e+00, 9.3732e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 5.4803e-01, 0.0000e+00, 1.5840e-01, 3.3736e-01,\n",
            "         2.4273e+00, 0.0000e+00, 1.1678e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.9940e-01, 4.2078e-01,\n",
            "         1.5047e+01, 0.0000e+00, 3.0435e+00, 4.7211e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2752e-01, 2.0251e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.1278e-01,\n",
            "         0.0000e+00, 1.4493e+00, 7.6703e-01, 1.4407e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.9565e-02, 2.1785e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 2.1013e-01, 0.0000e+00, 3.5959e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6074e+01, 7.1763e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 2.9205e-01]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76070583429592\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0532, 2.7562],\n",
            "        [0.9080, 0.0000, 0.0391,  ..., 0.0000, 0.5933, 0.0000],\n",
            "        [0.0000, 0.0000, 1.4735,  ..., 0.0000, 0.0000, 1.3653],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.6874,  ..., 0.0000, 0.9360, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.5675, 0.0000, 1.0915],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.4013, 1.4292]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76094488940652\n",
            "torch.Size([64, 48])\n",
            "tensor([[16.3720,  0.0000,  0.2249,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.4718,  0.0000,  0.0000,  ...,  0.0000,  0.3382,  0.0000],\n",
            "        [ 0.0423,  0.0000, 15.2836,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.5832,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 12.4963,  0.0000],\n",
            "        [ 0.5150,  0.0000,  0.0000,  ...,  0.0000,  2.3850,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76118370570082\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0816,  0.0000,  1.9577,  ...,  0.0000,  0.0000, 15.1552],\n",
            "        [ 0.5613,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.8005],\n",
            "        [ 0.0000,  0.0000,  1.1913,  ...,  0.1256,  0.0000, 11.2160],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.9459,  ...,  0.3502,  0.0000, 13.8958],\n",
            "        [ 0.0000,  0.0000,  0.7529,  ...,  0.0000,  0.4902, 15.7080],\n",
            "        [ 1.4984,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  96.875\n",
            "acc:  99.75830040529551\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 2.8267,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 3.5464,  0.0000,  0.0000,  ...,  1.7206,  0.0000,  1.7325],\n",
            "        [ 2.2994,  0.0000,  2.2801,  ..., 11.8555,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.4475,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.6613,  ...,  0.0000, 13.5507,  0.4774],\n",
            "        [ 0.0000,  0.0000,  0.4824,  ...,  0.0000,  0.5336, 13.1871]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75854186343209\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 1.4157,  0.0000,  0.9065,  ...,  0.0000,  0.0000,  1.0531],\n",
            "        [ 0.0269,  0.0000,  0.2567,  ...,  0.0000,  0.9077,  0.0000],\n",
            "        [ 0.2173,  0.0000,  0.0000,  ...,  0.6913,  0.0000,  3.2348],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.9674,  ...,  0.7266,  0.0000,  3.9461],\n",
            "        [ 1.7148,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.8506],\n",
            "        [ 0.0000,  0.0000, 14.9991,  ...,  0.0000,  0.3519,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75878308035173\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0496,  ...,  0.0000, 13.5700,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.1766,  ...,  0.0000,  0.3793, 14.9411],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.0868,  0.0000],\n",
            "        ...,\n",
            "        [ 0.8762,  0.0000,  0.1311,  ...,  0.0000,  0.0000,  0.5826],\n",
            "        [ 2.2806,  0.0000,  0.0000,  ...,  0.0000,  0.7072,  0.0000],\n",
            "        [ 1.2665,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75902405629544\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.4486,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0401, 0.0000, 0.0000,  ..., 1.0967, 0.0000, 3.1622],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.1171, 0.0000, 1.1283],\n",
            "        ...,\n",
            "        [0.4200, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.3766],\n",
            "        [0.0000, 0.0000, 1.0607,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.8870, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75926479150394\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         8.9320e-03, 0.0000e+00, 2.7959e-01, 1.8487e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 7.8917e-01, 0.0000e+00, 1.6279e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.2357e+00, 0.0000e+00, 0.0000e+00, 8.7686e-01],\n",
            "        [1.5318e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6791e+01, 2.6745e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.1229e+00, 9.4498e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         4.5191e-01, 0.0000e+00, 1.7718e+00, 1.4558e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 2.2607e+00, 0.0000e+00, 1.1419e+00, 1.4919e+01,\n",
            "         4.5065e-01, 1.1982e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.9179e-01,\n",
            "         0.0000e+00, 0.0000e+00, 9.8569e-01, 3.6559e-01, 0.0000e+00, 0.0000e+00,\n",
            "         5.1564e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [6.3850e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.4014e-02, 5.1814e-01,\n",
            "         1.5452e+01, 0.0000e+00, 4.1860e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.5056e+00, 0.0000e+00, 0.0000e+00, 2.5323e+00, 0.0000e+00,\n",
            "         1.5274e+00, 0.0000e+00, 0.0000e+00, 1.5277e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.8598e+00, 0.0000e+00, 0.0000e+00, 1.8131e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2073e+00, 0.0000e+00, 1.3219e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0877e+00,\n",
            "         1.5181e+01, 0.0000e+00, 7.0153e-01, 0.0000e+00],\n",
            "        [2.1526e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7503e+00, 0.0000e+00,\n",
            "         4.1271e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 5.9189e+00, 6.5103e-01, 0.0000e+00, 0.0000e+00, 4.2410e-02,\n",
            "         4.7438e-01, 0.0000e+00, 0.0000e+00, 1.6365e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 4.1710e-01, 2.8681e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 7.1811e-01, 1.0151e+00, 1.3587e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0243e-01, 0.0000e+00, 0.0000e+00,\n",
            "         7.6810e-01, 0.0000e+00, 5.8230e-01, 0.0000e+00],\n",
            "        [1.4088e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.3638e+00, 0.0000e+00,\n",
            "         0.0000e+00, 7.6243e-01, 9.9048e-01, 1.0033e+00, 0.0000e+00, 7.8199e-01,\n",
            "         5.4143e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 7.5669e-02, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 3.5251e-01, 1.9108e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 7.5821e-01, 4.3341e-01, 0.0000e+00, 0.0000e+00, 5.1681e-01,\n",
            "         0.0000e+00, 4.3167e-02, 0.0000e+00, 3.5100e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.4854e+01, 2.4568e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 5.4640e-01, 2.6972e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 2.6716e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.1105e-01, 0.0000e+00,\n",
            "         1.4258e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 2.4962e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.5561e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0512e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 7.0137e-01, 1.6342e-01, 4.3598e-01,\n",
            "         1.3635e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.7398e-01, 0.0000e+00,\n",
            "         7.3835e-01, 6.4693e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5433e+00,\n",
            "         0.0000e+00, 4.8503e-01, 7.4273e-01, 1.5230e+01, 7.8007e-01, 0.0000e+00,\n",
            "         1.0380e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.6701e+00, 0.0000e+00, 1.3092e+00, 4.1509e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6824e+01, 1.6471e+00, 2.4501e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5131e+00,\n",
            "         5.7463e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75950528621772\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 2.2908,  ..., 0.0000, 0.0000, 4.0024],\n",
            "        [0.0000, 0.0000, 0.7523,  ..., 0.0000, 0.2933, 0.0000],\n",
            "        [0.8674, 0.0000, 0.5570,  ..., 0.0000, 0.0000, 2.5224],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.2326,  ..., 0.3901, 0.0000, 0.0000],\n",
            "        [1.5208, 0.0000, 0.2143,  ..., 0.0000, 0.0000, 3.6520],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.4359, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75974554067705\n",
            "torch.Size([64, 48])\n",
            "tensor([[15.3962,  0.0000,  0.3803,  ...,  0.8678,  0.0000,  0.0000],\n",
            "        [ 1.6705,  0.0000,  0.0000,  ...,  0.0000,  1.3424,  0.0000],\n",
            "        [ 0.0000,  0.0000, 15.2950,  ...,  0.2561,  0.0000,  0.4346],\n",
            "        ...,\n",
            "        [ 0.8624,  0.0000,  0.2409,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.1045,  1.1868],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75998555512193\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.7801,  0.0000,  1.1619,  ...,  0.1412,  1.0945, 13.8626],\n",
            "        [ 0.0000,  0.0000,  0.5503,  ...,  0.0000,  0.9036,  0.0000],\n",
            "        [ 0.0603,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 14.0843],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.7029,  ...,  0.0000,  0.6122, 14.7627],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3602, 14.2387],\n",
            "        [ 1.2695,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.7586643907312\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.4644,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 2.2241,  0.0000,  0.0000,  ...,  0.1923,  0.0000,  1.9020],\n",
            "        [ 1.7560,  0.0000,  1.1850,  ...,  9.4036,  1.2165,  0.1797],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.4537,  ...,  0.0000, 14.6397,  0.0816],\n",
            "        [ 0.0000,  0.0000,  0.9342,  ...,  0.0000,  0.0000, 14.9549]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75890548524595\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.4642,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.4369],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.1398,  0.0000,  2.0582],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.4488,  1.0613],\n",
            "        [ 0.0000,  0.0000, 13.7263,  ...,  0.1704,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75914633890704\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 16.1631,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.6113,  ...,  0.0000,  0.0000, 14.6177],\n",
            "        [ 0.0000,  0.0000,  0.4390,  ...,  0.0000,  0.2331,  0.0000],\n",
            "        ...,\n",
            "        [ 0.5708,  0.0000,  0.0000,  ...,  0.0000,  1.2263,  1.0316],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.2506,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.5144]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7593869519551\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 2.5011,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [2.0031, 0.0000, 0.4490,  ..., 1.2964, 0.0000, 1.6114],\n",
            "        [1.3763, 0.0000, 0.0000,  ..., 0.6967, 0.0000, 3.2744],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0980, 0.0000, 0.6541,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 1.2863, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75962732463046\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         9.5752e-02, 0.0000e+00, 2.0671e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4784e+01, 1.6248e-01, 4.8520e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 6.4526e-01],\n",
            "        [2.1200e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3492e+01, 3.1274e-01,\n",
            "         1.3060e-01, 0.0000e+00, 0.0000e+00, 1.5367e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.6789e-01, 2.7610e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 3.9110e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 1.8155e+00, 0.0000e+00, 7.9564e-01, 1.4525e+01,\n",
            "         0.0000e+00, 2.1806e+00, 0.0000e+00, 8.9082e-01, 0.0000e+00, 1.8036e+00,\n",
            "         0.0000e+00, 0.0000e+00, 4.3284e-01, 0.0000e+00, 1.2292e-01, 0.0000e+00,\n",
            "         1.7077e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [6.8415e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.8803e-01,\n",
            "         1.6627e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.8337e+00, 3.3799e+00, 1.7152e+00, 4.7455e-01, 0.0000e+00, 5.2363e-01,\n",
            "         2.4089e+00, 2.4709e-01, 3.3129e-01, 2.9129e+00],\n",
            "        [2.0249e-01, 0.0000e+00, 0.0000e+00, 1.7891e+00, 2.7808e-01, 0.0000e+00,\n",
            "         7.5674e-01, 0.0000e+00, 0.0000e+00, 2.1805e+00, 0.0000e+00, 7.9050e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 8.1872e-01, 2.4622e+00,\n",
            "         1.5928e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         7.9567e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.1334e-01,\n",
            "         0.0000e+00, 2.0127e+00, 8.6477e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4767e+00, 0.0000e+00, 0.0000e+00, 1.5435e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 4.0866e-01, 4.8749e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 7.3942e-01, 1.2100e+00, 1.3268e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.1580e+00, 0.0000e+00, 0.0000e+00, 1.3627e+00, 0.0000e+00, 2.6796e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.4456e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.4968e+00, 0.0000e+00, 5.9961e-01, 0.0000e+00, 0.0000e+00, 2.7483e+00,\n",
            "         1.6772e+00, 0.0000e+00, 1.4754e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.2855e-01, 8.2345e-01, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6918e+00, 9.1342e-01,\n",
            "         0.0000e+00, 0.0000e+00, 5.7163e-01, 2.5499e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.4713e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3153e+00,\n",
            "         9.7216e-01, 0.0000e+00, 1.2454e+01, 2.1219e+00],\n",
            "        [1.1126e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7878e+00, 2.0733e+00,\n",
            "         5.8318e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.4033e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5224e+01, 0.0000e+00, 1.5716e-01, 1.8899e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 5.5673e-01, 6.0311e-03, 0.0000e+00, 0.0000e+00,\n",
            "         2.7029e+00, 0.0000e+00, 0.0000e+00, 8.5220e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.5363e-01, 0.0000e+00, 2.4944e-01, 1.0150e+00, 0.0000e+00,\n",
            "         1.1056e+01, 2.3681e-01, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 4.6249e-01, 7.2031e-01, 0.0000e+00,\n",
            "         3.2018e-01, 0.0000e+00, 0.0000e+00, 5.5139e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 5.3807e-01, 9.5304e-01, 1.2690e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 3.0877e-01, 0.0000e+00, 1.0454e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.4195e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.4516e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 2.0744e-01]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75986745717329\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.1370, 0.0000, 0.3697,  ..., 0.0000, 0.0000, 2.7343],\n",
            "        [0.3305, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.8036,  ..., 0.0000, 0.0000, 2.9254],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.6009, 0.0000],\n",
            "        [0.0000, 0.0000, 0.1610,  ..., 2.2302, 0.0000, 2.0248],\n",
            "        [0.0000, 0.0000, 1.3845,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76010734982346\n",
            "torch.Size([64, 48])\n",
            "tensor([[13.9589,  0.0000,  0.6618,  ...,  0.0180,  0.0000,  0.2756],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.3319,  0.0000],\n",
            "        [ 0.0000,  0.0000, 16.2494,  ...,  1.5580,  0.0000,  0.4961],\n",
            "        ...,\n",
            "        [ 1.2748,  0.0000,  1.5859,  ...,  0.0000,  0.0000,  0.7305],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.0243,  0.0000],\n",
            "        [ 1.0148,  0.0000,  0.0000,  ...,  0.0000,  1.0368,  0.6291]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76034700282064\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.1890,  2.9763,  ...,  0.0000,  0.6688, 12.1636],\n",
            "        [ 0.0000,  0.0000,  0.7871,  ...,  0.0000,  0.0723,  0.1932],\n",
            "        [ 0.0000,  0.0000,  0.8396,  ...,  0.0000,  2.5078, 13.3540],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  2.4623,  ...,  0.0000,  0.0000, 12.5574],\n",
            "        [ 0.0000,  0.0000,  0.4556,  ...,  0.0000,  0.0000, 13.0653],\n",
            "        [ 0.4527,  0.0000,  0.0000,  ...,  0.0000,  0.3344,  0.0556]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.7590254773433\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  2.6812,  0.2070],\n",
            "        [ 0.7373,  0.0000,  0.0000,  ...,  1.7760,  0.0000,  1.4742],\n",
            "        [ 0.0000,  0.0000,  1.1248,  ..., 10.4408,  0.0000,  1.5946],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.8028,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.1116,  ...,  0.0000, 15.7224,  2.3868],\n",
            "        [ 0.0000,  0.0000,  1.2626,  ...,  0.0000,  0.0000, 14.2387]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75926621113217\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  2.9654,  ...,  1.8587,  0.1847,  0.0000],\n",
            "        [ 1.9329,  0.0000,  0.4969,  ...,  0.0000,  1.2430,  0.0000],\n",
            "        [ 1.2659,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.8025],\n",
            "        ...,\n",
            "        [ 1.1836,  0.0000,  2.4631,  ...,  0.0000,  0.0000,  0.5934],\n",
            "        [ 0.7717,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  1.9658],\n",
            "        [ 0.0000,  0.0000, 15.6356,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75950670442775\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0995,  0.0000,  0.0000,  ...,  0.0000, 13.8063,  0.3111],\n",
            "        [ 0.0000,  0.0000,  0.2639,  ...,  0.0000,  0.0000, 14.2663],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.5240,  0.0000,  0.0000,  ...,  0.0000,  0.0937,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.5873,  ...,  0.3964,  0.6856,  0.1765],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0522,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75974695747028\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 1.7542,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.4606, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.6343],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 3.4346],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.5691,  ..., 0.0000, 0.3361, 0.0000],\n",
            "        [0.5152, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 1.0542,  ..., 0.0000, 2.8381, 1.1570]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75998697049978\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         4.9147e-01, 1.4751e+00, 7.9969e-01, 7.4059e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.4193e+00, 0.0000e+00, 1.5496e+01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7311e-01],\n",
            "        [4.5269e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6292e+01, 4.4300e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.9811e-01, 1.1139e+00, 0.0000e+00, 1.9209e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.7261e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 1.3299e+00, 0.0000e+00, 8.3417e-01, 1.5679e+01,\n",
            "         0.0000e+00, 2.1830e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.5416e-01,\n",
            "         0.0000e+00, 0.0000e+00, 2.0645e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         6.0909e-01, 0.0000e+00, 0.0000e+00, 1.8020e+00],\n",
            "        [1.2545e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.3993e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.7479e-01,\n",
            "         1.0205e+00, 1.8980e+00, 0.0000e+00, 6.1904e-01, 2.8931e-01, 0.0000e+00,\n",
            "         2.1729e+00, 1.0491e+00, 0.0000e+00, 5.6549e-01],\n",
            "        [8.0701e-01, 0.0000e+00, 0.0000e+00, 5.1599e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.4802e+00, 0.0000e+00, 8.5280e-01, 1.1741e+00, 0.0000e+00, 1.7284e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7125e+00,\n",
            "         1.4811e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.0359e-01,\n",
            "         1.9056e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.9701e+00, 0.0000e+00, 0.0000e+00, 2.0389e-01, 2.3224e-01,\n",
            "         3.7016e-01, 2.8668e-01, 0.0000e+00, 1.6583e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 8.5523e-01, 0.0000e+00, 0.0000e+00,\n",
            "         1.2464e+00, 0.0000e+00, 6.6380e-01, 1.5314e+01, 0.0000e+00, 0.0000e+00,\n",
            "         7.5621e-01, 0.0000e+00, 0.0000e+00, 2.0390e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 7.0312e-01, 0.0000e+00],\n",
            "        [1.4891e+01, 0.0000e+00, 2.4730e+00, 0.0000e+00, 1.3748e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 4.0377e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.0126e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1345e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 6.9136e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0501e+00, 0.0000e+00,\n",
            "         5.5639e-02, 0.0000e+00, 1.4502e+01, 7.3121e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1846e+00, 2.3628e+00,\n",
            "         2.7310e-01, 0.0000e+00, 0.0000e+00, 6.7712e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.4147e-01, 0.0000e+00, 0.0000e+00, 2.4626e-01, 0.0000e+00,\n",
            "         1.5945e+01, 0.0000e+00, 6.9919e-01, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 2.3338e-01, 0.0000e+00, 2.1380e-01,\n",
            "         2.2370e+00, 0.0000e+00, 5.8854e-01, 0.0000e+00, 0.0000e+00, 1.5543e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4950e-01,\n",
            "         1.5826e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 5.6967e-01, 2.7587e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7070e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2028e+00, 0.0000e+00, 0.0000e+00, 1.4897e+01, 1.4639e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.3503e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7592e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.5909e+01, 1.6922e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.2354e+00, 0.0000e+00, 5.5513e-01, 1.0156e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76022674375602\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.0040,  ..., 0.0000, 0.0000, 1.4461],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.7437],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.7635, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.3769, 0.0000, 3.2998],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76046627747854\n",
            "torch.Size([64, 48])\n",
            "tensor([[1.4880e+01, 0.0000e+00, 3.2018e-01,  ..., 2.3038e-01, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [4.5824e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.0746e+00,\n",
            "         0.0000e+00],\n",
            "        [9.6981e-01, 0.0000e+00, 1.5750e+01,  ..., 2.2856e+00, 0.0000e+00,\n",
            "         1.3711e-02],\n",
            "        ...,\n",
            "        [1.0841e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.3303e+01,\n",
            "         6.4360e-01],\n",
            "        [9.6969e-01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 6.6657e-01,\n",
            "         0.0000e+00]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76070557190664\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.7530,  ...,  0.0000,  0.6723, 15.4664],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.9064, 12.9798],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  2.1484,  ...,  0.0000,  0.0000, 16.0938],\n",
            "        [ 1.5690,  0.0000,  2.1143,  ...,  1.2800,  0.0000, 14.4551],\n",
            "        [ 2.9518,  0.0000,  0.0000,  ...,  0.0000,  1.9828,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  98.4375\n",
            "acc:  99.75938368821842\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.1234,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 2.2450,  0.0000,  0.0000,  ...,  1.2833,  0.0000,  1.3269],\n",
            "        [ 0.6332,  0.0000,  0.0000,  ..., 11.6867,  0.0000,  0.2548],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.1872,  ...,  0.0000,  0.0000,  0.8663],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 12.6302,  0.5826],\n",
            "        [ 0.0000,  0.0000,  0.9749,  ...,  0.0000,  0.6910, 13.4347]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75962406415427\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.2553,  0.0000,  0.8972,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.4762,  0.0000,  0.0000,  ...,  0.5365,  0.0000,  0.5392],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.4997,  ...,  0.0000,  0.0000,  1.8941],\n",
            "        [ 0.4081,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.1395,  0.0000, 15.6429,  ...,  0.0000,  0.8118,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75986419995431\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.0197,  0.0000,  0.0000,  ...,  0.0000, 16.4538,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 13.7739],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.7681,  0.8061],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.3697,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.5034,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76010409585845\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 1.2752,  ..., 0.2718, 0.5328, 0.0515],\n",
            "        [0.0000, 0.0000, 0.3710,  ..., 0.2004, 0.0000, 2.8782],\n",
            "        [1.4532, 0.0000, 0.0000,  ..., 0.0916, 0.0000, 2.6084],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [1.2769, 0.0000, 2.7654,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.4342, 0.0000, 0.0000,  ..., 0.0000, 1.6867, 0.6345]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76034375210635\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         6.8928e-01, 1.1635e+00, 2.4601e-01, 8.8057e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4436e+01, 0.0000e+00, 0.0000e+00,\n",
            "         3.6615e-01, 0.0000e+00, 0.0000e+00, 1.4076e+00],\n",
            "        [1.1482e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4350e+01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.7505e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 4.0207e-01, 0.0000e+00, 9.9659e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 8.4623e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 2.0518e+00, 0.0000e+00, 1.5863e+00, 1.6054e+01,\n",
            "         0.0000e+00, 2.1312e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.9322e+00, 0.0000e+00, 0.0000e+00, 1.0837e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.8098e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.7037e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.0048e+00, 0.0000e+00, 0.0000e+00, 1.0534e+00, 1.6827e-01, 0.0000e+00,\n",
            "         1.8432e+00, 4.6613e-01, 0.0000e+00, 5.2115e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 1.9524e-01, 0.0000e+00, 4.2968e-01, 0.0000e+00,\n",
            "         7.8830e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.1094e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.7600e+00,\n",
            "         1.4140e+01, 0.0000e+00, 2.0583e-01, 9.5056e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.0860e+00, 0.0000e+00, 1.5689e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 5.3604e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6500e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 2.5242e-01, 2.5891e-01, 1.0510e+00,\n",
            "         0.0000e+00, 1.9293e+00, 0.0000e+00, 1.3250e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.3127e+00, 0.0000e+00, 5.1658e-01, 2.8623e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.9407e+00, 0.0000e+00],\n",
            "        [1.6456e+01, 0.0000e+00, 7.3886e-01, 0.0000e+00, 1.1960e-01, 0.0000e+00,\n",
            "         2.9398e-01, 0.0000e+00, 8.4373e-01, 1.8001e+00, 0.0000e+00, 0.0000e+00,\n",
            "         4.1576e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 3.5569e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2610e-01, 2.4722e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.0060e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1075e+00,\n",
            "         1.2689e-01, 0.0000e+00, 1.3419e+01, 6.6095e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.0137e-01, 2.1692e-02,\n",
            "         0.0000e+00, 0.0000e+00, 8.2901e-01, 0.0000e+00, 0.0000e+00, 6.8173e-02,\n",
            "         0.0000e+00, 6.3935e-01, 0.0000e+00, 0.0000e+00, 3.6327e-01, 7.4146e-01,\n",
            "         1.4238e+01, 0.0000e+00, 0.0000e+00, 1.2183e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         6.9677e-01, 0.0000e+00, 0.0000e+00, 1.5506e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.9605e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.0719e-01,\n",
            "         1.5967e+01, 0.0000e+00, 0.0000e+00, 1.2675e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.0078e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5299e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.4822e-03, 0.0000e+00, 1.5716e+01, 0.0000e+00, 0.0000e+00,\n",
            "         3.5227e-02, 1.8404e-01, 2.8288e-01, 2.2527e+00],\n",
            "        [1.2920e-01, 0.0000e+00, 1.8433e+00, 0.0000e+00, 2.2746e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.1698e+01, 3.4532e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         7.2427e-01, 0.0000e+00, 2.2046e-01, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76058316893742\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.3173, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.1229],\n",
            "        [0.0000, 0.0000, 1.3989,  ..., 0.0000, 0.0962, 0.0000],\n",
            "        [0.0000, 0.0000, 0.4780,  ..., 0.1645, 0.0000, 2.3222],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.1996,  ..., 0.0000, 1.2398, 0.0000],\n",
            "        [2.4395, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.8611],\n",
            "        [0.1866, 0.0000, 0.0000,  ..., 0.0000, 1.7414, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76082234659083\n",
            "torch.Size([64, 48])\n",
            "tensor([[14.4158,  0.0000,  0.9294,  ...,  0.0000,  0.0000,  0.7344],\n",
            "        [ 2.1478,  0.0000,  0.0000,  ...,  0.1009,  0.5230,  0.0000],\n",
            "        [ 0.2531,  0.0000, 15.5417,  ...,  1.0504,  0.0000,  1.9306],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 13.4408,  0.0000],\n",
            "        [ 1.2850,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.2243]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76106128530552\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0982,  0.0000,  2.3974,  ...,  0.0000,  1.2928, 13.6135],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 14.7563],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.3307,  ...,  0.0000,  0.0000, 15.1341],\n",
            "        [ 0.0000,  0.0000,  2.3406,  ...,  0.0000,  0.0000, 11.9481],\n",
            "        [ 0.8214,  0.0000,  0.0000,  ...,  0.0000,  0.6088,  1.7671]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  96.875\n",
            "acc:  99.75817810719832\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 2.2561,  0.1521,  0.4826,  ...,  0.0000,  0.2658,  0.0000],\n",
            "        [ 1.4459,  0.1382,  0.0000,  ...,  0.0000,  0.0000,  1.6574],\n",
            "        [ 0.2166,  0.0000,  1.7840,  ..., 10.7906,  0.6480,  0.5561],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.1379,  0.0000],\n",
            "        [ 0.0424,  0.0000,  0.8809,  ...,  0.0000,  0.3333, 14.7537]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75841968751081\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.2051,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.3288,  ...,  0.0000,  0.0000,  1.9188],\n",
            "        ...,\n",
            "        [ 0.2117,  0.0000,  1.0167,  ...,  0.9088,  0.0000,  1.8427],\n",
            "        [ 0.4639,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  2.2168],\n",
            "        [ 0.3502,  0.0000, 15.0640,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75866102648432\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 1.1499,  0.0000,  1.7974,  ...,  0.0000, 14.4482,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.2930,  ...,  0.0000,  0.0000, 13.0002],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.7546,  0.0000,  0.0000,  ...,  0.0000,  2.4813,  0.9258],\n",
            "        [ 0.0000,  0.0000,  0.3465,  ...,  0.0000,  0.1614,  0.0000],\n",
            "        [ 0.3532,  0.0000,  0.0000,  ...,  0.0000,  0.1973,  0.3439]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75890212435996\n",
            "torch.Size([64, 78])\n",
            "tensor([[6.1594e-01, 0.0000e+00, 1.0481e+00,  ..., 0.0000e+00, 3.5580e-01,\n",
            "         0.0000e+00],\n",
            "        [8.3480e-02, 0.0000e+00, 0.0000e+00,  ..., 1.6494e+00, 0.0000e+00,\n",
            "         2.1217e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 4.9769e-02,  ..., 5.1037e-01, 0.0000e+00,\n",
            "         2.3145e+00],\n",
            "        ...,\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [2.1559e+00, 0.0000e+00, 0.0000e+00,  ..., 6.0293e-02, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 2.5363e+00,\n",
            "         6.7215e-04]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75914298137859\n",
            "torch.Size([13, 34])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.4564,  0.7557,\n",
            "          1.4510,  0.0000,  0.0000,  0.2364,  0.0000,  0.0885,  0.8057, 14.9248,\n",
            "          0.0000,  0.7802,  0.6993,  0.0000,  0.0000,  0.1258],\n",
            "        [ 0.6434,  0.0000,  0.0000,  0.0000, 17.0779,  2.3498,  0.0000,  0.0000,\n",
            "          0.8228,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  1.1450,  0.0000,  0.4728,  0.9827],\n",
            "        [ 0.0000,  0.0000,  0.9162,  0.0000,  0.0000, 15.4742,  0.0000,  2.4137,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.2198,  0.9922,  0.1006,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.3980,  0.0000,  2.0327,  0.1290, 12.9987,  0.4534,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.6089,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  1.1940,  0.0000,  2.0304],\n",
            "        [ 2.5143,  0.0000,  0.0000,  0.0000,  3.1751,  0.0000,  0.1281,  0.0000,\n",
            "          2.7923,  2.4194,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  2.1192, 15.4710,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.1269,  0.0000,  0.0000,  0.0000,  0.9786,  0.5686,  2.6217,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  1.0927,  0.0000,  1.7019,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.7224, 13.1964],\n",
            "        [ 0.0000,  0.0000,  0.0000,  1.3035,  2.3804,  0.0000,  0.0000,  1.9248,\n",
            "          1.6896, 14.4009,  0.0000,  0.0000,  2.1459,  0.0000,  0.0000,  3.2525,\n",
            "          0.0000,  0.5885,  0.0000,  0.0000,  1.1731,  0.0000],\n",
            "        [15.2279,  0.0000,  0.3572,  0.0000,  1.5859,  0.0000,  0.0757,  0.7528,\n",
            "          2.0496,  0.0000,  0.0000,  1.1846,  0.0000,  0.9721,  0.0000,  0.8031,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.6356,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  1.0020,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  1.3928,  0.0000,  0.0000, 13.7128,  0.3948],\n",
            "        [ 0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,\n",
            "          0.0000,  1.3076, 16.0576,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.9889,  0.0000,  0.0000,  0.0000,  0.0000,  0.0000,  2.1240,  0.0000,\n",
            "          0.4540,  0.0000,  0.0000,  0.0000,  0.0000,  0.4944,  0.0000,  0.5365,\n",
            "          0.6297,  0.0000, 15.7354,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0930,  0.0000,  0.0000,  0.0000,  0.1840,  0.1150,\n",
            "          0.0000,  0.1657,  0.0000,  1.7087,  0.0000,  0.0000,  0.0000, 15.9903,\n",
            "          0.6360,  0.0000,  0.0000,  0.0000,  0.0000,  0.1455],\n",
            "        [ 0.3377,  0.0000,  0.0000,  0.0000,  1.3260,  0.0000,  0.0000,  0.0000,\n",
            "         16.9269,  0.2238,  0.0000,  0.0000,  0.0000,  0.0727,  0.0000,  0.0000,\n",
            "          0.0000,  0.0000,  0.0000,  0.0000,  0.7443,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75938359778081\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.2809,  ..., 0.0000, 0.0000, 3.7664],\n",
            "        [0.7112, 0.0000, 0.8730,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 1.5634],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 2.0365,  ..., 0.0000, 0.4554, 0.0000],\n",
            "        [0.0903, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.7900],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75962397380701\n",
            "torch.Size([64, 48])\n",
            "tensor([[1.5223e+01, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         1.5136e+00],\n",
            "        [8.9559e-02, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00],\n",
            "        [2.9668e-01, 0.0000e+00, 1.3519e+01,  ..., 0.0000e+00, 0.0000e+00,\n",
            "         8.6133e-01],\n",
            "        ...,\n",
            "        [1.5113e-02, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 2.0814e-01,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 1.4986e+01,\n",
            "         0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00,  ..., 0.0000e+00, 2.9952e+00,\n",
            "         0.0000e+00]], device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.75986410969732\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.1891,  0.0000,  0.8850,  ...,  0.0000,  0.2634, 12.4203],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0330],\n",
            "        [ 0.0000,  0.0000,  1.2509,  ...,  0.0364,  0.0000, 14.6161],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000, 14.6688],\n",
            "        [ 0.0000,  0.0000,  0.0463,  ...,  0.0000,  0.0000, 13.3896],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.7351,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76010400569163\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 1.7493,  0.0000,  0.0000,  ...,  0.0000,  0.7397,  0.6019],\n",
            "        [ 2.7343,  0.0000,  0.0000,  ...,  0.3333,  0.0000,  2.3101],\n",
            "        [ 0.5263,  0.0000,  0.6880,  ..., 10.4873,  0.0000,  1.8570],\n",
            "        ...,\n",
            "        [ 0.0000,  0.1019,  0.0357,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.3434,  ...,  0.0000, 13.5075,  0.2466],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.1546, 15.1377]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7603436620296\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.0000,  0.0000,  0.6242,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.7002,  0.0000,  1.9177,  ...,  0.0586,  0.6732,  0.0000],\n",
            "        [ 0.4044,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  3.5621],\n",
            "        ...,\n",
            "        [ 0.5549,  0.0000,  0.9971,  ...,  0.0000,  0.0000,  2.0389],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.5571],\n",
            "        [ 0.0000,  0.0000, 14.3901,  ...,  0.4773,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76058307895065\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.1058,  0.0000,  0.8634,  ...,  0.0000, 19.0731,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.7879,  ...,  0.0000,  0.0000, 13.3177],\n",
            "        [ 0.6014,  0.0000,  0.0000,  ...,  0.0000,  1.4846,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.5358],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 1.6785,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76082225669396\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 1.1692,  ..., 0.0000, 1.2987, 0.0000],\n",
            "        [2.6425, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 2.0286],\n",
            "        [1.8858, 0.0000, 0.0000,  ..., 0.2168, 0.0000, 1.7565],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0140,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.7006,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76106119549846\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 0.0000e+00, 5.7565e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.1354e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3797e+01, 0.0000e+00, 0.0000e+00,\n",
            "         8.4844e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [3.9057e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6003e+01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 8.9507e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 8.3981e-01, 5.5890e-02, 0.0000e+00,\n",
            "         1.3834e+00, 0.0000e+00, 1.2654e+00, 6.5368e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 6.5076e-01, 8.7022e-01, 1.3020e+01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0398e-01,\n",
            "         0.0000e+00, 0.0000e+00, 2.7914e-03, 6.6068e-01, 0.0000e+00, 0.0000e+00,\n",
            "         7.6068e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.0173e-01,\n",
            "         1.5151e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.9741e-02, 0.0000e+00, 0.0000e+00, 8.5850e-02, 5.2515e-01, 0.0000e+00,\n",
            "         3.4283e-01, 1.2306e+00, 0.0000e+00, 1.6424e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 9.6025e-01, 0.0000e+00,\n",
            "         2.7431e+00, 0.0000e+00, 0.0000e+00, 3.2732e-01, 0.0000e+00, 1.3552e+00,\n",
            "         0.0000e+00, 1.6583e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.5946e-01,\n",
            "         1.5839e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.4448e+00, 0.0000e+00, 2.8881e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         3.5379e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.3293e+00,\n",
            "         0.0000e+00, 2.2323e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.2540e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 1.0814e+00, 2.5315e-01, 1.8891e+00,\n",
            "         0.0000e+00, 1.3133e+00, 0.0000e+00, 1.5226e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.8592e+00, 1.9602e-01, 0.0000e+00, 1.5181e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.9577e-01, 0.0000e+00, 6.3242e-01, 0.0000e+00],\n",
            "        [1.5498e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.3371e+00, 0.0000e+00, 0.0000e+00, 2.8223e+00,\n",
            "         8.2234e-01, 1.0982e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.0527e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [7.6872e-01, 0.0000e+00, 0.0000e+00, 1.2464e-01, 2.7008e+00, 9.0695e-01,\n",
            "         1.9323e-01, 0.0000e+00, 1.1849e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1942e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6021e+01, 7.8598e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 6.1526e-01, 7.9594e-01,\n",
            "         9.6365e-01, 0.0000e+00, 5.4828e-01, 0.0000e+00, 0.0000e+00, 1.5048e-01,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.3261e+01, 0.0000e+00, 4.2363e-01, 0.0000e+00],\n",
            "        [1.9311e-01, 0.0000e+00, 3.1509e-01, 1.2527e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 5.2763e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.3647e-01, 0.0000e+00,\n",
            "         1.5438e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 2.5971e-01, 1.3717e+00, 0.0000e+00, 2.1561e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.5757e+01, 1.2554e+00, 0.0000e+00,\n",
            "         1.2571e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.0598e+00, 3.4723e-01, 4.2292e-01, 0.0000e+00, 3.8404e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.4463e+01, 9.8101e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 6.6200e-01, 0.0000e+00]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76129989560286\n",
            "torch.Size([64, 57])\n",
            "tensor([[0.0000, 0.0000, 0.7530,  ..., 0.0000, 0.0000, 3.2060],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.8373, 0.0000, 0.1115,  ..., 0.2578, 0.0000, 1.7920],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.0000, 0.0000],\n",
            "        [0.0000, 0.0000, 0.1148,  ..., 0.7076, 0.0000, 1.7203],\n",
            "        [0.0000, 0.0000, 1.8324,  ..., 0.0000, 0.0000, 0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76153835724561\n",
            "torch.Size([64, 48])\n",
            "tensor([[14.3374,  0.0000,  0.6527,  ...,  0.8061,  0.0000,  0.8816],\n",
            "        [ 1.4904,  0.0000,  0.0000,  ...,  0.0000,  0.9398,  0.1395],\n",
            "        [ 0.0488,  0.0000, 13.9596,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 1.6366,  0.0000,  0.3431,  ...,  0.0000,  0.0000,  1.1902],\n",
            "        [ 0.0000,  0.0000,  1.3300,  ...,  0.2780, 15.9472,  0.3240],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  1.1347,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76177658066494\n",
            "torch.Size([64, 54])\n",
            "tensor([[ 0.0000,  0.0000,  1.4111,  ...,  0.0000,  0.5906, 16.7569],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000,  1.1071,  ...,  0.0000,  0.0000, 14.8358],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  1.5079,  ...,  0.0000,  0.8608, 15.6058],\n",
            "        [ 0.0000,  0.0000,  1.8395,  ...,  0.5007,  0.0000, 16.0049],\n",
            "        [ 1.3368,  0.0000,  0.0414,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76201456609884\n",
            "torch.Size([64, 73])\n",
            "tensor([[ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.8745,  0.0000],\n",
            "        [ 2.7754,  0.0000,  0.0000,  ...,  1.9170,  0.0000,  0.0000],\n",
            "        [ 1.0217,  0.0000,  0.0000,  ..., 11.3275,  0.0000,  0.0000],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0749],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000, 15.1170,  1.1208],\n",
            "        [ 0.0000,  0.0000,  0.4779,  ...,  0.0000,  0.0000, 14.5738]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76225231378505\n",
            "torch.Size([64, 59])\n",
            "tensor([[ 0.7133,  0.0000,  0.4392,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0370,  0.0000,  2.0611,  ...,  0.0000,  0.0708,  0.0000],\n",
            "        [ 0.2352,  0.0000,  0.0000,  ...,  1.1104,  0.0000,  4.7327],\n",
            "        ...,\n",
            "        [ 0.0000,  0.0000,  0.7651,  ...,  0.0000,  0.0000,  0.5340],\n",
            "        [ 0.8268,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0000, 14.6121,  ...,  0.0000,  0.0000,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76248982396109\n",
            "torch.Size([64, 75])\n",
            "tensor([[ 0.3443,  0.0000,  0.0000,  ...,  0.0000, 12.0067,  0.0000],\n",
            "        [ 0.0000,  0.0000,  0.0000,  ...,  0.0000,  0.1404, 15.6820],\n",
            "        [ 0.6335,  0.0000,  0.0000,  ...,  0.0000,  2.0007,  0.0000],\n",
            "        ...,\n",
            "        [ 0.6383,  0.0000,  0.0000,  ...,  0.0000,  0.8469,  0.4747],\n",
            "        [ 0.1393,  0.0000,  0.0000,  ...,  0.0000,  0.0000,  0.0000],\n",
            "        [ 0.0000,  0.0378,  0.3070,  ...,  0.0000,  0.9960,  0.0000]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.76272709686423\n",
            "torch.Size([64, 78])\n",
            "tensor([[0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.6012, 0.0000],\n",
            "        [2.3606, 0.0000, 0.2186,  ..., 0.1732, 0.0000, 2.5815],\n",
            "        [0.0000, 0.0000, 0.2710,  ..., 0.6154, 0.0000, 1.2338],\n",
            "        ...,\n",
            "        [0.0000, 0.0000, 0.1732,  ..., 0.0000, 0.0000, 1.1117],\n",
            "        [0.0000, 0.0000, 1.2486,  ..., 0.0000, 0.5069, 0.0000],\n",
            "        [0.0000, 0.0000, 0.0000,  ..., 0.0000, 0.3966, 0.4570]],\n",
            "       device='cuda:0', grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7629641327315\n",
            "torch.Size([13, 34])\n",
            "tensor([[0.0000e+00, 0.0000e+00, 1.3126e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         2.0643e-01, 0.0000e+00, 6.8352e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4154e+01, 6.1613e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 7.4797e-01],\n",
            "        [1.1844e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.6124e+01, 3.7797e-01,\n",
            "         2.1387e-01, 0.0000e+00, 7.6733e-01, 9.0477e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.1347e+00, 0.0000e+00, 3.0111e+00, 3.3051e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 4.2434e-02, 0.0000e+00, 2.5686e-01, 1.3417e+01,\n",
            "         0.0000e+00, 1.7374e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.9786e+00, 3.7872e-02, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 3.7652e-01, 0.0000e+00],\n",
            "        [6.3580e-01, 0.0000e+00, 1.5370e-01, 0.0000e+00, 6.8338e-01, 0.0000e+00,\n",
            "         1.5795e+01, 0.0000e+00, 3.8687e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         5.6802e-01, 0.0000e+00, 0.0000e+00, 4.1317e-01, 1.8469e-01, 0.0000e+00,\n",
            "         7.2260e-01, 0.0000e+00, 0.0000e+00, 8.2661e-01],\n",
            "        [1.0265e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.4829e-01, 0.0000e+00,\n",
            "         6.4539e-01, 0.0000e+00, 1.0830e+00, 8.1330e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.5761e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 3.1808e-01, 0.0000e+00, 0.0000e+00, 2.8343e-01,\n",
            "         2.3162e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 2.0963e+00, 4.2541e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         8.6032e-01, 0.0000e+00, 0.0000e+00, 1.4863e+01],\n",
            "        [0.0000e+00, 0.0000e+00, 3.6712e-01, 1.5757e+00, 1.3211e+00, 0.0000e+00,\n",
            "         0.0000e+00, 3.2464e+00, 0.0000e+00, 1.3430e+01, 0.0000e+00, 0.0000e+00,\n",
            "         1.3067e+00, 0.0000e+00, 1.9680e-01, 2.9444e+00, 3.1595e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [1.5207e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.8334e+00, 0.0000e+00,\n",
            "         1.3451e+00, 1.8721e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 5.6564e-01,\n",
            "         0.0000e+00, 6.7102e-01, 3.3168e-01, 0.0000e+00, 0.0000e+00, 3.0433e-01,\n",
            "         0.0000e+00, 9.3704e-01, 6.1339e-01, 1.5737e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 9.3146e-01, 9.1506e-01, 2.2553e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1585e-01, 0.0000e+00, 0.0000e+00,\n",
            "         9.0186e-01, 3.0155e-01, 0.0000e+00, 1.3603e+00, 0.0000e+00, 9.2694e-02,\n",
            "         7.4115e-03, 0.0000e+00, 1.2940e+01, 1.8035e-01],\n",
            "        [9.0124e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 2.2071e+00, 8.0321e-01,\n",
            "         3.0897e+00, 0.0000e+00, 3.5463e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.0945e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 7.1817e-01,\n",
            "         1.5361e+01, 0.0000e+00, 0.0000e+00, 0.0000e+00],\n",
            "        [0.0000e+00, 0.0000e+00, 5.6273e-01, 3.5181e-01, 0.0000e+00, 8.3257e-01,\n",
            "         1.7545e-01, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         1.0952e+00, 1.4572e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 1.1356e-01,\n",
            "         1.4503e+01, 0.0000e+00, 4.8128e-01, 9.4831e-01],\n",
            "        [0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 1.7334e+00, 0.0000e+00, 4.2780e-01, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 1.4613e+01, 4.8564e-01, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.0751e+00, 0.0000e+00],\n",
            "        [3.8138e-01, 0.0000e+00, 4.6072e-01, 9.2906e-01, 2.9027e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 1.6969e+01, 1.1694e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 9.6779e-02, 0.0000e+00, 0.0000e+00, 0.0000e+00, 0.0000e+00,\n",
            "         0.0000e+00, 0.0000e+00, 0.0000e+00, 9.4892e-01]], device='cuda:0',\n",
            "       grad_fn=<ReluBackward0>)\n",
            "acc_t:  100.0\n",
            "acc:  99.7632009317997\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "execution_count": null,
      "metadata": {
        "id": "Ru4W2WUatUbR",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 312
        },
        "outputId": "72de6415-7852-4fd7-921f-89e03b38af7f"
      },
      "outputs": [
        {
          "output_type": "error",
          "ename": "TypeError",
          "evalue": "ignored",
          "traceback": [
            "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
            "\u001b[0;31mTypeError\u001b[0m                                 Traceback (most recent call last)",
            "\u001b[0;32m<ipython-input-54-6a7114a20aa5>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0moutputs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mnew_model\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m**\u001b[0m\u001b[0mbatch\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      2\u001b[0m \u001b[0mprint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0moutputs\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mloss\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0moutputs\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mlogits\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mshape\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;32m/usr/local/lib/python3.7/dist-packages/torch/nn/modules/module.py\u001b[0m in \u001b[0;36m_call_impl\u001b[0;34m(self, *input, **kwargs)\u001b[0m\n\u001b[1;32m    720\u001b[0m             \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_slow_forward\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    721\u001b[0m         \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 722\u001b[0;31m             \u001b[0mresult\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mforward\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m*\u001b[0m\u001b[0minput\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    723\u001b[0m         for hook in itertools.chain(\n\u001b[1;32m    724\u001b[0m                 \u001b[0m_global_forward_hooks\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mvalues\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
            "\u001b[0;31mTypeError\u001b[0m: forward() got an unexpected keyword argument 'labels'"
          ]
        }
      ],
      "source": [
        "outputs = new_model(**batch)\n",
        "print(outputs.loss, outputs.logits.shape)"
      ]
    }
  ],
  "metadata": {
    "accelerator": "GPU",
    "colab": {
      "collapsed_sections": [],
      "machine_shape": "hm",
      "provenance": []
    },
    "gpuClass": "standard",
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "name": "python"
    },
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "b4fdb559d38444efbff9e1261d7e6fc9": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_8f43742fc54c44e89dfd75c0c6855c02",
              "IPY_MODEL_c5e41eaf1a204ab2ac67b2530d6051ae",
              "IPY_MODEL_0f33ae297a984ac5b48e1e642a93d564"
            ],
            "layout": "IPY_MODEL_08ce54756ea8417980a906da0897438b"
          }
        },
        "8f43742fc54c44e89dfd75c0c6855c02": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_cb35af01c76441d08fba5c46bed3bc14",
            "placeholder": "​",
            "style": "IPY_MODEL_44892a2a92d44a89b9e1e15a943a621e",
            "value": "Downloading: 100%"
          }
        },
        "c5e41eaf1a204ab2ac67b2530d6051ae": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_fd771a5502e44de789fcdcb36102a3d1",
            "max": 385,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_3ef0ae063c5943fe9f3cce6487289870",
            "value": 385
          }
        },
        "0f33ae297a984ac5b48e1e642a93d564": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_2f6ef6d4056c4d088664ac8d7b169355",
            "placeholder": "​",
            "style": "IPY_MODEL_6ca7e15e48164f118022784ded3593da",
            "value": " 385/385 [00:00&lt;00:00, 14.6kB/s]"
          }
        },
        "08ce54756ea8417980a906da0897438b": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "cb35af01c76441d08fba5c46bed3bc14": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "44892a2a92d44a89b9e1e15a943a621e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "fd771a5502e44de789fcdcb36102a3d1": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "3ef0ae063c5943fe9f3cce6487289870": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "2f6ef6d4056c4d088664ac8d7b169355": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "6ca7e15e48164f118022784ded3593da": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "b1684cf338d246dd8907741e464965fa": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_dda92dca05ca4897bf3bc5693af46f2a",
              "IPY_MODEL_a49db3987a3248199f9c6625d8c836c7",
              "IPY_MODEL_ed18e1b9370046f3895facecf86567e2"
            ],
            "layout": "IPY_MODEL_4f0d4c20e52c4acdbafb66c024f293ac"
          }
        },
        "dda92dca05ca4897bf3bc5693af46f2a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_c91cda385987475ba2329d9b77ff1b66",
            "placeholder": "​",
            "style": "IPY_MODEL_a393377c162045a5bcfaf003de3c5980",
            "value": "Downloading: 100%"
          }
        },
        "a49db3987a3248199f9c6625d8c836c7": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_c972ab1ab39f4d899728ad5fbad711ae",
            "max": 227845,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_77716b15c2a24aa6a1c9878093019fe9",
            "value": 227845
          }
        },
        "ed18e1b9370046f3895facecf86567e2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_0f1d3d53aa094f57bf926f60ea16797a",
            "placeholder": "​",
            "style": "IPY_MODEL_7c1df77dee5e464c93a72a46df370cd8",
            "value": " 228k/228k [00:00&lt;00:00, 591kB/s]"
          }
        },
        "4f0d4c20e52c4acdbafb66c024f293ac": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "c91cda385987475ba2329d9b77ff1b66": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "a393377c162045a5bcfaf003de3c5980": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "c972ab1ab39f4d899728ad5fbad711ae": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "77716b15c2a24aa6a1c9878093019fe9": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "0f1d3d53aa094f57bf926f60ea16797a": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "7c1df77dee5e464c93a72a46df370cd8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "ed4b7ab068d24cebb625f5f49b6f80c8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_b613b49bc52e4acc88160bbb06e2fd15",
              "IPY_MODEL_fd3a5a0ab26b47428dd77379c4082b2a",
              "IPY_MODEL_4ace639ae8e94fea99cc160c9d251c11"
            ],
            "layout": "IPY_MODEL_99fa9bcd15b748b29d359e2489775178"
          }
        },
        "b613b49bc52e4acc88160bbb06e2fd15": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_6a69ddf311fe48f9b9574a4b0b94b33b",
            "placeholder": "​",
            "style": "IPY_MODEL_c51b724a70ca4a5d994e936c8fc15cdf",
            "value": "100%"
          }
        },
        "fd3a5a0ab26b47428dd77379c4082b2a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_9ec8fedfc7024b6ca2eb938e7cb7b73c",
            "max": 2,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_eb55b4968f0f498fa933ce800c773d7f",
            "value": 2
          }
        },
        "4ace639ae8e94fea99cc160c9d251c11": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_66d88631b5fd4325be2be15aba751d24",
            "placeholder": "​",
            "style": "IPY_MODEL_9f5f06454fd84c84a1ab873085fcd0bb",
            "value": " 2/2 [00:01&lt;00:00,  1.92ba/s]"
          }
        },
        "99fa9bcd15b748b29d359e2489775178": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "6a69ddf311fe48f9b9574a4b0b94b33b": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "c51b724a70ca4a5d994e936c8fc15cdf": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "9ec8fedfc7024b6ca2eb938e7cb7b73c": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "eb55b4968f0f498fa933ce800c773d7f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "66d88631b5fd4325be2be15aba751d24": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "9f5f06454fd84c84a1ab873085fcd0bb": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "dce837e4859c41a083dd63071a68c073": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_e3d670b58d9f4971baa3e0e8a4a83ab1",
              "IPY_MODEL_aab3732ea3754f8c8b1872c40b74f817",
              "IPY_MODEL_c11754333c1a4ac98089ac32ff85a7f4"
            ],
            "layout": "IPY_MODEL_8c97dd8da81e40d7b74d9995b75cd541"
          }
        },
        "e3d670b58d9f4971baa3e0e8a4a83ab1": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_f88b657362a94062b9ea95a5a9f96b67",
            "placeholder": "​",
            "style": "IPY_MODEL_9aa537aab5af40359aeae44df5de79ca",
            "value": "Downloading: 100%"
          }
        },
        "aab3732ea3754f8c8b1872c40b74f817": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_6c9ed62bdd5247abb9d16d91297558cc",
            "max": 442221694,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_20737572eea442319ea8f8c4fea0efa9",
            "value": 442221694
          }
        },
        "c11754333c1a4ac98089ac32ff85a7f4": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_a77993573a5648899dc41c94f2854024",
            "placeholder": "​",
            "style": "IPY_MODEL_f35f5710f7a04b92a3ec324ed53803ff",
            "value": " 442M/442M [00:07&lt;00:00, 62.6MB/s]"
          }
        },
        "8c97dd8da81e40d7b74d9995b75cd541": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "f88b657362a94062b9ea95a5a9f96b67": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "9aa537aab5af40359aeae44df5de79ca": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "6c9ed62bdd5247abb9d16d91297558cc": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "20737572eea442319ea8f8c4fea0efa9": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "a77993573a5648899dc41c94f2854024": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "f35f5710f7a04b92a3ec324ed53803ff": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "156f3f87f3b247ecb7839a3123be81d0": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_0a7bd295fd9342de983ceb6e35fb1658",
              "IPY_MODEL_e70684f8312c49b3856ed0c76b30e9a8",
              "IPY_MODEL_4772151442624801ac915791babda002"
            ],
            "layout": "IPY_MODEL_5087b70e8f9840cf898a214f33c66da4"
          }
        },
        "0a7bd295fd9342de983ceb6e35fb1658": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_30ba6122bd0c4d83aa1599ed3a499b58",
            "placeholder": "​",
            "style": "IPY_MODEL_1b984b3ac62f4687b7540e750d966174",
            "value": ""
          }
        },
        "e70684f8312c49b3856ed0c76b30e9a8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_c3caf825190e4931afa2375d9c587e5c",
            "max": 60,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_fa88b46ec04d440cbd767f59ffd2e5a1",
            "value": 60
          }
        },
        "4772151442624801ac915791babda002": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_9a33df6cc7e747cc86f297ef14416822",
            "placeholder": "​",
            "style": "IPY_MODEL_440459c4178d4fada590659af5c73959",
            "value": " 7999/? [29:39&lt;00:00,  3.93it/s]"
          }
        },
        "5087b70e8f9840cf898a214f33c66da4": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "30ba6122bd0c4d83aa1599ed3a499b58": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "1b984b3ac62f4687b7540e750d966174": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "c3caf825190e4931afa2375d9c587e5c": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "fa88b46ec04d440cbd767f59ffd2e5a1": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "9a33df6cc7e747cc86f297ef14416822": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "440459c4178d4fada590659af5c73959": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        }
      }
    }
  },
  "nbformat": 4,
  "nbformat_minor": 0
}